Skip to main content <#content>

Thank you for visiting nature.com. You are using a browser version with
limited support for CSS. To obtain the best experience, we recommend you
use a more up to date browser (or turn off compatibility mode in
Internet Explorer). In the meantime, to ensure continued support, we are
displaying the site without styles and JavaScript.

Advertisement

Nature <https://www.nature.com/>

  * View all journals <https://www.nature.com/siteindex>
  * Search

    <javascript:;>


        Search

    Search articles by subject, keyword or author
    Show results from

    Search

    Advanced search <https://www.nature.com/search/advanced>


    Quick links

      o Explore articles by subject <https://www.nature.com/subjects>
      o Find a job <https://www.nature.com/naturecareers>
      o Guide to authors <https://www.nature.com/authors/index.html>
      o Editorial policies
        <https://www.nature.com/authors/editorial_policies/>
  * Log in
    <https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41586-022-05366-w>

  * Explore content

    <javascript:;>


    Explore content

      o Research articles <https://www.nature.com/nature/research-articles>
      o News <https://www.nature.com/news>
      o Opinion <https://www.nature.com/opinion>
      o Research Analysis <https://www.nature.com/research-analysis>
      o Careers <https://www.nature.com/careers>
      o Books & Culture <https://www.nature.com/books-culture>
      o Podcasts <https://www.nature.com/nature/podcasts>
      o Videos <https://www.nature.com/nature/videos>
      o Current issue <https://www.nature.com/nature/current-issue>
      o Browse issues <https://www.nature.com/nature/browse-issues>
      o Collections <https://www.nature.com/nature/collections>
      o Subjects <https://www.nature.com/nature/browse-subjects>
      o Follow us on Facebook <https://www.facebook.com/Nature>
      o Follow us on Twitter <https://twitter.com/nature>
      o Sign up for alerts

        <https://www.nature.com/my-account/alerts/subscribe-journal?list-id=1>
      o RSS feed <https://www.nature.com/nature.rss>
  * About the journal

    <javascript:;>


    About the journal

      o Journal Staff <https://www.nature.com/nature/journal-staff>
      o About the Editors <https://www.nature.com/nature/editors>
      o Journal Information
        <https://www.nature.com/nature/journal-information>
      o Our publishing models
        <https://www.nature.com/nature/our-publishing-models>
      o Editorial Values Statement
        <https://www.nature.com/nature/editorial-values-statement>
      o Journal Metrics <https://www.nature.com/nature/journal-impact>
      o Awards <https://www.nature.com/nature/awards>
      o Contact <https://www.nature.com/nature/contact>
      o Editorial policies
        <https://www.nature.com/nature/editorial-policies>
      o History of Nature <https://www.nature.com/nature/history-of-nature>
      o Send a news tip <https://www.nature.com/nature/send-a-news-tip>
  * Publish with us

    <javascript:;>


    Publish with us

      o For Authors <https://www.nature.com/nature/for-authors>
      o For Referees <https://www.nature.com/nature/for-referees>
      o Language editing services
        <https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022>
      o Submit manuscript

        <https://mts-nature.nature.com/>

  * Sign up for alerts

    <https://www.nature.com/my-account/alerts/subscribe-journal?list-id=1>
  * RSS feed <https://www.nature.com/nature.rss>

 1. nature <https://www.nature.com/>
 2. articles <https://www.nature.com/nature/articles?type=article>
 3. article

Bending forces and nucleotide state jointly regulate F-actin structure
Download PDF

<https://www.nature.com/articles/s41586-022-05366-w.pdf?pdf=button%20sticky>
Your content has downloaded


    Similar content being viewed by others

Slider with three content items shown per slide. Use the Previous and
Next buttons to navigate the slides or the slide controller buttons at
the end to navigate through each slide.


      F-actin architecture determines constraints on myosin thick
      filament motion <https://doi.org/10.1038/s41467-022-34715-6>

Article16 November 2022

Camelia G. Muresan, Zachary Gao Sun, … Michael P. Murrell


      Structural basis of actin filament assembly and aging
      <https://doi.org/10.1038/s41586-022-05241-8>

Article26 October 2022

Wout Oosterheert, Björn U. Klink, … Stefan Raunser


      Crowding alters F-actin secondary structure and hydration
      <https://doi.org/10.1038/s42003-023-05274-3>

Article02 September 2023

Xiaobing Chen, Steven J. Roeters, … Carlos R. Baiz


      Bound nucleotide can control the dynamic architecture of monomeric
      actin <https://doi.org/10.1038/s41594-022-00743-5>

Article24 March 2022

Rustam Ali, Jacob A. Zahm & Michael K. Rosen


      Allosteric regulation controls actin-bundling properties of human
      plastins <https://doi.org/10.1038/s41594-022-00771-1>

Article19 May 2022

Christopher L. Schwebach, Elena Kudryashova, … Dmitri S. Kudryashov


      Biased localization of actin binding proteins by actin filament
      conformation <https://doi.org/10.1038/s41467-020-19768-9>

Article25 November 2020

Andrew R. Harris, Pamela Jreij, … Daniel A. Fletcher


      A statistical mechanics model for determining the length
      distribution of actin filaments under cellular tensional
      homeostasis <https://doi.org/10.1038/s41598-022-18833-1>

Article24 August 2022

Yuika Ueda, Daiki Matsunaga & Shinji Deguchi


      Biochemical and mechanical regulation of actin dynamics
      <https://doi.org/10.1038/s41580-022-00508-4>

Article02 August 2022

Pekka Lappalainen, Tommi Kotila, … Guillaume Romet-Lemonne


      A functional family of fluorescent nucleotide analogues to
      investigate actin dynamics and energetics
      <https://doi.org/10.1038/s41467-020-20827-4>

Article22 January 2021

Jessica Colombo, Adrien Antkowiak, … Alphée Michelot

Download PDF

<https://www.nature.com/articles/s41586-022-05366-w.pdf>

  * Article
  * Open Access
    <https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research>
  * Published: 26 October 2022 <#article-info>


  Bending forces and nucleotide state jointly regulate F-actin structure

  * Matthew J. Reynolds <#auth-Matthew_J_-Reynolds-Aff1>  ORCID:
    orcid.org/0000-0002-2501-9280
    <http://orcid.org/0000-0002-2501-9280>^1 <#Aff1> ,
  * Carla Hachicho <#auth-Carla-Hachicho-Aff1>^1 <#Aff1> ,
  * Ayala G. Carl <#auth-Ayala_G_-Carl-Aff1-Aff2>^1 <#Aff1>,2 <#Aff2> ,
  * Rui Gong <#auth-Rui-Gong-Aff1>^1 <#Aff1> &
  * …
  * Gregory M. Alushin<#auth-Gregory_M_-Alushin-Aff1>^1 <#Aff1>  

Show authors

/Nature/ <https://www.nature.com/> *volume 611*, pages 380–386
(2022)Cite this article <#citeas>

  *

    12k Accesses

  *

    9 Citations

  *

    154 Altmetric

  *

    Metrics details
    <https://www.nature.com/articles/s41586-022-05366-w/metrics>


    Abstract

ATP-hydrolysis-coupled actin polymerization is a fundamental mechanism
of cellular force generation^1 <#ref-CR1>,2 <#ref-CR2>,3
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR3> . In turn,
force^4 <https://www.nature.com/articles/s41586-022-05366-w#ref-CR4>,5
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR5> and actin
filament (F-actin) nucleotide state^6
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR6> regulate
actin dynamics by tuning F-actin’s engagement of actin-binding proteins
through mechanisms that are unclear. Here we show that the nucleotide
state of actin modulates F-actin structural transitions evoked by
bending forces. Cryo-electron microscopy structures of ADP–F-actin and
ADP-P_i –F-actin with sufficient resolution to visualize bound solvent
reveal intersubunit interfaces bridged by water molecules that could
mediate filament lattice flexibility. Despite extensive ordered solvent
differences in the nucleotide cleft, these structures feature nearly
identical lattices and essentially indistinguishable protein backbone
conformations that are unlikely to be discriminable by actin-binding
proteins. We next introduce a machine-learning-enabled pipeline for
reconstructing bent filaments, enabling us to visualize both continuous
structural variability and side-chain-level detail. Bent F-actin
structures reveal rearrangements at intersubunit interfaces
characterized by substantial alterations of helical twist and
deformations in individual protomers, transitions that are distinct in
ADP–F-actin and ADP-P_i –F-actin. This suggests that phosphate
rigidifies actin subunits to alter the bending structural landscape of
F-actin. As bending forces evoke nucleotide-state dependent
conformational transitions of sufficient magnitude to be detected by
actin-binding proteins, we propose that actin nucleotide state can serve
as a co-regulator of F-actin mechanical regulation.


    Main

Actin polymerization powers fundamental cellular processes, including
cell migration, organelle dynamics and endocytosis^1
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR1> . The
propulsive assembly of branched actin networks is coupled to nucleotide
consumption by globular actin (G-actin) subunits, which hydrolyse bound
ATP concomitant with their incorporation into growing F-actin^1
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR1> . An
elastic Brownian rachet model, wherein bending filaments push against
membranes as they polymerize^2
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR2>,3
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR3> ,
quantitatively explains the conversion of this chemical energy into
mechanical work. Consistently, electron microscopy has resolved abundant
bent filaments featuring radii of curvature in the hundreds of
nanometres within polymerizing actin networks adjacent to surfaces in
vitro^7 <https://www.nature.com/articles/s41586-022-05366-w#ref-CR7> and
in cells^8 <#ref-CR8>,9 <#ref-CR9>,10
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR10> .

After incorporation into F-actin, subunits rapidly hydrolyse bound ATP
in seconds. This produces metastable ADP-P_i –F-actin that persists for
minutes before phosphate release, resulting in long-lived ADP–F-actin^1
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR1> .
Retrograde flow simultaneously pushes aged filaments away from the
membrane^11
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR11> ,
producing a spatial gradient of F-actin nucleotide states—a biochemical
marker of filament age^6
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR6> . Both
actin nucleotide state and mechanical forces^1
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR1>,4
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR4>,12
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR12>,13
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR13> modulate
interactions between F-actin and key actin-binding proteins (ABPs) that
govern actin network dynamics. The F-actin-depolymerizing protein
cofilin preferentially binds to and severs ADP–F-actin^14
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR14>,15
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR15> ,
facilitating the selective disassembly of membrane distal, aged F-actin.
Cofilin and the branched F-actin nucleator ARP2/3 also feature
mechanically tuned^5
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR5>
F-actin-binding activity^16 <#ref-CR16>,17 <#ref-CR17>,18
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR18> ,
specifically responding to filament bending^17
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR17>,19
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR19> . This may
contribute to force sensitivity in branched F-actin networks, which
display modified interfilament geometry^20
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR20>,21
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR21> and
enhanced force production^20
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR20> in the
presence of resistive loads. Several other ABPs detect the nucleotide
state of F-actin^13
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR13>,22
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR22> and force
across individual filaments^23 <#ref-CR23>,24 <#ref-CR24>,25
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR25> through
mechanisms that are unclear.

F-actin polymerization is coupled to the G-to-F transition, a
substantial flattening of the actin subunit that renders its nucleotide
cleft active site competent for hydrolysis^26
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR26> . By
contrast, cryo-electron microscopy (cryo-EM) studies at a resolution of
3–4 Å found only modest nucleotide-state-dependent changes in F-actin,
reporting either localized rearrangements^27
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR27>,28
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR28> in actin’s
flexible D-loop^29 <#ref-CR29>,30 <#ref-CR30>,31
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR31> or nearly
identical backbone conformations^32
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR32> . However,
binding by ABPs can evoke substantial rearrangements, notably cofilin,
which stabilizes an undertwisted F-actin lattice^33
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR33>,34
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR34> . This
discrepancy challenges a traditional allosteric model for F-actin
nucleotide state discrimination by ABPs, which cannot directly access
actin’s buried nucleotide cleft. The actin nucleotide state could also
modulate F-actin’s deformability and corresponding capacity for
rearrangements mediating ABP binding^13
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR13>,15
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR15> ,
consistent with micrometre-scale persistence length measurements showing
that ADP-P_i –F-actin is stiffer than ADP–F-actin^35
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR35> .
Molecular dynamics simulations support mechanical regulation of F-actin
structure^36
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR36> , with
models abstracted to the subunit level predicting coupling between
filament bending and helical lattice twist modulation (twist–bend
coupling)^37
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR37> . Recent
finer-grained simulations featuring subunits composed of multiple rigid
bodies exhibited cooperativity between ATP hydrolysis and phosphate
release through the filament lattice^38
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR38> ,
suggesting coupling between propagation of lattice deformations and
filament nucleotide state. However, in the absence of direct structural
visualization, it remains unclear whether mechanically evoked
conformational transitions in F-actin intersect with filament nucleotide
state to control ABP engagement.


    Solvent in the nucleotide cleft of F-actin

To investigate whether subtle F-actin conformational changes could
explain ABP nucleotide-state sensing, we determined the structures of
ADP–F-actin and ADP-P_i –F-actin at an improved resolution. As reported
protocols for preparing ADP-P_i –F-actin vary^13
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR13>,27
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR27>,32
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR32> , we
validated our approach (polymerizing G-actin in the presence of 15 mM
KH_2 PO_4 ) by monitoring cofilin severing using a fluorescence
microscopy assay (Methods
<https://www.nature.com/articles/s41586-022-05366-w#Sec8>). Consistent
with previous reports^14
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR14>,15
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR15> ,
ADP–F-actin rapidly depolymerized within 5 min of cofilin addition,
whereas ADP-P_i –F-actin largely remained intact. Replacing KH_2 PO_4
with K_2 SO_4 produced intermediate severing, suggesting that phosphate
specifically caused the substantially reduced severing of ADP-P_i
–F-actin rather than altering the buffer’s ionic strength (Extended Data
Fig. 1a,b <https://www.nature.com/articles/s41586-022-05366-w#Fig5>). We
therefore used cryo-EM and iterative helical real-space refinement as
implemented in RELION to determine the structures of ADP–F-actin and
ADP-P_i –F-actin at resolutions of 2.4 Å and 2.5 Å, respectively (Fig.
1a <https://www.nature.com/articles/s41586-022-05366-w#Fig1>, Methods
<https://www.nature.com/articles/s41586-022-05366-w#Sec8>, Extended Data
Fig. 1c,d <https://www.nature.com/articles/s41586-022-05366-w#Fig5> and
Supplementary Table 1
<https://www.nature.com/articles/s41586-022-05366-w#MOESM1>).

*Fig. 1: Nucleotide cleft water networks are remodelled upon phosphate
release by F-actin.

*

figure 1 <https://www.nature.com/articles/s41586-022-05366-w/figures/1>

*a*, Cryo-EM maps of ADP–F-actin (left, shades of blue) and ADP-P_i
–F-actin (right, shades of orange). ADP (green) and water (magenta)
densities are shown. BE, barbed end; PE, pointed end. *b*, Atomic models
of the ADP–F-actin (blue) and ADP-P_i –F-actin nucleotide (nuc.) clefts
(orange) are shown in Cα representation. Top, carved transparent grey
density is displayed for ADP (green), PO_4 ^3− (yellow), Mg^2+ (light
green) and water molecules (violet). In ADP/PO_4 ^3− , nitrogen atoms
are blue, oxygen atoms are red and phosphorous atoms are yellow. Bottom,
the backbone and side chain residues involved in putative
hydrogen-bonding networks (dashed lines) are displayed and coloured by
heteroatom. *c*, Superposition of individual ADP–F-actin (blue) and
ADP-P_i –F-actin (orange) protomers, displayed in Cα representation.

Full size image

<https://www.nature.com/articles/s41586-022-05366-w/figures/1>

These maps featured sufficient resolution to accurately build and refine
ADP, magnesium, phosphate and water molecules (Fig. 1b
<https://www.nature.com/articles/s41586-022-05366-w#Fig1>, Methods
<https://www.nature.com/articles/s41586-022-05366-w#Sec8>, Extended Data
Fig. 1e <https://www.nature.com/articles/s41586-022-05366-w#Fig5> and
Supplementary Video 1
<https://www.nature.com/articles/s41586-022-05366-w#MOESM4>). In both
nucleotide states, extensive water-mediated hydrogen-bonding networks
stabilize the ligands. Notably, in ADP–F-actin, four water molecules
displace the inorganic phosphate’s oxygen tetrahedron. Other nucleotide
cleft positions also featured small changes in the number and
positioning of water molecules. These F-actin active-site compositions
and stereochemistry are consistent with a contemporary study^39
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR39> ,
providing confidence in interpreting the cryo-EM maps.

Despite substantial changes in bound small molecules, differences
between the conformations of actin protomers themselves, which were
exquisitely resolved (Extended Data Fig. 2a
<https://www.nature.com/articles/s41586-022-05366-w#Fig6>), were
miniscule (0.206 Å Cα root mean squared deviation (r.m.s.d.); Fig. 1c
<https://www.nature.com/articles/s41586-022-05366-w#Fig1>). Per-residue
r.m.s.d. and per-residue strain pseudoenergy analysis (a metric that
highlights local deformations; Methods
<https://www.nature.com/articles/s41586-022-05366-w#Sec8>) confirmed the
absence of meaningful Cα deviations with the exception of the D-loop in
subdomain 2 (Extended Data Fig. 2b,c
<https://www.nature.com/articles/s41586-022-05366-w#Fig6>). This region,
which is well established to be flexible in F-actin^27
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR27>,32
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR32>,40
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR40>,41
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR41> , also
corresponded to the lowest-resolution region of our maps, and
differences may therefore be attributable to resolution-dependent
uncertainty in the atomic models. At the helical-lattice level, the
refined rise was slightly larger in ADP–F-actin (28.1 Å versus 27.8 Å in
ADP-P_i –F-actin), which accumulates into detectable differences in
subunit positioning at longer length scales (Extended Data Fig. 2d
<https://www.nature.com/articles/s41586-022-05366-w#Fig6>).
Nevertheless, these structures did not reveal changes of the magnitude
associated with ABP binding (for example, cofilin^33
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR33>,34
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR34> ).
This suggests that nucleotide-state sensitivity in ABPs is unlikely to
be mediated primarily by their detection of F-actin conformational
changes concomitant with phosphate release.


    Water molecules mediate intersubunit contacts

We next examined water molecules outside the nucleotide cleft, reasoning
that solvent at intersubunit interfaces could potentially facilitate
mechanical remodelling of the filament. As is typical for helical
reconstructions, the highest-resolution map regions were the filament
cores (Extended Data Fig. 1d
<https://www.nature.com/articles/s41586-022-05366-w#Fig5>), which
revealed a continuous solvated channel aligned with the filament axis.
Water positioning in this channel was largely similar between
ADP–F-actin and ADP-P_i –F-actin (Fig. 2a
<https://www.nature.com/articles/s41586-022-05366-w#Fig2> and Extended
Data Fig. 2e <https://www.nature.com/articles/s41586-022-05366-w#Fig6>).
An analysis of solvent-accessible pockets (Methods
<https://www.nature.com/articles/s41586-022-05366-w#Sec8>) indicates
that the filament core is accessible to bulk solvent and features small
channels that connect to the nucleotide clefts of the protomers. This
positions the filament core to contribute to extensive solvent-mediated
lateral interactions between strands, which occur at a vertex between
three subunits (Fig. 2b
<https://www.nature.com/articles/s41586-022-05366-w#Fig2> (top)). We
also observed ordered water molecules on the outer surface of F-actin
that mediate longitudinal interactions between protomers along the same
strand (Fig. 2b
<https://www.nature.com/articles/s41586-022-05366-w#Fig2> (bottom)). A
detailed analysis of specific interactions^42
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR42>,43
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR43> is
presented in the Supplementary Discussion
<https://www.nature.com/articles/s41586-022-05366-w#MOESM1>. In summary,
we found that intersubunit interfaces are extensively solvated, which we
hypothesize could lubricate mechanical rearrangements within the filament.

*Fig. 2: Water molecules mediate key longitudinal and lateral contacts
in F-actin.

*

figure 2 <https://www.nature.com/articles/s41586-022-05366-w/figures/2>

*a*, Water molecules (violet) contained within the filament core of
ADP–F-actin. Actin subunits are shown in transparent grey main-chain
representation. *b*, Solvent-mediated contacts at lateral (top) and
longitudinal (bottom) interfaces in ADP–F-actin (shades of blue) and
ADP-P_i –F-actin (shades of orange). Key side chains and backbone atoms
are displayed and coloured by heteroatom. Water molecules are shown in
violet, and putative hydrogen bonds are indicated by dashed
lines. Protomer indices are indicated.

Full size image

<https://www.nature.com/articles/s41586-022-05366-w/figures/2>


    Visualizing F-actin bending deformations

Actin filaments are exposed to thermal fluctuations and fluid forces^12
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR12>,44
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR44> during
cryo-EM grid preparation, producing visibly bent regions (Fig. 3a
<https://www.nature.com/articles/s41586-022-05366-w#Fig3>) that are
generally excluded during helical processing. Although other stably
curved protein filaments have been structurally characterized^45
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR45>,46
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR46> ,
non-equilibrium bending has not been visualized in atomistic detail. As
F-actin bending is energetically unfavourable, only a small filament
segment subpopulation should feature appreciable deformations. Moreover,
bending is a continuum, rendering it refractory to characterization by
traditional classification approaches.

*Fig. 3: Cryo-EM reconstructions of mechanically deformed F-actin reveal
bend–twist coupling.

*

figure 3 <https://www.nature.com/articles/s41586-022-05366-w/figures/3>

*a*, Representative cryo-EM micrograph of ADP–F-actin filaments
featuring high-curvature regions, low-pass filtered to 30 Å. Picked
segments are coloured by estimated curvature as indicated. Scale bar,
100 nm. *b*, Normalized curvature histograms of ADP–F-actin (blue, /n/ =
 374,942) and ADP-P_i –F-actin (orange, /n/ = 470,625) filament
segments, compared using a two-tailed Mann–Whitney /U/-test. The dashed
lines indicate curvature thresholds for straight (≤2.0 μm^−1 ) and bent
(≥2.5 μm^−1 ) segments. The colour bars correspond to the curvature key
in *a*. *c*, Helically symmetric ADP–F-actin (left map) and cryoDRGN
reconstructions sampling continuous bending of ADP–F-actin (right three
maps), low-pass filtered to 8 Å. Strands are coloured in shades of blue.
Scale bar, 10 nm. *d*, Stitched volumes of straight and bent maps from
*c*, aligned to the bottom 16 protomers. Scale bar, 100 nm. *e*,
Schematic of twist and rise measurements along a bent filament axis.
Protomer numbering is indicated. *f*, Twist and rise measurements of
cryoDRGN reconstructions sampled along the major variability component.
The solid and dashed curves correspond to measurements from even-to-odd
and odd-to-even protomer indices, respectively.

Source data <https://www.nature.com/articles/s41586-022-05366-w#MOESM10>

Full size image

<https://www.nature.com/articles/s41586-022-05366-w/figures/3>

We therefore developed a neural-network-based approach to identify bent
F-actin segments in micrographs (Methods
<https://www.nature.com/articles/s41586-022-05366-w#Sec8> and Extended
Data Fig. 3 <https://www.nature.com/articles/s41586-022-05366-w#Fig7>)
and estimate their instantaneous in-plane curvature. As anticipated,
most segments exhibited low curvature. However, a subset of filaments
featured regions of continuous curvature gradients indicative of elastic
bending (Fig. 3a
<https://www.nature.com/articles/s41586-022-05366-w#Fig3>). Consistent
with persistence length measurements^35
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR35> , segment
curvature distributions revealed significantly greater mean curvature
for ADP–F-actin compared with ADP-P_i –F-actin (ADP, 1.14 μm^−1 ;
ADP-P_i , 0.96 μm^−1 ), including a long tail of highly curved
ADP–F-actin segments (Fig. 3b
<https://www.nature.com/articles/s41586-022-05366-w#Fig3>). A minimal
Boltzmann model of thermal fluctuations approximately captures the
curvature distributions, yet systematic deviations are consistent with
fluid forces during cryo-EM sample preparation increasing the prevalence
of highly curved filaments^9
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR9>,10
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR10>,44
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR44> (Extended
Data Fig. 4 <https://www.nature.com/articles/s41586-022-05366-w#Fig8>
and Supplementary Discussion
<https://www.nature.com/articles/s41586-022-05366-w#MOESM1>).

To investigate these bending conformational landscapes, we examined
16-protomer segments featuring estimated curvatures greater than an
arbitrary 2.5 μm^−1 cut-off (hereafter, bent F-actin). After ab initio
reconstruction in cryoSPARC and subsequent processing with RELION, we
used the heterogeneity analysis tool cryoDRGN to generate volumes
spanning each dataset’s continuous bending deformations (Fig. 3c,d
<https://www.nature.com/articles/s41586-022-05366-w#Fig3>, Methods
<https://www.nature.com/articles/s41586-022-05366-w#Sec8> and Extended
Data Fig. 5 <https://www.nature.com/articles/s41586-022-05366-w#Fig9>).
CryoDRGN does not generate independent half-maps for resolution
assessment, but the maps featured well-resolved α-helices characteristic
of ~8 Å resolution. These reconstructions predominantly exhibited
in-plane bending (Fig. 3d
<https://www.nature.com/articles/s41586-022-05366-w#Fig3> and Extended
Data Fig. 5b,e
<https://www.nature.com/articles/s41586-022-05366-w#Fig9>), featuring
central axis curvatures that fell within the distributions of estimated
segment curvatures (ADP, 2.0–5.4 μm^−1 ; ADP-P_i , 3.2–4.4 μm^−1 ).
Consistently, the radii of curvature of volumes stitched from multiple
copies of these reconstructions were on the order of hundreds of
nanometres (Fig. 3d
<https://www.nature.com/articles/s41586-022-05366-w#Fig3>), similar to
bent F-actin observed in cells^7 <#ref-CR7>,8 <#ref-CR8>,9 <#ref-CR9>,10
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR10> .


    Bending remodels F-actin’s lattice

To examine lattice remodelling, we flexibly fit atomic models into these
maps using ISOLDE and measured instantaneous helical parameters (rise
and twist; Fig. 3e
<https://www.nature.com/articles/s41586-022-05366-w#Fig3> and
Supplementary Video 2
<https://www.nature.com/articles/s41586-022-05366-w#MOESM5>) along their
bent central axes (Methods
<https://www.nature.com/articles/s41586-022-05366-w#Sec8>). This
revealed striking bend–twist coupling, consistent with theoretical
predictions^37
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR37> , with
alternating overtwisting and undertwisting of strands that increased
with filament axis curvature. Symmetric twist deviations between strands
remain in-phase, retaining canonical F-actin’s twist as the
instantaneous average between odd/even protomers to maintain lattice
structural integrity. Bending evoked twist deviations up to 15°,
substantially greater than the twist deviation induced by cofilin
(~5°)^33
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR33>,34
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR34> . Each
strand’s bend–twist relationship could be modelled analytically as a
travelling wave along its component protomers (Methods
<https://www.nature.com/articles/s41586-022-05366-w#Sec8>), whereby
increasing curvature modulates the twist amplitude and phase (Fig. 3f
<https://www.nature.com/articles/s41586-022-05366-w#Fig3>, Extended Data
Fig. 6a <https://www.nature.com/articles/s41586-022-05366-w#Fig10> and
Supplementary Table 2
<https://www.nature.com/articles/s41586-022-05366-w#MOESM1>). Both
ADP–F-actin and ADP-P_i –F-actin exhibit nearly identical propagation
speeds in twist/protomer-index space, but ADP–F-actin features a
larger-amplitude coupling factor. This leads to the physical
interpretation that a given curvature produces greater twist deviations
in ADP–F-actin than ADP-P_i –F-actin, suggesting ADP F-actin’s lattice
is more deformable (Extended Data Fig. 6b
<https://www.nature.com/articles/s41586-022-05366-w#Fig10> and
Supplementary Table 2
<https://www.nature.com/articles/s41586-022-05366-w#MOESM1>).

We also observed modest rise deviations that manifest as apparent
standing rather than travelling waves, of which the amplitude also
increases with curvature (Fig. 3f
<https://www.nature.com/articles/s41586-022-05366-w#Fig3>). These rise
waves have wavelengths approximately half those of the twist waves and,
to our knowledge, they have not been theoretically modelled in detail.
Although the physical basis of rise deviations is less clear, we
hypothesize that they result from subunit deformations to accommodate
twist remodelling and increased elastic energy at intersubunit
interfaces. Consistently, the most highly curved 16-protomer models
featured nucleotide-state-dependent alterations in the distances and
angles between subdomains of longitudinally adjacent protomers along
each strand^47
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR47> (Extended
Data Fig. 6c <https://www.nature.com/articles/s41586-022-05366-w#Fig10>;
a detailed analysis is provided in the Supplementary Discussion
<https://www.nature.com/articles/s41586-022-05366-w#MOESM1>). Notably,
bent F-actin exhibited no clear systematic differences associated with
subunit flattening, the hallmark of the G-to-F transition, in either
nucleotide state, suggesting that F-actin bending has a distinct
structural landscape.

To validate the cryoDRGN results, we divided the ADP segments into three
straight bins, one low-curvature bin and one high-curvature bin, all of
approximately equivalent size. Asymmetric single-particle
reconstructions (Methods
<https://www.nature.com/articles/s41586-022-05366-w#Sec8>) of the
straight controls (7.0 Å, 7.1 Å and 7.2 Å resolution) revealed
negligible deviations from the helical parameters of canonical F-actin,
whereas the low-curvature (6.9 Å resolution) and high-curvature (6.4 Å)
reconstructions exhibited twist and rise patterns consistent with the
cryoDRGN models (Extended Data Fig. 5d,f,g
<https://www.nature.com/articles/s41586-022-05366-w#Fig9> and
Supplementary Video 3
<https://www.nature.com/articles/s41586-022-05366-w#MOESM6>).
Collectively, these data suggest that bending evokes substantial,
nucleotide-state-modulated F-actin conformational transitions.


    Nucleotide clefts feature shear strain

We next sought to visualize protomer structural deformations
accompanying filament bending. Reasoning that continuous conformational
flexibility was limiting the resolution of the 16-protomer
reconstruction, we focused on the central 7 subunits (Methods
<https://www.nature.com/articles/s41586-022-05366-w#Sec8>). Focused
refinement substantially improved the resolution of both bent F-actin
reconstructions (ADP–F-actin, 3.6 Å; ADP-P_i –F-actin, 3.7 Å; Fig. 4a
<https://www.nature.com/articles/s41586-022-05366-w#Fig4>, Extended Data
Fig. 7a–c <https://www.nature.com/articles/s41586-022-05366-w#Fig11> and
Supplementary Table 3
<https://www.nature.com/articles/s41586-022-05366-w#MOESM1>),
facilitating direct atomic model building and refinement. We also
reconstructed two straight ADP–F-actin bins featuring matched segment
numbers—both yielded 3.7 Å maps, into which we built control atomic
models (Extended Data Fig. 7
<https://www.nature.com/articles/s41586-022-05366-w#Fig11> and
Supplementary Table 3
<https://www.nature.com/articles/s41586-022-05366-w#MOESM1>).

*Fig. 4: Actin nucleotide state modulates subunit shearing during
filament bending.

*

figure 4 <https://www.nature.com/articles/s41586-022-05366-w/figures/4>

*a*, Cryo-EM maps of bent ADP–F-actin (blue) and ADP-P_i –F-actin
(orange), coloured by strand. Protomer numbering is indicated. The
dashed and solid lines represent the convex and concave sides of the
bent filament, respectively. *b*, Ribbon representation of an individual
actin protomer coloured by subdomain (left). The spheres connected by
bars indicate subdomain centroids. Right, protomer subdomain centroid
diagrams with vectors (scaled 100×), indicating subdomain-averaged
displacements from the corresponding helically symmetric model. *c*,
Plots of subdomain shear indices (representing coordinated
rearrangements) of subdomains 1 and 4 (top) and subdomains 2 and 3
(bottom). Blue lines, ADP; orange lines, ADP-P_i . The solid and dashed
lines represent even (concave side) and odd (convex side) protomers,
respectively. AU, arbitrary units. *d*, Cα representation of the
indicated protomers’ nucleotide clefts from bent ADP–F-actin and ADP-P_i
–F-actin, coloured by per-residue strain pseudoenergy. ADP (dark green),
magnesium (light green) and phosphate (orange) are shown in stick
representation. *e*, Quantification of each protomer’s nucleotide cleft
strain pseudoenergy, compared between nucleotide states and bending
conditions (top). Bottom, equivalent quantification of
solvent-accessible volume of nucleotide clefts. Data are mean ± 95%
confidence interval. /n/ = 7 (bent) and /n/ = 14 (straight). Statistical
comparison was performed using one-way analysis of variance with Tukey
post hoc analysis; /P/ values were corrected for multiple comparisons.
*f*, Cartoon model of the steric boundary mechanism for joint regulation
of F-actin by bending forces and nucleotide state.

Source data <https://www.nature.com/articles/s41586-022-05366-w#MOESM11>

Full size image

<https://www.nature.com/articles/s41586-022-05366-w/figures/4>

We compared these asymmetric models with our helically symmetric models
by superimposing their central protomers, revealing systematic
rearrangements solely in the bent models (Extended Data Fig. 7d
<https://www.nature.com/articles/s41586-022-05366-w#Fig11>). The
straight controls featured only small, randomly distributed deviations,
suggesting that the bent structures capture conformational
rearrangements rather than merely reflecting model building uncertainty
at 3.6–3.7 Å resolution. To examine the internal deformations of
individual subunits, we superimposed each protomer with a subunit from
its nucleotide-state-matched helically symmetric model and examined
subdomain-averaged Cα displacements (Fig. 4b
<https://www.nature.com/articles/s41586-022-05366-w#Fig4>, Methods
<https://www.nature.com/articles/s41586-022-05366-w#Sec8> and
Supplementary Video 4
<https://www.nature.com/articles/s41586-022-05366-w#MOESM7>). This
revealed complex displacement patterns that were dependent on a
protomer’s strand membership and lattice position, primarily
characterized by subunit shearing around the nucleotide cleft. These
rearrangements are captured by a shear index (Fig. 4c
<https://www.nature.com/articles/s41586-022-05366-w#Fig4>; the dot
product of the indicated vectors), which reveals increased shearing near
the barbed ends of our reconstructions, probably due to the local
orientation of the lattices relative to the plane of curvature. Although
subdomain displacement patterns were highly similar between ADP and
ADP-P_i (Fig. 4b
<https://www.nature.com/articles/s41586-022-05366-w#Fig4>), the shearing
magnitude was greater in ADP (Fig. 4c
<https://www.nature.com/articles/s41586-022-05366-w#Fig4>), consistent
with the higher overall average curvature of bent ADP segments and
enhanced deformability of ADP subunits.

To identify specific structural elements experiencing deformations, we
once again performed per-residue strain pseudoenergy analysis (Methods
<https://www.nature.com/articles/s41586-022-05366-w#Sec8>), which is
insensitive to rigid-body displacements. This revealed three primary
sites of strain: the H-plug (Extended Data Fig. 8a
<https://www.nature.com/articles/s41586-022-05366-w#Fig12> (left)), the
D-loop (Extended Data Fig. 8a
<https://www.nature.com/articles/s41586-022-05366-w#Fig12> (right)) and
the nucleotide cleft (Fig. 4d
<https://www.nature.com/articles/s41586-022-05366-w#Fig4>). Strain in
the H-plug and D-loop is consistent with their roles as primary
mediators of intersubunit interactions. Notably, bent ADP–F-actin
displayed significantly higher total strain in residues adjacent to the
nucleotide (7.5 Å distance cut-off) compared with bent ADP-P_i –F-actin
or the straight ADP–F-actin controls (Fig. 4d,e
<https://www.nature.com/articles/s41586-022-05366-w#Fig4>).
Consistently, solvent-accessible volume measurements (Methods
<https://www.nature.com/articles/s41586-022-05366-w#Sec8>) also revealed
expanded nucleotide clefts in the bent ADP–F-actin protomers relative to
bent ADP-P_i –F-actin and straight ADP–F-actin controls (Fig. 4e
<https://www.nature.com/articles/s41586-022-05366-w#Fig4>). We also
examined detailed rearrangements within the subunits (Extended Data
Figs. 8b <https://www.nature.com/articles/s41586-022-05366-w#Fig12> and
9 <https://www.nature.com/articles/s41586-022-05366-w#Fig13>,
Supplementary Discussion
<https://www.nature.com/articles/s41586-022-05366-w#MOESM1> and
Supplementary Videos 5
<https://www.nature.com/articles/s41586-022-05366-w#MOESM8> and 6
<https://www.nature.com/articles/s41586-022-05366-w#MOESM9>). These were
broadly consistent with actin’s nucleotide cleft functioning as a
deformable locus that coordinates mechanical rearrangements, the
rigidity of which is dependent on phosphate occupancy. This provides a
structural mechanistic explanation for actin nucleotide state’s
modulation of F-actin bending mechanics.


    Discussion

Our direct structural visualization of a mechanically regulated F-actin
conformational landscape modulated by actin nucleotide state reveals
substantial remodelling of helical lattice twist, producing substantial
rearrangements at protomer–protomer contacts. We therefore speculate
that F-actin bending could modulate binding by numerous ABPs, as their
binding sites generally span two longitudinally adjacent protomers.
Indeed, mapping the known binding sites of representative force- and
nucleotide-state sensitive ABPs on bent F-actin suggests that they are
likely to be impacted by these structural transitions (Extended Data
Fig. 10 <https://www.nature.com/articles/s41586-022-05366-w#Fig14>).
Although it is possible that nucleotide-state-sensing ABPs detect the
very small structural changes (most plausibly, the 0.3 Å rise increase)
allosterically triggered by phosphate release from ADP-P_i –F-actin^22
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR22>,27
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR27> , our
high-resolution structures suggest that this is unlikely to be the
primary mechanism. Our bent actin structures support an alternative
model in which phosphate both rigidifies F-actin (consistent with
previous studies)^33
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR33> and
modifies the structural landscape evoked by bending forces, enabling
discrimination by ABPs. This model is furthermore compatible with the
rigidification of canonical F-actin by phosphate inhibiting engagement
by ABPs that must substantially deform the lattice to bind, such as
cofilin^15
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR15>,32
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR32>,48
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR48> .

Our studies also provide insights into the mechanisms of F-actin
bending, a model for regulation of protein structure by mechanical
force. As force is a delocalized perturbation across the filament, it
has been unclear how it is transduced into conformational remodelling of
component subunits. We propose a ‘steric boundaries’ conceptual model of
mechanical regulation, wherein lattice rearrangements reposition a
subunit’s contacting neighbours (Fig. 4f
<https://www.nature.com/articles/s41586-022-05366-w#Fig4>). This
remodels the physical space available for it to occupy, inducing it to
deform to minimize clashes. The model predicts coupling between a
subunit’s lattice position, local filament curvature and its resultant
conformation, consistent with our observations. Ligand binding/chemical
modifications can also alter a subunit’s deformation landscape,
providing a framework for intersecting biochemical and mechanical
regulation. As shown here for actin and phosphate, this does not require
ligand engagement to modify the protein’s ground-state conformation, a
departure from traditional allosteric regulation that bears resemblance
to dynamic allostery^49
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR49> . However,
in the steric boundaries framework, the ligand co-regulates mechanical
deformations rather than altering the protein’s intrinsic conformational
fluctuations (a non-exclusive mode of regulation). Consistently, we
found that actin’s nucleotide cleft mediates subunit shearing
deformations, enabling its biochemical content to modulate F-actin’s
mechanics. This additionally suggests that force could modulate
F-actin’s nucleotide hydrolysis and phosphate release kinetics, as has
previously been predicted^50
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR50> . Beyond
F-actin, we anticipate the steric boundaries mechanism could also
explain joint mechanical and biochemical regulation of other
multisubunit complexes—a subject for future research.


    Methods


      Protein preparation

Chicken skeletal muscle actin was purified as previously described^51
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR51> . In
brief, 1 g of chicken skeletal muscle acetone powder was resuspended in
20 ml of G-Ca buffer (G buffer: 2 mM Tris-Cl pH 8.0, 0.5 mM DTT, 0.2 M
ATP, 0.01% NaN_3 , supplemented with 0.1 mM CaCl_2 ) and mixed by
inversion for 30 min. The suspension was centrifuged in a Beckman Ti70
rotor at 42,500 rpm (79,766/g/) for 30 min. Then, 50 mM KCl and 2 mM
MgCl_2 were added to the supernatant, containing G-actin monomers, to
stimulate F-actin polymerization for 1 h. KCl (0.8 M) was then added,
and the solution was incubated for 30 min to facilitate dissociation of
contaminants from F-actin. The solution was then centrifuged in a Ti70
rotor at 42,500 rpm (79,766/g/) for 3 h. The pellet was resuspended in 2
 ml of G-Ca buffer and incubated overnight. The mixture was then
homogenized in a Dounce chamber for 10–15 passes, consecutively sheared
through 26G and 30G needles, then dialysed in 1 l of G-Ca buffer
overnight in Spectra/Por 1 dialysis tubing (MWCO 6–8 kDa). The actin
solution was then sheared through a 30G needle again before dialysis in
1 l of fresh G-Ca buffer for another day. It was then centrifuged in a
Beckman Ti90 rotor at 70,000 rpm (187,354/g/) for 3 h. The top
two-thirds of the supernatant was then loaded onto a HiLoad 16/600
Superdex 200 column (Cytiva) for size-exclusion chromatography. Purified
G-actin was maintained in G-Ca buffer at 4 °C before use.

A Flag–GFP tagged version of human myosin VI-S1 (used to anchor actin
filaments to coverslips for cofilin severing assays) was purified as
described previously^24
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR24> ,
flash-frozen in liquid nitrogen and maintained at −80 °C. Lyophilized
human cofilin 1 was purchased from Cytoskeleton (CF01) and reconstituted
in MB buffer (20 mM MOPS pH 7.4, 5 mM MgCl_2 , 0.1 mM EGTA, 50 mM KCl, 1
 mM DTT), then incubated overnight at 4 °C . Aliquots were then
flash-frozen in liquid nitrogen and maintained at −80 °C. Aliquots (20
 μg) of lyophilized rhodamine actin (Cytoskeleton AR05) were resuspended
in 18 μl of G-Ca buffer and 2 μl of MilliQ water, incubated at 4 °C for
at least 1 h, then clarified by ultracentrifugation in a Beckman TLA100
rotor at 100,000 rpm (335,400/g/) for 20 min.


      Cofilin severing assays

Glass coverslips (Corning 22 × 50 mm, 1½) were cleaned for 30 min using
100% acetone, 10 min using 100% ethanol and 2 h using 2% Hellmanex III
liquid cleaning concentrate (HellmaAnalytics) in a bath sonicator
followed by rinsing with MilliQ water. The cleaned glass coverslips were
coated with 1 mg ml^−1 mPEG5K-Silane (Sigma-Aldrich) in a 96% ethanol,
10 mM HCl solution for at least 16 h with rocking. After coating, the
coverslips were rinsed with 96% ethanol and water, then air-dried and
stored at 4 °C until use.

Rhodamine-labelled ADP–F-actin (20%) was prepared by diluting unlabelled
G-actin and rhodamine G-actin stocks to 0.9 μM and 0.1 μM, respectively,
in KMEH buffer (50 mM KCl, 1 mM MgCl_2 , 1 mM EGTA, 10 mM HEPES pH 7.0,
1 mM DTT) supplemented with G-Mg (G Buffer + 0.1 mM MgCl_2 ), for a
final actin concentration of 1 μM. The mixture was incubated at 25 °C
for 1 h, then 4 °C overnight. Before use, the F-actin was pelleted by
centrifugation at 60,000 rpm (120,744/g/) in a TLA100 rotor for 20 min
and resuspended in fresh KMEH to remove any free phosphate ions.
Rhodamine-labelled ADP-P_i –F-actin (20%) was prepared as described
above, except KMEH + G-Mg was supplemented with 15 mM K_2 HPO_4 (pH
 7.0). The mixture was incubated at 25 °C for 1 h, then placed on ice
and used immediately. Rhodamine-labelled ADP–F-actin (20%) in the
presence of K_2 SO_4 was prepared identically, except 15 mM K_2 SO_4 (pH
 7.0) was substituted for K_2 HPO_4 .

For preparation of total internal reflection fluorescence (TIRF)
samples, a PDMS gasket (Grace Bio-Labs, 103380) was placed onto the
cover slip and 20 μl of 0.25 μM rigor myosin VI S1 in MB buffer was
added to the well and incubated for 2 min, followed by blocking with 20
 μl of 0.1% polyvinylpyrrolidone (Sigma-Aldrich, 9003-39-8) in MB buffer
for 1 min. Next, 20 μl of 1 μM F-actin was added to the well for 30 s.
The actin was then washed with 20 μl MB buffer (for ADP–F-actin) or MB
buffer + 15 mM KH_2 PO_4 /K_2 SO_4 (for ADP-P_i –F-actin/ADP–F-actin in
the presence of sulfate).

TIRF videos were recorded using Nikon’s NIS-Elements software at either
a 1 s or 2 s frame rate on a Nikon H-TIRF system using a CFI Apo ×60
TIRF oil-immersion objective (NA 1.49), a quad filter (Chroma) and an
iXon EMCCD camera (Andor). Rhodamine was excited by a 561 nm laser.
Filaments were initially imaged for 2 min, the video was paused, and 20
 μl of 2 μM cofilin in MB buffer (1 μM final concentration) was added to
the well. The video was then resumed and filament severing was recorded
for an additional 8 min.


      Cofilin severing quantification

Videos were analysed using custom Python scripts that measured the
change in filament intensity over the course of the experiments. Video
regions containing F-actin were identified and masks were generated by
operating on a projection of the first 50 frames of the video using the
functions in the scikit-image Python package^52
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR52> . This
projection’s background (computed using a rolling ball radius of 50
 pixels) was subtracted and subjected to a gaussian blur with a filter
size of 2 pixels. A Li adaptive threshold was used to binarize the
projection, morphological objects with an area smaller than 100 pixels
were removed and the remaining binarized image was dilated by 1 pixel.
This set of masks for each video was applied to all frames of the video,
and quantification of actin intensity was performed on a per-mask basis.

For each mask, the summed pixel intensity was measured for each frame
and normalized by dividing by the 90th percentile intensity. The maximum
intensity was not used for normalization because the intensity often
spiked with the addition of buffer/cofilin at time 0 s. The intensity
traces for each mask of each video of the same experimental condition
were pooled and their average was plotted (Extended Data Fig. 1a,b
<https://www.nature.com/articles/s41586-022-05366-w#Fig5>).


      Cryo-EM grid preparation

ADP-P_i –F-actin was prepared as described above (without incorporation
of rhodamine actin), then diluted to 0.5 μM in KMEH + 15 mM KH_2 PO_4
supplemented with 0.01% Nonidet P40 (NP40) substitute (Roche), an
additive that we have found improves our ability to achieve thin
vitreous ice films. Solution (3 μl) was applied to a plasma-cleaned
C-flat 1.2/1.3 holey carbon Au 300 mesh grid (Electron Microscopy
Sciences) in a Leica EM GP plunge freezer operating at 25 °C. After
incubation for 60 s, the grid was blotted from the back using a Whatman
no. 5 filter paper for 4 s, then flash-frozen in liquid ethane.

The ADP–F-actin sample corresponds to a pre-existing dataset described
in a recent study^41
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR41> .
ADP–F-actin was prepared as described above, except KH_2 PO_4 was
omitted and KMEI buffer (50 mM KCl, 1 mM MgCl_2 , 1 mM EGTA, 10 mM
imidazole pH 7.0, 1 mM DTT) + 0.01% NP40 substitute was used instead of
KMEH.


      Cryo-EM data collection

ADP–F-actin and ADP-P_i –F-actin datasets were collected on the same FEI
Titan Krios system operating at 300 kV and equipped with a Gatan
K2-Summit direct electron detector using super-resolution mode. Videos
were recorded using the SerialEM software suite^53
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR53> at a
nominal magnification of ×29,000, corresponding to a calibrated pixel
size of 1.03 Å at the specimen level (super-resolution pixel size of
0.515 Å per pixel). Each 10 s exposure was dose-fractionated across 40
frames, with a total electron dose of 60 e^−  Å^−2 (1.5 e^−  Å^−2 per
frame), with defocus values ranging from −1.5 to −3.5 μm underfocus. For
the ADP-P_i –F-actin dataset, beam-image shift was used to collect 4,834
single exposures from nine holes in a 3-by-3 grid per each stage
translation. For the ADP–F-actin dataset, which has previously been
reported^41
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR41> and was
reprocessed here, 1,548 exposures were directly targeted using stage
translations with a single exposure per hole.


      Micrograph pre-processing

Movies were aligned with MotionCor2 using 5 × 5 patches^54
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR54> , and
dose-weighting sums^55
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR55> were
generated from twofold binned frames with Fourier cropping, resulting in
a pixel size of 1.03 Å in the images. Non-dose-weighted sums were used
for contrast transfer function (CTF) parameter estimation using
CTFFIND4^56 <https://www.nature.com/articles/s41586-022-05366-w#ref-CR56> .


      Synthetic dataset generation

Accurate nanoscale curvature measurements of F-actin in noisy cryo-EM
micrographs requires high-quality, pixel-wise image segmentation.
Traditional cross-correlation-based approaches for filament particle
picking use templates derived from 2D class averages or projections of a
straight F-actin map. This strategy features limitations, notably that
cross-correlation will be lower between straight templates and highly
curved filament segments in experimental images. Moreover,
discrimination of filaments from background or non-protein signal may be
poor. To achieve high-quality semantic segmentation, we implemented a
convolutional neural-network-based approach to identify filament
segments of all curvatures. Although other machine-learning-based
pickers have recently been introduced^57 <#ref-CR57>,58 <#ref-CR58>,59
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR59> , to our
knowledge, they do not explicitly focus on detecting or flagging
instantaneous curvature within a filamentous assembly. From semantically
segmented micrographs, we identified filaments and measured their
instantaneous in-plane 2D curvature.

To train the neural networks, synthetic pairs of noisy and noiseless
projections that mimicked experimental data were used. Plausible 3D
synthetic models of F-actin bent around a circular central axis were
generated using a custom Python script that loaded and operated on
Protein Data Bank (PDB) models using functions from the ProDy package^60
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR60> .
Individual actin protomers were treated as rigid objects and positioned
using a toroidal helix function:

$$\gamma (t)=(\begin{array}{c}r\,\cos (\omega t+\phi )+{d}_{1}\\ (\cos
(c\,t\,/\,R))(r\,\sin (\omega t+\phi )+{d}_{2}+R)-R\\ (\sin
(c\,t\,/\,R))(r\,\sin (\omega t+\phi )+{d}_{2}+R)+{d}_{3}\end{array}),$$

where the parameters are defined as follows: /γ/ is the position in 3D
space along the toroid, /r/ is the filament radius, /ω/ is the average
twist, /t/ is the parameterized position along the helical curve, /φ/ is
the phase of the twist, /d/_1 , /d/_2 , and /d/_3 are the displacements
of the toroid from the origin, /c/ is the rise parameter and /R/ is the
toroid’s radius of curvature. Note that this function converges to a
canonical F-actin helix when the curvature is zero. Furthermore, the
equation does not explicitly encode emergent architectural remodelling
phenomena such as twist–bend coupling. Using this synthetic filament
generation scheme, a library of 135 bent actin models consisting of 35
protomers and of systematically varying curvature and rotation about the
central filament axis were generated. These models were then converted
to volume files using the PDB2MRC function in EMAN2^61
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR61> . These
volumes were saved within 256-voxel boxes (voxel size, 4.12 Å). The
volumes were rotated about the phi and rot angles by a random, uniformly
sampled value between 0° and 359°, and the tilt was randomly sampled
from a Gaussian probability distribution centred at 90° with a s.d. of
2.5°, then randomly translated around the box by ±250 Å and projected
along the /z/ axis to generate a noiseless projection. A paired noisy
projection was generated by adding pink noise in Fourier space, as
implemented in EMAN2’s Python package to generate realistic-looking
synthetic data^61
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR61> . These
projection images were cropped to a smaller box size of 128 pixels to
ensure that filaments would span the image. Two-channel stacks of
semantic maps associated with the noisy–noiseless projection pairs were
generated by low-pass filtering the noiseless projection to 40 Å and
binarizing it.


      Network architecture and training

A denoising autoencoder (DAE) was trained using the architecture
outlined in Extended Data Fig. 3a
<https://www.nature.com/articles/s41586-022-05366-w#Fig7>. Each
trainable layer had a ReLU activation function, except for the final
layer, which had a linear activation function. The negative of the
cross-correlation coefficient was used as the loss function. For
training, the weights were initialized using the default initialization
in TensorFlow^62
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR62> . The
model was trained using the Adam optimizer version of stochastic
gradient descent with a learning rate of 0.00005 and minibatch size of
16 until the model converged (no improvement in validation loss for 3
epochs). After network convergence, the weights from the best epoch were
restored. For training, 800,000 noisy–noiseless projection pairs with
box sizes of 128 × 128 were generated, 90% of which were used for
training and 10% for validation. After network convergence, the DAE had
an average cross-correlation coefficient of 0.9887 on the validation set.

After training the model as a DAE, a semantic segmentation network was
trained by copying the convolutional encoding layers and weights of the
DAE while adding convolutional layers. The final layer was a two-channel
layer with sigmoid activation and default TensorFlow initialization.
This semantic segmentation network was then trained with a learning rate
of 0.001. For training, 30,000 pairs of noisy inputs and semantically
segmented targets of dimension 128 × 128 and 128 × 128 × 2,
respectively, were used with a minibatch size of 32; 90% of the
synthetic data were used for training and 10% for validation. The loss
function was binary cross-entropy and, after network convergence, the
model had a loss of 0.0651 on the validation set. Example network
performance on synthetic data is shown in Extended Data Fig. 3
<https://www.nature.com/articles/s41586-022-05366-w#Fig7>.

Models were trained on a single NVIDIA Titan XP GPU with 12 GB of VRAM.
Training required approximately 1 h per epoch for the DAE and 3 min per
epoch for the semantic segmentation network. After initiating this
project, we continued developing deep-learning-based filament particle
pickers. The architectures described here have been superseded by a
U-net architecture, which we found produces better segmentation with a
smaller training set in a shorter time^63
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR63> .


      Particle picking

A custom Python script was used to pass images to the fully
convolutional neural network for semantic segmentation (FCN-SS) and
execute curvature-sensitive filament picking. Each micrograph was binned
by 4 to a pixel size of 4.12 Å per pixel, then 128-pixel tiles featuring
32 pixels of overlap were extracted and passed as inputs to the network.
The outputs were stitched together by maximum intensity projection at
the overlaps, producing a semantic segmentation map of the micrograph.
These maps were then binarized using a fixed, empirically determined
threshold of 0.9 and skeletonized. Branches shorter than 8 pixels were
pruned, and pixels within a radius of 48 pixels from filament
intersections were removed. Continuous filaments were then identified by
matching tracks with common end points, and 2D splines were fit through
the filaments for curvature estimation. To prevent spuriously high
curvature values due to edge effects, the terminal 50 pixels of the
spline were omitted from picking. From the remaining filament sections,
the instantaneous curvature was measured along the spline at 56 Å
intervals (corresponding to a step size of the length of one protomer)
and used for segment selection. For picking segments from the identified
filaments for asymmetric reconstructions, a step size equivalent to two
short-pitch helical rise steps (56 Å) was used for extracting segments.
Filament segments with a curvature greater than or equal to 2.5 μm^−1
were considered to be bent, whereas those with a curvature of less than
or equal to 2.0 μm^−1 were considered to be straight. To select segments
from the identified filaments for high-resolution helical
reconstructions, a step size of three times the helical rise was used
(83.4 Å), and segments that were members of the same filament were
flagged in the output metadata (a RELION-formatted STAR file).


      Helical image processing

High-resolution reconstructions were determined using the iterative
helical real space refinement^64
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR64> approach
as implemented in RELION^65
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR65> . We
reprocessed our ADP–F-actin dataset that previously produced a map at
2.8 Å resolution^41
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR41> , and the
ADP-P_i –F-actin dataset from this work in parallel. Our
neural-network-based picker eliminated intersections where filaments
overlapped. After initial picking, particles were extracted without
binning in 512-pixel boxes using RELION, with 81 Å (3 protomers) of
non-overlap. Our picking scheme did not include psi (in-plane rotation)
angle estimates, so an initial refinement was performed with global
angular searches and a bare actin reference (EMD-24321) low-pass
filtered to 35 Å. After this initial alignment, the psi angles were
changed to a psi prior, all poses were removed from the metadata file
and the tilt prior was set to 90. This dataset was then processed for a
previously described F-actin cryo-EM data processing workflow^23
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR23>,41
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR41> in
RELION-3.1^66
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR66> , with
minor modifications described below. In brief, initial 2D classification
was performed to remove junk particles (only 0.4% of picked particles
for ADP–F-actin and 11.2% of particles for ADP-P_i –F-actin), followed
by 3D classification with alignment and five classes. For alignment, a
search range of 15° around the tilt and psi priors was used, and global
searches of the rot angle with an angular sampling of 7.5° was used. For
ADP–F-actin, no particles were excluded at the 3D classification stage
because all five classes were of high quality. For ADP-P_i –F-actin, two
classes comprising 23% of the remaining segments (128,533 segments
total) were excluded because their helical parameters were at the border
of the search range and the classes appeared to be abnormal. Selected
particles were then processed for unmasked 3D auto-refinement using the
same angular search range described above (with local angular sampling
starting at 1.7°) with helical symmetry searches as implemented in
RELION-3.1. This yielded a map at 4.2 Å-resolution for ADP–F-actin and
4.1 Å-resolution map for ADP-P_i –F-actin. Post-processing was performed
using a loose mask trimmed to 50% of the box size along the helical axis
(/z/-length), which resulted in a resolution of 3.4 Å for ADP–F-actin
and 3.5 Å for ADP-P_i –F-actin.

Then, several iterative rounds of CTF refinement, Bayesian polishing and
3D auto-refinement were performed. For both datasets, CTF refinement was
initially performed by estimating the anisotropic magnification for each
optics group. Defocus values were next fit on a per-particle basis and
astigmatism was fit on a per-micrograph basis, along with beam tilt
estimation. For the ADP-P_i –F-actin dataset, because beam-image shift
was used during data collection, the data were processed in nine optics
groups. Only a single optics group was used for the ADP–F-actin dataset,
which was collected with stage translations. After CTF refinement,
Bayesian polishing was used to improve the data’s motion correction on a
per-particle basis. The initial helical parameters for RELION’s symmetry
search were updated and the mask used for post-processing was used to
run another round of 3D auto-refinement. This process was repeated using
a 30% /z/-length mask, including estimation of trefoil and fourth order
aberrations during CTF refinement. After the second round of particle
polishing, a third round of CTF refinement was performed.

After the last iteration of CTF refinement, estimated defocus values
were smoothed over the length of each continuous filament, similar to
previously reported approaches^46
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR46> . Finally,
a single round of masked refinement using a 30% /z/-length mask with
local angular and translational searches was performed using
solvent-flattened Fourier shell correlation (FSC) resolution assessment.
The final reconstructions converged with resolutions of 2.43 Å for
ADP–F-actin and 2.51 Å for ADP-P_i –F-actin.


      Image processing of bent F-actin

Selected bent segments were extracted in RELION with a box size of 512 ×
 512 pixels and pixel size of 1.03 Å per pixel (bin 1), initially with
filament overlap. To avoid reference bias, ab initio initial model
generation was performed using cryoSPARC^67
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR67> (v.2.11.0)
using the subset of ADP–F-actin segments with an estimated curvature of
greater than 4.0 μm^−1 . Subsequent homogeneous refinement of these
particles in cryoSPARC produced an asymmetric map with clear curvature.
The data were then imported into RELION-3.0, and 2D classification
without alignment was performed to remove junk particles, followed by
unsupervised 3D classification with three classes using global angular
searches. Two clearly bent, low-resolution classes curved in opposite
directions and one junk class were produced. The particles in the bent
classes were then processed for supervised classification using the two
bent classes as references and one straight F-actin reference as a decoy
using global angular alignment. Only 0.3% of particles were assigned to
the decoy, consistent with the selected segments almost exclusively
featuring bent F-actin. Alignment of the two bent classes revealed that
they were nearly identical but displaced by one protomer, making them
appear to bend in opposite directions. Their particles were therefore
pooled for homogeneous refinement in cryoSPARC using global searches and
the first bent class as a reference, low-pass filtered to 30 Å. These
particles were then reimported to RELION, and underwent 3D
auto-refinement using the cryoSPARC map low-pass filtered to 10 Å as a
reference, local angular searches, a loose 70% /z/-length mask and
solvent-flattened FSCs. This process was repeated to generate a
less-bent map from segments with measured curvatures in the 2.5 μm^−1 to
4.0 μm^−1 range.

After demonstrating the feasibility of reconstructing bent filaments,
all segments with a curvature above 2.5 μm^−1 were then processed for
homogeneous refinement in cryoSPARC using the highly bent RELION
refinement result as an initial reference, low-pass filtered to 30 Å.
This was done separately for ADP–F-actin and ADP-P_i –F-actin in
parallel. The data were then reimported into RELION for masked 3D
auto-refinement using local searches. Successive rounds of CTF
refinement, Bayesian particle polishing and 3D auto-refinement using a
70% /z/-length mask were performed until resolution improvement
plateaued. For the bent ADP–F-actin dataset, four rounds of CTF
refinement and three rounds of Bayesian polishing were performed. For
the bent ADP-P_i –F-actin dataset, three rounds of CTF refinement and
two rounds of Bayesian polishing were performed. From this stage, the
data were either processed for high-resolution asymmetric analysis or
continuous conformational variability analysis.

For high-resolution analysis, segment overlap within 360 pixels
(corresponding to 7 protomers) was removed and the particles were
processed for a final masked (70% /z/-length) 3D auto-refinement with
local searches and solvent-flattened FSC calculations. Resolution
anisotropy of these maps was assessed with the 3DFSC server^68
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR68> . For
continuous conformational variability analysis, segment overlap within
the entire 512-pixel box (corresponding to 16 protomers) was removed
before final masked (90% /z/-length) 3D auto-refinement (also with local
searches and solvent-flattened FSC calculations). These segments and
their assigned poses were then used for training of neural networks in
cryoDRGN^69
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR69> to assess
conformational variability.

Asymmetric ADP–F-actin straight controls were generated using a similar
method. All filament segments with a measured curvature less than or
equal to 2.0 μm^−1 were subjected to ab initio map generation and
homogeneous refinement in cryoSPARC. They were then imported to RELION
for subsequent rounds of local 3D auto-refinement, CTF refinement and
Bayesian polishing as described above. Then, all segment overlap was
removed within 360 pixels, and two random subsets of sufficient size to
generate maps of comparable resolution to the bent asymmetric maps were
generated. These two subsets of particles underwent a final local 3D
auto-refinement as described for the bent asymmetric reconstructions,
and the resulting ~3.7 Å maps were used as controls for model building
and analysis.


      Variability analysis of bent F-actin

To perform variability analysis on the consensus, 16-protomer asymmetric
bent reconstructions, the particles were downsampled by 2 to a box size
of 256 and a pixel size of 2.06 Å. Two cryoDRGN neural networks were
trained, one for the bent ADP–F-actin dataset and one for the bent
ADP-P_i –F-actin dataset. In both cases, the network had a variational
auto-encoder architecture of seven 1,024-dimensional encoding layers and
seven 1,024-dimensional decoding layers with a 10-dimensional latent
space. All of the other parameters were set to the default. The networks
were trained for 40 epochs. Using four NVIDIA Titan XP GPUs, the average
epoch time was ~12 min. Principal component analysis of the individual
particle embeddings in the trained 10-dimensional latent space revealed
the major variability present in the dataset to be flexible
conformational heterogeneity due to bending deformations. Predicted
reconstructions sampled along this trajectory in the latent space were
used in subsequent analysis.


      High-resolution atomic models

Before model building and refinement, maps were processed for density
modification with phenix.resolve_cryo_em^70
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR70> , using
only the maps as inputs, then resampled onto a grid featuring 0.2575 Å
voxels through fourfold Fourier unbinning with the program resample.exe
(distributed with FREALIGN^71
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR71> ).

Our previous model of bare ADP–F-actin (PDB: 7R8V
<https://doi.org/10.2210/pdb7R8V/pdb>) was copied and rigid-body fit
into the central three protomers in the ADP–F-actin and ADP-P_i –F-actin
maps using UCSF Chimera^72
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR72> . Atomic
models for each nucleotide state were then built and refined
independently. Manual adjustments were made to the central protomer
using Coot^73
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR73> , and the
other two chains were replaced with this updated protomer. These models,
containing three actin protomers with associated ADP, and Mg^2+ (and
PO_4 ^3− ) ligands were refined using PHENIX real-space refinement^74
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR74> with
non-crystallographic symmetry (NCS) restraints. After real-space
refinement in phenix, initial solvent water molecules were placed using
phenix.douse^75
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR75> with the
mean scale parameter set to 0.4. Approximately 140 water molecules per
protomer were initially placed with this automatic function. The maps
and models were then manually inspected in Coot, and water molecules
were added or pruned. As the map resolution decreased radially from the
filament core, phenix.douse was unable to reliably detect water peaks in
all of the map regions using a single threshold. After manual
adjustments, all water molecules outside a central slab 28 Å along the
filament axis (the approximate span of a single helical rise) were
deleted. The water molecules within the slab were then symmetrized to
make an 84-Å-long slab containing waters, and two protomer chains were
added to the model to fully satisfy all neighbour contacts for the
central protomer. Water molecules were then associated with the closest
protomer. This central protomer was copied twice and each protomer with
its ligands were then fit as rigid bodies into the map to form a new
trimer model. Each water molecule was then manually inspected in Coot
and adjusted to fit into the map density if needed. A final PHENIX
real-space refinement was performed with NCS restraints on the protein
chains but not the solvent waters. A summary of validation statistics is
provided in Supplementary Table 1
<https://www.nature.com/articles/s41586-022-05366-w#MOESM1>.


      Atomistic models from variability analysis

From each cryoDRGN frame, 16 copies of the corresponding helically
symmetric central actin protomer model were rigid-body fit into the
central 16 protomer sites in the map and combined into a single model.
The map and initial model were then adjusted using the molecular
dynamics flexible fitting^76
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR76> -based
modelling software ISOLDE^77
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR77>
implemented in ChimeraX^78
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR78> , using
secondary structure distance and torsional restraints. Owing to the
relatively low resolution of the cryoDRGN predictions (by visual
inspection estimated to be ~8 Å), the map weight was reduced to 10% of
the automatically determined weight. The simulation temperature was set
to 120 K, and the flexible fitting simulation was run for five real-time
clock minutes before lowering the simulation temperature to 0 K and
stopping it. These models were subsequently used for measurement of
helical parameters and subdomain distance/angle measurements.


      Bent and control F-actin atomic models

Before model building, maps were processed for standard post-processing
using RELION. Models were built into the ~3.6 Å asymmetric
reconstructions by rigid-body fitting the central protomer from the
corresponding helically symmetric model into each of the seven central
protomer sites in the map along the filament length. Initial,
large-scale adjustments to the model were performed using ISOLDE. For
each condition, the map and model were loaded without applying any
restraints, and the simulation temperature was set to 120 K. The
simulation was then run for 5 min before lowering the simulation
temperature to 0 K and ending the simulation. These models were then
processed for PHENIX real-space refinement without using NCS. A summary
of validation statistics is provided in Supplementary Table 3
<https://www.nature.com/articles/s41586-022-05366-w#MOESM1>.


      Extended F-actin model for rise analysis

The extended 31-protomer helically symmetric actin filament models for
ADP–F-actin and ADP-P_i –F-actin were generated using UCSF Chimera as
previously described^41
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR41> . Starting
from the modelled actin trimer, a copy was generated and the two
terminal protomers on the barbed end were superimposed onto the two
terminal protomers on the pointed end. These two protomers of the newly
generated trimer were deleted, and the remaining protomer was combined
with the original model, extending it by one protomer. This process was
repeated iteratively until a 31-protomer filament was generated.


      Central pore and nucleotide cleft analysis

The CASTp web server^79
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR79> was used
to identify continuous solvent-accessible pockets within the
high-resolution helically symmetric F-actin structures. To eliminate
boundary effects, each model was extended to five protomers as described
above. Using an initial probe size of 1.4 Å revealed a
solvent-accessible core that connected through narrow channels to the
nucleotide pocket and broader channels to the filament’s exterior and
bulk solvent. Increasing the probe size to 1.6 Å isolated the central
solvent channel from these pockets. Water molecules contained within
this discrete pocket at the filament’s core are displayed in Fig. 2a
<https://www.nature.com/articles/s41586-022-05366-w#Fig2> and Extended
Data Fig. 2e <https://www.nature.com/articles/s41586-022-05366-w#Fig6>.

The CASTp server was also used with a probe size of 1.4 Å to measure the
volume of the solvent-accessible nucleotide pocket in the asymmetric,
seven-protomer F-actin models. The nucleotide pocket volume of
individual protomers from each model was measured to separate the
nucleotide pocket from the filament’s central core.


      Analytical modelling of thermal fluctuations

Boltzmann modelling of thermal bending fluctuations was performed as
described in the Supplementary Discussion
<https://www.nature.com/articles/s41586-022-05366-w#MOESM1>. Residuals
between the modelled curves and experimental curvature histograms (Fig.
3b <https://www.nature.com/articles/s41586-022-05366-w#Fig3>) were
calculated by computing the difference between the modelled probability
distribution and the histogram heights at the centre of each 0.2 μm^−1
bin. An adjusted Boltzmann distribution with a multiplicative constant
/α/ was fit to each dataset by minimizing the sum of the squared
residuals between the adjusted model and the data, converging to /α/
values of 0.80 for ADP–F-actin and 0.93 for ADP-P_i –F-actin. Model
distributions featuring integer persistence lengths between 5 and 15 μm
were additionally calculated to visualize the effects of varying
persistence length on the curvature distribution.


      Rise, twist and curvature measurements

Rise and twist were measured along deformed actin filament axes using
custom Python scripts, implementing an approach similar to previously
described helix deformations^80 <#ref-CR80>,81 <#ref-CR81>,82
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR82> . First, a
central axis was defined using a 3D spline fit. To minimize edge
effects, the model was extended by copying the model twice, aligning the
terminal three subunits of one copy’s barbed end with the terminal three
subunits of the original model’s pointed end and then aligning the
terminal three subunits of the other copy’s pointed end with the
terminal three subunits of the original model’s barbed end. The
overlapping subunits from the copied models were deleted to generate
a final, 42-protomer model. This was used to define the central spline
while sufficiently minimizing edge effects.

To define the 3D spline for each filament’s central axis, an iterative,
orientation-independent approach was implemented using a set of
waypoints. The initial 41 waypoints were defined as the set of 3D
coordinates corresponding to the centroid of 2 consecutive subunits in a
rolling window along the filament. A 3D cubic spline with a natural
boundary condition was then fit through the set of waypoints to generate
the initial filament axis. A set of line segments with a length equal to
the filament’s radius and one end positioned at each subunit’s centroid
was aligned to minimize the free end’s distance from the spline. The
waypoints were then updated to become the Euclidean average of two of
these consecutive free ends. The process of updating the 3D cubic
spline, defining new line segment extensions from the subunit centroids,
and updating waypoints was repeated 500 times to obtain the final
central axis spline.

Rise was measured by computing the distance travelled along the path of
the central axis spline between protomer centroids. For each protomer,
the point on the central axis that was the closest to the subunit’s
centroid was stored, and the distance along the spline path was
calculated to the next protomer. The twist between protomers along the
deformed short-pitch F-actin helix was measured in the context of the
moving Frenet–Serret frame of reference. The Frenet–Serret frame of
reference was defined by the orthonormal basis of the unit tangent, unit
normal and unit binormal vectors along the length of the spline. The set
of unit tangent vectors sampled at the positions along the 3D cubic
spline corresponding to each subunit (as determined during the rise
measurements) was calculated. A vector with a magnitude equal to the
filament’s radius oriented along the normal axis in the Frenet–Serret
frame and its tail at the origin in the Frenet–Serret frame was then
rotated in the normal-binormal plane until the distance between its head
and the corresponding subunit centroid was minimized. This rotation
angle defined the absolute angular twist for the protomer. To measure
the twist along the short-pitch helix, the difference between
consecutive absolute angular twists was calculated.


      Travelling-wave analytical model

Inspection of instantaneous twist versus protomer index plots for bent
filaments revealed a clear sinusoidal pattern along each strand.
Furthermore, as the curvature increased along the cryoDRGN trajectory,
both the magnitude and position of this sinusoidal pattern changed. We
therefore modelled the bend–twist phenomenon for each strand as
travelling waves using the equation:

$$u(x,t)=Ax{\rm{\sin }}(kx+\omega t+\phi )+B,$$

where /u/(/x/,/t/) is the instantaneous twist, /A/ is a coupling factor
between curvature and twist amplitude, /k/ is the propagation factor
that determines how rapidly the twist wave travels along the filament’s
length with bending, /x/ is the curvature, /ω/ is the period of twist,
/t/ is the position along the central axis (parameterized to protomer
index), /φ/ is the phase shift of twist and /B/ is the overall average
twist. This equation was jointly fit against the measured twist values
and estimated curvatures for each of the 16-protomer cryoDRGN models.
Curvature for each model was measured as the average of the
instantaneous 3D curvatures of the central axis spline. For the ADP
nucleotide state, all frames were used. For the ADP-P_i state, the first
three frames had low curvature and had fluctuating curvature
measurements, so they were omitted due to the inaccurate average
curvature measurement of the central spline. The fit values for the
model parameters are presented in Supplementary Table 2
<https://www.nature.com/articles/s41586-022-05366-w#MOESM1>, and example
fit functions through experimental data are presented in Extended Data
Fig. 6a,b <https://www.nature.com/articles/s41586-022-05366-w#Fig10>.


      Analysing central axis deformations

Analysis of the bending deformations of filaments were performed on the
central axes of the 16-protomer cryoDRGN models. Principal component
analysis was performed on the coordinates sampled along the axis spline
in Euclidean space. The plane defined by the first and second principal
components represents the plane of maximum filament curvature. The plane
formed by the first and third principal components represents an
orthogonal plane capturing the 3D character of the bent filament.
Central axis deformation was also analysed by measuring the deviation of
the central axis from a straight line fit. For each curved cryoDRGN
model, a straight line was aligned to the terminal 56 Å of its central
axis at the barbed end. Discrete 0.28 Å sampling steps were then made
along the central axis, and the distance from the sampled point and the
straight line was plotted in Extended Data Fig. 5e
<https://www.nature.com/articles/s41586-022-05366-w#Fig9>.


      Actin subdomain measurements

Actin subdomains were defined using previously established residue
assignment conventions^83
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR83> :
subdomain 1 (SD1): amino acids 5–32, 70–144, 338–375; SD2: amino acids
33–69; SD3: amino acids 145–180, 270–337; SD4: amino acids 181–269.
Using a custom Python script, the Euclidean distances, angles, and
dihedral angles between subdomains indicated in Extended Data Fig. 6a
<https://www.nature.com/articles/s41586-022-05366-w#Fig10> were
measured. The protomer indexing started at the pointed end and
progressed to the barbed end. For measurements that spanned multiple
protomers, the protomer index corresponded to the most pointed-end protomer.


      Subunit shear measurements

For shear measurements, each protomer of the asymmetric F-actin models
was aligned to the protomer of the corresponding high-resolution,
helically symmetric model of the same nucleotide state. The average
displacement vector for each subdomain between these models was then
computed. Observing anti-correlated displacements between non-adjacent
subdomains led us to define two shear indices to describe these
coordinated deformations: shear index 1, the dot product of the
subdomain 1 and 4 displacement vectors, and shear index 2, the dot
product of the subdomain 2 and 3 displacement vectors. Shear indices of
pairs of subdomain displacement vectors that have large individual
magnitudes and opposing directionality will have large negative values,
indicative of shear, whereas small displacements or lack of correlated
subdomain displacements will produce values near zero.


      Strain analysis

To quantify protein deformations not explained by rigid-body motions,
strain pseudoenergy analysis was performed using a custom Python script,
implementing a previously described approach^84
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR84>,85
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR85> . In
brief, a reference helical F-actin protomer was rigid-body fit into each
protomer of the model to which it was being compared. The local
deformation matrix within an 8 Å neighbourhood was estimated for each
alpha-carbon of the reference protomer. The Eulerian strain tensor is
computed using a first order approximation of the deformation matrix’s
spatial derivative. The shear strain energy is then calculated directly
from this strain tensor. This approach to protein deformation has the
major advantage of being rotationally invariant and distinguishing
rigid-body motions from internal deformation. However, the local
deformation estimation can be inaccurate for very large deformations,
which limited our strain analysis to individual protomers. Furthermore,
the first-order approximation assumes continuous, as opposed to
granular, deformations, which makes the measurements relative
pseudoenergies.


      Plots, statistics and molecular graphics

Plots were generated using GraphPad Prism or Matplotlib^86
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR86> .
Statistical tests were performed using GraphPad Prism. Molecular
graphics were prepared using UCSF Chimera^72
<https://www.nature.com/articles/s41586-022-05366-w#ref-CR72> and UCSF
ChimeraX^78 <https://www.nature.com/articles/s41586-022-05366-w#ref-CR78> .


      Reporting summary

Further information on research design is available in the Nature
Research Reporting Summary
<https://www.nature.com/articles/s41586-022-05366-w#MOESM2> linked to
this article.


    Data availability

Cryo-EM density maps and corresponding atomic models have been deposited
in the PDB and EMDB with the following accession codes: helically
symmetric ADP–F-actin (PDB: 8D13 <https://doi.org/10.2210/pdb8D13/pdb>,
EMDB: EMD-27114 <https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-27114>);
helically symmetric ADP-P_i –F-actin (PDB: 8D14
<https://doi.org/10.2210/pdb8D14/pdb>, EMDB: EMD-27115
<https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-27115>); asymmetric bent
ADP–F-actin (PDB: 8D15 <https://doi.org/10.2210/pdb8D15/pdb>, EMDB:
EMD-27116 <https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-27116>); asymmetric
bent ADP-P_i –F-actin (PDB: 8D16 <https://doi.org/10.2210/pdb8D16/pdb>,
EMDB: EMD-27117 <https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-27117>);
asymmetric straight ADP–F-actin control 1 (PDB: 8D17
<https://doi.org/10.2210/pdb8D17/pdb>, EMDB: EMD-27118
<https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-27118>); asymmetric straight
ADP–F-actin control 2 (PDB: 8D18 <https://doi.org/10.2210/pdb8D18/pdb>,
EMDB: EMD-27119 <https://www.ebi.ac.uk/pdbe/entry/emdb/EMD-27119>).
Cryo-EM datasets have been deposited in the EMPIAR with the following
accession codes: ADP–F-actin (EMPIAR-11128
<https://www.ebi.ac.uk/pdbe/emdb/empiar/entry/11128/>); ADP-P_i –F-actin
(EMPIAR-11129 <https://www.ebi.ac.uk/pdbe/emdb/empiar/entry/11129>).
These depositions include the raw movies and processed particle stacks
used to generate the final reconstructions deposited in the EMDB.
Datasets for cryoDRGN analysis, neural network training and cofilin
severing assays are available at Zenodo. Synthetic datasets used to
train denoising auto-encoder and semantic segmentation neural networks
as well as the trained networks are available at
https://doi.org/10.5281/zenodo.6917913
<https://doi.org/10.5281/zenodo.6917913>. CryoDRGN reconstructions,
fitted models, trained cryoDRGN networks and the data required to train
the cryoDRGN networks are available at
https://doi.org/10.5281/zenodo.6928604
<https://doi.org/10.5281/zenodo.6928604>. Cofilin TIRF microscopy data
are available at https://doi.org/10.5281/zenodo.6929148
<https://doi.org/10.5281/zenodo.6929148>. All other data required to
assess this study’s conclusions are presented in the manuscript.
Materials are available from the corresponding author without
restriction.  Source data
<https://www.nature.com/articles/s41586-022-05366-w#Sec38> are provided
with this paper.


    Code availability

All custom code associated with this study is open source and is
available for download without restriction. Cryo-EM analysis software is
available at GitHub (https://github.com/alushinlab/bent_actin
<https://github.com/alushinlab/bent_actin>), and scripts for analysing
TIRF videos are available at Zenodo
(https://doi.org/10.5281/zenodo.6929148
<https://doi.org/10.5281/zenodo.6929148>). 


    References

 1.

    Pollard, T. D. & Borisy, G. G. Cellular motility driven by assembly
    and disassembly of actin filaments. /Cell/ *112*, 453–465 (2003).

    Article <https://doi.org/10.1016%2FS0092-8674%2803%2900120-X>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXhs1SnsL0%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12600310>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Cellular%20motility%20driven%20by%20assembly%20and%20disassembly%20of%20actin%20filaments&journal=Cell&doi=10.1016%2FS0092-8674%2803%2900120-X&volume=112&pages=453-465&publication_year=2003&author=Pollard%2CTD&author=Borisy%2CGG> 

 2.

    Mogilner, A. & Oster, G. Cell motility driven by actin
    polymerization. /Biophys. J./ *71*, 3030–3045 (1996).

    Article <https://doi.org/10.1016%2FS0006-3495%2896%2979496-1>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=1996BpJ....71.3030M>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28XnsFKltLs%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8968574>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1233792>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Cell%20motility%20driven%20by%20actin%20polymerization&journal=Biophys.%20J.&doi=10.1016%2FS0006-3495%2896%2979496-1&volume=71&pages=3030-3045&publication_year=1996&author=Mogilner%2CA&author=Oster%2CG> 

 3.

    Mogilner, A. & Oster, G. Force generation by actin polymerization
    II: the elastic ratchet and tethered filaments. /Biophys. J./ *84*,
    1591–1605 (2003).

    Article <https://doi.org/10.1016%2FS0006-3495%2803%2974969-8>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2003BpJ....84.1591M>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXit1ehsr4%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12609863>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1302730>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Force%20generation%20by%20actin%20polymerization%20II%3A%20the%20elastic%20ratchet%20and%20tethered%20filaments&journal=Biophys.%20J.&doi=10.1016%2FS0006-3495%2803%2974969-8&volume=84&pages=1591-1605&publication_year=2003&author=Mogilner%2CA&author=Oster%2CG> 

 4.

    Jégou, A. & Romet-Lemonne, G. Mechanically tuning actin filaments to
    modulate the action of actin-binding proteins. /Curr. Opin. Cell
    Biol./ *68*, 72–80 (2021).

    Article <https://doi.org/10.1016%2Fj.ceb.2020.09.002>  PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33160108>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Mechanically%20tuning%20actin%20filaments%20to%20modulate%20the%20action%20of%20actin-binding%20proteins&journal=Curr.%20Opin.%20Cell%20Biol.&doi=10.1016%2Fj.ceb.2020.09.002&volume=68&pages=72-80&publication_year=2021&author=J%C3%A9gou%2CA&author=Romet-Lemonne%2CG> 

 5.

    Sun, X. & Alushin, G. M. Cellular force-sensing through actin
    filaments. /FEBS J./ https://doi.org/10.1111/febs.16568
    <https://doi.org/10.1111/febs.16568> (2022).

 6.

    Pollard, T. D., Blanchoin, L. & Mullins, R. D. Molecular mechanisms
    controlling actin filament dynamics in nonmuscle cells. /Annu. Rev.
    Biophys. Biomol. Struct./ *29*, 545–576 (2000).

    Article <https://doi.org/10.1146%2Fannurev.biophys.29.1.545>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXlsFOnu78%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10940259>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Molecular%20mechanisms%20controlling%20actin%20filament%20dynamics%20in%20nonmuscle%20cells&journal=Annu.%20Rev.%20Biophys.%20Biomol.%20Struct.&doi=10.1146%2Fannurev.biophys.29.1.545&volume=29&pages=545-576&publication_year=2000&author=Pollard%2CTD&author=Blanchoin%2CL&author=Mullins%2CRD> 

 7.

    Cameron, L. A., Svitkina, T. M., Vignjevic, D., Theriot, J. A. &
    Borisy, G. G. Dendritic organization of actin comet tails. /Curr.
    Biol./ *11*, 130–135 (2001).

    Article <https://doi.org/10.1016%2FS0960-9822%2801%2900022-7>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXhtVSgur0%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11231131>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Dendritic%20organization%20of%20actin%20comet%20tails&journal=Curr.%20Biol.&doi=10.1016%2FS0960-9822%2801%2900022-7&volume=11&pages=130-135&publication_year=2001&author=Cameron%2CLA&author=Svitkina%2CTM&author=Vignjevic%2CD&author=Theriot%2CJA&author=Borisy%2CGG> 

 8.

    Svitkina, T. M., Verkhovsky, A. B., McQuade, K. M. & Borisy, G. G.
    Analysis of the actin–myosin II system in fish epidermal
    keratocytes: mechanism of cell body translocation. /J. Cell Biol./
    *139*, 397–415 (1997).

    Article <https://doi.org/10.1083%2Fjcb.139.2.397>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXmvV2gurg%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9334344>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2139803>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Analysis%20of%20the%20actin%E2%80%93myosin%20II%20system%20in%20fish%20epidermal%20keratocytes%3A%20mechanism%20of%20cell%20body%20translocation&journal=J.%20Cell%20Biol.&doi=10.1083%2Fjcb.139.2.397&volume=139&pages=397-415&publication_year=1997&author=Svitkina%2CTM&author=Verkhovsky%2CAB&author=McQuade%2CKM&author=Borisy%2CGG> 

 9.

    Akamatsu, M. et al. Principles of self-organization and load
    adaptation by the actin cytoskeleton during clathrin-mediated
    endocytosis. /eLife/ *9*, e49840 (2020).

    Article <https://doi.org/10.7554%2FeLife.49840>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlarsLvO>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31951196>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7041948>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Principles%20of%20self-organization%20and%20load%20adaptation%20by%20the%20actin%20cytoskeleton%20during%20clathrin-mediated%20endocytosis&journal=eLife&doi=10.7554%2FeLife.49840&volume=9&publication_year=2020&author=Akamatsu%2CM> 

10.

    Jasnin, M. et al. Elasticity of podosome actin networks produces
    nanonewton protrusive forces. /Nat. Commun./ *13*, 3842 (2022).

    Article <https://doi.org/10.1038%2Fs41467-022-30652-6>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2022NatCo..13.3842J>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xhslegu7fN>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35789161>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9253342>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Elasticity%20of%20podosome%20actin%20networks%20produces%20nanonewton%20protrusive%20forces&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-022-30652-6&volume=13&publication_year=2022&author=Jasnin%2CM> 

11.

    Ponti, A., Machacek, M., Gupton, S. L., Waterman-Storer, C. M. &
    Danuser, G. Two distinct actin networks drive the protrusion of
    migrating cells. /Science/ *305*, 1782–1786 (2004).

    Article <https://doi.org/10.1126%2Fscience.1100533>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2004Sci...305.1782P>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXnsFajur4%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15375270>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Two%20distinct%20actin%20networks%20drive%20the%20protrusion%20of%20migrating%20cells&journal=Science&doi=10.1126%2Fscience.1100533&volume=305&pages=1782-1786&publication_year=2004&author=Ponti%2CA&author=Machacek%2CM&author=Gupton%2CSL&author=Waterman-Storer%2CCM&author=Danuser%2CG> 

12.

    Galkin, V. E., Orlova, A. & Egelman, E. H. Actin filaments as
    tension sensors. /Curr. Biol./ *22*, R96–R101 (2012).

    Article <https://doi.org/10.1016%2Fj.cub.2011.12.010>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvFeit7s%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22321312>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3277726>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20filaments%20as%20tension%20sensors&journal=Curr.%20Biol.&doi=10.1016%2Fj.cub.2011.12.010&volume=22&pages=R96-R101&publication_year=2012&author=Galkin%2CVE&author=Orlova%2CA&author=Egelman%2CEH> 

13.

    Zimmermann, D., Santos, A., Kovar, D. R. & Rock, R. S. Actin age
    orchestrates myosin-5 and myosin-6 run lengths. /Curr. Biol./ *25*,
    2057–2062 (2015).

    Article <https://doi.org/10.1016%2Fj.cub.2015.06.033>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtF2ksL3N>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26190073>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4556227>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20age%20orchestrates%20myosin-5%20and%20myosin-6%20run%20lengths&journal=Curr.%20Biol.&doi=10.1016%2Fj.cub.2015.06.033&volume=25&pages=2057-2062&publication_year=2015&author=Zimmermann%2CD&author=Santos%2CA&author=Kovar%2CDR&author=Rock%2CRS> 

14.

    Blanchoin, L. & Pollard, T. D. Mechanism of interaction of
    acanthamoeba actophorin (ADF/cofilin) with actin filaments. /J.
    Biol. Chem./ *274*, 15538–15546 (1999).

    Article <https://doi.org/10.1074%2Fjbc.274.22.15538>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXjsV2gsLc%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10336448>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Mechanism%20of%20interaction%20of%20acanthamoeba%20actophorin%20%28ADF%2Fcofilin%29%20with%20actin%20filaments.&journal=J.%20Biol.%20Chem.&doi=10.1074%2Fjbc.274.22.15538&volume=274&pages=15538-15546&publication_year=1999&author=Blanchoin%2CL&author=Pollard%2CTD> 

15.

    Suarez, C. et al. Cofilin tunes the nucleotide state of actin
    filaments and severs at bare and decorated segment boundaries.
    /Curr. Biol./ *21*, 862–868 (2011).

    Article <https://doi.org/10.1016%2Fj.cub.2011.03.064>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmsVOqtLc%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21530260>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3100394>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Cofilin%20tunes%20the%20nucleotide%20state%20of%20actin%20filaments%20and%20severs%20at%20bare%20and%20decorated%20segment%20boundaries&journal=Curr.%20Biol.&doi=10.1016%2Fj.cub.2011.03.064&volume=21&pages=862-868&publication_year=2011&author=Suarez%2CC> 

16.

    Hayakawa, K., Tatsumi, H. & Sokabe, M. Actin filaments function as a
    tension sensor by tension-dependent binding of cofilin to the
    filament. /J. Cell Biol./ *195*, 721–727 (2011).

    Article <https://doi.org/10.1083%2Fjcb.201102039>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs1Shsb3E>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22123860>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3257564>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20filaments%20function%20as%20a%20tension%20sensor%20by%20tension-dependent%20binding%20of%20cofilin%20to%20the%20filament&journal=J.%20Cell%20Biol.&doi=10.1083%2Fjcb.201102039&volume=195&pages=721-727&publication_year=2011&author=Hayakawa%2CK&author=Tatsumi%2CH&author=Sokabe%2CM> 

17.

    Wioland, H., Jegou, A. & Romet-Lemonne, G. Torsional stress
    generated by ADF/cofilin on cross-linked actin filaments boosts
    their severing. /Proc. Natl Acad. Sci. USA/ *116*, 2595–2602 (2019).

    Article <https://doi.org/10.1073%2Fpnas.1812053116>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019PNAS..116.2595W>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXivVajsro%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30692249>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6377502>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Torsional%20stress%20generated%20by%20ADF%2Fcofilin%20on%20cross-linked%20actin%20filaments%20boosts%20their%20severing&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1812053116&volume=116&pages=2595-2602&publication_year=2019&author=Wioland%2CH&author=Jegou%2CA&author=Romet-Lemonne%2CG> 

18.

    Pandit, N. G. et al. Force and phosphate release from Arp2/3 complex
    promote dissociation of actin filament branches. /Proc. Natl Acad.
    Sci. USA/ *117*, 13519–13528 (2020).

    Article <https://doi.org/10.1073%2Fpnas.1911183117>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020PNAS..11713519P>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtF2it7jL>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32461373>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7306818>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Force%20and%20phosphate%20release%20from%20Arp2%2F3%20complex%20promote%20dissociation%20of%20actin%20filament%20branches&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1911183117&volume=117&pages=13519-13528&publication_year=2020&author=Pandit%2CNG> 

19.

    Risca, V. I. et al. Actin filament curvature biases branching
    direction. /Proc. Natl Acad. Sci. USA/ *109*, 2913–2918 (2012).

    Article <https://doi.org/10.1073%2Fpnas.1114292109>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2012PNAS..109.2913R>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xjs1Wmsbg%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22308368>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3286980>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20filament%20curvature%20biases%20branching%20direction&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1114292109&volume=109&pages=2913-2918&publication_year=2012&author=Risca%2CVI> 

20.

    Bieling, P. et al. Force feedback controls motor activity and
    mechanical properties of self-assembling branched actin networks.
    /Cell/ *164*, 115–127 (2016).

    Article <https://doi.org/10.1016%2Fj.cell.2015.11.057>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtFShsbc%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26771487>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5033619>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Force%20feedback%20controls%20motor%20activity%20and%20mechanical%20properties%20of%20self-assembling%20branched%20actin%20networks&journal=Cell&doi=10.1016%2Fj.cell.2015.11.057&volume=164&pages=115-127&publication_year=2016&author=Bieling%2CP> 

21.

    Mueller, J. et al. Load adaptation of lamellipodial actin networks.
    /Cell/ *171*, 188–200 (2017).

    Article <https://doi.org/10.1016%2Fj.cell.2017.07.051>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsVert7jE>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28867286>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Load%20adaptation%20of%20lamellipodial%20actin%20networks&journal=Cell&doi=10.1016%2Fj.cell.2017.07.051&volume=171&pages=188-200&publication_year=2017&author=Mueller%2CJ> 

22.

    Ge, P., Durer, Z. A. O., Kudryashov, D., Zhou, Z. H. & Reisler, E.
    Cryo-EM reveals different coronin binding modes for ADP– and
    ADP–BeFx actin filaments. /Nat. Struct. Mol. Biol./ *21*, 1075–1081
    (2014).

    Article <https://doi.org/10.1038%2Fnsmb.2907>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVGiurfF>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25362487>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4388421>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Cryo-EM%20reveals%20different%20coronin%20binding%20modes%20for%20ADP%E2%80%93%20and%20ADP%E2%80%93BeFx%20actin%20filaments&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.2907&volume=21&pages=1075-1081&publication_year=2014&author=Ge%2CP&author=Durer%2CZAO&author=Kudryashov%2CD&author=Zhou%2CZH&author=Reisler%2CE> 

23.

    Mei, L. et al. Molecular mechanism for direct actin force-sensing by
    α-catenin. /eLife/ *9*, e62514 (2020).

    Article <https://doi.org/10.7554%2FeLife.62514>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFCrurrJ>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32969337>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7588232>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Molecular%20mechanism%20for%20direct%20actin%20force-sensing%20by%20%CE%B1-catenin&journal=eLife&doi=10.7554%2FeLife.62514&volume=9&publication_year=2020&author=Mei%2CL> 

24.

    Sun, X. et al. Mechanosensing through direct binding of tensed
    F-actin by LIM domains. /Dev. Cell/ *55*, 468–482 (2020).

    Article <https://doi.org/10.1016%2Fj.devcel.2020.09.022>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFSns7zP>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33058779>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7686152>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Mechanosensing%20through%20direct%20binding%20of%20tensed%20F-actin%20by%20LIM%20domains&journal=Dev.%20Cell&doi=10.1016%2Fj.devcel.2020.09.022&volume=55&pages=468-482&publication_year=2020&author=Sun%2CX> 

25.

    Winkelman, J. D., Anderson, C. A., Suarez, C., Kovar, D. R. &
    Gardel, M. L. Evolutionarily diverse LIM domain-containing proteins
    bind stressed actin filaments through a conserved mechanism. /Proc.
    Natl Acad. Sci. USA/ *117*, 25532–25542 (2020).

    Article <https://doi.org/10.1073%2Fpnas.2004656117>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020PNAS..11725532W>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVKkt7jF>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32989126>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7568268>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Evolutionarily%20diverse%20LIM%20domain-containing%20proteins%20bind%20stressed%20actin%20filaments%20through%20a%20conserved%20mechanism&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.2004656117&volume=117&pages=25532-25542&publication_year=2020&author=Winkelman%2CJD&author=Anderson%2CCA&author=Suarez%2CC&author=Kovar%2CDR&author=Gardel%2CML> 

26.

    Oda, T., Iwasa, M., Aihara, T., Maéda, Y. & Narita, A. The nature of
    the globular- to fibrous-actin transition. /Nature/ *457*, 441–445
    (2009).

    Article <https://doi.org/10.1038%2Fnature07685>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009Natur.457..441O>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXotlOnsQ%3D%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19158791>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20nature%20of%20the%20globular-%20to%20fibrous-actin%20transition&journal=Nature&doi=10.1038%2Fnature07685&volume=457&pages=441-445&publication_year=2009&author=Oda%2CT&author=Iwasa%2CM&author=Aihara%2CT&author=Ma%C3%A9da%2CY&author=Narita%2CA> 

27.

    Merino, F. et al. Structural transitions of F-actin upon ATP
    hydrolysis at near-atomic resolution revealed by cryo-EM. /Nat.
    Struct. Mol. Biol./ *25*, 528–537 (2018).

    Article <https://doi.org/10.1038%2Fs41594-018-0074-0>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlKqu7nP>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29867215>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20transitions%20of%20F-actin%20upon%20ATP%20hydrolysis%20at%20near-atomic%20resolution%20revealed%20by%20cryo-EM&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fs41594-018-0074-0&volume=25&pages=528-537&publication_year=2018&author=Merino%2CF> 

28.

    Pospich, S., Merino, F. & Raunser, S. Structural effects and
    functional implications of phalloidin and jasplakinolide binding to
    actin filaments. /Structure/ *28*, 437–449 (2020).

    Article <https://doi.org/10.1016%2Fj.str.2020.01.014>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjsVyisr0%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32084355>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20effects%20and%20functional%20implications%20of%20phalloidin%20and%20jasplakinolide%20binding%20to%20actin%20filaments&journal=Structure&doi=10.1016%2Fj.str.2020.01.014&volume=28&pages=437-449&publication_year=2020&author=Pospich%2CS&author=Merino%2CF&author=Raunser%2CS> 

29.

    Dominguez, R. & Holmes, K. C. Actin structure and function. /Annu.
    Rev. Biophys./ *40*, 169–186 (2011).

    Article <https://doi.org/10.1146%2Fannurev-biophys-042910-155359> 
    CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXnvFaitbk%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21314430>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3130349>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20structure%20and%20function&journal=Annu.%20Rev.%20Biophys.&doi=10.1146%2Fannurev-biophys-042910-155359&volume=40&pages=169-186&publication_year=2011&author=Dominguez%2CR&author=Holmes%2CKC> 

30.

    Galkin, V. E., Orlova, A., Schröder, G. F. & Egelman, E. H.
    Structural polymorphism in F-actin. /Nat. Struct. Mol. Biol./ *17*,
    1318–1323 (2010).

    Article <https://doi.org/10.1038%2Fnsmb.1930>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXht1Kms7rO>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20935633>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2988880>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20polymorphism%20in%20F-actin&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.1930&volume=17&pages=1318-1323&publication_year=2010&author=Galkin%2CVE&author=Orlova%2CA&author=Schr%C3%B6der%2CGF&author=Egelman%2CEH> 

31.

    Oda, T., Takeda, S., Narita, A. & Maéda, Y. Structural polymorphism
    of actin. /J. Mol. Biol./ *431*, 3217–3228 (2019).

    Article <https://doi.org/10.1016%2Fj.jmb.2019.05.048>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFOktrvM>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31181287>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20polymorphism%20of%20actin&journal=J.%20Mol.%20Biol.&doi=10.1016%2Fj.jmb.2019.05.048&volume=431&pages=3217-3228&publication_year=2019&author=Oda%2CT&author=Takeda%2CS&author=Narita%2CA&author=Ma%C3%A9da%2CY> 

32.

    Chou, S. Z. & Pollard, T. D. Mechanism of actin polymerization
    revealed by cryo-EM structures of actin filaments with three
    different bound nucleotides. /Proc. Natl Acad. Sci. USA/ *116*,
    4265–4274 (2019).

    Article <https://doi.org/10.1073%2Fpnas.1807028115>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019PNAS..116.4265C>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXktFeis7k%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30760599>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6410863>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Mechanism%20of%20actin%20polymerization%20revealed%20by%20cryo-EM%20structures%20of%20actin%20filaments%20with%20three%20different%20bound%20nucleotides&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1807028115&volume=116&pages=4265-4274&publication_year=2019&author=Chou%2CSZ&author=Pollard%2CTD> 

33.

    McGough, A., Pope, B., Chiu, W. & Weeds, A. Cofilin changes the
    twist of F-actin: implications for actin filament dynamics and
    cellular function. /J. Cell Biol./ *138*, 771–781 (1997).

    Article <https://doi.org/10.1083%2Fjcb.138.4.771>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXlsleitbY%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9265645>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2138052>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Cofilin%20changes%20the%20twist%20of%20F-actin%3A%20implications%20for%20actin%20filament%20dynamics%20and%20cellular%20function&journal=J.%20Cell%20Biol.&doi=10.1083%2Fjcb.138.4.771&volume=138&pages=771-781&publication_year=1997&author=McGough%2CA&author=Pope%2CB&author=Chiu%2CW&author=Weeds%2CA> 

34.

    Tanaka, K. et al. Structural basis for cofilin binding and actin
    filament disassembly. /Nat. Commun./ *9*, 1860 (2018).

    Article <https://doi.org/10.1038%2Fs41467-018-04290-w>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2018NatCo...9.1860T>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29749375>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5945598>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20for%20cofilin%20binding%20and%20actin%20filament%20disassembly&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-018-04290-w&volume=9&publication_year=2018&author=Tanaka%2CK> 

35.

    Isambert, H. et al. Flexibility of actin filaments derived from
    thermal fluctuations: effect of bound nucleotide, phalloidin, and
    muscle regulatory proteins. /J. Biol. Chem./ *270*, 11437–11444 (1995).

    Article <https://doi.org/10.1074%2Fjbc.270.19.11437>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXls1eis7s%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=7744781>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Flexibility%20of%20actin%20filaments%20derived%20from%20thermal%20fluctuations%3A%20effect%20of%20bound%20nucleotide%2C%20phalloidin%2C%20and%20muscle%20regulatory%20proteins&journal=J.%20Biol.%20Chem.&doi=10.1074%2Fjbc.270.19.11437&volume=270&pages=11437-11444&publication_year=1995&author=Isambert%2CH> 

36.

    Saunders, M. G. & Voth, G. A. Comparison between actin filament
    models: coarse-graining reveals essential differences. /Structure/
    *20*, 641–653 (2012).

    Article <https://doi.org/10.1016%2Fj.str.2012.02.008>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xltl2ktbo%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22483111>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Comparison%20between%20actin%20filament%20models%3A%20coarse-graining%20reveals%20essential%20differences&journal=Structure&doi=10.1016%2Fj.str.2012.02.008&volume=20&pages=641-653&publication_year=2012&author=Saunders%2CMG&author=Voth%2CGA> 

37.

    De La Cruz, E. M., Roland, J., McCullough, B. R., Blanchoin, L. &
    Martiel, J.-L. Origin of twist-bend coupling in actin filaments.
    /Biophys. J./ *99*, 1852–1860 (2010).

    Article <https://doi.org/10.1016%2Fj.bpj.2010.07.009>  Google
    Scholar
    <http://scholar.google.com/scholar_lookup?&title=Origin%20of%20twist-bend%20coupling%20in%20actin%20filaments&journal=Biophys.%20J.&doi=10.1016%2Fj.bpj.2010.07.009&volume=99&pages=1852-1860&publication_year=2010&author=Cruz%2CEM&author=Roland%2CJ&author=McCullough%2CBR&author=Blanchoin%2CL&author=Martiel%2CJ-L> 

38.

    Katkar, H. H. et al. Insights into the cooperative nature of ATP
    hydrolysis in actin filaments. /Biophys. J./ *115*, 1589–1602 (2018).

    Article <https://doi.org/10.1016%2Fj.bpj.2018.08.034>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2018BpJ...115.1589K>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslCju7zM>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30249402>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6260209>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Insights%20into%20the%20cooperative%20nature%20of%20ATP%20hydrolysis%20in%20actin%20filaments&journal=Biophys.%20J.&doi=10.1016%2Fj.bpj.2018.08.034&volume=115&pages=1589-1602&publication_year=2018&author=Katkar%2CHH> 

39.

    Oosterheert, W., Klink, B. U., Belyy, A., Pospich, S. & Raunser, S.
    Structural basis of actin filament assembly and aging. Preprint at
    /bioRxiv/ https://doi.org/10.1101/2022.03.29.486216
    <https://doi.org/10.1101/2022.03.29.486216> (2022).

40.

    Das, S. et al. D-loop dynamics and near-atomic-resolution cryo-EM
    structure of phalloidin-bound F-actin. /Structure/ *28*, 586–593 (2020).

    Article <https://doi.org/10.1016%2Fj.str.2020.04.004>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXosVWjsbs%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32348747>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7316398>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=D-loop%20dynamics%20and%20near-atomic-resolution%20cryo-EM%20structure%20of%20phalloidin-bound%20F-actin&journal=Structure&doi=10.1016%2Fj.str.2020.04.004&volume=28&pages=586-593&publication_year=2020&author=Das%2CS> 

41.

    Gong, R. et al. Structural basis for tunable control of actin
    dynamics by myosin-15 in mechanosensory stereocilia. /Sci. Adv./
    *8*, eabl4733 (2022).

    Article <https://doi.org/10.1126%2Fsciadv.abl4733>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitFSmsbrM>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35857845>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9299544>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20for%20tunable%20control%20of%20actin%20dynamics%20by%20myosin-15%20in%20mechanosensory%20stereocilia&journal=Sci.%20Adv.&doi=10.1126%2Fsciadv.abl4733&volume=8&publication_year=2022&author=Gong%2CR> 

42.

    Saunders, M. G. & Voth, G. A. Water molecules in the nucleotide
    binding cleft of actin: effects on subunit conformation and
    implications for ATP hydrolysis. /J. Mol. Biol./ *413*, 279–291 (2011).

    Article <https://doi.org/10.1016%2Fj.jmb.2011.07.068>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1Grtr3K>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21856312>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Water%20molecules%20in%20the%20nucleotide%20binding%20cleft%20of%20actin%3A%20effects%20on%20subunit%20conformation%20and%20implications%20for%20ATP%20hydrolysis&journal=J.%20Mol.%20Biol.&doi=10.1016%2Fj.jmb.2011.07.068&volume=413&pages=279-291&publication_year=2011&author=Saunders%2CMG&author=Voth%2CGA> 

43.

    Kang, H. et al. Identification of cation-binding sites on actin that
    drive polymerization and modulate bending stiffness. /Proc. Natl
    Acad. Sci. USA/ *109*, 16923–16927 (2012).

    Article <https://doi.org/10.1073%2Fpnas.1211078109>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2012PNAS..10916923K>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xhs1WksL%2FL>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23027950>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3479481>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Identification%20of%20cation-binding%20sites%20on%20actin%20that%20drive%20polymerization%20and%20modulate%20bending%20stiffness&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1211078109&volume=109&pages=16923-16927&publication_year=2012&author=Kang%2CH> 

44.

    Armstrong, M. et al. Microscale fluid behavior during cryo-EM sample
    blotting. /Biophys. J./ *118*, 708–719 (2020).

    Article <https://doi.org/10.1016%2Fj.bpj.2019.12.017>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020BpJ...118..708A>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtF2iu74%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31952802>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Microscale%20fluid%20behavior%20during%20cryo-EM%20sample%20blotting&journal=Biophys.%20J.&doi=10.1016%2Fj.bpj.2019.12.017&volume=118&pages=708-719&publication_year=2020&author=Armstrong%2CM> 

45.

    Shibata, S., Matsunami, H., Aizawa, S.-I. & Wolf, M. Torque
    transmission mechanism of the curved bacterial flagellar hook
    revealed by cryo-EM. /Nat. Struct. Mol. Biol./ *26*, 941–945 (2019).

    Article <https://doi.org/10.1038%2Fs41594-019-0301-3>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvVOqu7bN>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31570877>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Torque%20transmission%20mechanism%20of%20the%20curved%20bacterial%20flagellar%20hook%20revealed%20by%20cryo-EM&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fs41594-019-0301-3&volume=26&pages=941-945&publication_year=2019&author=Shibata%2CS&author=Matsunami%2CH&author=Aizawa%2CS-I&author=Wolf%2CM> 

46.

    Gibson, K. H. et al. An asymmetric sheath controls flagellar
    supercoiling and motility in the leptospira spirochete. /eLife/ *9*,
    e53672 (2020).

    Article <https://doi.org/10.7554%2FeLife.53672>  PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32157997>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7065911>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=An%20asymmetric%20sheath%20controls%20flagellar%20supercoiling%20and%20motility%20in%20the%20leptospira%20spirochete&journal=eLife&doi=10.7554%2FeLife.53672&volume=9&publication_year=2020&author=Gibson%2CKH> 

47.

    Orlova, A. & Egelman, E. H. A conformational change in the actin
    subunit can change the flexibility of the actin filament. /J. Mol.
    Biol./ *232*, 334–341 (1993).

    Article <https://doi.org/10.1006%2Fjmbi.1993.1393>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXmtF2qt7k%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8345515>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=A%20conformational%20change%20in%20the%20actin%20subunit%20can%20change%20the%20flexibility%20of%20the%20actin%20filament&journal=J.%20Mol.%20Biol.&doi=10.1006%2Fjmbi.1993.1393&volume=232&pages=334-341&publication_year=1993&author=Orlova%2CA&author=Egelman%2CEH> 

48.

    McCullough, B. R., Blanchoin, L., Martiel, J.-L. & De La Cruz, E. M.
    Cofilin increases the bending flexibility of actin filaments:
    implications for severing and cell mechanics. /J. Mol. Biol./ *381*,
    550–558 (2008).

    Article <https://doi.org/10.1016%2Fj.jmb.2008.05.055>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXpt1KisLo%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18617188>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2753234>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Cofilin%20increases%20the%20bending%20flexibility%20of%20actin%20filaments%3A%20implications%20for%20severing%20and%20cell%20mechanics&journal=J.%20Mol.%20Biol.&doi=10.1016%2Fj.jmb.2008.05.055&volume=381&pages=550-558&publication_year=2008&author=McCullough%2CBR&author=Blanchoin%2CL&author=Martiel%2CJ-L&author=Cruz%2CEM> 

49.

    Guo, J. & Zhou, H.-X. Protein allostery and conformational dynamics.
    /Chem. Rev./ *116*, 6503–6515 (2016).

    Article <https://doi.org/10.1021%2Facs.chemrev.5b00590>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XisVKjsr8%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26876046>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5011433>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Protein%20allostery%20and%20conformational%20dynamics&journal=Chem.%20Rev.&doi=10.1021%2Facs.chemrev.5b00590&volume=116&pages=6503-6515&publication_year=2016&author=Guo%2CJ&author=Zhou%2CH-X> 

50.

    Mani, S., Katkar, H. H. & Voth, G. A. Compressive and tensile
    deformations alter ATP hydrolysis and phosphate release rates in
    actin filaments. /J. Chem. Theory Comput./ *17*, 1900–1913 (2021).

    Article <https://doi.org/10.1021%2Facs.jctc.0c01186>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXktFemsbc%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33596075>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Compressive%20and%20tensile%20deformations%20alter%20ATP%20hydrolysis%20and%20phosphate%20release%20rates%20in%20actin%20filaments&journal=J.%20Chem.%20Theory%20Comput.&doi=10.1021%2Facs.jctc.0c01186&volume=17&pages=1900-1913&publication_year=2021&author=Mani%2CS&author=Katkar%2CHH&author=Voth%2CGA> 

51.

    Pardee, J. D. & Spudich, J. A. in /Methods in Enzymology/ Vol. 85
    (eds Frederiksen, D. W. & Cunningham, L. W.) Ch. 18, 164–181
    (Academic, 1982).

52.

    Walt, Svander et al. scikit-image: image processing in Python.
    /PeerJ/ *2*, e453 (2014).

    Article <https://doi.org/10.7717%2Fpeerj.453>  PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25024921>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4081273>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=scikit-image%3A%20image%20processing%20in%20Python&journal=PeerJ&doi=10.7717%2Fpeerj.453&volume=2&publication_year=2014&author=Walt%2CSvander> 

53.

    Mastronarde, D. N. Automated electron microscope tomography using
    robust prediction of specimen movements. /J. Struct. Biol./ *152*,
    36–51 (2005).

    Article <https://doi.org/10.1016%2Fj.jsb.2005.07.007>  PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16182563>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Automated%20electron%20microscope%20tomography%20using%20robust%20prediction%20of%20specimen%20movements&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2005.07.007&volume=152&pages=36-51&publication_year=2005&author=Mastronarde%2CDN> 

54.

    Zheng, S. Q. et al. MotionCor2: anisotropic correction of
    beam-induced motion for improved cryo-electron microscopy. /Nat.
    Methods/ *14*, 331–332 (2017).

    Article <https://doi.org/10.1038%2Fnmeth.4193>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXjt1ags7g%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28250466>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5494038>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=MotionCor2%3A%20anisotropic%20correction%20of%20beam-induced%20motion%20for%20improved%20cryo-electron%20microscopy&journal=Nat.%20Methods&doi=10.1038%2Fnmeth.4193&volume=14&pages=331-332&publication_year=2017&author=Zheng%2CSQ> 

55.

    Grant, T. & Grigorieff, N. Measuring the optimal exposure for single
    particle cryo-EM using a 2.6 Å reconstruction of rotavirus VP6.
    /eLife/ *4*, e06980 (2015).

    Article <https://doi.org/10.7554%2FeLife.06980>  PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26023829>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4471936>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Measuring%20the%20optimal%20exposure%20for%20single%20particle%20cryo-EM%20using%20a%202.6%E2%80%89%C3%85%20reconstruction%20of%20rotavirus%20VP6&journal=eLife&doi=10.7554%2FeLife.06980&volume=4&publication_year=2015&author=Grant%2CT&author=Grigorieff%2CN> 

56.

    Rohou, A. & Grigorieff, N. CTFFIND4: fast and accurate defocus
    estimation from electron micrographs. /J. Struct. Biol./ *192*,
    216–221 (2015).

    Article <https://doi.org/10.1016%2Fj.jsb.2015.08.008>  PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26278980>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6760662>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=CTFFIND4%3A%20fast%20and%20accurate%20defocus%20estimation%20from%20electron%20micrographs&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2015.08.008&volume=192&pages=216-221&publication_year=2015&author=Rohou%2CA&author=Grigorieff%2CN> 

57.

    Wang, F. et al. DeepPicker: a deep learning approach for fully
    automated particle picking in cryo-EM. /J. Struct. Biol./ *195*,
    325–336 (2016).

    Article <https://doi.org/10.1016%2Fj.jsb.2016.07.006>  PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27424268>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=DeepPicker%3A%20a%20deep%20learning%20approach%20for%20fully%20automated%20particle%20picking%20in%20cryo-EM&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2016.07.006&volume=195&pages=325-336&publication_year=2016&author=Wang%2CF> 

58.

    Wagner, T. et al. SPHIRE-crYOLO is a fast and accurate fully
    automated particle picker for cryo-EM. /Commun. Biol./ *2*, 218 (2019).

    Article <https://doi.org/10.1038%2Fs42003-019-0437-z>  PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31240256>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6584505>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=SPHIRE-crYOLO%20is%20a%20fast%20and%20accurate%20fully%20automated%20particle%20picker%20for%20cryo-EM&journal=Commun.%20Biol.&doi=10.1038%2Fs42003-019-0437-z&volume=2&publication_year=2019&author=Wagner%2CT> 

59.

    Bepler, T. et al. Positive-unlabeled convolutional neural networks
    for particle picking in cryo-electron micrographs. /Nat. Methods/
    *16*, 1153–1160 (2019).

    Article <https://doi.org/10.1038%2Fs41592-019-0575-8>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFejs7%2FM>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31591578>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6858545>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Positive-unlabeled%20convolutional%20neural%20networks%20for%20particle%20picking%20in%20cryo-electron%20micrographs&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0575-8&volume=16&pages=1153-1160&publication_year=2019&author=Bepler%2CT> 

60.

    Bakan, A., Meireles, L. M. & Bahar, I. ProDy: protein dynamics
    inferred from theory and experiments. /Bioinformatics/ *27*,
    1575–1577 (2011).

    Article <https://doi.org/10.1093%2Fbioinformatics%2Fbtr168>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmvVajs7s%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21471012>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3102222>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=ProDy%3A%20protein%20dynamics%20inferred%20from%20theory%20and%20experiments&journal=Bioinformatics&doi=10.1093%2Fbioinformatics%2Fbtr168&volume=27&pages=1575-1577&publication_year=2011&author=Bakan%2CA&author=Meireles%2CLM&author=Bahar%2CI> 

61.

    Tang, G. et al. EMAN2: an extensible image processing suite for
    electron microscopy. /J. Struct. Biol./ *157*, 38–46 (2007).

    Article <https://doi.org/10.1016%2Fj.jsb.2006.05.009>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xhtlalt77M>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16859925>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=EMAN2%3A%20an%20extensible%20image%20processing%20suite%20for%20electron%20microscopy&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2006.05.009&volume=157&pages=38-46&publication_year=2007&author=Tang%2CG> 

62.

    TensorFlow Developers. TensorFlow (Zenodo, 2022);
    https://doi.org/10.5281/zenodo.6574269
    <https://doi.org/10.5281/zenodo.6574269>

63.

    Mei, L. et al. Structural mechanism for bidirectional actin
    cross-linking by T-plastin. /Proc. Natl Acad. Sci. USA/ *119*,
    e2205370119 (2022).

    Article <https://doi.org/10.1073%2Fpnas.2205370119>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisFags77J>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36067297>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9478642>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20mechanism%20for%20bidirectional%20actin%20cross-linking%20by%20T-plastin&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.2205370119&volume=119&publication_year=2022&author=Mei%2CL> 

64.

    Egelman, E. H. The iterative helical real space reconstruction
    method: surmounting the problems posed by real polymers. /J. Struct.
    Biol./ *157*, 83–94 (2007).

    Article <https://doi.org/10.1016%2Fj.jsb.2006.05.015>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xhtlalt77J>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16919474>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20iterative%20helical%20real%20space%20reconstruction%20method%3A%20surmounting%20the%20problems%20posed%20by%20real%20polymers&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2006.05.015&volume=157&pages=83-94&publication_year=2007&author=Egelman%2CEH> 

65.

    He, S. & Scheres, S. H. W. Helical reconstruction in RELION. /J.
    Struct. Biol./ *198*, 163–176 (2017).

    Article <https://doi.org/10.1016%2Fj.jsb.2017.02.003>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXjt1Klsrk%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28193500>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5479445>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Helical%20reconstruction%20in%20RELION&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2017.02.003&volume=198&pages=163-176&publication_year=2017&author=He%2CS&author=Scheres%2CSHW> 

66.

    Zivanov, J. et al. New tools for automated high-resolution cryo-EM
    structure determination in RELION-3. /eLife/ *7*, e42166 (2018).

    Article <https://doi.org/10.7554%2FeLife.42166>  PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30412051>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6250425>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=New%20tools%20for%20automated%20high-resolution%20cryo-EM%20structure%20determination%20in%20RELION-3&journal=eLife&doi=10.7554%2FeLife.42166&volume=7&publication_year=2018&author=Zivanov%2CJ> 

67.

    Punjani, A., Rubinstein, J. L., Fleet, D. J. & Brubaker, M. A.
    cryoSPARC: algorithms for rapid unsupervised cryo-EM structure
    determination. /Nat. Methods/ *14*, 290–296 (2017).

    Article <https://doi.org/10.1038%2Fnmeth.4169>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXitlGisbs%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28165473>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=cryoSPARC%3A%20algorithms%20for%20rapid%20unsupervised%20cryo-EM%20structure%20determination&journal=Nat.%20Methods&doi=10.1038%2Fnmeth.4169&volume=14&pages=290-296&publication_year=2017&author=Punjani%2CA&author=Rubinstein%2CJL&author=Fleet%2CDJ&author=Brubaker%2CMA> 

68.

    Tan, Y. Z. et al. Addressing preferred specimen orientation in
    single-particle cryo-EM through tilting. /Nat. Methods/ *14*,
    793–796 (2017).

    Article <https://doi.org/10.1038%2Fnmeth.4347>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFSmtrjO>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28671674>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5533649>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Addressing%20preferred%20specimen%20orientation%20in%20single-particle%20cryo-EM%20through%20tilting&journal=Nat.%20Methods&doi=10.1038%2Fnmeth.4347&volume=14&pages=793-796&publication_year=2017&author=Tan%2CYZ> 

69.

    Zhong, E. D., Bepler, T., Berger, B. & Davis, J. H. CryoDRGN:
    reconstruction of heterogeneous cryo-EM structures using neural
    networks. /Nat. Methods/ *18*, 176–185 (2021).

    Article <https://doi.org/10.1038%2Fs41592-020-01049-4>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjt1SmsLw%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33542510>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8183613>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=CryoDRGN%3A%20reconstruction%20of%20heterogeneous%20cryo-EM%20structures%20using%20neural%20networks&journal=Nat.%20Methods&doi=10.1038%2Fs41592-020-01049-4&volume=18&pages=176-185&publication_year=2021&author=Zhong%2CED&author=Bepler%2CT&author=Berger%2CB&author=Davis%2CJH> 

70.

    Terwilliger, T. C., Ludtke, S. J., Read, R. J., Adams, P. D. &
    Afonine, P. V. Improvement of cryo-EM maps by density modification.
    /Nat. Methods/ *17*, 923–927 (2020).

    Article <https://doi.org/10.1038%2Fs41592-020-0914-9>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1WrurbL>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32807957>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7484085>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Improvement%20of%20cryo-EM%20maps%20by%20density%20modification&journal=Nat.%20Methods&doi=10.1038%2Fs41592-020-0914-9&volume=17&pages=923-927&publication_year=2020&author=Terwilliger%2CTC&author=Ludtke%2CSJ&author=Read%2CRJ&author=Adams%2CPD&author=Afonine%2CPV> 

71.

    Grigorieff, N. in /Methods in Enzymology/ Vol. 579 (ed. Crowther, R.
    A.) Ch. 8, 191–226 (Academic, 2016).

72.

    Pettersen, E. F. et al. UCSF Chimera—a visualization system for
    exploratory research and analysis. /J. Comput. Chem./ *25*,
    1605–1612 (2004).

    Article <https://doi.org/10.1002%2Fjcc.20084>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXmvVOhsbs%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15264254>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=UCSF%20Chimera%E2%80%94a%20visualization%20system%20for%20exploratory%20research%20and%20analysis&journal=J.%20Comput.%20Chem.&doi=10.1002%2Fjcc.20084&volume=25&pages=1605-1612&publication_year=2004&author=Pettersen%2CEF> 

73.

    Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and
    development of Coot. /Acta Crystalogry. D/ *66*, 486–501 (2010).

    Article <https://doi.org/10.1107%2FS0907444910007493>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXksFKisb8%3D>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Features%20and%20development%20of%20Coot&journal=Acta%20Crystalogry.%20D&doi=10.1107%2FS0907444910007493&volume=66&pages=486-501&publication_year=2010&author=Emsley%2CP&author=Lohkamp%2CB&author=Scott%2CWG&author=Cowtan%2CK> 

74.

    Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM
    and crystallography. /Acta Crystalogr. D/ *74*, 531–544 (2018).

    Article <https://doi.org/10.1107%2FS2059798318006551>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2ltbzP>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Real-space%20refinement%20in%20PHENIX%20for%20cryo-EM%20and%20crystallography&journal=Acta%20Crystalogr.%20D&doi=10.1107%2FS2059798318006551&volume=74&pages=531-544&publication_year=2018&author=Afonine%2CPV> 

75.

    Liebschner, D. et al. Macromolecular structure determination using
    X-rays, neutrons and electrons: recent developments in Phenix. /Acta
    Crystalogr. D/ *75*, 861–877 (2019).

    Article <https://doi.org/10.1107%2FS2059798319011471>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFWkurrO>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Macromolecular%20structure%20determination%20using%20X-rays%2C%20neutrons%20and%20electrons%3A%20recent%20developments%20in%20Phenix&journal=Acta%20Crystalogr.%20D&doi=10.1107%2FS2059798319011471&volume=75&pages=861-877&publication_year=2019&author=Liebschner%2CD> 

76.

    Phillips, J. C. et al. Scalable molecular dynamics with NAMD. /J.
    Comput. Chem./ *26*, 1781–1802 (2005).

    Article <https://doi.org/10.1002%2Fjcc.20289>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXht1SlsbbJ>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16222654>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2486339>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Scalable%20molecular%20dynamics%20with%20NAMD&journal=J.%20Comput.%20Chem.&doi=10.1002%2Fjcc.20289&volume=26&pages=1781-1802&publication_year=2005&author=Phillips%2CJC> 

77.

    Croll, T. I. ISOLDE: a physically realistic environment for model
    building into low-resolution electron-density maps. /Acta
    Crystalogr. D/ *74*, 519–530 (2018).

    Article <https://doi.org/10.1107%2FS2059798318002425>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2ltbzI>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=ISOLDE%3A%20a%20physically%20realistic%20environment%20for%20model%20building%20into%20low-resolution%20electron-density%20maps&journal=Acta%20Crystalogr.%20D&doi=10.1107%2FS2059798318002425&volume=74&pages=519-530&publication_year=2018&author=Croll%2CTI> 

78.

    Goddard, T. D. et al. UCSF ChimeraX: meeting modern challenges in
    visualization and analysis. /Protein Sci./ *27*, 14–25 (2018).

    Article <https://doi.org/10.1002%2Fpro.3235>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsVertLjN>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28710774>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=UCSF%20ChimeraX%3A%20meeting%20modern%20challenges%20in%20visualization%20and%20analysis&journal=Protein%20Sci.&doi=10.1002%2Fpro.3235&volume=27&pages=14-25&publication_year=2018&author=Goddard%2CTD> 

79.

    Tian, W., Chen, C., Lei, X., Zhao, J. & Liang, J. CASTp 3.0:
    computed atlas of surface topography of proteins. /Nucleic Acids
    Res./ *46*, W363–W367 (2018).

    Article <https://doi.org/10.1093%2Fnar%2Fgky473>  CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyrurs%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29860391>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6031066>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=CASTp%203.0%3A%20computed%20atlas%20of%20surface%20topography%20of%20proteins&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky473&volume=46&pages=W363-W367&publication_year=2018&author=Tian%2CW&author=Chen%2CC&author=Lei%2CX&author=Zhao%2CJ&author=Liang%2CJ> 

80.

    Panyukov, S. & Rabin, Y. Fluctuating filaments: statistical
    mechanics of helices. /Phys. Rev. E/ *62*, 7135–7146 (2000).

    Article <https://doi.org/10.1103%2FPhysRevE.62.7135>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2000PhRvE..62.7135P>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXnvFans74%3D>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Fluctuating%20filaments%3A%20statistical%20mechanics%20of%20helices&journal=Phys.%20Rev.%20E&doi=10.1103%2FPhysRevE.62.7135&volume=62&pages=7135-7146&publication_year=2000&author=Panyukov%2CS&author=Rabin%2CY> 

81.

    Rohou, A. & Grigorieff, N. Frealix: model-based refinement of
    helical filament structures from electron micrographs. /J. Struct.
    Biol./ *186*, 234–244 (2014).

    Article <https://doi.org/10.1016%2Fj.jsb.2014.03.012>  PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24657230>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4067737>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Frealix%3A%20model-based%20refinement%20of%20helical%20filament%20structures%20from%20electron%20micrographs&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2014.03.012&volume=186&pages=234-244&publication_year=2014&author=Rohou%2CA&author=Grigorieff%2CN> 

82.

    De La Cruz, E. M., Martiel, J.-L. & Blanchoin, L. Mechanical
    heterogeneity favors fragmentation of strained actin filaments.
    /Biophys. J./ *108*, 2270–2281 (2015).

    Article <https://doi.org/10.1016%2Fj.bpj.2015.03.058>  Google
    Scholar
    <http://scholar.google.com/scholar_lookup?&title=Mechanical%20heterogeneity%20favors%20fragmentation%20of%20strained%20actin%20filaments&journal=Biophys.%20J.&doi=10.1016%2Fj.bpj.2015.03.058&volume=108&pages=2270-2281&publication_year=2015&author=Cruz%2CEM&author=Martiel%2CJ-L&author=Blanchoin%2CL> 

83.

    Zsolnay, V., Katkar, H. H., Chou, S. Z., Pollard, T. D. & Voth, G.
    A. Structural basis for polarized elongation of actin filaments.
    /Proc. Natl Acad. Sci. USA/ *117*, 30458–30464 (2020).

    Article <https://doi.org/10.1073%2Fpnas.2011128117>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020PNAS..11730458Z>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVKnsbvL>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33199648>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7720195>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20for%20polarized%20elongation%20of%20actin%20filaments&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.2011128117&volume=117&pages=30458-30464&publication_year=2020&author=Zsolnay%2CV&author=Katkar%2CHH&author=Chou%2CSZ&author=Pollard%2CTD&author=Voth%2CGA> 

84.

    Mitchell, M. R., Tlusty, T. & Leibler, S. Strain analysis of protein
    structures and low dimensionality of mechanical allosteric
    couplings. /Proc. Natl Acad. Sci. USA/ *113*, E5847–E5855 (2016).

    Article <https://doi.org/10.1073%2Fpnas.1609462113>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016PNAS..113E5847M>  CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFertb3I>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27655887>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5056043>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Strain%20analysis%20of%20protein%20structures%20and%20low%20dimensionality%20of%20mechanical%20allosteric%20couplings&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1609462113&volume=113&pages=E5847-E5855&publication_year=2016&author=Mitchell%2CMR&author=Tlusty%2CT&author=Leibler%2CS> 

85.

    Mitchell, M. R. & Leibler, S. Elastic strain and twist analysis of
    protein structural data and allostery of the transmembrane channel
    KcsA. /Phys. Biol./ *15*, 036004 (2018).

    Article <https://doi.org/10.1088%2F1478-3975%2Faa9906>  ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2018PhBio..15c6004M>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29116053>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Elastic%20strain%20and%20twist%20analysis%20of%20protein%20structural%20data%20and%20allostery%20of%20the%20transmembrane%20channel%20KcsA&journal=Phys.%20Biol.&doi=10.1088%2F1478-3975%2Faa9906&volume=15&publication_year=2018&author=Mitchell%2CMR&author=Leibler%2CS> 

86.

    Hunter, J. D. Matplotlib: a 2D graphics environment. /Comput. Sci.
    Eng./ *9*, 90–95 (2007).

    Article <https://doi.org/10.1109%2FMCSE.2007.55>  Google Scholar
    <http://scholar.google.com/scholar_lookup?&title=Matplotlib%3A%20a%202D%20graphics%20environment&journal=Comput.%20Sci.%20Eng.&doi=10.1109%2FMCSE.2007.55&volume=9&pages=90-95&publication_year=2007&author=Hunter%2CJD> 

Download
references<https://citation-needed.springer.com/v2/references/10.1038/s41586-022-05366-w?format=refman&flavour=references>


    Acknowledgements

We thank S. Espinosa de los Reyes for preparation of ADP–F-actin grids;
R. Ranganathan for strain pseudoenergy calculation pseudocode; and H.
Ng, J. Sotiris and M. Ebrahim for assistance with cryo-EM data
collection at the Rockefeller University Cryo-electron Microscopy
Resource Center. This work was funded by a National Institutes of Health
grant (R01GM141044), an Irma T. Hirschl and Monique Weill-Caulier
Research Award, and a Pew Biomedical Scholar Award to G.M.A. R.G. was
supported by an H. Li Memorial Fellowship, and A.G.C. was supported by
NIH T32 GM115327.


    Author information


      Authors and Affiliations

 1.

    Laboratory of Structural Biophysics and Mechanobiology, The
    Rockefeller University, New York, NY, USA

    Matthew J. Reynolds, Carla Hachicho, Ayala G. Carl, Rui
    Gong & Gregory M. Alushin

 2.

    Tri-Institutional Program in Chemical Biology, The Rockefeller
    University, New York, NY, USA

    Ayala G. Carl

Authors

 1. Matthew J. Reynolds
    View author publications
    <https://www.nature.com/search?author=Matthew%20J.%20Reynolds>

    You can also search for this author in PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Matthew%20J.%20Reynolds> Google Scholar <http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Matthew%20J.%20Reynolds%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en>

 2. Carla Hachicho
    View author publications
    <https://www.nature.com/search?author=Carla%20Hachicho>

    You can also search for this author in PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Carla%20Hachicho> Google Scholar <http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Carla%20Hachicho%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en>

 3. Ayala G. Carl
    View author publications
    <https://www.nature.com/search?author=Ayala%20G.%20Carl>

    You can also search for this author in PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Ayala%20G.%20Carl> Google Scholar <http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Ayala%20G.%20Carl%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en>

 4. Rui Gong
    View author publications
    <https://www.nature.com/search?author=Rui%20Gong>

    You can also search for this author in PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Rui%20Gong> Google Scholar <http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Rui%20Gong%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en>

 5. Gregory M. Alushin
    View author publications
    <https://www.nature.com/search?author=Gregory%20M.%20Alushin>

    You can also search for this author in PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Gregory%20M.%20Alushin> Google Scholar <http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Gregory%20M.%20Alushin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en>


      Contributions

M.J.R. and G.M.A. conceived the project. M.J.R. developed custom cryo-EM
analysis software and performed all data analysis presented in the
paper. C.H. prepared proteins, performed cofilin severing assays and
prepared cryo-EM specimens. A.G.C. and R.G. prepared cryo-EM specimens
and collected cryo-EM data. M.J.R. and G.M.A. wrote the paper, with
input from all of the authors.


      Corresponding author

Correspondence to Gregory M. Alushin <mailto:galushin@rockefeller.edu>.


    Ethics declarations


      Competing interests

The authors declare no competing interests.


    Peer review


      Peer review information

/Nature/ thanks Glen Hocky and the other, anonymous, reviewer(s) for
their contribution to the peer review of this work. Peer reviewer
reports <https://www.nature.com/articles/s41586-022-05366-w#MOESM3> are
available.


    Additional information

*Publisher’s note* Springer Nature remains neutral with regard to
jurisdictional claims in published maps and institutional affiliations.


    Extended data figures and tables


      Extended Data Fig. 1 Validation of ADP–P_i -F-actin preparation
      and helically symmetric reconstructions.
      <https://www.nature.com/articles/s41586-022-05366-w/figures/5>

*a*, Representative TIRF-microscopy video frames from cofilin severing
assays. Cofilin-free controls are shown in the top row of each
condition, indicated by a darker border. Scale bar, 40 μm. *b*,
Quantification of TIRF videos showing the average normalized actin
channel intensity. Error margin in graph indicates +/− 95% CI.
Half-lives represent exponential decay at time 0 s, with 95% CI, and n
values represent independent experiments: ADP–F-actin - cofilin (778 ±
 24 s, n = 3); ADP-P_i –F-actin - cofilin (454 ± 14 s, n = 3);
ADP-sulfate–F-actin - cofilin (348 ± 16 s, n = 3); ADP–F-actin + cofilin
(50.4 ± 2.1 s, n = 4); ADP-P_i –F-actin + cofilin (177.5 ± 10.3 s, n =
 3); ADP-sulfate–F-actin + cofilin (86.8 ± 2.5 s, n = 3). *c*, Half-map
(left) and map-to-model (right) Fourier Shell Correlation (FSC) curves
for helically symmetric reconstructions of ADP- and ADP-P_i -F-actin.
*d*, Local resolution assessment of helically symmetric ADP–F-actin and
ADP-P_i –F-actin. PE: pointed end; BE: barbed end. *e*, Potential
hydrogen-bonding networks adjacent to the nucleosidyl region of ADP. Key
side chains and back bone atoms participating in hydrogen-bonding
networks are displayed and coloured by heteroatom.

Source data <https://www.nature.com/articles/s41586-022-05366-w#MOESM12>


      Extended Data Fig. 2 Additional analysis of helically symmetric
      ADP–F-actin and ADP-P_i –F-actin models.
      <https://www.nature.com/articles/s41586-022-05366-w/figures/6>

*a*, Example cryo-EM map density superimposed with atomic model residues
A131–A135 from ADP–F-actin (top) and ADP–P_i -F-actin (bottom). *b*,
Individual ADP–F-actin protomer shown in C_α representation, coloured by
per-residue RMSD between ADP–F-actin and ADP-P_i –F-actin. *c*, Same as
*b*, but coloured by per-residue strain pseudo-energy. *d*,
Superposition of extended 31-protomer ADP- (blue) and ADP-P_i -F-actin
(orange) models, aligned at the terminal barbed end protomer. *e*, Water
molecules (violet) contained within the ADP-P_i –F-actin filament’s
core. Actin subunits are shown in transparent grey backbone
representation. PE: pointed end; BE: barbed end.


      Extended Data Fig. 3 Neural network architecture and example
      performance.
      <https://www.nature.com/articles/s41586-022-05366-w/figures/7>

*a*, Neural network architecture diagram for denoising auto-encoder.
Example network input and output is displayed for a representative
extracted segment. *b*, Network architecture diagram for semantic
segmentation fully convolutional network. Example input and output for a
representative extracted segment is shown. *c*, Representative network
performance on filament segments from synthetic projections (top) and
experimental cryo-EM micrographs (bottom). Scale bars, 20 nm.


      Extended Data Fig. 4 Boltzmann modelling of filament segment
      curvature distributions.
      <https://www.nature.com/articles/s41586-022-05366-w/figures/8>

*a*, Curvature distributions (left column) of ADP–F-actin (top) and
ADP–P_i -F-actin (bottom) with modelled thermal bending probability
distributions. Residual plots (right column) between the measured and
theoretical distributions. Grey curves correspond to adjusted bending
models fit with a multiplicative parameter in the energy term. *b*,
Theoretical probability distributions of 500 Å elastic rods bending due
to thermal fluctuations, modelled as Boltzmann distributions. Varying
the persistence length between 5 μm and 15 μm demonstrates the effect of
filament bending stiffness.

Source data <https://www.nature.com/articles/s41586-022-05366-w#MOESM13>


      Extended Data Fig. 5 Additional analysis of filament bending
      deformations.
      <https://www.nature.com/articles/s41586-022-05366-w/figures/9>

*a*, Helically symmetric ADP-P_i –F-actin (left map) and cryoDRGN
reconstructions sampling ADP-P_i –F-actin bending (right three maps).
Maps are lowpass filtered to 8 Å, and strands are coloured in shades of
orange. PE: pointed end; BE: barbed end. Scale bar, 10 nm. *b*, Stitched
volumes of straight and bent maps from *a*, aligned on the bottom 16
protomers. Scale bar, 100 nm. *c*, Projections of zeroth (cyan) and
ninth (magenta) cryoDRGN reconstructions from ADP–F-actin (left) and
ADP-P_i –F-actin (right) aligned on the bottom protomer and oriented to
display maximum displacement. *d*, Asymmetric reconstructions of
ADP–F-actin from indicated curvature bins. Scale bar, 10 nm. *e*, Plots
of central axis deviations from straight lines in ADP–F-actin and
ADP-P_i –F-actin cryoDRGN reconstructions. First and second columns show
principal component analysis of the cryoDRGN reconstructions’ central
axes. Third column shows displacement of the cryoDRGN reconstructions’
central axes from straight lines which were aligned to the barbed-end
terminal 56 Å of the central axes. *f*, Half-map Fourier Shell
Correlation (FSC) curves for control asymmetric 16-protomer
reconstructions. *g*, Twist and rise measurements of control asymmetric
16-protomer reconstructions.

Source data <https://www.nature.com/articles/s41586-022-05366-w#MOESM14>


      Extended Data Fig. 6 Quantitation of lattice architectural
      remodelling during filament bending.
      <https://www.nature.com/articles/s41586-022-05366-w/figures/10>

*a*, Plots of travelling wave equation fits (black lines) with measured
twist values (coloured points) from cryoDRGN reconstructions of
different curvatures. *b*, Plots of twist travelling wave function at
various curvatures, separated by strand. *c*, Plots of intra-strand,
inter-strand, and intra-protomer subdomain distances and angles from
ISOLDE models of the most curved cryoDRGN reconstructions of
ADP–F-actin (blue) and ADP-P_i –F-actin (orange). Solid and dashed lines
represent even (concave side) and odd (convex side) protomers, respectively.

Source data <https://www.nature.com/articles/s41586-022-05366-w#MOESM15>


      Extended Data Fig. 7 Resolution assessment and validation of
      high-resolution bent F-actin asymmetric reconstructions.
      <https://www.nature.com/articles/s41586-022-05366-w/figures/11>

*a*, Local resolution assessment of bent ADP–F-actin and ADP-P_i
–F-actin reconstructions, as well as two independent straight
ADP–F-actin controls. PE: pointed end; BE: barbed end. *b*, Half-map
(left) and map-to-model (right) Fourier Shell Correlation (FSC) curves
for asymmetric bent ADP–F-actin, ADP-P_i –F-actin, and straight
ADP–F-actin controls. *c*, 3D-FSC curves for asymmetric reconstructions,
which indicate equivalently isotropic resolution between bent and
straight reconstructions. Dotted green lines indicate +/− 1 s.d. from
average FSC. *d*, Vector plots (coloured by direction and scaled 6X)
representing C_α displacements between helically symmetric models and
those built into indicated asymmetric reconstructions, aligned on the
central protomer.

Source data <https://www.nature.com/articles/s41586-022-05366-w#MOESM16>


      Extended Data Fig. 8 Strain and flexibility of inter-protomer
      contact sites in bent F-actin.
      <https://www.nature.com/articles/s41586-022-05366-w/figures/12>

*a*, Grey C_α representations of indicated protomers from bent F-actin,
locally coloured by computed per-residue strain pseudo-energy relative
to helically symmetric models on their H-plugs (left) and D-loops
(right). *b*, D-loop heterogeneity in asymmetric F-actin maps. Local
density around each of the unique D-loops from the asymmetric
reconstructions are shown with a docked PDB model of ADP–F-actin (7R8V)
featuring both “in” and “out” D-loop conformations.

Source data <https://www.nature.com/articles/s41586-022-05366-w#MOESM17>


      Extended Data Fig. 9 Steric encounters at inter-subunit interfaces
      transduce filament bending strain to protomers.
      <https://www.nature.com/articles/s41586-022-05366-w/figures/13>

*a*, Longitudinal interfaces and *b*, lateral interfaces of bent
ADP–F-actin (left) and ADP-P_i –F-actin (right). Solid and dashed
borders represent inside (concave) and outside (convex) of curve,
respectively. Transparent arrows represent individual C_α displacements
from helically symmetric models scaled 6X, and solid arrows show
averaged displacements of indicated regions scaled 20X. PE: pointed end;
BE: barbed end.


      Extended Data Fig. 10 F-actin bending remodels inter-subunit
      interfaces engaged by ABPs.
      <https://www.nature.com/articles/s41586-022-05366-w/figures/14>

*a*, C_α representations of inner strand protomer 6-protomer 8 (concave,
cornflower blue) and outer strand protomer 7-protomer 9 (convex, light
blue) longitudinal interfaces extracted from the most curved ADP–F-actin
cryoDRGN 16-protomter atomic model, superimposed on the barbed end
subunit of 2 protomers extracted from the ADP–F-actin helical model
(grey). Box encloses a region featuring major contacts by ABPs. BE:
barbed end; PE: pointed end. Actin subdomains are indicated. *b*,
Binding interfaces of indicated ABPs (pink space filling model) are
displayed superimposed on the bent interfaces from *a*. Cofilin, PDB
5YU8; coronin, EMDB 6100 / PDB 2AQ5; ARP2/3, PDB 7TPT; α-catenin, PDB
6UPV; myosin-5, PDB 7PLT.


    Supplementary information


      Supplementary Information
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM1_ESM.pdf>

Supplementary Tables 1–3: cryo-EM data collection, refinement and
validation statistics, and twist travelling wave equation parameters.
Supplementary Discussion: detailed structural analysis and thermal
fluctuation modelling of bending deformations.


      Reporting Summary
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM2_ESM.pdf>


      Peer Review File
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM3_ESM.pdf>


      Supplementary Video 1 Solvated nucleotide clefts in ADP–F-actin
      and ADP-P_i –F-actin.
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM4_ESM.mp4>

Views of ADP–F-actin (blue) and ADP-P_i –F-actin (orange) nucleotide
clefts. Atomic models are shown in Cα representation. Transparent grey
density from the maps is displayed. Green, ADP; yellow, PO_4 ^3− ; light
green, Mg^2+ ; violet, water. In ADP/PO_4 ^3− , nitrogen atoms are blue,
oxygen atoms are red and phosphorous atoms are yellow.


      Supplementary Video 2 Morphs along bending trajectories of
      cryo-DRGN reconstructions.
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM5_ESM.mp4>

Morphs of cryo-DRGN reconstructions sampling continuous bending of
ADP–F-actin and ADP-P_i –F-actin, low-pass filtered to 8 Å and aligned
at two bottom protomers, as well as associated helical parameters.
Reconstructed ADP–F-actin strands are coloured in shades of blue, and
reconstructed ADP-P_i –F-actin strands are coloured in shades of orange.
Rise and twist measurements are separated by strand and darken in colour
as curvature increases. The vertical dashed lines indicate helical
parameters of canonical F-actin.


      Supplementary Video 3 Morphs of asymmetric bent and straight
      ADP–F-actin reconstructions.
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM6_ESM.mp4>

Morphs of asymmetrically reconstructed ADP maps, aligned at the central
protomer. Both maps start from asymmetric, straight ADP map 1. The left
morph interpolates to the bent ADP map; the right morph interpolates to
straight ADP map 2.


      Supplementary Video 4 Morphs between straight and bent F-actin
      highlighting subunit shearing.
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM7_ESM.mp4>

Morphs of protomer subdomain shearing from the helically symmetric
protomer model to bent protomer models. Vectors (scaled 100×) indicate
subdomain-averaged displacements.


      Supplementary Video 5 Morphs highlighting longitudinal interface
      deformations during bending.
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM8_ESM.mp4>

Morphs of longitudinal interfaces of ADP–F-actin (left) and ADP-P_i
–F-actin (right) models from helically symmetric models to bent models.
The transparent arrows represent individual Cα displacements from
helically symmetric models scaled 6×, and the solid arrows show averaged
displacements of indicated regions scaled 20×.


      Supplementary Video 6 Morphs highlighting lateral interface
      deformations during bending.
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM9_ESM.mp4>

Morphs of lateral interfaces of ADP–F-actin (left) and ADP-P_i –F-actin
(right) models from helically symmetric models to bent models. The
transparent arrows represent individual Cα displacements from helically
symmetric models scaled 6×, and the solid arrows show averaged
displacements of the indicated regions scaled 20×.


    Source data


      Source Data Fig. 3
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM10_ESM.xlsx>


      Source Data Fig. 4
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM11_ESM.xlsx>


      Source Data Extended Data Fig. 1
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM12_ESM.xlsx>


      Source Data Extended Data Fig. 4
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM13_ESM.xlsx>


      Source Data Extended Data Fig. 5
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM14_ESM.xlsx>


      Source Data Extended Data Fig. 6
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM15_ESM.xlsx>


      Source Data Extended Data Fig. 7
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM16_ESM.xlsx>


      Source Data Extended Data Fig. 8
      <https://static-content.springer.com/esm/art%3A10.1038%2Fs41586-022-05366-w/MediaObjects/41586_2022_5366_MOESM17_ESM.xlsx>


    Rights and permissions

*Open Access* This article is licensed under a Creative Commons
Attribution 4.0 International License, which permits use, sharing,
adaptation, distribution and reproduction in any medium or format, as
long as you give appropriate credit to the original author(s) and the
source, provide a link to the Creative Commons license, and indicate if
changes were made. The images or other third party material in this
article are included in the article’s Creative Commons license, unless
indicated otherwise in a credit line to the material. If material is not
included in the article’s Creative Commons license and your intended use
is not permitted by statutory regulation or exceeds the permitted use,
you will need to obtain permission directly from the copyright holder.
To view a copy of this license, visit
http://creativecommons.org/licenses/by/4.0/
<http://creativecommons.org/licenses/by/4.0/>.

Reprints and Permissions
<https://s100.copyright.com/AppDispatchServlet?title=Bending%20forces%20and%20nucleotide%20state%20jointly%20regulate%20F-actin%20structure&author=Matthew%20J.%20Reynolds%20et%20al&contentID=10.1038%2Fs41586-022-05366-w&copyright=The%20Author%28s%29&publication=0028-0836&publicationDate=2022-10-26&publisherName=SpringerNature&orderBeanReset=true&oa=CC%20BY>


    About this article

Check for updates. Verify currency and authenticity via CrossMark
<https://crossmark.crossref.org/dialog/?doi=10.1038/s41586-022-05366-w>


      Cite this article

Reynolds, M.J., Hachicho, C., Carl, A.G. /et al./ Bending forces and
nucleotide state jointly regulate F-actin structure. /Nature/ *611*,
380–386 (2022). https://doi.org/10.1038/s41586-022-05366-w

Download
citation<https://citation-needed.springer.com/v2/references/10.1038/s41586-022-05366-w?format=refman&flavour=citation>

  *

    Received: 02 June 2022

  *

    Accepted: 20 September 2022

  *

    Published: 26 October 2022

  *

    Issue Date: 10 November 2022

  *

    DOI: https://doi.org/10.1038/s41586-022-05366-w


      Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative


      Subjects

  * Actin <https://www.nature.com/subjects/actin>
  * Cryoelectron microscopy
    <https://www.nature.com/subjects/cryoelectron-microscopy>


    This article is cited by

  *


          Distinct inter-domain interactions of dimeric versus monomeric
          α-catenin link cell junctions to filaments
          <https://doi.org/10.1038/s42003-023-04610-x>

      o Erumbi S. Rangarajan
      o Emmanuel W. Smith
      o Tina Izard

    /Communications Biology/ (2023)

  *


          Catching actin proteins in action
          <https://doi.org/10.1038/d41586-022-03343-x>

      o Pilar Cossio
      o Glen M. Hocky

    /Nature/ (2022)


    Comments

By submitting a comment you agree to abide by our Terms
<https://www.nature.com/info/tandc.html> and Community Guidelines
<https://www.nature.com/info/community-guidelines.html>. If you find
something abusive or that does not comply with our terms or guidelines
please flag it as inappropriate.

Download PDF

<https://www.nature.com/articles/s41586-022-05366-w.pdf>


    Associated Content


      Catching actin proteins in action
      <https://www.nature.com/articles/d41586-022-03343-x>

  * Pilar Cossio
  * Glen M. Hocky

Nature News & Views 26 Oct 2022

  * Sections
  * Figures
  * References

  * Abstract <#Abs1>
  * Main <#Sec1>
  * Solvent in the nucleotide cleft of F-actin <#Sec2>
  * Water molecules mediate intersubunit contacts <#Sec3>
  * Visualizing F-actin bending deformations <#Sec4>
  * Bending remodels F-actin’s lattice <#Sec5>
  * Nucleotide clefts feature shear strain <#Sec6>
  * Discussion <#Sec7>
  * Methods <#Sec8>
  * Data availability <#data-availability>
  * Code availability <#code-availability>
  * References <#Bib1>
  * Acknowledgements <#Ack1>
  * Author information <#author-information>
  * Ethics declarations <#ethics>
  * Peer review <#peer-review>
  * Additional information <#additional-information>
  * Extended data figures and tables <#Sec36>
  * Supplementary information <#Sec37>
  * Source data <#Sec38>
  * Rights and permissions <#rightslink>
  * About this article <#article-info>
  * This article is cited by <#further-reading>
  * Comments <#article-comments>

Advertisement

  *
    *Fig. 1: Nucleotide cleft water networks are remodelled upon
    phosphate release by F-actin.

    *

    figure 1

    View in article <#Fig1>Full size image

    <https://www.nature.com/articles/s41586-022-05366-w/figures/1>

  *
    *Fig. 2: Water molecules mediate key longitudinal and lateral
    contacts in F-actin.

    *

    figure 2

    View in article <#Fig2>Full size image

    <https://www.nature.com/articles/s41586-022-05366-w/figures/2>

  *
    *Fig. 3: Cryo-EM reconstructions of mechanically deformed F-actin
    reveal bend–twist coupling.

    *

    figure 3

    View in article <#Fig3>Full size image

    <https://www.nature.com/articles/s41586-022-05366-w/figures/3>

  *
    *Fig. 4: Actin nucleotide state modulates subunit shearing during
    filament bending.

    *

    figure 4

    View in article <#Fig4>Full size image

    <https://www.nature.com/articles/s41586-022-05366-w/figures/4>

  *
    *Extended Data Fig. 1 Validation of ADP–P_i -F-actin preparation and
    helically symmetric reconstructions.

    *

    extended data figure 5

    View in article <#Fig5>Full size image

    <https://www.nature.com/articles/s41586-022-05366-w/figures/5>

  *
    *Extended Data Fig. 2 Additional analysis of helically symmetric
    ADP–F-actin and ADP-P_i –F-actin models.

    *

    extended data figure 6

    View in article <#Fig6>Full size image

    <https://www.nature.com/articles/s41586-022-05366-w/figures/6>

  *
    *Extended Data Fig. 3 Neural network architecture and example
    performance.

    *

    extended data figure 7

    View in article <#Fig7>Full size image

    <https://www.nature.com/articles/s41586-022-05366-w/figures/7>

  *
    *Extended Data Fig. 4 Boltzmann modelling of filament segment
    curvature distributions.

    *

    extended data figure 8

    View in article <#Fig8>Full size image

    <https://www.nature.com/articles/s41586-022-05366-w/figures/8>

  *
    *Extended Data Fig. 5 Additional analysis of filament bending
    deformations.

    *

    extended data figure 9

    View in article <#Fig9>Full size image

    <https://www.nature.com/articles/s41586-022-05366-w/figures/9>

  *
    *Extended Data Fig. 6 Quantitation of lattice architectural
    remodelling during filament bending.

    *

    extended data figure 10

    View in article <#Fig10>Full size image

    <https://www.nature.com/articles/s41586-022-05366-w/figures/10>

  *
    *Extended Data Fig. 7 Resolution assessment and validation of
    high-resolution bent F-actin asymmetric reconstructions.

    *

    extended data figure 11

    View in article <#Fig11>Full size image

    <https://www.nature.com/articles/s41586-022-05366-w/figures/11>

  *
    *Extended Data Fig. 8 Strain and flexibility of inter-protomer
    contact sites in bent F-actin.

    *

    extended data figure 12

    View in article <#Fig12>Full size image

    <https://www.nature.com/articles/s41586-022-05366-w/figures/12>

  *
    *Extended Data Fig. 9 Steric encounters at inter-subunit interfaces
    transduce filament bending strain to protomers.

    *

    extended data figure 13

    View in article <#Fig13>Full size image

    <https://www.nature.com/articles/s41586-022-05366-w/figures/13>

  *
    *Extended Data Fig. 10 F-actin bending remodels inter-subunit
    interfaces engaged by ABPs.

    *

    extended data figure 14

    View in article <#Fig14>Full size image

    <https://www.nature.com/articles/s41586-022-05366-w/figures/14>

 1.

    Pollard, T. D. & Borisy, G. G. Cellular motility driven by assembly
    and disassembly of actin filaments. /Cell/ *112*, 453–465 (2003).

    Article <https://doi.org/10.1016%2FS0092-8674%2803%2900120-X> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXhs1SnsL0%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12600310> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Cellular%20motility%20driven%20by%20assembly%20and%20disassembly%20of%20actin%20filaments&journal=Cell&doi=10.1016%2FS0092-8674%2803%2900120-X&volume=112&pages=453-465&publication_year=2003&author=Pollard%2CTD&author=Borisy%2CGG> 

 2.

    Mogilner, A. & Oster, G. Cell motility driven by actin
    polymerization. /Biophys. J./ *71*, 3030–3045 (1996).

    Article <https://doi.org/10.1016%2FS0006-3495%2896%2979496-1> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=1996BpJ....71.3030M> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28XnsFKltLs%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8968574> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1233792> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Cell%20motility%20driven%20by%20actin%20polymerization&journal=Biophys.%20J.&doi=10.1016%2FS0006-3495%2896%2979496-1&volume=71&pages=3030-3045&publication_year=1996&author=Mogilner%2CA&author=Oster%2CG> 

 3.

    Mogilner, A. & Oster, G. Force generation by actin polymerization
    II: the elastic ratchet and tethered filaments. /Biophys. J./ *84*,
    1591–1605 (2003).

    Article <https://doi.org/10.1016%2FS0006-3495%2803%2974969-8> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2003BpJ....84.1591M> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXit1ehsr4%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12609863> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1302730> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Force%20generation%20by%20actin%20polymerization%20II%3A%20the%20elastic%20ratchet%20and%20tethered%20filaments&journal=Biophys.%20J.&doi=10.1016%2FS0006-3495%2803%2974969-8&volume=84&pages=1591-1605&publication_year=2003&author=Mogilner%2CA&author=Oster%2CG> 

 4.

    Jégou, A. & Romet-Lemonne, G. Mechanically tuning actin filaments to
    modulate the action of actin-binding proteins. /Curr. Opin. Cell
    Biol./ *68*, 72–80 (2021).

    Article <https://doi.org/10.1016%2Fj.ceb.2020.09.002> PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33160108> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Mechanically%20tuning%20actin%20filaments%20to%20modulate%20the%20action%20of%20actin-binding%20proteins&journal=Curr.%20Opin.%20Cell%20Biol.&doi=10.1016%2Fj.ceb.2020.09.002&volume=68&pages=72-80&publication_year=2021&author=J%C3%A9gou%2CA&author=Romet-Lemonne%2CG> 

 5.

    Sun, X. & Alushin, G. M. Cellular force-sensing through actin
    filaments. /FEBS J./ https://doi.org/10.1111/febs.16568
    <https://doi.org/10.1111/febs.16568> (2022).
 6.

    Pollard, T. D., Blanchoin, L. & Mullins, R. D. Molecular mechanisms
    controlling actin filament dynamics in nonmuscle cells. /Annu. Rev.
    Biophys. Biomol. Struct./ *29*, 545–576 (2000).

    Article <https://doi.org/10.1146%2Fannurev.biophys.29.1.545> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXlsFOnu78%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10940259> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Molecular%20mechanisms%20controlling%20actin%20filament%20dynamics%20in%20nonmuscle%20cells&journal=Annu.%20Rev.%20Biophys.%20Biomol.%20Struct.&doi=10.1146%2Fannurev.biophys.29.1.545&volume=29&pages=545-576&publication_year=2000&author=Pollard%2CTD&author=Blanchoin%2CL&author=Mullins%2CRD> 

 7.

    Cameron, L. A., Svitkina, T. M., Vignjevic, D., Theriot, J. A. &
    Borisy, G. G. Dendritic organization of actin comet tails. /Curr.
    Biol./ *11*, 130–135 (2001).

    Article <https://doi.org/10.1016%2FS0960-9822%2801%2900022-7> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXhtVSgur0%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11231131> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Dendritic%20organization%20of%20actin%20comet%20tails&journal=Curr.%20Biol.&doi=10.1016%2FS0960-9822%2801%2900022-7&volume=11&pages=130-135&publication_year=2001&author=Cameron%2CLA&author=Svitkina%2CTM&author=Vignjevic%2CD&author=Theriot%2CJA&author=Borisy%2CGG> 

 8.

    Svitkina, T. M., Verkhovsky, A. B., McQuade, K. M. & Borisy, G. G.
    Analysis of the actin–myosin II system in fish epidermal
    keratocytes: mechanism of cell body translocation. /J. Cell Biol./
    *139*, 397–415 (1997).

    Article <https://doi.org/10.1083%2Fjcb.139.2.397> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXmvV2gurg%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9334344> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2139803> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Analysis%20of%20the%20actin%E2%80%93myosin%20II%20system%20in%20fish%20epidermal%20keratocytes%3A%20mechanism%20of%20cell%20body%20translocation&journal=J.%20Cell%20Biol.&doi=10.1083%2Fjcb.139.2.397&volume=139&pages=397-415&publication_year=1997&author=Svitkina%2CTM&author=Verkhovsky%2CAB&author=McQuade%2CKM&author=Borisy%2CGG> 

 9.

    Akamatsu, M. et al. Principles of self-organization and load
    adaptation by the actin cytoskeleton during clathrin-mediated
    endocytosis. /eLife/ *9*, e49840 (2020).

    Article <https://doi.org/10.7554%2FeLife.49840> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlarsLvO> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31951196> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7041948> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Principles%20of%20self-organization%20and%20load%20adaptation%20by%20the%20actin%20cytoskeleton%20during%20clathrin-mediated%20endocytosis&journal=eLife&doi=10.7554%2FeLife.49840&volume=9&publication_year=2020&author=Akamatsu%2CM> 

10.

    Jasnin, M. et al. Elasticity of podosome actin networks produces
    nanonewton protrusive forces. /Nat. Commun./ *13*, 3842 (2022).

    Article <https://doi.org/10.1038%2Fs41467-022-30652-6> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2022NatCo..13.3842J> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xhslegu7fN> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35789161> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9253342> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Elasticity%20of%20podosome%20actin%20networks%20produces%20nanonewton%20protrusive%20forces&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-022-30652-6&volume=13&publication_year=2022&author=Jasnin%2CM> 

11.

    Ponti, A., Machacek, M., Gupton, S. L., Waterman-Storer, C. M. &
    Danuser, G. Two distinct actin networks drive the protrusion of
    migrating cells. /Science/ *305*, 1782–1786 (2004).

    Article <https://doi.org/10.1126%2Fscience.1100533> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2004Sci...305.1782P> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXnsFajur4%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15375270> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Two%20distinct%20actin%20networks%20drive%20the%20protrusion%20of%20migrating%20cells&journal=Science&doi=10.1126%2Fscience.1100533&volume=305&pages=1782-1786&publication_year=2004&author=Ponti%2CA&author=Machacek%2CM&author=Gupton%2CSL&author=Waterman-Storer%2CCM&author=Danuser%2CG> 

12.

    Galkin, V. E., Orlova, A. & Egelman, E. H. Actin filaments as
    tension sensors. /Curr. Biol./ *22*, R96–R101 (2012).

    Article <https://doi.org/10.1016%2Fj.cub.2011.12.010> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvFeit7s%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22321312> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3277726> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20filaments%20as%20tension%20sensors&journal=Curr.%20Biol.&doi=10.1016%2Fj.cub.2011.12.010&volume=22&pages=R96-R101&publication_year=2012&author=Galkin%2CVE&author=Orlova%2CA&author=Egelman%2CEH> 

13.

    Zimmermann, D., Santos, A., Kovar, D. R. & Rock, R. S. Actin age
    orchestrates myosin-5 and myosin-6 run lengths. /Curr. Biol./ *25*,
    2057–2062 (2015).

    Article <https://doi.org/10.1016%2Fj.cub.2015.06.033> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtF2ksL3N> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26190073> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4556227> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20age%20orchestrates%20myosin-5%20and%20myosin-6%20run%20lengths&journal=Curr.%20Biol.&doi=10.1016%2Fj.cub.2015.06.033&volume=25&pages=2057-2062&publication_year=2015&author=Zimmermann%2CD&author=Santos%2CA&author=Kovar%2CDR&author=Rock%2CRS> 

14.

    Blanchoin, L. & Pollard, T. D. Mechanism of interaction of
    acanthamoeba actophorin (ADF/cofilin) with actin filaments. /J.
    Biol. Chem./ *274*, 15538–15546 (1999).

    Article <https://doi.org/10.1074%2Fjbc.274.22.15538> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK1MXjsV2gsLc%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10336448> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Mechanism%20of%20interaction%20of%20acanthamoeba%20actophorin%20%28ADF%2Fcofilin%29%20with%20actin%20filaments.&journal=J.%20Biol.%20Chem.&doi=10.1074%2Fjbc.274.22.15538&volume=274&pages=15538-15546&publication_year=1999&author=Blanchoin%2CL&author=Pollard%2CTD> 

15.

    Suarez, C. et al. Cofilin tunes the nucleotide state of actin
    filaments and severs at bare and decorated segment boundaries.
    /Curr. Biol./ *21*, 862–868 (2011).

    Article <https://doi.org/10.1016%2Fj.cub.2011.03.064> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmsVOqtLc%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21530260> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3100394> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Cofilin%20tunes%20the%20nucleotide%20state%20of%20actin%20filaments%20and%20severs%20at%20bare%20and%20decorated%20segment%20boundaries&journal=Curr.%20Biol.&doi=10.1016%2Fj.cub.2011.03.064&volume=21&pages=862-868&publication_year=2011&author=Suarez%2CC> 

16.

    Hayakawa, K., Tatsumi, H. & Sokabe, M. Actin filaments function as a
    tension sensor by tension-dependent binding of cofilin to the
    filament. /J. Cell Biol./ *195*, 721–727 (2011).

    Article <https://doi.org/10.1083%2Fjcb.201102039> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs1Shsb3E> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22123860> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3257564> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20filaments%20function%20as%20a%20tension%20sensor%20by%20tension-dependent%20binding%20of%20cofilin%20to%20the%20filament&journal=J.%20Cell%20Biol.&doi=10.1083%2Fjcb.201102039&volume=195&pages=721-727&publication_year=2011&author=Hayakawa%2CK&author=Tatsumi%2CH&author=Sokabe%2CM> 

17.

    Wioland, H., Jegou, A. & Romet-Lemonne, G. Torsional stress
    generated by ADF/cofilin on cross-linked actin filaments boosts
    their severing. /Proc. Natl Acad. Sci. USA/ *116*, 2595–2602 (2019).

    Article <https://doi.org/10.1073%2Fpnas.1812053116> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019PNAS..116.2595W> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXivVajsro%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30692249> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6377502> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Torsional%20stress%20generated%20by%20ADF%2Fcofilin%20on%20cross-linked%20actin%20filaments%20boosts%20their%20severing&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1812053116&volume=116&pages=2595-2602&publication_year=2019&author=Wioland%2CH&author=Jegou%2CA&author=Romet-Lemonne%2CG> 

18.

    Pandit, N. G. et al. Force and phosphate release from Arp2/3 complex
    promote dissociation of actin filament branches. /Proc. Natl Acad.
    Sci. USA/ *117*, 13519–13528 (2020).

    Article <https://doi.org/10.1073%2Fpnas.1911183117> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020PNAS..11713519P> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtF2it7jL> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32461373> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7306818> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Force%20and%20phosphate%20release%20from%20Arp2%2F3%20complex%20promote%20dissociation%20of%20actin%20filament%20branches&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1911183117&volume=117&pages=13519-13528&publication_year=2020&author=Pandit%2CNG> 

19.

    Risca, V. I. et al. Actin filament curvature biases branching
    direction. /Proc. Natl Acad. Sci. USA/ *109*, 2913–2918 (2012).

    Article <https://doi.org/10.1073%2Fpnas.1114292109> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2012PNAS..109.2913R> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xjs1Wmsbg%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22308368> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3286980> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20filament%20curvature%20biases%20branching%20direction&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1114292109&volume=109&pages=2913-2918&publication_year=2012&author=Risca%2CVI> 

20.

    Bieling, P. et al. Force feedback controls motor activity and
    mechanical properties of self-assembling branched actin networks.
    /Cell/ *164*, 115–127 (2016).

    Article <https://doi.org/10.1016%2Fj.cell.2015.11.057> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtFShsbc%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26771487> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5033619> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Force%20feedback%20controls%20motor%20activity%20and%20mechanical%20properties%20of%20self-assembling%20branched%20actin%20networks&journal=Cell&doi=10.1016%2Fj.cell.2015.11.057&volume=164&pages=115-127&publication_year=2016&author=Bieling%2CP> 

21.

    Mueller, J. et al. Load adaptation of lamellipodial actin networks.
    /Cell/ *171*, 188–200 (2017).

    Article <https://doi.org/10.1016%2Fj.cell.2017.07.051> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsVert7jE> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28867286> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Load%20adaptation%20of%20lamellipodial%20actin%20networks&journal=Cell&doi=10.1016%2Fj.cell.2017.07.051&volume=171&pages=188-200&publication_year=2017&author=Mueller%2CJ> 

22.

    Ge, P., Durer, Z. A. O., Kudryashov, D., Zhou, Z. H. & Reisler, E.
    Cryo-EM reveals different coronin binding modes for ADP– and
    ADP–BeFx actin filaments. /Nat. Struct. Mol. Biol./ *21*, 1075–1081
    (2014).

    Article <https://doi.org/10.1038%2Fnsmb.2907> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVGiurfF> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25362487> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4388421> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Cryo-EM%20reveals%20different%20coronin%20binding%20modes%20for%20ADP%E2%80%93%20and%20ADP%E2%80%93BeFx%20actin%20filaments&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.2907&volume=21&pages=1075-1081&publication_year=2014&author=Ge%2CP&author=Durer%2CZAO&author=Kudryashov%2CD&author=Zhou%2CZH&author=Reisler%2CE> 

23.

    Mei, L. et al. Molecular mechanism for direct actin force-sensing by
    α-catenin. /eLife/ *9*, e62514 (2020).

    Article <https://doi.org/10.7554%2FeLife.62514> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFCrurrJ> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32969337> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7588232> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Molecular%20mechanism%20for%20direct%20actin%20force-sensing%20by%20%CE%B1-catenin&journal=eLife&doi=10.7554%2FeLife.62514&volume=9&publication_year=2020&author=Mei%2CL> 

24.

    Sun, X. et al. Mechanosensing through direct binding of tensed
    F-actin by LIM domains. /Dev. Cell/ *55*, 468–482 (2020).

    Article <https://doi.org/10.1016%2Fj.devcel.2020.09.022> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFSns7zP> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33058779> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7686152> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Mechanosensing%20through%20direct%20binding%20of%20tensed%20F-actin%20by%20LIM%20domains&journal=Dev.%20Cell&doi=10.1016%2Fj.devcel.2020.09.022&volume=55&pages=468-482&publication_year=2020&author=Sun%2CX> 

25.

    Winkelman, J. D., Anderson, C. A., Suarez, C., Kovar, D. R. &
    Gardel, M. L. Evolutionarily diverse LIM domain-containing proteins
    bind stressed actin filaments through a conserved mechanism. /Proc.
    Natl Acad. Sci. USA/ *117*, 25532–25542 (2020).

    Article <https://doi.org/10.1073%2Fpnas.2004656117> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020PNAS..11725532W> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVKkt7jF> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32989126> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7568268> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Evolutionarily%20diverse%20LIM%20domain-containing%20proteins%20bind%20stressed%20actin%20filaments%20through%20a%20conserved%20mechanism&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.2004656117&volume=117&pages=25532-25542&publication_year=2020&author=Winkelman%2CJD&author=Anderson%2CCA&author=Suarez%2CC&author=Kovar%2CDR&author=Gardel%2CML> 

26.

    Oda, T., Iwasa, M., Aihara, T., Maéda, Y. & Narita, A. The nature of
    the globular- to fibrous-actin transition. /Nature/ *457*, 441–445
    (2009).

    Article <https://doi.org/10.1038%2Fnature07685> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2009Natur.457..441O> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXotlOnsQ%3D%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19158791> Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20nature%20of%20the%20globular-%20to%20fibrous-actin%20transition&journal=Nature&doi=10.1038%2Fnature07685&volume=457&pages=441-445&publication_year=2009&author=Oda%2CT&author=Iwasa%2CM&author=Aihara%2CT&author=Ma%C3%A9da%2CY&author=Narita%2CA> 

27.

    Merino, F. et al. Structural transitions of F-actin upon ATP
    hydrolysis at near-atomic resolution revealed by cryo-EM. /Nat.
    Struct. Mol. Biol./ *25*, 528–537 (2018).

    Article <https://doi.org/10.1038%2Fs41594-018-0074-0> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlKqu7nP> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29867215> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20transitions%20of%20F-actin%20upon%20ATP%20hydrolysis%20at%20near-atomic%20resolution%20revealed%20by%20cryo-EM&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fs41594-018-0074-0&volume=25&pages=528-537&publication_year=2018&author=Merino%2CF> 

28.

    Pospich, S., Merino, F. & Raunser, S. Structural effects and
    functional implications of phalloidin and jasplakinolide binding to
    actin filaments. /Structure/ *28*, 437–449 (2020).

    Article <https://doi.org/10.1016%2Fj.str.2020.01.014> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjsVyisr0%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32084355> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20effects%20and%20functional%20implications%20of%20phalloidin%20and%20jasplakinolide%20binding%20to%20actin%20filaments&journal=Structure&doi=10.1016%2Fj.str.2020.01.014&volume=28&pages=437-449&publication_year=2020&author=Pospich%2CS&author=Merino%2CF&author=Raunser%2CS> 

29.

    Dominguez, R. & Holmes, K. C. Actin structure and function. /Annu.
    Rev. Biophys./ *40*, 169–186 (2011).

    Article
    <https://doi.org/10.1146%2Fannurev-biophys-042910-155359> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXnvFaitbk%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21314430> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3130349> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20structure%20and%20function&journal=Annu.%20Rev.%20Biophys.&doi=10.1146%2Fannurev-biophys-042910-155359&volume=40&pages=169-186&publication_year=2011&author=Dominguez%2CR&author=Holmes%2CKC> 

30.

    Galkin, V. E., Orlova, A., Schröder, G. F. & Egelman, E. H.
    Structural polymorphism in F-actin. /Nat. Struct. Mol. Biol./ *17*,
    1318–1323 (2010).

    Article <https://doi.org/10.1038%2Fnsmb.1930> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXht1Kms7rO> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20935633> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2988880> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20polymorphism%20in%20F-actin&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fnsmb.1930&volume=17&pages=1318-1323&publication_year=2010&author=Galkin%2CVE&author=Orlova%2CA&author=Schr%C3%B6der%2CGF&author=Egelman%2CEH> 

31.

    Oda, T., Takeda, S., Narita, A. & Maéda, Y. Structural polymorphism
    of actin. /J. Mol. Biol./ *431*, 3217–3228 (2019).

    Article <https://doi.org/10.1016%2Fj.jmb.2019.05.048> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFOktrvM> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31181287> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20polymorphism%20of%20actin&journal=J.%20Mol.%20Biol.&doi=10.1016%2Fj.jmb.2019.05.048&volume=431&pages=3217-3228&publication_year=2019&author=Oda%2CT&author=Takeda%2CS&author=Narita%2CA&author=Ma%C3%A9da%2CY> 

32.

    Chou, S. Z. & Pollard, T. D. Mechanism of actin polymerization
    revealed by cryo-EM structures of actin filaments with three
    different bound nucleotides. /Proc. Natl Acad. Sci. USA/ *116*,
    4265–4274 (2019).

    Article <https://doi.org/10.1073%2Fpnas.1807028115> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2019PNAS..116.4265C> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXktFeis7k%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30760599> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6410863> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Mechanism%20of%20actin%20polymerization%20revealed%20by%20cryo-EM%20structures%20of%20actin%20filaments%20with%20three%20different%20bound%20nucleotides&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1807028115&volume=116&pages=4265-4274&publication_year=2019&author=Chou%2CSZ&author=Pollard%2CTD> 

33.

    McGough, A., Pope, B., Chiu, W. & Weeds, A. Cofilin changes the
    twist of F-actin: implications for actin filament dynamics and
    cellular function. /J. Cell Biol./ *138*, 771–781 (1997).

    Article <https://doi.org/10.1083%2Fjcb.138.4.771> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXlsleitbY%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=9265645> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2138052> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Cofilin%20changes%20the%20twist%20of%20F-actin%3A%20implications%20for%20actin%20filament%20dynamics%20and%20cellular%20function&journal=J.%20Cell%20Biol.&doi=10.1083%2Fjcb.138.4.771&volume=138&pages=771-781&publication_year=1997&author=McGough%2CA&author=Pope%2CB&author=Chiu%2CW&author=Weeds%2CA> 

34.

    Tanaka, K. et al. Structural basis for cofilin binding and actin
    filament disassembly. /Nat. Commun./ *9*, 1860 (2018).

    Article <https://doi.org/10.1038%2Fs41467-018-04290-w> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2018NatCo...9.1860T> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29749375> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5945598> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20for%20cofilin%20binding%20and%20actin%20filament%20disassembly&journal=Nat.%20Commun.&doi=10.1038%2Fs41467-018-04290-w&volume=9&publication_year=2018&author=Tanaka%2CK> 

35.

    Isambert, H. et al. Flexibility of actin filaments derived from
    thermal fluctuations: effect of bound nucleotide, phalloidin, and
    muscle regulatory proteins. /J. Biol. Chem./ *270*, 11437–11444 (1995).

    Article <https://doi.org/10.1074%2Fjbc.270.19.11437> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2MXls1eis7s%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=7744781> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Flexibility%20of%20actin%20filaments%20derived%20from%20thermal%20fluctuations%3A%20effect%20of%20bound%20nucleotide%2C%20phalloidin%2C%20and%20muscle%20regulatory%20proteins&journal=J.%20Biol.%20Chem.&doi=10.1074%2Fjbc.270.19.11437&volume=270&pages=11437-11444&publication_year=1995&author=Isambert%2CH> 

36.

    Saunders, M. G. & Voth, G. A. Comparison between actin filament
    models: coarse-graining reveals essential differences. /Structure/
    *20*, 641–653 (2012).

    Article <https://doi.org/10.1016%2Fj.str.2012.02.008> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xltl2ktbo%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22483111> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Comparison%20between%20actin%20filament%20models%3A%20coarse-graining%20reveals%20essential%20differences&journal=Structure&doi=10.1016%2Fj.str.2012.02.008&volume=20&pages=641-653&publication_year=2012&author=Saunders%2CMG&author=Voth%2CGA> 

37.

    De La Cruz, E. M., Roland, J., McCullough, B. R., Blanchoin, L. &
    Martiel, J.-L. Origin of twist-bend coupling in actin filaments.
    /Biophys. J./ *99*, 1852–1860 (2010).

    Article <https://doi.org/10.1016%2Fj.bpj.2010.07.009> Google Scholar
    <http://scholar.google.com/scholar_lookup?&title=Origin%20of%20twist-bend%20coupling%20in%20actin%20filaments&journal=Biophys.%20J.&doi=10.1016%2Fj.bpj.2010.07.009&volume=99&pages=1852-1860&publication_year=2010&author=Cruz%2CEM&author=Roland%2CJ&author=McCullough%2CBR&author=Blanchoin%2CL&author=Martiel%2CJ-L> 

38.

    Katkar, H. H. et al. Insights into the cooperative nature of ATP
    hydrolysis in actin filaments. /Biophys. J./ *115*, 1589–1602 (2018).

    Article <https://doi.org/10.1016%2Fj.bpj.2018.08.034> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2018BpJ...115.1589K> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslCju7zM> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30249402> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6260209> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Insights%20into%20the%20cooperative%20nature%20of%20ATP%20hydrolysis%20in%20actin%20filaments&journal=Biophys.%20J.&doi=10.1016%2Fj.bpj.2018.08.034&volume=115&pages=1589-1602&publication_year=2018&author=Katkar%2CHH> 

39.

    Oosterheert, W., Klink, B. U., Belyy, A., Pospich, S. & Raunser, S.
    Structural basis of actin filament assembly and aging. Preprint at
    /bioRxiv/ https://doi.org/10.1101/2022.03.29.486216
    <https://doi.org/10.1101/2022.03.29.486216> (2022).
40.

    Das, S. et al. D-loop dynamics and near-atomic-resolution cryo-EM
    structure of phalloidin-bound F-actin. /Structure/ *28*, 586–593 (2020).

    Article <https://doi.org/10.1016%2Fj.str.2020.04.004> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXosVWjsbs%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32348747> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7316398> Google Scholar <http://scholar.google.com/scholar_lookup?&title=D-loop%20dynamics%20and%20near-atomic-resolution%20cryo-EM%20structure%20of%20phalloidin-bound%20F-actin&journal=Structure&doi=10.1016%2Fj.str.2020.04.004&volume=28&pages=586-593&publication_year=2020&author=Das%2CS> 

41.

    Gong, R. et al. Structural basis for tunable control of actin
    dynamics by myosin-15 in mechanosensory stereocilia. /Sci. Adv./
    *8*, eabl4733 (2022).

    Article <https://doi.org/10.1126%2Fsciadv.abl4733> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitFSmsbrM> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=35857845> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9299544> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20for%20tunable%20control%20of%20actin%20dynamics%20by%20myosin-15%20in%20mechanosensory%20stereocilia&journal=Sci.%20Adv.&doi=10.1126%2Fsciadv.abl4733&volume=8&publication_year=2022&author=Gong%2CR> 

42.

    Saunders, M. G. & Voth, G. A. Water molecules in the nucleotide
    binding cleft of actin: effects on subunit conformation and
    implications for ATP hydrolysis. /J. Mol. Biol./ *413*, 279–291 (2011).

    Article <https://doi.org/10.1016%2Fj.jmb.2011.07.068> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1Grtr3K> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21856312> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Water%20molecules%20in%20the%20nucleotide%20binding%20cleft%20of%20actin%3A%20effects%20on%20subunit%20conformation%20and%20implications%20for%20ATP%20hydrolysis&journal=J.%20Mol.%20Biol.&doi=10.1016%2Fj.jmb.2011.07.068&volume=413&pages=279-291&publication_year=2011&author=Saunders%2CMG&author=Voth%2CGA> 

43.

    Kang, H. et al. Identification of cation-binding sites on actin that
    drive polymerization and modulate bending stiffness. /Proc. Natl
    Acad. Sci. USA/ *109*, 16923–16927 (2012).

    Article <https://doi.org/10.1073%2Fpnas.1211078109> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2012PNAS..10916923K> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xhs1WksL%2FL> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23027950> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3479481> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Identification%20of%20cation-binding%20sites%20on%20actin%20that%20drive%20polymerization%20and%20modulate%20bending%20stiffness&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1211078109&volume=109&pages=16923-16927&publication_year=2012&author=Kang%2CH> 

44.

    Armstrong, M. et al. Microscale fluid behavior during cryo-EM sample
    blotting. /Biophys. J./ *118*, 708–719 (2020).

    Article <https://doi.org/10.1016%2Fj.bpj.2019.12.017> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020BpJ...118..708A> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtF2iu74%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31952802> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Microscale%20fluid%20behavior%20during%20cryo-EM%20sample%20blotting&journal=Biophys.%20J.&doi=10.1016%2Fj.bpj.2019.12.017&volume=118&pages=708-719&publication_year=2020&author=Armstrong%2CM> 

45.

    Shibata, S., Matsunami, H., Aizawa, S.-I. & Wolf, M. Torque
    transmission mechanism of the curved bacterial flagellar hook
    revealed by cryo-EM. /Nat. Struct. Mol. Biol./ *26*, 941–945 (2019).

    Article <https://doi.org/10.1038%2Fs41594-019-0301-3> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvVOqu7bN> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31570877> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Torque%20transmission%20mechanism%20of%20the%20curved%20bacterial%20flagellar%20hook%20revealed%20by%20cryo-EM&journal=Nat.%20Struct.%20Mol.%20Biol.&doi=10.1038%2Fs41594-019-0301-3&volume=26&pages=941-945&publication_year=2019&author=Shibata%2CS&author=Matsunami%2CH&author=Aizawa%2CS-I&author=Wolf%2CM> 

46.

    Gibson, K. H. et al. An asymmetric sheath controls flagellar
    supercoiling and motility in the leptospira spirochete. /eLife/ *9*,
    e53672 (2020).

    Article <https://doi.org/10.7554%2FeLife.53672> PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32157997> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7065911> Google Scholar <http://scholar.google.com/scholar_lookup?&title=An%20asymmetric%20sheath%20controls%20flagellar%20supercoiling%20and%20motility%20in%20the%20leptospira%20spirochete&journal=eLife&doi=10.7554%2FeLife.53672&volume=9&publication_year=2020&author=Gibson%2CKH> 

47.

    Orlova, A. & Egelman, E. H. A conformational change in the actin
    subunit can change the flexibility of the actin filament. /J. Mol.
    Biol./ *232*, 334–341 (1993).

    Article <https://doi.org/10.1006%2Fjmbi.1993.1393> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3sXmtF2qt7k%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8345515> Google Scholar <http://scholar.google.com/scholar_lookup?&title=A%20conformational%20change%20in%20the%20actin%20subunit%20can%20change%20the%20flexibility%20of%20the%20actin%20filament&journal=J.%20Mol.%20Biol.&doi=10.1006%2Fjmbi.1993.1393&volume=232&pages=334-341&publication_year=1993&author=Orlova%2CA&author=Egelman%2CEH> 

48.

    McCullough, B. R., Blanchoin, L., Martiel, J.-L. & De La Cruz, E. M.
    Cofilin increases the bending flexibility of actin filaments:
    implications for severing and cell mechanics. /J. Mol. Biol./ *381*,
    550–558 (2008).

    Article <https://doi.org/10.1016%2Fj.jmb.2008.05.055> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXpt1KisLo%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18617188> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2753234> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Cofilin%20increases%20the%20bending%20flexibility%20of%20actin%20filaments%3A%20implications%20for%20severing%20and%20cell%20mechanics&journal=J.%20Mol.%20Biol.&doi=10.1016%2Fj.jmb.2008.05.055&volume=381&pages=550-558&publication_year=2008&author=McCullough%2CBR&author=Blanchoin%2CL&author=Martiel%2CJ-L&author=Cruz%2CEM> 

49.

    Guo, J. & Zhou, H.-X. Protein allostery and conformational dynamics.
    /Chem. Rev./ *116*, 6503–6515 (2016).

    Article <https://doi.org/10.1021%2Facs.chemrev.5b00590> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XisVKjsr8%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26876046> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5011433> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Protein%20allostery%20and%20conformational%20dynamics&journal=Chem.%20Rev.&doi=10.1021%2Facs.chemrev.5b00590&volume=116&pages=6503-6515&publication_year=2016&author=Guo%2CJ&author=Zhou%2CH-X> 

50.

    Mani, S., Katkar, H. H. & Voth, G. A. Compressive and tensile
    deformations alter ATP hydrolysis and phosphate release rates in
    actin filaments. /J. Chem. Theory Comput./ *17*, 1900–1913 (2021).

    Article <https://doi.org/10.1021%2Facs.jctc.0c01186> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXktFemsbc%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33596075> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Compressive%20and%20tensile%20deformations%20alter%20ATP%20hydrolysis%20and%20phosphate%20release%20rates%20in%20actin%20filaments&journal=J.%20Chem.%20Theory%20Comput.&doi=10.1021%2Facs.jctc.0c01186&volume=17&pages=1900-1913&publication_year=2021&author=Mani%2CS&author=Katkar%2CHH&author=Voth%2CGA> 

51.

    Pardee, J. D. & Spudich, J. A. in /Methods in Enzymology/ Vol. 85
    (eds Frederiksen, D. W. & Cunningham, L. W.) Ch. 18, 164–181
    (Academic, 1982).
52.

    Walt, Svander et al. scikit-image: image processing in Python.
    /PeerJ/ *2*, e453 (2014).

    Article <https://doi.org/10.7717%2Fpeerj.453> PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25024921> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4081273> Google Scholar <http://scholar.google.com/scholar_lookup?&title=scikit-image%3A%20image%20processing%20in%20Python&journal=PeerJ&doi=10.7717%2Fpeerj.453&volume=2&publication_year=2014&author=Walt%2CSvander> 

53.

    Mastronarde, D. N. Automated electron microscope tomography using
    robust prediction of specimen movements. /J. Struct. Biol./ *152*,
    36–51 (2005).

    Article <https://doi.org/10.1016%2Fj.jsb.2005.07.007> PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16182563> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Automated%20electron%20microscope%20tomography%20using%20robust%20prediction%20of%20specimen%20movements&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2005.07.007&volume=152&pages=36-51&publication_year=2005&author=Mastronarde%2CDN> 

54.

    Zheng, S. Q. et al. MotionCor2: anisotropic correction of
    beam-induced motion for improved cryo-electron microscopy. /Nat.
    Methods/ *14*, 331–332 (2017).

    Article <https://doi.org/10.1038%2Fnmeth.4193> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXjt1ags7g%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28250466> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5494038> Google Scholar <http://scholar.google.com/scholar_lookup?&title=MotionCor2%3A%20anisotropic%20correction%20of%20beam-induced%20motion%20for%20improved%20cryo-electron%20microscopy&journal=Nat.%20Methods&doi=10.1038%2Fnmeth.4193&volume=14&pages=331-332&publication_year=2017&author=Zheng%2CSQ> 

55.

    Grant, T. & Grigorieff, N. Measuring the optimal exposure for single
    particle cryo-EM using a 2.6 Å reconstruction of rotavirus VP6.
    /eLife/ *4*, e06980 (2015).

    Article <https://doi.org/10.7554%2FeLife.06980> PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26023829> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4471936> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Measuring%20the%20optimal%20exposure%20for%20single%20particle%20cryo-EM%20using%20a%202.6%E2%80%89%C3%85%20reconstruction%20of%20rotavirus%20VP6&journal=eLife&doi=10.7554%2FeLife.06980&volume=4&publication_year=2015&author=Grant%2CT&author=Grigorieff%2CN> 

56.

    Rohou, A. & Grigorieff, N. CTFFIND4: fast and accurate defocus
    estimation from electron micrographs. /J. Struct. Biol./ *192*,
    216–221 (2015).

    Article <https://doi.org/10.1016%2Fj.jsb.2015.08.008> PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26278980> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6760662> Google Scholar <http://scholar.google.com/scholar_lookup?&title=CTFFIND4%3A%20fast%20and%20accurate%20defocus%20estimation%20from%20electron%20micrographs&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2015.08.008&volume=192&pages=216-221&publication_year=2015&author=Rohou%2CA&author=Grigorieff%2CN> 

57.

    Wang, F. et al. DeepPicker: a deep learning approach for fully
    automated particle picking in cryo-EM. /J. Struct. Biol./ *195*,
    325–336 (2016).

    Article <https://doi.org/10.1016%2Fj.jsb.2016.07.006> PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27424268> Google Scholar <http://scholar.google.com/scholar_lookup?&title=DeepPicker%3A%20a%20deep%20learning%20approach%20for%20fully%20automated%20particle%20picking%20in%20cryo-EM&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2016.07.006&volume=195&pages=325-336&publication_year=2016&author=Wang%2CF> 

58.

    Wagner, T. et al. SPHIRE-crYOLO is a fast and accurate fully
    automated particle picker for cryo-EM. /Commun. Biol./ *2*, 218 (2019).

    Article <https://doi.org/10.1038%2Fs42003-019-0437-z> PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31240256> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6584505> Google Scholar <http://scholar.google.com/scholar_lookup?&title=SPHIRE-crYOLO%20is%20a%20fast%20and%20accurate%20fully%20automated%20particle%20picker%20for%20cryo-EM&journal=Commun.%20Biol.&doi=10.1038%2Fs42003-019-0437-z&volume=2&publication_year=2019&author=Wagner%2CT> 

59.

    Bepler, T. et al. Positive-unlabeled convolutional neural networks
    for particle picking in cryo-electron micrographs. /Nat. Methods/
    *16*, 1153–1160 (2019).

    Article <https://doi.org/10.1038%2Fs41592-019-0575-8> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFejs7%2FM> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=31591578> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6858545> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Positive-unlabeled%20convolutional%20neural%20networks%20for%20particle%20picking%20in%20cryo-electron%20micrographs&journal=Nat.%20Methods&doi=10.1038%2Fs41592-019-0575-8&volume=16&pages=1153-1160&publication_year=2019&author=Bepler%2CT> 

60.

    Bakan, A., Meireles, L. M. & Bahar, I. ProDy: protein dynamics
    inferred from theory and experiments. /Bioinformatics/ *27*,
    1575–1577 (2011).

    Article <https://doi.org/10.1093%2Fbioinformatics%2Fbtr168> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXmvVajs7s%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21471012> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3102222> Google Scholar <http://scholar.google.com/scholar_lookup?&title=ProDy%3A%20protein%20dynamics%20inferred%20from%20theory%20and%20experiments&journal=Bioinformatics&doi=10.1093%2Fbioinformatics%2Fbtr168&volume=27&pages=1575-1577&publication_year=2011&author=Bakan%2CA&author=Meireles%2CLM&author=Bahar%2CI> 

61.

    Tang, G. et al. EMAN2: an extensible image processing suite for
    electron microscopy. /J. Struct. Biol./ *157*, 38–46 (2007).

    Article <https://doi.org/10.1016%2Fj.jsb.2006.05.009> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xhtlalt77M> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16859925> Google Scholar <http://scholar.google.com/scholar_lookup?&title=EMAN2%3A%20an%20extensible%20image%20processing%20suite%20for%20electron%20microscopy&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2006.05.009&volume=157&pages=38-46&publication_year=2007&author=Tang%2CG> 

62.

    TensorFlow Developers. TensorFlow (Zenodo, 2022);
    https://doi.org/10.5281/zenodo.6574269
    <https://doi.org/10.5281/zenodo.6574269>
63.

    Mei, L. et al. Structural mechanism for bidirectional actin
    cross-linking by T-plastin. /Proc. Natl Acad. Sci. USA/ *119*,
    e2205370119 (2022).

    Article <https://doi.org/10.1073%2Fpnas.2205370119> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XisFags77J> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=36067297> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9478642> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20mechanism%20for%20bidirectional%20actin%20cross-linking%20by%20T-plastin&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.2205370119&volume=119&publication_year=2022&author=Mei%2CL> 

64.

    Egelman, E. H. The iterative helical real space reconstruction
    method: surmounting the problems posed by real polymers. /J. Struct.
    Biol./ *157*, 83–94 (2007).

    Article <https://doi.org/10.1016%2Fj.jsb.2006.05.015> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xhtlalt77J> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16919474> Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20iterative%20helical%20real%20space%20reconstruction%20method%3A%20surmounting%20the%20problems%20posed%20by%20real%20polymers&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2006.05.015&volume=157&pages=83-94&publication_year=2007&author=Egelman%2CEH> 

65.

    He, S. & Scheres, S. H. W. Helical reconstruction in RELION. /J.
    Struct. Biol./ *198*, 163–176 (2017).

    Article <https://doi.org/10.1016%2Fj.jsb.2017.02.003> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXjt1Klsrk%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28193500> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5479445> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Helical%20reconstruction%20in%20RELION&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2017.02.003&volume=198&pages=163-176&publication_year=2017&author=He%2CS&author=Scheres%2CSHW> 

66.

    Zivanov, J. et al. New tools for automated high-resolution cryo-EM
    structure determination in RELION-3. /eLife/ *7*, e42166 (2018).

    Article <https://doi.org/10.7554%2FeLife.42166> PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=30412051> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6250425> Google Scholar <http://scholar.google.com/scholar_lookup?&title=New%20tools%20for%20automated%20high-resolution%20cryo-EM%20structure%20determination%20in%20RELION-3&journal=eLife&doi=10.7554%2FeLife.42166&volume=7&publication_year=2018&author=Zivanov%2CJ> 

67.

    Punjani, A., Rubinstein, J. L., Fleet, D. J. & Brubaker, M. A.
    cryoSPARC: algorithms for rapid unsupervised cryo-EM structure
    determination. /Nat. Methods/ *14*, 290–296 (2017).

    Article <https://doi.org/10.1038%2Fnmeth.4169> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXitlGisbs%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28165473> Google Scholar <http://scholar.google.com/scholar_lookup?&title=cryoSPARC%3A%20algorithms%20for%20rapid%20unsupervised%20cryo-EM%20structure%20determination&journal=Nat.%20Methods&doi=10.1038%2Fnmeth.4169&volume=14&pages=290-296&publication_year=2017&author=Punjani%2CA&author=Rubinstein%2CJL&author=Fleet%2CDJ&author=Brubaker%2CMA> 

68.

    Tan, Y. Z. et al. Addressing preferred specimen orientation in
    single-particle cryo-EM through tilting. /Nat. Methods/ *14*,
    793–796 (2017).

    Article <https://doi.org/10.1038%2Fnmeth.4347> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFSmtrjO> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28671674> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5533649> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Addressing%20preferred%20specimen%20orientation%20in%20single-particle%20cryo-EM%20through%20tilting&journal=Nat.%20Methods&doi=10.1038%2Fnmeth.4347&volume=14&pages=793-796&publication_year=2017&author=Tan%2CYZ> 

69.

    Zhong, E. D., Bepler, T., Berger, B. & Davis, J. H. CryoDRGN:
    reconstruction of heterogeneous cryo-EM structures using neural
    networks. /Nat. Methods/ *18*, 176–185 (2021).

    Article <https://doi.org/10.1038%2Fs41592-020-01049-4> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjt1SmsLw%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33542510> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8183613> Google Scholar <http://scholar.google.com/scholar_lookup?&title=CryoDRGN%3A%20reconstruction%20of%20heterogeneous%20cryo-EM%20structures%20using%20neural%20networks&journal=Nat.%20Methods&doi=10.1038%2Fs41592-020-01049-4&volume=18&pages=176-185&publication_year=2021&author=Zhong%2CED&author=Bepler%2CT&author=Berger%2CB&author=Davis%2CJH> 

70.

    Terwilliger, T. C., Ludtke, S. J., Read, R. J., Adams, P. D. &
    Afonine, P. V. Improvement of cryo-EM maps by density modification.
    /Nat. Methods/ *17*, 923–927 (2020).

    Article <https://doi.org/10.1038%2Fs41592-020-0914-9> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1WrurbL> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=32807957> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7484085> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Improvement%20of%20cryo-EM%20maps%20by%20density%20modification&journal=Nat.%20Methods&doi=10.1038%2Fs41592-020-0914-9&volume=17&pages=923-927&publication_year=2020&author=Terwilliger%2CTC&author=Ludtke%2CSJ&author=Read%2CRJ&author=Adams%2CPD&author=Afonine%2CPV> 

71.

    Grigorieff, N. in /Methods in Enzymology/ Vol. 579 (ed. Crowther, R.
    A.) Ch. 8, 191–226 (Academic, 2016).
72.

    Pettersen, E. F. et al. UCSF Chimera—a visualization system for
    exploratory research and analysis. /J. Comput. Chem./ *25*,
    1605–1612 (2004).

    Article <https://doi.org/10.1002%2Fjcc.20084> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXmvVOhsbs%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15264254> Google Scholar <http://scholar.google.com/scholar_lookup?&title=UCSF%20Chimera%E2%80%94a%20visualization%20system%20for%20exploratory%20research%20and%20analysis&journal=J.%20Comput.%20Chem.&doi=10.1002%2Fjcc.20084&volume=25&pages=1605-1612&publication_year=2004&author=Pettersen%2CEF> 

73.

    Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and
    development of Coot. /Acta Crystalogry. D/ *66*, 486–501 (2010).

    Article <https://doi.org/10.1107%2FS0907444910007493> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXksFKisb8%3D> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Features%20and%20development%20of%20Coot&journal=Acta%20Crystalogry.%20D&doi=10.1107%2FS0907444910007493&volume=66&pages=486-501&publication_year=2010&author=Emsley%2CP&author=Lohkamp%2CB&author=Scott%2CWG&author=Cowtan%2CK> 

74.

    Afonine, P. V. et al. Real-space refinement in PHENIX for cryo-EM
    and crystallography. /Acta Crystalogr. D/ *74*, 531–544 (2018).

    Article <https://doi.org/10.1107%2FS2059798318006551> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2ltbzP> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Real-space%20refinement%20in%20PHENIX%20for%20cryo-EM%20and%20crystallography&journal=Acta%20Crystalogr.%20D&doi=10.1107%2FS2059798318006551&volume=74&pages=531-544&publication_year=2018&author=Afonine%2CPV> 

75.

    Liebschner, D. et al. Macromolecular structure determination using
    X-rays, neutrons and electrons: recent developments in Phenix. /Acta
    Crystalogr. D/ *75*, 861–877 (2019).

    Article <https://doi.org/10.1107%2FS2059798319011471> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFWkurrO> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Macromolecular%20structure%20determination%20using%20X-rays%2C%20neutrons%20and%20electrons%3A%20recent%20developments%20in%20Phenix&journal=Acta%20Crystalogr.%20D&doi=10.1107%2FS2059798319011471&volume=75&pages=861-877&publication_year=2019&author=Liebschner%2CD> 

76.

    Phillips, J. C. et al. Scalable molecular dynamics with NAMD. /J.
    Comput. Chem./ *26*, 1781–1802 (2005).

    Article <https://doi.org/10.1002%2Fjcc.20289> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXht1SlsbbJ> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16222654> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2486339> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Scalable%20molecular%20dynamics%20with%20NAMD&journal=J.%20Comput.%20Chem.&doi=10.1002%2Fjcc.20289&volume=26&pages=1781-1802&publication_year=2005&author=Phillips%2CJC> 

77.

    Croll, T. I. ISOLDE: a physically realistic environment for model
    building into low-resolution electron-density maps. /Acta
    Crystalogr. D/ *74*, 519–530 (2018).

    Article <https://doi.org/10.1107%2FS2059798318002425> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2ltbzI> Google Scholar <http://scholar.google.com/scholar_lookup?&title=ISOLDE%3A%20a%20physically%20realistic%20environment%20for%20model%20building%20into%20low-resolution%20electron-density%20maps&journal=Acta%20Crystalogr.%20D&doi=10.1107%2FS2059798318002425&volume=74&pages=519-530&publication_year=2018&author=Croll%2CTI> 

78.

    Goddard, T. D. et al. UCSF ChimeraX: meeting modern challenges in
    visualization and analysis. /Protein Sci./ *27*, 14–25 (2018).

    Article <https://doi.org/10.1002%2Fpro.3235> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsVertLjN> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28710774> Google Scholar <http://scholar.google.com/scholar_lookup?&title=UCSF%20ChimeraX%3A%20meeting%20modern%20challenges%20in%20visualization%20and%20analysis&journal=Protein%20Sci.&doi=10.1002%2Fpro.3235&volume=27&pages=14-25&publication_year=2018&author=Goddard%2CTD> 

79.

    Tian, W., Chen, C., Lei, X., Zhao, J. & Liang, J. CASTp 3.0:
    computed atlas of surface topography of proteins. /Nucleic Acids
    Res./ *46*, W363–W367 (2018).

    Article <https://doi.org/10.1093%2Fnar%2Fgky473> CAS
    <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyrurs%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29860391> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6031066> Google Scholar <http://scholar.google.com/scholar_lookup?&title=CASTp%203.0%3A%20computed%20atlas%20of%20surface%20topography%20of%20proteins&journal=Nucleic%20Acids%20Res.&doi=10.1093%2Fnar%2Fgky473&volume=46&pages=W363-W367&publication_year=2018&author=Tian%2CW&author=Chen%2CC&author=Lei%2CX&author=Zhao%2CJ&author=Liang%2CJ> 

80.

    Panyukov, S. & Rabin, Y. Fluctuating filaments: statistical
    mechanics of helices. /Phys. Rev. E/ *62*, 7135–7146 (2000).

    Article <https://doi.org/10.1103%2FPhysRevE.62.7135> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2000PhRvE..62.7135P> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXnvFans74%3D> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Fluctuating%20filaments%3A%20statistical%20mechanics%20of%20helices&journal=Phys.%20Rev.%20E&doi=10.1103%2FPhysRevE.62.7135&volume=62&pages=7135-7146&publication_year=2000&author=Panyukov%2CS&author=Rabin%2CY> 

81.

    Rohou, A. & Grigorieff, N. Frealix: model-based refinement of
    helical filament structures from electron micrographs. /J. Struct.
    Biol./ *186*, 234–244 (2014).

    Article <https://doi.org/10.1016%2Fj.jsb.2014.03.012> PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24657230> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4067737> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Frealix%3A%20model-based%20refinement%20of%20helical%20filament%20structures%20from%20electron%20micrographs&journal=J.%20Struct.%20Biol.&doi=10.1016%2Fj.jsb.2014.03.012&volume=186&pages=234-244&publication_year=2014&author=Rohou%2CA&author=Grigorieff%2CN> 

82.

    De La Cruz, E. M., Martiel, J.-L. & Blanchoin, L. Mechanical
    heterogeneity favors fragmentation of strained actin filaments.
    /Biophys. J./ *108*, 2270–2281 (2015).

    Article <https://doi.org/10.1016%2Fj.bpj.2015.03.058> Google Scholar
    <http://scholar.google.com/scholar_lookup?&title=Mechanical%20heterogeneity%20favors%20fragmentation%20of%20strained%20actin%20filaments&journal=Biophys.%20J.&doi=10.1016%2Fj.bpj.2015.03.058&volume=108&pages=2270-2281&publication_year=2015&author=Cruz%2CEM&author=Martiel%2CJ-L&author=Blanchoin%2CL> 

83.

    Zsolnay, V., Katkar, H. H., Chou, S. Z., Pollard, T. D. & Voth, G.
    A. Structural basis for polarized elongation of actin filaments.
    /Proc. Natl Acad. Sci. USA/ *117*, 30458–30464 (2020).

    Article <https://doi.org/10.1073%2Fpnas.2011128117> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2020PNAS..11730458Z> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVKnsbvL> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=33199648> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7720195> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20for%20polarized%20elongation%20of%20actin%20filaments&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.2011128117&volume=117&pages=30458-30464&publication_year=2020&author=Zsolnay%2CV&author=Katkar%2CHH&author=Chou%2CSZ&author=Pollard%2CTD&author=Voth%2CGA> 

84.

    Mitchell, M. R., Tlusty, T. & Leibler, S. Strain analysis of protein
    structures and low dimensionality of mechanical allosteric
    couplings. /Proc. Natl Acad. Sci. USA/ *113*, E5847–E5855 (2016).

    Article <https://doi.org/10.1073%2Fpnas.1609462113> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2016PNAS..113E5847M> CAS <https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFertb3I> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27655887> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5056043> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Strain%20analysis%20of%20protein%20structures%20and%20low%20dimensionality%20of%20mechanical%20allosteric%20couplings&journal=Proc.%20Natl%20Acad.%20Sci.%20USA&doi=10.1073%2Fpnas.1609462113&volume=113&pages=E5847-E5855&publication_year=2016&author=Mitchell%2CMR&author=Tlusty%2CT&author=Leibler%2CS> 

85.

    Mitchell, M. R. & Leibler, S. Elastic strain and twist analysis of
    protein structural data and allostery of the transmembrane channel
    KcsA. /Phys. Biol./ *15*, 036004 (2018).

    Article <https://doi.org/10.1088%2F1478-3975%2Faa9906> ADS
    <http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&bibcode=2018PhBio..15c6004M> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29116053> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Elastic%20strain%20and%20twist%20analysis%20of%20protein%20structural%20data%20and%20allostery%20of%20the%20transmembrane%20channel%20KcsA&journal=Phys.%20Biol.&doi=10.1088%2F1478-3975%2Faa9906&volume=15&publication_year=2018&author=Mitchell%2CMR&author=Leibler%2CS> 

86.

    Hunter, J. D. Matplotlib: a 2D graphics environment. /Comput. Sci.
    Eng./ *9*, 90–95 (2007).

    Article <https://doi.org/10.1109%2FMCSE.2007.55> Google Scholar
    <http://scholar.google.com/scholar_lookup?&title=Matplotlib%3A%20a%202D%20graphics%20environment&journal=Comput.%20Sci.%20Eng.&doi=10.1109%2FMCSE.2007.55&volume=9&pages=90-95&publication_year=2007&author=Hunter%2CJD> 

Nature (/Nature/) ISSN 1476-4687 (online) ISSN 0028-0836 (print)


    nature.com sitemap


      About Nature Portfolio

  * About us <https://www.nature.com/npg_/company_info/index.html>
  * Press releases
    <https://www.nature.com/npg_/press_room/press_releases.html>
  * Press office <https://press.nature.com/>
  * Contact us <https://support.nature.com/support/home>


      Discover content

  * Journals A-Z <https://www.nature.com/siteindex>
  * Articles by subject <https://www.nature.com/subjects/>
  * Nano <https://nano.nature.com/>
  * Protocol Exchange <https://www.nature.com/protocolexchange/>
  * Nature Index <https://www.natureindex.com/>


      Publishing policies

  * Nature portfolio policies
    <https://www.nature.com/authors/editorial_policies/>
  * Open access <https://www.nature.com/nature-research/open-access>


      Author & Researcher services

  * Reprints & permissions <https://www.nature.com/reprints/>
  * Research data <https://www.springernature.com/gp/authors/research-data>
  * Language editing
    <https://authorservices.springernature.com/language-editing/>
  * Scientific editing
    <https://authorservices.springernature.com/scientific-editing/>
  * Nature Masterclasses <https://masterclasses.nature.com/>
  * Live Expert Trainer-led workshops
    <https://masterclasses.nature.com/live-expert-trainer-led/23649702>
  * Research Solutions <https://solutions.springernature.com/>


      Libraries & institutions

  * Librarian service & tools
    <https://www.springernature.com/gp/librarians/tools-services>
  * Librarian portal
    <https://www.springernature.com/gp/librarians/manage-your-account/librarianportal>
  * Open research
    <https://www.nature.com/openresearch/about-open-access/information-for-institutions/>
  * Recommend to library
    <https://www.springernature.com/gp/librarians/recommend-to-your-library>


      Advertising & partnerships

  * Advertising
    <https://partnerships.nature.com/product/digital-advertising/>
  * Partnerships & Services <https://partnerships.nature.com/>
  * Media kits <https://partnerships.nature.com/media-kits/>
  * Branded content
    <https://partnerships.nature.com/product/branded-content-native-advertising/>


      Career development

  * Nature Careers <https://www.nature.com/naturecareers>
  * NatureConferences <https://conferences.nature.com/>
  * Natureevents <https://www.nature.com/natureevents/>


      Regional websites

  * Nature Africa <https://www.nature.com/natafrica>
  * Nature China <http://www.naturechina.com/>
  * Nature India <https://www.nature.com/nindia>
  * Nature Italy <https://www.nature.com/natitaly>
  * Nature Japan <https://www.natureasia.com/ja-jp/>
  * Nature Korea <https://www.natureasia.com/ko-kr/>
  * Nature Middle East <https://www.nature.com/nmiddleeast/>

  * Privacy Policy <https://www.nature.com/info/privacy>
  * Use of cookies <https://www.nature.com/info/cookies>
  * Your privacy choices/Manage cookies
  * Legal notice <https://www.nature.com/info/legal-notice>
  * Accessibility statement
    <https://www.nature.com/info/accessibility-statement>
  * Terms & Conditions <https://www.nature.com/info/terms-and-conditions>
  * Your US state privacy rights <https://www.springernature.com/ccpa>

Springer Nature <https://www.springernature.com/>

© 2023 Springer Nature Limited

Close banner Close
Nature Briefing

Sign up for the /Nature Briefing/ newsletter — what matters in science,
free to your inbox daily.

Email address

Sign up
I agree my information will be processed in accordance with the /Nature/
and Springer Nature Limited Privacy Policy
<https://www.nature.com/info/privacy>.

Close banner Close
Get the most important science stories of the day, free in your inbox.
Sign up for Nature Briefing <https://www.nature.com/briefing/signup/>
