Skip to Content <#maincontent>
eLife logo

eLife home page <https://elifesciences.org/>

  * Menu <#mainMenu>
  * Home <https://elifesciences.org/>
  * Magazine <https://elifesciences.org/magazine>
  * Community <https://elifesciences.org/community>
  * About <https://elifesciences.org/about>

  * Search <https://elifesciences.org/search>
  * Alerts <https://elifesciences.org/content-alerts>
  * Submit your research <https://elifesciences.org/submit-your-research>

Search by keyword or author
Reset form

Search

Limit my search to Cell Biology

Close

Skip to Content <#maincontent>
eLife logo

eLife home page <https://elifesciences.org/>


      Menu

  * Home <https://elifesciences.org/>
  * Magazine <https://elifesciences.org/magazine>
  * Community <https://elifesciences.org/community>
  * About <https://elifesciences.org/about>
  * Research categories <https://elifesciences.org/subjects>
  * Inside eLife <https://elifesciences.org/inside-elife>
  * Search <https://elifesciences.org/search>
  * Subscribe to alerts <https://elifesciences.org/content-alerts>
  * Submit your research <https://elifesciences.org/submit-your-research>
  * Author guide
    <https://reviewer.elifesciences.org/author-guide/editorial-process>
  * Reviewer guide
    <https://reviewer.elifesciences.org/reviewer-guide/review-process>

Back to top <#siteHeader>

  * Research Article <https://elifesciences.org/articles/research-article>

 1. Cell Biology <https://elifesciences.org/subjects/cell-biology>


  Cooperation between tropomyosin and α-actinin inhibits fimbrin
  association with actin filament networks in fission yeast

 1. Jenna R Christensen <https://elifesciences.org/articles/47279#xaa457a99>
 2. Kaitlin E Homa <https://elifesciences.org/articles/47279#xbb8e3ca2>
 3. Alisha N Morganthaler
    <https://elifesciences.org/articles/47279#xee183223>
 4. Rachel R Brown <https://elifesciences.org/articles/47279#x34e19a24>
 5. Cristian Suarez <https://elifesciences.org/articles/47279#x6a55e6af>
 6. Alyssa J Harker <https://elifesciences.org/articles/47279#x1e4f837f>
 7. Meghan E O'Connell <https://elifesciences.org/articles/47279#x59af7267>
 8. David R Kovar
    <https://elifesciences.org/articles/47279#xbe3c5e36> Is a
    corresponding author

 1. The University of Chicago, United States;

Jun 10, 2019
https://doi.org/10.7554/eLife.47279 <https://doi.org/10.7554/eLife.47279>

  * Open access <https://en.wikipedia.org/wiki/Open_access>
  * Copyright information <https://creativecommons.org/licenses/by/4.0/>


          Version of Record

Accepted for publication after peer review and revision.

 1. Download <#downloads>
    A two-part list of links to download the article, or parts of the
    article, in various formats.


          Downloads(link to download the article as PDF)

      * Article PDF
        <https://elifesciences.org/download/aHR0cHM6Ly9jZG4uZWxpZmVzY2llbmNlcy5vcmcvYXJ0aWNsZXMvNDcyNzkvZWxpZmUtNDcyNzktdjEucGRmP2Nhbm9uaWNhbFVyaT1odHRwczovL2VsaWZlc2NpZW5jZXMub3JnL2FydGljbGVzLzQ3Mjc5/elife-47279-v1.pdf?_hash=6YdQk3A3qXO9A4jNcGNT8C4Ozxh7e2qbbo0QKVzkzgU%3D>
      * Figures PDF
        <https://elifesciences.org/download/aHR0cHM6Ly9jZG4uZWxpZmVzY2llbmNlcy5vcmcvYXJ0aWNsZXMvNDcyNzkvZWxpZmUtNDcyNzktZmlndXJlcy12MS5wZGY-/elife-47279-figures-v1.pdf?_hash=HHAnaFGzEmXOlZ88FBL7YL61WQHnqhxfaQ3VW84%2Fa6A%3D>


          Open citations(links to open the citations from this article
          in various online reference manager services)

      * Mendeley <https://www.mendeley.com/import?doi=10.7554/eLife.47279>
      *


          Cite this article(links to download the citations from this
          article in formats compatible with various reference manager
          tools)

     1. Jenna R Christensen
     2. Kaitlin E Homa
     3. Alisha N Morganthaler
     4. Rachel R Brown
     5. Cristian Suarez
     6. Alyssa J Harker
     7. Meghan E O'Connell
     8. David R Kovar
    (2019) Cooperation between tropomyosin and α-actinin inhibits
    fimbrin association with actin filament networks in fission yeast
    /eLife/ *8*:e47279.
    https://doi.org/10.7554/eLife.47279
      * Download BibTeX <https://elifesciences.org/articles/47279.bib>
      * Download .RIS <https://elifesciences.org/articles/47279.ris>
 2. Cite <#cite-this-article>
 3. Share <#share>
 4.
    CommentOpen annotations (there are currently 0 annotations on this
    page). <#comment>

  * 1,409 views <https://elifesciences.org/articles/47279#metrics>
  * 14 citations <https://elifesciences.org/articles/47279#metrics>

Version of Record published
    June 10, 2019 (This version)
Accepted
    May 29, 2019
Received
    March 30, 2019

<#>

 1.
     1. Of interest


            BATF relieves hepatic steatosis by inhibiting PD1 and
            promoting energy metabolism
            <https://elifesciences.org/articles/88521>

    Zhiwang Zhang, Qichao Liao ... Lei Zhou

    Research Article
    <https://elifesciences.org/articles/research-article> Sep 15, 2023

 2. Further reading <#listing> 


            Share this article

Doi

Copy to clipboard

  * <mailto:?subject=Cooperation%20between%20tropomyosin%20and%20%CE%B1-actinin%20inhibits%20fimbrin%20association%20with%20actin%20filament%20networks%20in%20fission%20yeast&body=https%3A%2F%2Fdoi.org%2F10.7554%2FeLife.47279>
  * <https://twitter.com/intent/tweet/?text=In%20%40eLife%3A%20Cooperation%20between%20tropomyosin%20and%20%CE%B1-actinin%20inhibits%20fimbrin%20association%20with%20actin%20filament%20networks%20in%20fission%20yeast&url=https%3A%2F%2Fdoi.org%2F10.7554%2FeLife.47279>
  * <https://facebook.com/sharer/sharer.php?u=https%3A%2F%2Fdoi.org%2F10.7554%2FeLife.47279>
  * <https://www.linkedin.com/shareArticle?title=Cooperation%20between%20tropomyosin%20and%20%CE%B1-actinin%20inhibits%20fimbrin%20association%20with%20actin%20filament%20networks%20in%20fission%20yeast&url=https%3A%2F%2Fdoi.org%2F10.7554%2FeLife.47279>
  * <https://reddit.com/submit/?title=Cooperation%20between%20tropomyosin%20and%20%CE%B1-actinin%20inhibits%20fimbrin%20association%20with%20actin%20filament%20networks%20in%20fission%20yeast&url=https%3A%2F%2Fdoi.org%2F10.7554%2FeLife.47279>
  * <https://toot.kytta.dev/?text=Cooperation%20between%20tropomyosin%20and%20%CE%B1-actinin%20inhibits%20fimbrin%20association%20with%20actin%20filament%20networks%20in%20fission%20yeast%20https%3A%2F%2Fdoi.org%2F10.7554%2FeLife.47279>


            Cite this article

 1. Jenna R Christensen
 2. Kaitlin E Homa
 3. Alisha N Morganthaler
 4. Rachel R Brown
 5. Cristian Suarez
 6. Alyssa J Harker
 7. Meghan E O'Connell
 8. David R Kovar

(2019) Cooperation between tropomyosin and α-actinin inhibits fimbrin
association with actin filament networks in fission yeast
/eLife/ *8*:e47279.
https://doi.org/10.7554/eLife.47279

 1. Copy to clipboard
 2. Download BibTeX <https://elifesciences.org/articles/47279.bib>
 3. Download .RIS <https://elifesciences.org/articles/47279.ris>

  * Full text <https://elifesciences.org/articles/47279#content>
  * Figuresand data
    <https://elifesciences.org/articles/47279/figures#content>
  * Side by side <https://lens.elifesciences.org/47279>

  * Abstract <#abstract>
  * Introduction <#s1>
  * Results <#s2>
  * Discussion <#s3>
  * Materials and methods <#s4>
  * Data availability <#data>
  * References <#references>
  * Decision letter <#SA1>
  * Author response <#SA2>
  * Article and author information <#info>
  * Metrics <#metrics>


    Abstract

We previously discovered that competition between fission yeast actin
binding proteins (ABPs) for binding F-actin facilitates their sorting to
different cellular networks. Specifically, competition between endocytic
actin patch ABPs fimbrin Fim1 and cofilin Adf1 enhances their
activities, and prevents tropomyosin Cdc8’s association with actin
patches. However, these interactions do not explain how Fim1 is
prevented from associating strongly with other F-actin networks such as
the contractile ring. Here, we identified α-actinin Ain1, a contractile
ring ABP, as another Fim1 competitor. Fim1 competes with Ain1 for
association with F-actin, which is dependent upon their F-actin
residence time. While Fim1 outcompetes both Ain1 and Cdc8 individually,
Cdc8 enhances the F-actin bundling activity of Ain1, allowing Ain1 to
generate F-actin bundles that Cdc8 can bind in the presence of Fim1.
Therefore, the combination of contractile ring ABPs Ain1 and Cdc8 is
capable of inhibiting Fim1’s association with F-actin networks.

https://doi.org/10.7554/eLife.47279.001
<https://doi.org/10.7554/eLife.47279.001>


    Introduction

<#>

As in many cell types, the unicellular fission yeast assembles diverse
actin filament (F-actin) networks within a crowded cytoplasm to
facilitate different cellular functions such as cytokinesis (contractile
ring), endocytosis (actin patches) and polarization (actin cables).
These F-actin networks each possess a defined set of actin-binding
proteins (ABPs) that regulate the formation, organization, and dynamics
of the actin filaments within the network. However, the mechanisms by
which specific sets of ABPs sort to particular F-actin networks are less
clear. We hypothesize that a combination of competitive and cooperative
interactions between different ABPs for association with F-actin is
critical for establishing and maintaining their sorting. We previously
identified competitive binding interactions between three fission yeast
ABPs with distinct network localizations—fimbrin Fim1 and ADF/cofilin
Adf1 (endocytic actin patches) and tropomyosin Cdc8 (cytokinetic
contractile ring) (hereafter called Fim1, Adf1 and Cdc8)—that help
facilitate their sorting to the proper F-actin networks (Christensen et
al., 2017 <#bib4>; Skau and Kovar, 2010 <#bib34>). Specifically,
synergistic activities between Fim1 and Adf1 rapidly displace Cdc8 from
F-actin networks such as actin patches (Christensen et al., 2017
<#bib4>; Skau and Kovar, 2010 <#bib34>). However, these interactions do
not explain how Fim1 is prevented from strongly associating with other
F-actin networks such as the contractile ring. Therefore, we sought to
determine whether other ABPs at the contractile ring prevent Fim1
association. In this study, we demonstrate that Fim1 competes with the
contractile ring ABP α-actinin Ain1 (hereafter called Ain1) for
association with F-actin, and that their ability to compete is dependent
upon their residence time on F-actin. Additionally, we show that
although Fim1 outcompetes both Cdc8 and Ain1 individually, Cdc8 and Ain1
have synergistic effects. Cdc8 enhances Ain1-mediated F-actin bundling,
and Ain1 facilitates association of Cdc8 with Fim1-bound F-actin. These
combined effects allow the combination of Cdc8 and Ain1 to successfully
compete with Fim1 for association with F-actin.


    Results

<#>


      F-actin crosslinking proteins fimbrin Fim1 and α-actinin Ain1
      compete at the contractile ring and at actin patches

We previously found that fimbrin Fim1 and ADF/cofilin Adf1 synergize to
displace tropomyosin Cdc8 from F-actin (Christensen et al., 2017
<#bib4>), which helps explain why Fim1 is highly concentrated on actin
patches, whereas Cdc8 is not (Skau and Kovar, 2010 <#bib34>).
Conversely, despite Fim1 being a dominant competitor for F-actin
(Christensen et al., 2017 <#bib4>; Skau and Kovar, 2010 <#bib34>), Fim1
is present at much lower concentrations at the contractile ring, an
F-actin network where Cdc8 is abundant (Arai et al., 1998 <#bib2>;
Nakano et al., 2001 <#bib24>; Wu et al., 2001 <#bib39>). We hypothesized
that competition with additional contractile ring ABPs may prevent Fim1
from strongly associating with the contractile ring. As Fim1 is highly
concentrated in actin patches, we speculated that depletion of actin
patches by the Arp2/3 complex inhibitor CK-666 (Burke et al., 2014
<#bib3>; Nolen et al., 2009 <#bib25>) would result in a rapid increase
of free Fim1 in the cytoplasm that might subsequently allow Fim1 to
outcompete its contractile ring ABP competitors. Upon treating fission
yeast cells expressing the general F-actin marker Lifeact-GFP with
CK-666, we observed a depletion of actin patches and the formation of
excessive formin-mediated ‘ectopic’ actin cables and contractile ring
material (Figure 1A <#fig1>) (Burke et al., 2014 <#bib3>).

Figure 1 with 3 supplements
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9paWlmLmVsaWZlc2NpZW5jZXMub3JnL2xheC80NzI3OSUyRmVsaWZlLTQ3Mjc5LWZpZzEtdjEudGlmL2Z1bGwvZnVsbC8wL2RlZmF1bHQuanBn/elife-47279-fig1-v1.jpg?_hash=w%2BfgabTRr9PSP27lTKLDiHe7AymXdkqE1W%2FUrrtL6u4%3D> Open asset <https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig1-v1.tif/full/1500,/0/default.jpg>

<https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig1-v1.tif/full/1500,/0/default.jpg>


            Fimbrin Fim1 and α-actinin Ain1 compete for association with
            the contractile ring.

(*A*) Fluorescence micrographs of fission yeast cells expressing
Lifeact-GFP following treatment with DMSO (control, top) or 200 µM
Arp2/3 complex inhibitor CK-666 (bottom). Scale bar, 5 μm. (B-D, top …
see more

https://doi.org/10.7554/eLife.47279.002
<https://doi.org/10.7554/eLife.47279.002>

Figure 1—figure supplement 3
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9paWlmLmVsaWZlc2NpZW5jZXMub3JnL2xheC80NzI3OSUyRmVsaWZlLTQ3Mjc5LWZpZzEtZmlnc3VwcDMtdjEudGlmL2Z1bGwvZnVsbC8wL2RlZmF1bHQuanBn/elife-47279-fig1-figsupp3-v1.jpg?_hash=5LKzfICtW81jt8FaZkXZAji36rVqJWGElbsTTPlGLaI%3D> Open asset <https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig1-figsupp3-v1.tif/full/,1500/0/default.jpg>

<https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig1-figsupp3-v1.tif/full/,1500/0/default.jpg>


            Fimbrin Fim1 displaces α-actinin Ain1 from the contractile
            ring following CK-666 treatment.

(A-C, top) Fluorescent micrographs of fission yeast cells expressing
spindle pole body marker Sad1-tdTomato and Fim1-GFP (*A*), Ain1-GFP
(*B*), or Ain1-GFP in a /fim1-1Δ/ background (*C*), following … see more

https://doi.org/10.7554/eLife.47279.005
<https://doi.org/10.7554/eLife.47279.005>

Figure 1—figure supplement 2
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9paWlmLmVsaWZlc2NpZW5jZXMub3JnL2xheC80NzI3OSUyRmVsaWZlLTQ3Mjc5LWZpZzEtZmlnc3VwcDItdjEudGlmL2Z1bGwvZnVsbC8wL2RlZmF1bHQuanBn/elife-47279-fig1-figsupp2-v1.jpg?_hash=ZbOBfbmmilYOAe4rbmxeEzYkqqyGSLlUkQddn5BSfMk%3D> Open asset <https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig1-figsupp2-v1.tif/full/full/0/default.jpg>

<https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig1-figsupp2-v1.tif/full/full/0/default.jpg>


            Tropomyosin Cdc8 does not leave the contractile ring
            following CK-666 treatment.

Mean anti-Cdc8 contractile ring fluorescence normalized to total cell
fluorescence. Error bars = s.e. Two-tailed t-test for data sets with
unequal variance yielded p-value=0.9338. n ≥ 10 cells.

https://doi.org/10.7554/eLife.47279.004
<https://doi.org/10.7554/eLife.47279.004>

Figure 1—figure supplement 1
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9paWlmLmVsaWZlc2NpZW5jZXMub3JnL2xheC80NzI3OSUyRmVsaWZlLTQ3Mjc5LWZpZzEtZmlnc3VwcDEtdjEudGlmL2Z1bGwvZnVsbC8wL2RlZmF1bHQuanBn/elife-47279-fig1-figsupp1-v1.jpg?_hash=1d5cm2XrWQgottyqwQmzRFQGgsP6lMchkkjmqn%2BurKc%3D> Open asset <https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig1-figsupp1-v1.tif/full/1500,/0/default.jpg>

<https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig1-figsupp1-v1.tif/full/1500,/0/default.jpg>


            Contractile ring ABP localization following CK-666 treatment.

Fluorescent micrographs of fission yeast cells either immunostained
(anti-Cdc8) or expressing the indicated fluorescently tagged contractile
ring ABPs from their endogenous locus: formin Cdc12-GFP, … see more

https://doi.org/10.7554/eLife.47279.003
<https://doi.org/10.7554/eLife.47279.003>

In control (DMSO-treated) fission yeast cells, Fim1-GFP localizes
predominantly to actin patches, with only a small amount associating
with the contractile ring (Figure 1B <#fig1>, left) (Wu et al., 2001
<#bib39>). However, in cells treated with CK-666, Fim1-GFP strongly
associates with the contractile ring and to a subset of ectopic F-actin
(Figure 1B <#fig1>, right). The localization of most contractile ring
ABPs, including formin Cdc12, type II myosin Myo2, myosin regulatory
light chain Rlc1, the IQGAP Rng2, and tropomyosin Cdc8, is unaffected by
CK-666 treatment, (Figure 1—figure supplement 1
<https://elifesciences.org/articles/47279/figures#fig1s1> and Figure
1—figure supplement 2
<https://elifesciences.org/articles/47279/figures#fig1s2>). Conversely,
less α-actinin Ain1 associates with the contractile ring in cells
treated with CK-666 (Figure 1C <#fig1>, Figure 1—figure supplement 1
<https://elifesciences.org/articles/47279/figures#fig1s1>). Therefore,
we hypothesized that Fim1 and Ain1 are competitors, and that enhanced
Fim1 association with the contractile ring in cells treated with CK-666
displaces Ain1. We tested this hypothesis by observing Ain1 localization
in a strain lacking Fim1 (/fim1-1Δ,/ Ain1-GFP). In /fim1-1Δ/ cells,
similar amounts of Ain1-GFP are associated with the contractile ring in
control and CK-666-treated cells (Figure 1D <#fig1>), suggesting that
the absence of competitor Fim1 allows Ain1 to remain associated with the
contractile ring in the presence of CK-666. At all stages of contractile
ring assembly and constriction, Fim1 similarly localizes to the
contractile ring and displaces Ain1 following CK-666 treatment (Figure
1—figure supplement 3
<https://elifesciences.org/articles/47279/figures#fig1s3>), although it
is most prominent in stages with fully-developed contractile rings
(stages 2 and 3).

If competition between Fim1 and Ain1 is a primary driver of their
sorting to distinct F-actin networks, we expected that Ain1-GFP might
erroneously localize to actin patches in the absence of Fim1. However,
Ain1-GFP is observed at actin patches in less than 1% of /fim1-1Δ/ cells
(Figures 1D <#fig1> and 2C <#fig2>, Figure 2—video 1
<https://elifesciences.org/articles/47279/figures#fig2video1>). It is
possible that a combination of the low number of Ain1 molecules
(~3,600 ± 500 [Wu and Pollard, 2005 <#bib41>]), and the high density of
F-actin in actin patches (5,000–7,000 actin molecules in each of 30–50
actin patches [Sirotkin et al., 2010 <#bib32>; Wu and Pollard, 2005
<#bib41>]), may dilute the Ain1-GFP signal beyond detection. Therefore,
increasing the concentration of Ain1-GFP could allow observable Ain1-GFP
at actin patches, but only in a /fim1-1Δ/ background. To increase the
expression of Ain1-GFP, we introduced an additional copy of Ain1-GFP at
the leu1-32 locus under the medium-strength 41Xnmt promoter (Li et al.,
2016 <#bib19>). We first quantified the cellular expression of
endogenously tagged Ain1-GFP and the overexpressed Ain1-GFP constructs
used in this study. We observed that while most Ain1-GFP constructs are
expressed similarly to endogenously-tagged Ain1, the amount of Ain1-GFP
overexpressed under the 41Xnmt promoter is almost two-fold higher
(Figure 2—figure supplement 1
<https://elifesciences.org/articles/47279/figures#fig2s1>). As
predicted, despite this two-fold increase in expression, Ain1-GFP
localizes to actin patches in only 10% of WT cells expressing endogenous
Fim1 (Figure 2A,C <#fig2>, Figure 2—video 1
<https://elifesciences.org/articles/47279/figures#fig2video1>). In
contrast, overexpressed Ain1-GFP associates with actin patches in ~67%
of /fim1-1Δ/ cells (Figure 2B,C <#fig2>, Figure 2—video 1
<https://elifesciences.org/articles/47279/figures#fig2video1>).
Therefore, Fim1 and Ain1 appear to compete for association with F-actin
at both actin patches and the contractile ring.

Figure 2 with 2 supplements
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9paWlmLmVsaWZlc2NpZW5jZXMub3JnL2xheC80NzI3OSUyRmVsaWZlLTQ3Mjc5LWZpZzItdjEudGlmL2Z1bGwvZnVsbC8wL2RlZmF1bHQuanBn/elife-47279-fig2-v1.jpg?_hash=5zTYsztGg7kEo%2FQB%2FhGPDoKNzLlzLgqk1XmdnXA97NM%3D> Open asset <https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig2-v1.tif/full/full/0/default.jpg>

<https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig2-v1.tif/full/full/0/default.jpg>


            Fimbrin Fim1 and α-actinin Ain1 compete for association with
            actin patches.

(*A–B*) Time-lapse fluorescent micrographs of fission yeast cells
expressing ArpC5-mCherry (bottom) and overexpressing GFP-tagged
α-actinin Ain1 from the 41Xnmt promoter (top) for 20 hr in a wild-type …
see more

https://doi.org/10.7554/eLife.47279.006
<https://doi.org/10.7554/eLife.47279.006>

Figure 2—video 1
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9zdGF0aWMtbW92aWUtdXNhLmdsZW5jb2Vzb2Z0d2FyZS5jb20vbXA0LzEwLjc1NTQvNjQ0L2FkOWIyZDE3MTk3NDdkYzQxYmY4OTM0YmNhOWI2OTY5ODg0NWQwNDkvZWxpZmUtNDcyNzktZmlnMi12aWRlbzEubXA0/elife-47279-fig2-video1.mp4?_hash=kY%2Fvx5gdR1XIqsvRpyJR3MtbGJTnJys6q%2BJZABU1YUI%3D>
posterframe for video

This video cannot be played in place because your browser does support
HTML5 video. You may still download the video for offline viewing.

Download as MPEG-4
<https://static-movie-usa.glencoesoftware.com/mp4/10.7554/644/ad9b2d1719747dc41bf8934bca9b69698845d049/elife-47279-fig2-video1.mp4>
Download as WebM
<https://static-movie-usa.glencoesoftware.com/webm/10.7554/644/ad9b2d1719747dc41bf8934bca9b69698845d049/elife-47279-fig2-video1.webm>
Download as Ogg
<https://static-movie-usa.glencoesoftware.com/ogv/10.7554/644/ad9b2d1719747dc41bf8934bca9b69698845d049/elife-47279-fig2-video1.ogv>


            Overexpressed α-actinin Ain1-GFP localizes to actin patches
            in the absence of fimbrin, related to Figure 1 <#fig1>.

Fission yeast cells expressing actin patch marker ArpC5-mCherry (right)
with Ain1-GFP (left) at endogenous or overexpressed levels. (Top panels)
Endogenous Ain1-GFP in a /fim1-1Δ/ background. (Middle … see more

https://doi.org/10.7554/eLife.47279.008
<https://doi.org/10.7554/eLife.47279.008>

Figure 2—figure supplement 1
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9paWlmLmVsaWZlc2NpZW5jZXMub3JnL2xheC80NzI3OSUyRmVsaWZlLTQ3Mjc5LWZpZzItZmlnc3VwcDEtdjEudGlmL2Z1bGwvZnVsbC8wL2RlZmF1bHQuanBn/elife-47279-fig2-figsupp1-v1.jpg?_hash=jFdfffC3oA23gctimfRvnYI7dwcoZqMKi5KVGucAjWc%3D> Open asset <https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig2-figsupp1-v1.tif/full/full/0/default.jpg>

<https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig2-figsupp1-v1.tif/full/full/0/default.jpg>


            Cellular expression of GFP-tagged α-actinin Ain1 constructs.

Fluorescence intensity of GFP-tagged Ain1 constructs for cells in
cytokinesis (with contractile ring) or interphase (no contractile ring).
Error bars = s.e. A single factor ANOVA yielded p-values … see more

https://doi.org/10.7554/eLife.47279.007
<https://doi.org/10.7554/eLife.47279.007>


      Less dynamic α-actinin Ain1 associates with actin patches

Ultrastructural and mutational studies of fimbrin/plastin and α-actinin
from several organisms demonstrate that they bind to a similar site on
F-actin (Galkin et al., 2010 <#bib9>; Galkin et al., 2008 <#bib8>;
Holtzman et al., 1994 <#bib10>; Honts et al., 1994 <#bib11>; McGough et
al., 1994 <#bib20>). However, while fission yeast fimbrin Fim1 is
relatively stable on single actin filaments (k_off  = 0.043 ± 0.001 s^−1
) and very stable on F-actin bundles (k_off  = 0.023 ± 0.003 s^−1 )
(Skau et al., 2011 <#bib33>), α-actinin Ain1 has not been observed to
associate with single actin filaments and is extremely dynamic on
F-actin bundles (k_off  = 3.33 s^−1 on two-filament and three-filament
bundles) (Li et al., 2016 <#bib19>). Therefore, we hypothesized that
Fim1’s longer residence time on F-actin bundles may explain its ability
to outcompete Ain1 for a similar F-actin binding site. To test this
possibility, we took advantage of the Ain1 mutant Ain1(R216E), which is
5- to 10-fold less dynamic on F-actin bundles (k_off  = 0.67 s^−1 and
k_off  = 0.33 s^−1 on two- and three-filament bundles, respectively) (Li
et al., 2016 <#bib19>), and assessed its ability to compete with Fim1 in
vitro and in vivo.

We utilized multi-color TIRF microscopy (TIRFM) to directly visualize
the association of fluorescently labeled Fim1 with actin filaments in
vitro in either the presence or absence of unlabeled Ain1. Compared to
50 nM Fim1-TMR alone, which fully decorates F-actin bundles, less
Fim1-TMR associates with two filament F-actin bundles in the presence of
either 1 μM wild-type Ain1 or Ain1(R216E) (Figure 3A,B <#fig3>).
However, there is little difference in the amount of Fim1-TMR associated
with two-filament F-actin bundles in the presence of wild-type Ain1 or
mutant Ain1(R216E).

Figure 3
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9paWlmLmVsaWZlc2NpZW5jZXMub3JnL2xheC80NzI3OSUyRmVsaWZlLTQ3Mjc5LWZpZzMtdjEudGlmL2Z1bGwvZnVsbC8wL2RlZmF1bHQuanBn/elife-47279-fig3-v1.jpg?_hash=TjZv6GeXaGZIrIlz5iKSLr0%2BCp1%2F8FUyY1WbeJLBs1s%3D> Open asset <https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig3-v1.tif/full/1500,/0/default.jpg>

<https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig3-v1.tif/full/1500,/0/default.jpg>


            Fimbrin Fim1 and α-actinin Ain1 compete for F-actin binding
            F-actin in vitro.

(*A–B*) Two-color TIRFM of 1.5 μM Mg-ATP actin (15% Alexa 488-labeled)
with 50 nM fimbrin Fim1 (Cy5-labeled) alone, or with either 1 μM
wild-type α-actinin Ain1 or mutant Ain1(R216E). Scale bars, 5 … see more

https://doi.org/10.7554/eLife.47279.009
<https://doi.org/10.7554/eLife.47279.009>

Although we did not detect that Ain1(R216E) competes with Fim1 better
than does wild-type Ain1 in vitro (Figure 3 <#fig3>), the less dynamic
Ain1(R216E) mutant is better than wild-type Ain1 at associating with
Fim1-bound actin patches in vivo (Figure 4 <#fig4>). In fission yeast
cells expressing endogenously tagged Fim1-mCherry, overexpressed
wild-type Ain1-GFP localizes to actin patches in only ~9% of cells
(Figure 4A,C <#fig4>, Figure 4—video 1
<https://elifesciences.org/articles/47279/figures#fig4video1>), whereas
overexpressed mutant Ain1(R216E)-GFP localizes to actin patches in 100%
of cells (Figure 4B,C <#fig4>, Figure 4—video 1
<https://elifesciences.org/articles/47279/figures#fig4video1>). The
disparity between Ain1(R216E)’s ability to compete with Fim1 in vitro
versus in vivo potentially suggests that slight differences in dynamics
may have a bigger effect in a cellular context. In particular, the
dynamics of ABPs such as Ain1 may be finely tuned to allow for proper
sorting given the large number of actin interacting proteins, with a
small change in dynamics skewing the sorting to a dramatic degree.
Alternatively, it is possible that our simplified in vitro system does
not fully mimic in vivo conditions.

Figure 4 with 1 supplement
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9paWlmLmVsaWZlc2NpZW5jZXMub3JnL2xheC80NzI3OSUyRmVsaWZlLTQ3Mjc5LWZpZzQtdjEudGlmL2Z1bGwvZnVsbC8wL2RlZmF1bHQuanBn/elife-47279-fig4-v1.jpg?_hash=PxEsCLQMdD1Om4sPAfIJ0pWo0OQuAhTwPIw97w7v0V0%3D> Open asset <https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig4-v1.tif/full/full/0/default.jpg>

<https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig4-v1.tif/full/full/0/default.jpg>


            Fimbrin Fim1 and α-actinin Ain1 competition at actin patches
            is driven by their residence time on F-actin.

(A,B, top) Fluorescence micrographs of fission yeast in an /ain1-1Δ/
background overexpressing GFP-tagged wild-type Ain1 (*A*) or mutant
Ain1(R216E) (*B*) from the 41Xnmt1 promoter. Scale bars, 5 μm. … see more

https://doi.org/10.7554/eLife.47279.010
<https://doi.org/10.7554/eLife.47279.010>

Figure 4—video 1
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9zdGF0aWMtbW92aWUtdXNhLmdsZW5jb2Vzb2Z0d2FyZS5jb20vbXA0LzEwLjc1NTQvNjQ0L2FkOWIyZDE3MTk3NDdkYzQxYmY4OTM0YmNhOWI2OTY5ODg0NWQwNDkvZWxpZmUtNDcyNzktZmlnNC12aWRlbzEubXA0/elife-47279-fig4-video1.mp4?_hash=ihzZqeJ8YP3qwPOGNfghEliaCjbmwQLUvJM5d9Yw5fc%3D>
posterframe for video

This video cannot be played in place because your browser does support
HTML5 video. You may still download the video for offline viewing.

Download as MPEG-4
<https://static-movie-usa.glencoesoftware.com/mp4/10.7554/644/ad9b2d1719747dc41bf8934bca9b69698845d049/elife-47279-fig4-video1.mp4>
Download as WebM
<https://static-movie-usa.glencoesoftware.com/webm/10.7554/644/ad9b2d1719747dc41bf8934bca9b69698845d049/elife-47279-fig4-video1.webm>
Download as Ogg
<https://static-movie-usa.glencoesoftware.com/ogv/10.7554/644/ad9b2d1719747dc41bf8934bca9b69698845d049/elife-47279-fig4-video1.ogv>


            Overexpressed mutant α-actinin Ain1(R216E)-GFP, but not
            Ain1-GFP, localizes to actin patches in the presence of
            fimbrin Fim1, related to Figure 4 <#fig4>.

(Top) Fission yeast cells with Ain1-GFP overexpressed under the 41xnmt
promoter (left) and expressing Fim1-mCherry at the endogenous locus
(right). (Bottom) Fission yeast cells with Ain1(R216E)-GFP … see more

https://doi.org/10.7554/eLife.47279.011
<https://doi.org/10.7554/eLife.47279.011>


      Tropomyosin Cdc8 and α-actinin Ain1 do not compete for association
      with actin filaments

We previously reported that tropomyosin Cdc8, an F-actin side-binding
protein that associates with the contractile ring, is displaced from
F-actin by fimbrin Fim1 in vitro and is thereby prevented from
associating with actin patches in fission yeast cells (Christensen et
al., 2017 <#bib4>; Skau and Kovar, 2010 <#bib34>). Although
fimbrin/plastin and α-actinin isoforms bind to a similar site on
F-actin, Cdc8 and α-actinin Ain1 both associate with the contractile
ring. Therefore, we considered whether Ain1 also displaces Cdc8 from
F-actin, or if they can simultaneously associate with F-actin. In
multi-color in vitro TIRFM assays, Cdc8 is not displaced from Ain1- or
Ain1(R216E)-bundled F-actin networks (Figure 5A,B <#fig5>, Figure
5—video 1
<https://elifesciences.org/articles/47279/figures#fig5video1>), but it
is displaced from Fim1-bundled networks (Figure 5C <#fig5>; Christensen
et al., 2017 <#bib4>; Skau and Kovar, 2010 <#bib34>). Thus, Ain1 and
Cdc8 are capable of co-existing on the same F-actin network in vitro as
they do at the contractile ring in cells.

Figure 5 with 1 supplement
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9paWlmLmVsaWZlc2NpZW5jZXMub3JnL2xheC80NzI3OSUyRmVsaWZlLTQ3Mjc5LWZpZzUtdjEudGlmL2Z1bGwvZnVsbC8wL2RlZmF1bHQuanBn/elife-47279-fig5-v1.jpg?_hash=LGclJt7qZHxxKXQ00Fzo9r0bAG2OkxBYSniB8jvKSOc%3D> Open asset <https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig5-v1.tif/full/1500,/0/default.jpg>

<https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig5-v1.tif/full/1500,/0/default.jpg>


            α-actinin Ain1 does not displace tropomyosin Cdc8 from
            F-actin bundles in vitro.

(A-C, top) Two-color TIRFM of 1.5 μM Mg-ATP actin (15% Alexa
488-labeled) with 2.5 µM tropomyosin Cdc8 (TMR-labeled) and unlabeled
500 nM (*A*) wild-type α-actinin Ain1, (*B*) mutant Ain1(R216E), or
(*C*) … see more

https://doi.org/10.7554/eLife.47279.012
<https://doi.org/10.7554/eLife.47279.012>

Figure 5—video 1
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9zdGF0aWMtbW92aWUtdXNhLmdsZW5jb2Vzb2Z0d2FyZS5jb20vbXA0LzEwLjc1NTQvNjQ0L2FkOWIyZDE3MTk3NDdkYzQxYmY4OTM0YmNhOWI2OTY5ODg0NWQwNDkvZWxpZmUtNDcyNzktZmlnNS12aWRlbzEubXA0/elife-47279-fig5-video1.mp4?_hash=dqlT%2BjUjiAtvlO0naGPqwHKQ%2BdifgB%2F%2Fho6W5aVg7EQ%3D>
posterframe for video

This video cannot be played in place because your browser does support
HTML5 video. You may still download the video for offline viewing.

Download as MPEG-4
<https://static-movie-usa.glencoesoftware.com/mp4/10.7554/644/ad9b2d1719747dc41bf8934bca9b69698845d049/elife-47279-fig5-video1.mp4>
Download as WebM
<https://static-movie-usa.glencoesoftware.com/webm/10.7554/644/ad9b2d1719747dc41bf8934bca9b69698845d049/elife-47279-fig5-video1.webm>
Download as Ogg
<https://static-movie-usa.glencoesoftware.com/ogv/10.7554/644/ad9b2d1719747dc41bf8934bca9b69698845d049/elife-47279-fig5-video1.ogv>


            α-actinin Ain1 does not displace tropomyosin Cdc8 from
            F-actin bundles, related to Figure 5 <#fig5>.

TIRFM of 1.5 μM actin (Alexa-488 labeled) (left column) with 2.5 μM
tropomyosin Cdc8 (TMR- or Cy5-labeled) (middle column) and 500 nM Ain1
(top row), 500 nM Ain1(R216E) (middle row), or 500 nM Fim1 … see more

https://doi.org/10.7554/eLife.47279.013
<https://doi.org/10.7554/eLife.47279.013>


      Tropomyosin Cdc8 enhances the bundling activity of α-actinin Ain1

Compared to fimbrin Fim1, α-actinin Ain1 is a relatively weak F-actin
bundling protein (Addario et al., 2016 <#bib1>; Li et al., 2016
<#bib19>; Morita et al., 2017 <#bib23>). Remarkably, low-speed
sedimentation (Figure 6A,B <#fig6>) and TIRFM assays (Figure 6C,D
<#fig6>, Figure 6—video 1
<https://elifesciences.org/articles/47279/figures#fig6video1>) revealed
that tropomyosin Cdc8 significantly enhances the F-actin bundling
ability of Ain1. In single-color TIRFM assays with unlabeled ABPs, 500
nM Cdc8 increases Ain1-mediated F-actin bundling 10-fold over Ain1 alone
(Figure 6D <#fig6>, Figure 6—video 1
<https://elifesciences.org/articles/47279/figures#fig6video1>). This
surprising finding suggests that the combination of Ain1 and Cdc8 may
allow for significant F-actin bundling to occur in the context of the
contractile ring despite Ain1’s poor bundling ability alone.

Figure 6 with 1 supplement
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9paWlmLmVsaWZlc2NpZW5jZXMub3JnL2xheC80NzI3OSUyRmVsaWZlLTQ3Mjc5LWZpZzYtdjEudGlmL2Z1bGwvZnVsbC8wL2RlZmF1bHQuanBn/elife-47279-fig6-v1.jpg?_hash=gJFUOP%2FviqFL1H5djG2Zo812EHReP9qVqwbYljL9JNQ%3D> Open asset <https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig6-v1.tif/full/,1500/0/default.jpg>

<https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig6-v1.tif/full/,1500/0/default.jpg>


            Tropomyosin Cdc8 enhances α-actinin Ain1-mediated F-actin
            bundling in vitro.

(*A–B*) Low-speed (10,000 x /g/) sedimentation assays of 5 µM Mg-ATP
preassembled actin filaments, 500 nM Ain1, and increasing concentrations
of Cdc8 (0–5 µM). (A) Coomassie blue-stained gel of pellets … see more

https://doi.org/10.7554/eLife.47279.014
<https://doi.org/10.7554/eLife.47279.014>

Figure 6—video 1
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9zdGF0aWMtbW92aWUtdXNhLmdsZW5jb2Vzb2Z0d2FyZS5jb20vbXA0LzEwLjc1NTQvNjQ0L2FkOWIyZDE3MTk3NDdkYzQxYmY4OTM0YmNhOWI2OTY5ODg0NWQwNDkvZWxpZmUtNDcyNzktZmlnNi12aWRlbzEubXA0/elife-47279-fig6-video1.mp4?_hash=OylAxTCA30ICTrtWG%2B1sOFGyt%2Fi9UjfXzfzgUTY4qKY%3D>
posterframe for video

This video cannot be played in place because your browser does support
HTML5 video. You may still download the video for offline viewing.

Download as MPEG-4
<https://static-movie-usa.glencoesoftware.com/mp4/10.7554/644/ad9b2d1719747dc41bf8934bca9b69698845d049/elife-47279-fig6-video1.mp4>
Download as WebM
<https://static-movie-usa.glencoesoftware.com/webm/10.7554/644/ad9b2d1719747dc41bf8934bca9b69698845d049/elife-47279-fig6-video1.webm>
Download as Ogg
<https://static-movie-usa.glencoesoftware.com/ogv/10.7554/644/ad9b2d1719747dc41bf8934bca9b69698845d049/elife-47279-fig6-video1.ogv>


            Tropomyosin Cdc8 enhances α-actinin Ain1-mediated bundling,
            related to Figure 6 <#fig6>.

TIRFM of 1.5 μM actin (Alexa-488 labeled) with the indicated varying
concentrations of unlabeled Ain1 and Cdc8. Scale bar, 5 μm. Time in s.

https://doi.org/10.7554/eLife.47279.015
<https://doi.org/10.7554/eLife.47279.015>

Cdc8’s enhancement of Ain1-mediated F-actin bundling could potentially
arise from Cdc8 increasing the number of Ain1 binding events on F-actin.
To test this possibility, we analyzed the single molecule dynamics of
Ain1 by performing TIRFM experiments with sparsely-labeled (0.5%
TMR-labeled) Ain1 on uncoated versus Cdc8-coated F-actin. Three-fold
more Ain1 binding events were observed on Cdc8-coated F-actin compared
to uncoated F-actin (Figure 6E–G <#fig6>), suggesting that Cdc8 enhances
the binding of Ain1 to F-actin, thereby increasing the F-actin bundling
ability of Ain1.


      Tropomyosin Cdc8 and α-actinin Ain1 cooperate to displace fimbrin
      Fim1 from actin filaments

On their own, both α-actinin Ain1 and tropomyosin Cdc8 are outcompeted
by fimbrin Fim1 for binding to F-actin. Furthermore, there are ~86,500
Fim1 polypeptides in the cell, but only ~3600 Ain1 molecules (Wu and
Pollard, 2005 <#bib41>), raising the question as to why Fim1 is not
associated more strongly with the contractile ring in wild-type cells.
Given that Cdc8 enhances the bundling ability of Ain1, we speculated
that the combination of Cdc8 and Ain1 might inhibit Fim1 association
with actin filaments. To test this possibility, we first performed low
speed F-actin sedimentation assays in the presence of Cdc8 and either
Fim1, Ain1, or both (Figure 7A,B <#fig7>). Almost two-fold more Cdc8
pellets with Ain1-mediated F-actin bundles than with Fim1-mediated
bundles (Figure 7B <#fig7>). Additionally, in the presence of both Fim1
and Ain1, an intermediate amount of Cdc8 pellets in the presence of both
Fim1 and Ain1, suggesting that Ain1 allows Cdc8 to better associate with
F-actin in the presence of Fim1.

Figure 7
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9paWlmLmVsaWZlc2NpZW5jZXMub3JnL2xheC80NzI3OSUyRmVsaWZlLTQ3Mjc5LWZpZzctdjEudGlmL2Z1bGwvZnVsbC8wL2RlZmF1bHQuanBn/elife-47279-fig7-v1.jpg?_hash=mwbKdYc7H%2F8r1RnsGRkcI8ZJBcrzPDilhP8Fdv30SaA%3D> Open asset <https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig7-v1.tif/full/1500,/0/default.jpg>

<https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig7-v1.tif/full/1500,/0/default.jpg>


            α-actinin Ain1 facilitates the association of tropomyosin
            Cdc8 with bundled F-actin in the presence of fimbrin Fim1 in
            vitro.

(*A,B*) Low-speed sedimentation comparing Cdc8 in the pellet with
F-actin bundled by Fim1 (left), Ain1 (middle), or both (right). (*A*)
Coomassie blue-stained gel of representative triplicate samples of … see
more

https://doi.org/10.7554/eLife.47279.016
<https://doi.org/10.7554/eLife.47279.016>

To directly investigate the effect of Ain1 and Cdc8 cooperation on
competition with Fim1, we performed four-color TIRFM with fluorescently
labeled ABPs and quantified Fim1 association with F-actin in the
presence of Cdc8 and/or Ain1. In the absence of Ain1, Cdc8 is displaced
from F-actin bundles by Fim1 in a cooperative manner, with most F-actin
bundles completely devoid of Cdc8, concurrent with regions of high Fim1
localization (Figure 8A <#fig8>, Figure 8—video 1
<https://elifesciences.org/articles/47279/figures#fig8video1>;
Christensen et al., 2017 <#bib4>). Conversely, in reactions containing
Ain1, over 100-fold more Cdc8 is associated with F-actin bundles (Figure
8B,C <#fig8>, Figure 8—video 1
<https://elifesciences.org/articles/47279/figures#fig8video1>).
Furthermore,~40% less Fim1 is observed to associate with these bundles
(Figure 8B,D <#fig8>, Figure 8—video 1
<https://elifesciences.org/articles/47279/figures#fig8video1>),
suggesting that the combination of Ain1 and Cdc8 is capable of
inhibiting Fim1 association with F-actin networks. Additionally, in the
presence of Cdc8, Fim1 and Ain1 sort into mutually exclusive domains
along F-actin bundles (Figure 8B <#fig8>), similar to what has been
observed previously between human Fascin1 and α-actinin-4 (Winkelman et
al., 2016 <#bib38>). We speculate that competition for the same binding
site allows Ain1 to prevent long stretches of Fim1 from forming that
might be capable of displacing Cdc8. Thus, Fim1 poorly associates with
F-actin in the presence of both Cdc8 and Ain1 because 1) Cdc8 and Ain1
prevent Fim1 from cooperatively associating with F-actin and 2) Cdc8 and
Ain1 synergize to compete with Fim1. The synergy between Cdc8 and Ain1
may explain why Fim1 only poorly associates with contractile rings in
fission yeast cells (Figure 8E <#fig8>).

Figure 8 with 1 supplement
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9paWlmLmVsaWZlc2NpZW5jZXMub3JnL2xheC80NzI3OSUyRmVsaWZlLTQ3Mjc5LWZpZzgtdjEudGlmL2Z1bGwvZnVsbC8wL2RlZmF1bHQuanBn/elife-47279-fig8-v1.jpg?_hash=FLeiJdMKVl7pVltP7SkvADBzKT8fJHzt975hiVgiMm4%3D> Open asset <https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig8-v1.tif/full/,1500/0/default.jpg>

<https://iiif.elifesciences.org/lax/47279%2Felife-47279-fig8-v1.tif/full/,1500/0/default.jpg>


            Tropomyosin Cdc8 and α-actinin Ain1 cooperate to compete
            with fimbrin Fim1 for association with F-actin in vitro.

(*A–D*) Four-color TIRFM of 1.5 μM Mg-ATP actin (15% Alexa 488-labeled)
with 50 nM fimbrin Fim1 (Alexa 405-labeled) and 2.5 μM tropomyosin Cdc8
(TMR-labeled) in the (*A*) absence or (*B*) presence of 500 … see more

https://doi.org/10.7554/eLife.47279.017
<https://doi.org/10.7554/eLife.47279.017>

Figure 8—video 1
Download asset
<https://elifesciences.org/download/aHR0cHM6Ly9zdGF0aWMtbW92aWUtdXNhLmdsZW5jb2Vzb2Z0d2FyZS5jb20vbXA0LzEwLjc1NTQvNjQ0L2FkOWIyZDE3MTk3NDdkYzQxYmY4OTM0YmNhOWI2OTY5ODg0NWQwNDkvZWxpZmUtNDcyNzktZmlnOC12aWRlbzEubXA0/elife-47279-fig8-video1.mp4?_hash=JieKiIpAE9B4MvvxTfz0gtydXCgWkPWlbM2IPC%2FHDyM%3D>
posterframe for video

This video cannot be played in place because your browser does support
HTML5 video. You may still download the video for offline viewing.

Download as MPEG-4
<https://static-movie-usa.glencoesoftware.com/mp4/10.7554/644/ad9b2d1719747dc41bf8934bca9b69698845d049/elife-47279-fig8-video1.mp4>
Download as WebM
<https://static-movie-usa.glencoesoftware.com/webm/10.7554/644/ad9b2d1719747dc41bf8934bca9b69698845d049/elife-47279-fig8-video1.webm>
Download as Ogg
<https://static-movie-usa.glencoesoftware.com/ogv/10.7554/644/ad9b2d1719747dc41bf8934bca9b69698845d049/elife-47279-fig8-video1.ogv>


            Tropomyosin Cdc8 and α-actinin Ain1 cooperate to compete
            with fimbrin Fim1 for bundling F-actin, related to Figure 5
            <#fig5>.

Four-color TIRFM of 1.5 μM actin (Alexa-488 labeled), 2.5 μM tropomyosin
Cdc8 (TMR-labeled) and 50 nM fimbrin Fim1 (Alexa 405-labeled) with
(bottom) or without (top) 500 nM Ain1 (Cy5-labeled). Scale … see more

https://doi.org/10.7554/eLife.47279.019
<https://doi.org/10.7554/eLife.47279.019>

Table 1


            Fission yeast strains used in this study.

https://doi.org/10.7554/eLife.47279.018
<https://doi.org/10.7554/eLife.47279.018>

Strain name	Genotype	Reference
 KV91	h+, kanMX6::myo2p::gfp-myo2p+, ade6-M210, leu1-32, ura4-D18	Wu et
al., 2003 <#bib40>
 KV343	h?, cdc12-mGFP::KanR	This study
 KV459	h+, rlc1-tdTomato-natMX6 ade6-M210 leu1-32 ura4-D18	This study
 KV579	h-, ain1-GFP-KanMX6, URA+	Wu et al., 2001 <#bib39>
 KV588	h+, pAct1 Lifeact-GFP::Leu+; ade6-m216; leu1-32; ura4-D18	Huang
et al., 2012 <#bib12>
Laporte et al., 2011 <#bib18> KV707	h-, leu1-32, his3-D1, ura4-D18,
ade6-M216, Pnmt41-SpAin1-mGFP::Leu+	This study
 KV804	h?, fim1-mCherry-natMX6, ain1-Δ1:: kanMX6,
Pnmt41-SpAin1-mGFP::Leu+	This study
 KV818	h + kanMX6-Prng2-mEGFP-rng2 ade6-M210 leu1-32 ura4-D18	Laporte et
al., 2011 <#bib18>
 KV856	h?, ain1-Δ1:: kanMX6, Pnmt41-SpAin1(R216E)-mGFP::Leu+,
fim1-mCherry-natMX6, ade6, leu1-32, ura4-D18	This study
 KV861	h?, ain1-GFP-kanMX6, sad1-tdTomato-natMX6, ade6-m21?, leu1-32,
ura4-D18	This study
 KV878	h+, fim1-mGFP-kanMX6, sad1-tdTomato-natMX6	This study
 KV908	h? fim1-1Δ-kanMX6, ain1-GFP-kanMX6, sad1-tdTomato::natMX6	This study
 KV963	h?, fim1-1Δ::kanMX6, Pnmt41-SpAin1-mGFP::Leu+	This study
 KV968	h? Pnmt41-SpAin1-mGFP::Leu+, ARPC5-mCherry-natMX6	This study


    Discussion

<#>

We have discovered that fimbrin Fim1 is inhibited from associating with
actin filaments by the combined efforts of contractile ring ABPs
α-actinin Ain1 and tropomyosin Cdc8. Although we have demonstrated that
competition between ABPs is one key mechanism that drives their sorting
to different F-actin networks, we suspect that there are several
non-mutually exclusive mechanisms that drive the sorting of Fim1 and
other ABPs to the correct F-actin network. Specifically, our data
suggest that Fim1’s association with the contractile ring is inhibited
in part by both (1) a preferred association with actin patches and (2)
competition with Cdc8 and Ain1.


      A ‘sink’ model for fimbrin Fim1 association with actin patches

If fimbrin Fim1 preferentially associates with actin patches over other
F-actin networks, actin patches could act as a ‘sink’ for Fim1, thereby
sequestering Fim1 and limiting the concentration of free Fim1 capable of
associating with other F-actin networks. Indeed, although only small
amounts of Fim1 are present on the contractile ring of wild-type cells
(Nakano et al., 2001 <#bib24>; Wu et al., 2001 <#bib39>), depletion of
actin patches (the ‘sink’) by the Arp2/3 complex inhibitor CK-666
results in significantly more Fim1 associating with the contractile ring
(Figure 1B <#fig1>). However, a sink model alone likely cannot account
for Fim1’s primary localization to actin patches. Only 30% (~25,000) of
the ~86,500 total Fim1 molecules in a fission yeast cell associate
with ~50 actin patches (Wu and Pollard, 2005 <#bib41>; Sirotkin et al.,
2010 <#bib32>). Therefore, ~70–75% of Fim1 molecules are either
cytoplasmic or associated with another F-actin network. We suspect that
other mechanisms (conformational changes in the actin filament, ABP
dynamics, post-translational modifications) may dictate both Fim1’s
preferential association with actin patches, as well as the observed
competitive and cooperative interactions amongst Fim1, α-actinin Ain1,
and tropomyosin Cdc8.


      Conformational changes in the actin filament

Actin forms a flexible polymer capable of adopting a variety of
conformations (Galkin et al., 2010 <#bib9>; Oda and Maéda, 2010
<#bib26>), and different F-actin conformations have been demonstrated to
both result from or influence the association of ABPs (Michelot and
Drubin, 2011 <#bib22>; Papp et al., 2006 <#bib27>; Risca et al., 2012
<#bib28>). Fimbrin Fim1’s inherent preference for actin patches could
result from a preference for either a branched F-actin network or for a
particular twist or conformational change imparted by Arp2/3
complex-mediated actin assembly. We suspect that conformational changes
in the actin filament may also affect the observed increase of α-actinin
Ain1-mediated bundling in the presence of tropomyosin Cdc8 (Figure 6A–D
<#fig6>). Tropomyosins have been found to increase the persistence
length of F-actin (Fujime and Ishiwata, 1971 <#bib7>; Isambert et al.,
1995 <#bib13>), suggesting that they are capable of altering the
conformation of an actin filament. Therefore, Cdc8 may mediate a slight
conformational change more amenable to Ain1 binding, increasing the
density of Ain1 on the actin filaments and the corresponding degree of
bundling. Tropomyosin’s increase of the persistence length of F-actin
may have an additional effect on F-actin bundling by promoting bundle
stability and inter-filament contacts.


      ABP dynamics affect their ability to compete

We discovered that the contractile ring ABP α-actinin Ain1 competes with
fimbrin Fim1, and that dynamic association of both Ain1 and Fim1 with
actin filaments affects their ability to compete at different F-actin
networks. In particular, we found that a less dynamic Ain1(R216E)
mislocalizes to F-actin patches even in the presence of Fim1. However,
in addition to ectopic localization to actin patches, cells expressing
Ain1(R216E) also have cytokinesis defects (Li et al., 2016 <#bib19>).
These findings may explain why fission yeast requires two F-actin
bundling proteins. Actin patches require a stable F-actin bundler such
as Fim1 to crosslink the branched F-actin network, prevent tropomyosin
Cdc8 association, and create boundaries that enhance cofilin
Adf1-mediated severing (Christensen et al., 2017 <#bib4>; Skau and
Kovar, 2010 <#bib34>). On the other hand, a dynamic F-actin bundler is
required at the contractile ring to facilitate anti-parallel F-actin
contacts while still allowing Cdc8 association and myosin-mediated actin
filament sliding and contraction (Li et al., 2016 <#bib19>). For these
reasons, we assume that the dynamic Ain1 is an ideal actin crosslinker
for the contractile ring (Figure 6E–G <#fig6>). Though our results
suggest that Ain1 is more dynamic on uncoated actin filaments, we showed
that wild-type Ain1 is still a dynamic bundler on Cdc8-coated actin
filaments and is likely capable of allowing contractile ring ABPs to
remain associated with and function optimally at the contractile ring.

Additionally, the binding dynamics of Ain1 and Fim1 on F-actin seem to
mediate their competition with Cdc8. We demonstrated that Fim1, but not
Ain1, displaces Cdc8 from F-actin bundles. However, Fim1 competes with
Cdc8 specifically at regions where it binds stably (F-actin bundles),
and does not compete as strongly with Cdc8 on single filaments
(Christensen et al., 2017 <#bib4>). Therefore, the presence of a dynamic
bundling protein (Fim1 on single filaments and Ain1 on both single and
bundled filaments) may allow Cdc8 to remain associated with F-actin in
those circumstances. Future studies will seek to build a fuller picture
of how Fim1 and Ain1 differentially affect the activity of other ABPs
present at the contractile ring such as myosin-II and Adf1.


      Post-translational modifications and other sorting mechanisms

In vitro, the combined efforts of α-actinin Ain1 and tropomyosin Cdc8
prevent ~40% of fimbrin Fim1 association with F-actin bundles. It should
be noted that although Ain1 and Cdc8 work together to compete with Fim1,
our reactions contain low concentrations of Fim1 (50 nM) compared to
Ain1 (500 nM or 1 μM) and Cdc8 (2.5 μM). Given the potent F-actin
binding and bundling abilities of Fim1 and the high cellular
concentration of Fim1, a reasonable assumption is that Cdc8 and Ain1 may
only prevent a portion of Fim1 polypeptides from associating with
contractile ring F-actin in vivo. Therefore, other non-mutually
exclusive mechanisms likely contribute to ABP sorting. One possibility
is that additional ABPs inhibit Fim1 from associating with the
contractile ring. A second possibility is that Fim1 is
post-translationally modified, regulating its activity. Budding yeast
fimbrin Sac6 is phosphorylated at different stages of the cell cycle,
which affects its ability to bind F-actin (Miao et al., 2016 <#bib21>).
Fission yeast Fim1 might be similarly post-translationally modified
(Kettenbach et al., 2015 <#bib16>; Swaffer et al., 2018 <#bib37>), and
therefore a portion of the cytoplasmic Fim1 pool might be more or less
active. A third possibility is that mechanical stresses applied to actin
filaments, such as the contractile forces applied by the molecular motor
type-II myosin Myo2 during contractile ring assembly and constriction,
might differentially alter the F-actin binding and/or activity of actin
patch and contractile ring ABPs (Romet-Lemonne and Jégou, 2013 <#bib29>).


      A similar combination of mechanisms could drive ABP sorting to
      other F-actin networks

We expect that similar fundamental mechanistic principles also promote
ABP sorting at other F-actin networks, both in fission yeast and in
multicellular organisms. One possibility is that actin assembly factors,
such as the Arp2/3 complex (actin patches), formin Cdc12 (contractile
rings), and formin For3 (actin cables) assemble actin filaments with
intrinsic differences (such as a specific filament conformation) that
initiate the differential sorting of certain ABPs to diverse networks
(Kovar et al., 2011 <#bib17>; Michelot and Drubin, 2011 <#bib22>). The
initial actin assembly factor-dependent recruitment of ‘upstream’ ABPs
may then subsequently recruit and/or inhibit other ABPs, defining both
the subset of ABPs that associate with a particular F-actin network as
well as the network’s corresponding architecture and dynamics. For
example, although fission yeast has a single tropomyosin isoform, Cdc8,
it can be either acetylated or unacetylated (Skoumpla et al., 2007
<#bib35>). While the acetylated form of Cdc8 associates with the
contractile ring, the unacetylated form associates with actin cables
(Coulton et al., 2010 <#bib5>). Unacetylated Cdc8 preferentially
associates with For3-assembled F-actin regardless of where it is
assembled (Johnson et al., 2014 <#bib14>). We used a contractile ring
acetylation mimic form of Cdc8 (AlaSer-Cdc8) in the in vitro assays
reported here (Christensen et al., 2017 <#bib4>). However, it is
possible that the acetylated and unacetylated forms of Cdc8
differentially compete or cooperate with other ABPs in vitro and/or in
vivo, influencing the sorting of distinct sets of ABPs to each network.
Interestingly, how ABPs influence one another could also depend on the
cellular type or context, as tropomyosin and α-actinin-4 exhibit an
antagonistic relationship during stress fiber formation in MDCK cells
(Kemp and Brieher, 2018 <#bib15>). Future work will involve
investigating the potential role of actin assembly factors in regulating
ABP sorting and understanding the competitive and cooperation
interactions between ABPs in other cell types.


    Materials and methods

<#>


      Strain construction and growth

Request a detailed protocol
<https://bio-protocol.org/eLIFErap47279?item=s4-1>

Fission yeast strains were created by genetic crossing on SPA5S plates
followed by tetrad dissection on YE5S plates. Strains were screened for
auxotrophic (leu, ura) or antibiotic (nat, kan) markers and maintained
on YE5S plates. Glycerol stocks were created by pelleting cells and
resuspending in 750 μL media and 250 μL of 50% sterile glycerol. The
strains used in this study are indicated in Table 1 <#table1>.


      Cell imaging and treatment with CK-666

Request a detailed protocol
<https://bio-protocol.org/eLIFErap47279?item=s4-2>

For live cell imaging, cells were grown in YE5S media overnight at 25°C,
subcultured into EMM5S media without thiamine, and kept in log phase for
20–22 hr at 25°C. Cells were imaged directly on glass slides. Z-stacks
of 10 slices, 0.5 μm apart were acquired with a 100x, 1.4 NA objective
on a Zeiss Axiovert 200M equipped with a Yokogawa CSU-10 spinning-disk
unit (McBain, Simi Valley, CA) illuminated with a 50-milliwatt 473 nm
DPSS laser, and a Cascade 512B EM-CCD camera (Photometrics, Tucson, AZ)
controlled by MetaMorph software (Molecular Devices, Sunnyvale, CA). For
CK-666 treatments, CK-666 powder stock (Sigma, St. Louis, MO) was
diluted to 10 mM in DMSO. Cells were grown as stated above, and
incubated with CK-666 or an equivalent volume of DMSO (control) in a
rotator at 25°C for 30 min prior to imaging. Cells were then immediately
imaged as above.


      Contractile ring fluorescence quantification

Request a detailed protocol
<https://bio-protocol.org/eLIFErap47279?item=s4-3>

Contractile ring maturation was divided into three stages by measuring
the distance between spindle pole bodies (SPBs, visualized by
Sad1-tdTomato) and noting constriction of the contractile ring. Stage 1
cells had SPBs less than 6 μm apart, with no observable ring
constriction. Stage 2 cells had SPBs greater than 6 μm apart, with no
observable ring constriction. Stage 3 cells had SPBs less than 9 μm
apart, with evident ring constriction. Quantification of ABP association
with all contractile rings (stages 1–3) is shown in Figure 1B–D <#fig1>,
and quantification at each distinct ring stage is show in Figure
1—figure supplement 3
<https://elifesciences.org/articles/47279/figures#fig1s3>. The
contractile ring region was determined by visually examining the z-stack
for the ring site. Normalized contractile ring fluorescence was taken by
drawing a region of interest (ROI) around the observed ring and around
the whole cell using ImageJ. To obtain a fluorescence value for cells
without a visible contractile ring (for example, DMSO-treated Fim1-GFP
cells), mitotic cells were determined by the presence of two spindle
pole body markers. These fission yeast cells were measured along their
length and a 4-pixel width line was drawn across the exact center of the
fission yeast cell (denoted as a yellow dotted-line in Figure 1B
<#fig1>). The mean fluorescence of the ring divided by the whole cell
was then determined. A value of 1.00 indicates no increased fluorescence
at the site of the contractile ring, while values > 1 indicate increased
fluorescence at the ring. Maximum projections were used for images in
figures and sum projections were used for quantification.


      Tropomyosin Cdc8 antibody staining

Request a detailed protocol
<https://bio-protocol.org/eLIFErap47279?item=s4-4>

Following standard growth and culturing protocols for live cell imaging,
fission yeast cells were stained with anti-Cdc8p (Cranz-Mileva et al.,
2015 <#bib6>). Cells were fixed in 16% formaldehyde for 5 min at 20°C.
Cells were then washed in cold 1X PBS and resuspended in 140 μL 1.2M
sorbitol. 60 μL fresh protoplasting solution (3 mg/ml zymolase 100T in
1.2M sorbitol) was added and cells were incubated for 7 min on a rotator
at room temperature. 1 mL of 1% Triton-X was then added to the cells and
incubation continued for 2 min. Cells were then pelleted and resuspended
in 0.5 mL PBAL (10% BSA, 100 mM lysine monohydrochloride, 1 mM NaN_3 ,
50 ng/ml ampicillin in PBS) and incubated for 2.5 hr on a rotator at
room temperature. Cells were resuspended in 100 μL of anti-Cdc8p 1:10 in
PBAL (gift of Sarah Hitchcock-DeGregori) and incubated overnight at 4°C
on a rotator. Following incubation with primary antibody, cells were
washed three times with 0.5 mL PBAL and resuspended in 50 μL Alexa-Flour
555 goat anti-rabbit secondary antibody (Thermo-Fisher Scientific,
Carlsbad, CA) (1:100 in PBAL) and incubated for 90 min at room
temperature on a shaker in the dark. Cells were then washed five times
with 0.5 mL PBAL and resuspended in 20–30 μL PBAL for imaging. Cells
were stored at 4°C and imaged within 48 hr of staining.


      Protein purification

Request a detailed protocol
<https://bio-protocol.org/eLIFErap47279?item=s4-5>

Chicken skeletal muscle actin was purified as described previously
(Spudich and Watt, 1971 <#bib36>). Fimbrin Fim1 and acetylation mimic
tropomyosin AlaSer-Cdc8 (WT and I76C mutant) were expressed in
BL21-Codon Plus (DE3)-RP (Agilent Technologies, Santa Clara, CA).
His-tagged Fim1 was purified using Talon Metal Affinity Resin (Clontech,
Mountain View, CA) (Skau and Kovar, 2010 <#bib34>). Cdc8 was purified by
boiling the cell lysate, performing an ammonium sulfate cut, and running
on an anion exchange column (Skau and Kovar, 2010 <#bib34>). His-tagged
wild-type α-actinin Ain1 and mutant Ain1(R216E) were expressed in High
Five insect cells using baculovirus expression and purified using Talon
Metal Affinity Resin (Li et al., 2016 <#bib19>).

The A_280 of purified proteins was taken with a Nanodrop 2000c
Spectrophotometer (Thermo-Scientific, Waltham, MA). Protein
concentration was calculated using extinction coefficients Fim1: 55,140
M^−1 cm^−1 , Cdc8 (WT and I76C mutant): 2,980 M^−1 cm^−1 , Ain1 and
Ain1(R216E): 86477 M^−1 cm^−1 . Proteins were labeled with
TMR-6-maleimide (Life Technologies, Grand Island, NY) or
Cy5-monomaleimide (GE Healthcare, Little Chalfont, UK) dyes following
manufacturer’s protocols following purification. Proteins were
flash-frozen in liquid nitrogen and stored at −80°C.


      Low-speed sedimentation assays

Request a detailed protocol
<https://bio-protocol.org/eLIFErap47279?item=s4-6>

To perform low-speed sedimentation assays, 15 µM Mg-ATP actin monomers
were spontaneously assembled in 10 mM imidazole, pH 7.0 50 mM KCl, 5 mM
MgCl_2 , 1 mM EGTA, 0.5 mM DTT, 0.2 mM ATP, and 90 µM CaCl_2 for 1 hr to
generate F-actin. F-actin was then incubated with 500 nM Ain1 and a
range of concentrations of Cdc8 (Figure 6A–B <#fig6>), or 5 µM Cdc8 and
0.5 µM Fim1-SNAP and/or 5 µM Ain1 (Figure 7A–B <#fig7>). This incubation
occurred for 20 min at 25°C, followed by a 10,000 g spin for 20 min at
room temperature. Supernatant and pellets were separated by 15% SDS-PAGE
gel electrophoresis and stained for 30 min with Coomassie Blue,
destained for 16 hr, and analyzed by densitometry with ImageJ.


      TIRF microscopy (TIRFM)

Request a detailed protocol
<https://bio-protocol.org/eLIFErap47279?item=s4-7>

Time-lapse TIRFM movies were obtained using an Olympus IX-71 microscope
with through-the-objective TIRF illumination, iXon EMCCD camera (Andor
Technology), and a cellTIRF 4Line system (Olympus). The actin binding
proteins (ABPs) of interest were initially added to a polymerization mix
(10 mM imidazole (pH 7.0), 50 mM KCl, 1 mM MgCl_2 , 1 mM EGTA, 50 mM
DTT, 0.2 mM ATP, 50 μM CaCl_2 , 15 mM glucose, 20 μg/mL catalase, 100
μg/mL glucose oxidase, and 0.5% (400 centipoise) methylcellulose). This
ABP/polymerization mix was then added to Mg-ATP-actin (15% Alexa
488-labeled) to induce F-actin assembly in the presence of the ABPs of
interest (Zimmermann et al., 2016 <#bib42>). The mixture was then added
to a flow chamber and imaged at room temperature at 5 s intervals
(unless otherwise noted).


      Quantification of bundling

Request a detailed protocol
<https://bio-protocol.org/eLIFErap47279?item=s4-8>

The percentage of actin filaments bundled was quantified at similar
actin filament densities (between 2095 and 2295 μm total filament
length) for each experiment. The total actin filament length in the
chamber was measured manually by creating ROIs for every actin filament
and measuring total actin filament length in FIJI (Schindelin et al.,
2012 <#bib30>; Schneider et al., 2012 <#bib31>). ROIs for every segment
of actin filament present in a bundle were then created and total
bundled filament length measured. A ‘bundled’ segment was identified by
looking at the history of the TIRF movie and determining that two actin
filaments were associated and had been associated for at least three
consecutive frames (at least 15 s) including the frame quantified. The
ratio of actin filament present in a bundle vs. total actin filament
length was then calculated.


      Quantification of fluorescence intensity on actin filaments or bundles

Request a detailed protocol
<https://bio-protocol.org/eLIFErap47279?item=s4-9>

Fluorescence intensity on actin filaments was quantified on movies taken
under the same microscope conditions (laser intensity and angle,
exposure time) and with the same protein batches. Fluorescence intensity
was quantified at the same time point in each compared movie. The actin
channel was used to identify single actin filaments or two-filament
actin bundles, and ROIs of a three-pixel segmented line were created
along all single filaments or two-filament bundles in the selected
frame. The mean fluorescence for each segment was then measured using
ImageJ.


      Quantifying number of cells with Ain1 in actin patches

Request a detailed protocol
<https://bio-protocol.org/eLIFErap47279?item=s4-10>

To quantify the number of cells containing Ain1-GFP in actin patches, 1
min timelapse movies of 1 frame per second were taken, imaging both
Ain1-GFP and an actin patch marker (ArpC5-mCherry or Fim1-mCherry).
Movie files for independent experiments and replicates were blinded and
independently analyzed for number of cells containing Ain1-GFP in actin
patches using FIJI (Schindelin et al., 2012 <#bib30>; Schneider et al.,
2012 <#bib31>). For a single cell to count as positively containing
Ain1-GFP in actin patches, three criteria had to be met: 1) at least one
distinguishable actin patch containing Ain1-GFP was observed, 2) the
observed actin patch(es) contained Ain1-GFP for at least three frames
and 3) the Ain1-GFP signal trajectory matched the channel expressing
either ArpC5-mCherry or Fim1-mCherry. Total number of cells and cells
with actin patches containing Ain1-GFP were then calculated to obtain
percent of cells containing Ain1-GFP in actin patches.


      Quantification of Ain1 spot density

Request a detailed protocol
<https://bio-protocol.org/eLIFErap47279?item=s4-11>

TIRFM images were collected with a cellTIRF 4Line system (Olympus)
fitted to an Olympus IX-71 microscope with through-the-objective TIRF
illumination and an iXon EMCCD camera (Andor Technology, Belfast, UK).
1.5 μM Mg-ATP-actin (10% Alexa 488 labeled) was mixed with
polymerization TIRF buffer [10 mM imidazole (pH 7.0), 50 mM KCl, 1 mM
MgCl_2 , 1 mM EGTA, 50 mM DTT, 0.2 mM ATP, 50 μM CaCl_2 , 15 mM glucose,
20 μg/mL catalase, 100 μg/mL glucose oxidase, and 0.5% (400 centipoise)
methylcellulose] to induce F-actin assembly, 0.5 μM 0.5% TMR-labeled
α-actinin, and 2 μM (monomer 1 μM dimer) unlabeled tropomyosin. This
mixture was transferred to a flow cell for imaging at room temperature.
Once the actin had polymerized and formed bundles, we imaged once in the
488 channel to visualize the labeled actin (one frame, 488 nm excitation
for 50 ms) and then continuously imaged in the 561 channel to visualize
the sparsely labeled α-actinin (100 frames, 561 nm excitation for 50
ms,~110 ms interval).

To measure α-actinin Ain1 spot density, we created kymographs on bundles
for each experiment using ImageJ. Ain1 spots were detected in the
kymograph as lines at least four pixels wide with a fluorescence value
above 1.25 times that of background fluorescence. Spot density was
normalized to the length of actin filaments in the bundle. Ain1 spot
density was determined using the following formula:

n÷(L×r)tn÷(L×r)t

where /n/ is the number of Ain1 spots detected, /L/ is the length of the
bundle in µm, /r/ is the actin fluorescence ratio (total amount of
fluorescence in the actin bundle divided by the average fluorescence of
single actin filaments), and /t/ is the time of measurement in seconds.

Add a comment +Open annotations. The current annotation count on this
page is 0.


    Data availability

<#>

All data generated or analyzed during the study are included in the
manuscript.


    References

<#>

 1.
     1. Addario B
        <https://scholar.google.com/scholar?q=%22author:Addario+B%22>
     2. Sandblad L
        <https://scholar.google.com/scholar?q=%22author:Sandblad+L%22>
     3. Persson K
        <https://scholar.google.com/scholar?q=%22author:Persson+K%22>
     4. Backman L
        <https://scholar.google.com/scholar?q=%22author:Backman+L%22>
    (2016) Characterisation of /Schizosaccharomyces pombe/ α-actinin
    <https://doi.org/10.7717/peerj.1858> /PeerJ/ *4*:e1858.
    https://doi.org/10.7717/peerj.1858 <https://doi.org/10.7717/peerj.1858>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/27069798>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Characterisation+of%C2%A0Schizosaccharomyces+pombe%C2%A0%CE%B1-actinin&author=Addario+B&author=Sandblad+L&author%5B2%5D=Persson+K&author%5B3%5D=Backman+L&publication_year=2016&journal=PeerJ&volume=4&pages=e1858&pmid=27069798>
 2.
     1. Arai R <https://scholar.google.com/scholar?q=%22author:Arai+R%22>
     2. Nakano K
        <https://scholar.google.com/scholar?q=%22author:Nakano+K%22>
     3. Mabuchi I
        <https://scholar.google.com/scholar?q=%22author:Mabuchi+I%22>
    (1998) Subcellular localization and possible function of actin,
    tropomyosin and actin-related protein 3 (Arp3) in the fission
    yeast /Schizosaccharomyces pombe/
    <https://doi.org/10.1016/S0171-9335(98)80007-1> /European Journal of
    Cell Biology/ *76*:288–295.
    https://doi.org/10.1016/S0171-9335(98)80007-1
    <https://doi.org/10.1016/S0171-9335(98)80007-1>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/9765059>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Subcellular+localization+and+possible+function+of+actin%2C+tropomyosin+and+actin-related+protein+3+%28Arp3%29+in+the+fission+yeast%C2%A0Schizosaccharomyces+pombe&author=Arai+R&author=Nakano+K&author%5B2%5D=Mabuchi+I&publication_year=1998&journal=European+Journal+of+Cell+Biology&volume=76&pages=pp.+288%E2%80%93295&pmid=9765059>
 3.
     1. Burke TA
        <https://scholar.google.com/scholar?q=%22author:Burke+TA%22>
     2. Christensen JR
        <https://scholar.google.com/scholar?q=%22author:Christensen+JR%22>
     3. Barone E
        <https://scholar.google.com/scholar?q=%22author:Barone+E%22>
     4. Suarez C
        <https://scholar.google.com/scholar?q=%22author:Suarez+C%22>
     5. Sirotkin V
        <https://scholar.google.com/scholar?q=%22author:Sirotkin+V%22>
     6. Kovar DR
        <https://scholar.google.com/scholar?q=%22author:Kovar+DR%22>
    (2014) Homeostatic actin cytoskeleton networks are regulated by
    assembly factor competition for monomers
    <https://doi.org/10.1016/j.cub.2014.01.072> /Current Biology/
    *24*:579–585.
    https://doi.org/10.1016/j.cub.2014.01.072
    <https://doi.org/10.1016/j.cub.2014.01.072>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/24560576>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Homeostatic+actin+cytoskeleton+networks+are+regulated+by+assembly+factor+competition+for+monomers&author=Burke+TA&author=Christensen+JR&author%5B2%5D=Barone+E&author%5B3%5D=Suarez+C&author%5B4%5D=Sirotkin+V&author%5B5%5D=Kovar+DR&publication_year=2014&journal=Current+Biology&volume=24&pages=pp.+579%E2%80%93585&pmid=24560576>
 4.
     1. Christensen JR
        <https://scholar.google.com/scholar?q=%22author:Christensen+JR%22>
     2. Hocky GM
        <https://scholar.google.com/scholar?q=%22author:Hocky+GM%22>
     3. Homa KE <https://scholar.google.com/scholar?q=%22author:Homa+KE%22>
     4. Morganthaler AN
        <https://scholar.google.com/scholar?q=%22author:Morganthaler+AN%22>
     5. Hitchcock-DeGregori SE
        <https://scholar.google.com/scholar?q=%22author:Hitchcock-DeGregori+SE%22>
     6. Voth GA <https://scholar.google.com/scholar?q=%22author:Voth+GA%22>
     7. Kovar DR
        <https://scholar.google.com/scholar?q=%22author:Kovar+DR%22>
    (2017) Competition between tropomyosin, fimbrin, and ADF/Cofilin
    drives their sorting to distinct actin filament networks
    <https://doi.org/10.7554/eLife.23152> /eLife/ *6*:e23152.
    https://doi.org/10.7554/eLife.23152
    <https://doi.org/10.7554/eLife.23152>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/28282023>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Competition+between+tropomyosin%2C+fimbrin%2C+and+ADF%2FCofilin+drives+their+sorting+to+distinct+actin+filament+networks&author=Christensen+JR&author=Hocky+GM&author%5B2%5D=Homa+KE&author%5B3%5D=Morganthaler+AN&author%5B4%5D=Hitchcock-DeGregori+SE&author%5B5%5D=Voth+GA&author%5B6%5D=Kovar+DR&publication_year=2017&journal=eLife&volume=6&pages=e23152&pmid=28282023>
 5.
     1. Coulton AT
        <https://scholar.google.com/scholar?q=%22author:Coulton+AT%22>
     2. East DA <https://scholar.google.com/scholar?q=%22author:East+DA%22>
     3. Galinska-Rakoczy A
        <https://scholar.google.com/scholar?q=%22author:Galinska-Rakoczy+A%22>
     4. Lehman W
        <https://scholar.google.com/scholar?q=%22author:Lehman+W%22>
     5. Mulvihill DP
        <https://scholar.google.com/scholar?q=%22author:Mulvihill+DP%22>
    (2010) The recruitment of acetylated and unacetylated tropomyosin to
    distinct actin polymers permits the discrete regulation of specific
    myosins in fission yeast <https://doi.org/10.1242/jcs.069971>
    /Journal of Cell Science/ *123*:3235–3243.
    https://doi.org/10.1242/jcs.069971 <https://doi.org/10.1242/jcs.069971>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=The+recruitment+of+acetylated+and+unacetylated+tropomyosin+to+distinct+actin+polymers+permits+the+discrete+regulation+of+specific+myosins+in+fission+yeast&author=Coulton+AT&author=East+DA&author%5B2%5D=Galinska-Rakoczy+A&author%5B3%5D=Lehman+W&author%5B4%5D=Mulvihill+DP&publication_year=2010&journal=Journal+of+Cell+Science&volume=123&pages=pp.+3235%E2%80%933243>
 6.
     1. Cranz-Mileva S
        <https://scholar.google.com/scholar?q=%22author:Cranz-Mileva+S%22>
     2. MacTaggart B
        <https://scholar.google.com/scholar?q=%22author:MacTaggart+B%22>
     3. Russell J
        <https://scholar.google.com/scholar?q=%22author:Russell+J%22>
     4. Hitchcock-DeGregori SE
        <https://scholar.google.com/scholar?q=%22author:Hitchcock-DeGregori+SE%22>
    (2015) Evolutionarily conserved sites in yeast tropomyosin function
    in cell polarity, transport and contractile ring formation
    <https://doi.org/10.1242/bio.012609> /Biology Open/ *4*:1040–1051.
    https://doi.org/10.1242/bio.012609 <https://doi.org/10.1242/bio.012609>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/26187949>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Evolutionarily+conserved+sites+in+yeast+tropomyosin+function+in+cell+polarity%2C+transport+and+contractile+ring+formation&author=Cranz-Mileva+S&author=MacTaggart+B&author%5B2%5D=Russell+J&author%5B3%5D=Hitchcock-DeGregori+SE&publication_year=2015&journal=Biology+Open&volume=4&pages=pp.+1040%E2%80%931051&pmid=26187949>
 7.
     1. Fujime S
        <https://scholar.google.com/scholar?q=%22author:Fujime+S%22>
     2. Ishiwata S
        <https://scholar.google.com/scholar?q=%22author:Ishiwata+S%22>
    (1971) Dynamic study of F-actin by quasielastic scattering of laser
    light <https://doi.org/10.1016/0022-2836(71)90144-6> /Journal of
    Molecular Biology/ *62*:251–265.
    https://doi.org/10.1016/0022-2836(71)90144-6
    <https://doi.org/10.1016/0022-2836(71)90144-6>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/4945533>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Dynamic+study+of+F-actin+by+quasielastic+scattering+of+laser+light&author=Fujime+S&author=Ishiwata+S&publication_year=1971&journal=Journal+of+Molecular+Biology&volume=62&pages=pp.+251%E2%80%93265&pmid=4945533>
 8.
     1. Galkin VE
        <https://scholar.google.com/scholar?q=%22author:Galkin+VE%22>
     2. Orlova A
        <https://scholar.google.com/scholar?q=%22author:Orlova+A%22>
     3. Cherepanova O
        <https://scholar.google.com/scholar?q=%22author:Cherepanova+O%22>
     4. Lebart MC
        <https://scholar.google.com/scholar?q=%22author:Lebart+MC%22>
     5. Egelman EH
        <https://scholar.google.com/scholar?q=%22author:Egelman+EH%22>
    (2008) High-resolution cryo-EM structure of the
    F-actin-fimbrin/plastin ABD2 complex
    <https://doi.org/10.1073/pnas.0708667105> /PNAS/ *105*:1494–1498.
    https://doi.org/10.1073/pnas.0708667105
    <https://doi.org/10.1073/pnas.0708667105>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/18234857>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=High-resolution+cryo-EM+structure+of+the+F-actin-fimbrin%2Fplastin+ABD2+complex&author=Galkin+VE&author=Orlova+A&author%5B2%5D=Cherepanova+O&author%5B3%5D=Lebart+MC&author%5B4%5D=Egelman+EH&publication_year=2008&journal=PNAS&volume=105&pages=pp.+1494%E2%80%931498&pmid=18234857>
 9.
     1. Galkin VE
        <https://scholar.google.com/scholar?q=%22author:Galkin+VE%22>
     2. Orlova A
        <https://scholar.google.com/scholar?q=%22author:Orlova+A%22>
     3. Salmazo A
        <https://scholar.google.com/scholar?q=%22author:Salmazo+A%22>
     4. Djinovic-Carugo K
        <https://scholar.google.com/scholar?q=%22author:Djinovic-Carugo+K%22>
     5. Egelman EH
        <https://scholar.google.com/scholar?q=%22author:Egelman+EH%22>
    (2010) Opening of tandem calponin homology domains regulates their
    affinity for F-actin <https://doi.org/10.1038/nsmb.1789> /Nature
    Structural & Molecular Biology/ *17*:614–616.
    https://doi.org/10.1038/nsmb.1789 <https://doi.org/10.1038/nsmb.1789>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/20383143>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Opening+of+tandem+calponin+homology+domains+regulates+their+affinity+for+F-actin&author=Galkin+VE&author=Orlova+A&author%5B2%5D=Salmazo+A&author%5B3%5D=Djinovic-Carugo+K&author%5B4%5D=Egelman+EH&publication_year=2010&journal=Nature+Structural+%26+Molecular+Biology&volume=17&pages=pp.+614%E2%80%93616&pmid=20383143>
10.
     1. Holtzman DA
        <https://scholar.google.com/scholar?q=%22author:Holtzman+DA%22>
     2. Wertman KF
        <https://scholar.google.com/scholar?q=%22author:Wertman+KF%22>
     3. Drubin DG
        <https://scholar.google.com/scholar?q=%22author:Drubin+DG%22>
    (1994) Mapping actin surfaces required for functional interactions
    in vivo <https://doi.org/10.1083/jcb.126.2.423> /The Journal of Cell
    Biology/ *126*:423–432.
    https://doi.org/10.1083/jcb.126.2.423
    <https://doi.org/10.1083/jcb.126.2.423>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/8034743>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Mapping+actin+surfaces+required+for+functional+interactions+in+vivo&author=Holtzman+DA&author=Wertman+KF&author%5B2%5D=Drubin+DG&publication_year=1994&journal=The+Journal+of+Cell+Biology&volume=126&pages=pp.+423%E2%80%93432&pmid=8034743>
11.
     1. Honts JE
        <https://scholar.google.com/scholar?q=%22author:Honts+JE%22>
     2. Sandrock TS
        <https://scholar.google.com/scholar?q=%22author:Sandrock+TS%22>
     3. Brower SM
        <https://scholar.google.com/scholar?q=%22author:Brower+SM%22>
     4. O'Dell JL
        <https://scholar.google.com/scholar?q=%22author:O%27Dell+JL%22>
     5. Adams AE
        <https://scholar.google.com/scholar?q=%22author:Adams+AE%22>
    (1994) Actin mutations that show suppression with fimbrin mutations
    identify a likely fimbrin-binding site on actin
    <https://doi.org/10.1083/jcb.126.2.413> /The Journal of Cell
    Biology/ *126*:413–422.
    https://doi.org/10.1083/jcb.126.2.413
    <https://doi.org/10.1083/jcb.126.2.413>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/8034742>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Actin+mutations+that+show+suppression+with+fimbrin+mutations+identify+a+likely+fimbrin-binding+site+on+actin&author=Honts+JE&author=Sandrock+TS&author%5B2%5D=Brower+SM&author%5B3%5D=O%27Dell+JL&author%5B4%5D=Adams+AE&publication_year=1994&journal=The+Journal+of+Cell+Biology&volume=126&pages=pp.+413%E2%80%93422&pmid=8034742>
12.
     1. Huang J <https://scholar.google.com/scholar?q=%22author:Huang+J%22>
     2. Huang Y <https://scholar.google.com/scholar?q=%22author:Huang+Y%22>
     3. Yu H <https://scholar.google.com/scholar?q=%22author:Yu+H%22>
     4. Subramanian D
        <https://scholar.google.com/scholar?q=%22author:Subramanian+D%22>
     5. Padmanabhan A
        <https://scholar.google.com/scholar?q=%22author:Padmanabhan+A%22>
     6. Thadani R
        <https://scholar.google.com/scholar?q=%22author:Thadani+R%22>
     7. Tao Y <https://scholar.google.com/scholar?q=%22author:Tao+Y%22>
     8. Tang X <https://scholar.google.com/scholar?q=%22author:Tang+X%22>
     9. Wedlich-Soldner R
        <https://scholar.google.com/scholar?q=%22author:Wedlich-Soldner+R%22>
    10. Balasubramanian MK
        <https://scholar.google.com/scholar?q=%22author:Balasubramanian+MK%22>
    (2012) Nonmedially assembled F-actin cables incorporate into the
    actomyosin ring in fission yeast
    <https://doi.org/10.1083/jcb.201209044> /The Journal of Cell
    Biology/ *199*:831–847.
    https://doi.org/10.1083/jcb.201209044
    <https://doi.org/10.1083/jcb.201209044>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/23185032>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Nonmedially+assembled+F-actin+cables+incorporate+into+the+actomyosin+ring+in+fission+yeast&author=Huang+J&author=Huang+Y&author%5B2%5D=Yu+H&author%5B3%5D=Subramanian+D&author%5B4%5D=Padmanabhan+A&author%5B5%5D=Thadani+R&author%5B6%5D=Tao+Y&author%5B7%5D=Tang+X&author%5B8%5D=Wedlich-Soldner+R&author%5B9%5D=Balasubramanian+MK&publication_year=2012&journal=The+Journal+of+Cell+Biology&volume=199&pages=pp.+831%E2%80%93847&pmid=23185032>
13.
     1. Isambert H
        <https://scholar.google.com/scholar?q=%22author:Isambert+H%22>
     2. Venier P
        <https://scholar.google.com/scholar?q=%22author:Venier+P%22>
     3. Maggs AC
        <https://scholar.google.com/scholar?q=%22author:Maggs+AC%22>
     4. Fattoum A
        <https://scholar.google.com/scholar?q=%22author:Fattoum+A%22>
     5. Kassab R
        <https://scholar.google.com/scholar?q=%22author:Kassab+R%22>
     6. Pantaloni D
        <https://scholar.google.com/scholar?q=%22author:Pantaloni+D%22>
     7. Carlier MF
        <https://scholar.google.com/scholar?q=%22author:Carlier+MF%22>
    (1995) Flexibility of actin filaments derived from thermal
    fluctuations. effect of bound nucleotide, Phalloidin, and muscle
    regulatory proteins <https://doi.org/10.1074/jbc.270.19.11437>
    /Journal of Biological Chemistry/ *270*:11437–11444.
    https://doi.org/10.1074/jbc.270.19.11437
    <https://doi.org/10.1074/jbc.270.19.11437>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/7744781>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Flexibility+of+actin+filaments+derived+from+thermal+fluctuations.+effect+of+bound+nucleotide%2C+Phalloidin%2C+and+muscle+regulatory+proteins&author=Isambert+H&author=Venier+P&author%5B2%5D=Maggs+AC&author%5B3%5D=Fattoum+A&author%5B4%5D=Kassab+R&author%5B5%5D=Pantaloni+D&author%5B6%5D=Carlier+MF&publication_year=1995&journal=Journal+of+Biological+Chemistry&volume=270&pages=pp.+11437%E2%80%9311444&pmid=7744781>
14.
     1. Johnson M
        <https://scholar.google.com/scholar?q=%22author:Johnson+M%22>
     2. East DA <https://scholar.google.com/scholar?q=%22author:East+DA%22>
     3. Mulvihill DP
        <https://scholar.google.com/scholar?q=%22author:Mulvihill+DP%22>
    (2014) Formins determine the functional properties of actin
    filaments in yeast <https://doi.org/10.1016/j.cub.2014.05.034>
    /Current Biology/ *24*:1525–1530.
    https://doi.org/10.1016/j.cub.2014.05.034
    <https://doi.org/10.1016/j.cub.2014.05.034>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/24954052>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Formins+determine+the+functional+properties+of+actin+filaments+in+yeast&author=Johnson+M&author=East+DA&author%5B2%5D=Mulvihill+DP&publication_year=2014&journal=Current+Biology&volume=24&pages=pp.+1525%E2%80%931530&pmid=24954052>
15.
     1. Kemp JP <https://scholar.google.com/scholar?q=%22author:Kemp+JP%22>
     2. Brieher WM
        <https://scholar.google.com/scholar?q=%22author:Brieher+WM%22>
    (2018) The actin filament bundling protein α-actinin-4 actually
    suppresses actin stress fibers by permitting actin turnover
    <https://doi.org/10.1074/jbc.RA118.004345> /Journal of Biological
    Chemistry/ *293*:14520–14533.
    https://doi.org/10.1074/jbc.RA118.004345
    <https://doi.org/10.1074/jbc.RA118.004345>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/30049798>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=The+actin+filament+bundling+protein+%CE%B1-actinin-4+actually+suppresses+actin+stress+fibers+by+permitting+actin+turnover&author=Kemp+JP&author=Brieher+WM&publication_year=2018&journal=Journal+of+Biological+Chemistry&volume=293&pages=pp.+14520%E2%80%9314533&pmid=30049798>
16.
     1. Kettenbach AN
        <https://scholar.google.com/scholar?q=%22author:Kettenbach+AN%22>
     2. Deng L <https://scholar.google.com/scholar?q=%22author:Deng+L%22>
     3. Wu Y <https://scholar.google.com/scholar?q=%22author:Wu+Y%22>
     4. Baldissard S
        <https://scholar.google.com/scholar?q=%22author:Baldissard+S%22>
     5. Adamo ME
        <https://scholar.google.com/scholar?q=%22author:Adamo+ME%22>
     6. Gerber SA
        <https://scholar.google.com/scholar?q=%22author:Gerber+SA%22>
     7. Moseley JB
        <https://scholar.google.com/scholar?q=%22author:Moseley+JB%22>
    (2015) Quantitative phosphoproteomics reveals pathways for
    coordination of cell growth and division by the conserved fission
    yeast kinase pom1 <https://doi.org/10.1074/mcp.M114.045245>
    /Molecular & Cellular Proteomics/ *14*:1275–1287.
    https://doi.org/10.1074/mcp.M114.045245
    <https://doi.org/10.1074/mcp.M114.045245>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/25720772>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Quantitative+phosphoproteomics+reveals+pathways+for+coordination+of+cell+growth+and+division+by+the+conserved+fission+yeast+kinase+pom1&author=Kettenbach+AN&author=Deng+L&author%5B2%5D=Wu+Y&author%5B3%5D=Baldissard+S&author%5B4%5D=Adamo+ME&author%5B5%5D=Gerber+SA&author%5B6%5D=Moseley+JB&publication_year=2015&journal=Molecular+%26+Cellular+Proteomics&volume=14&pages=pp.+1275%E2%80%931287&pmid=25720772>
17.
     1. Kovar DR
        <https://scholar.google.com/scholar?q=%22author:Kovar+DR%22>
     2. Sirotkin V
        <https://scholar.google.com/scholar?q=%22author:Sirotkin+V%22>
     3. Lord M <https://scholar.google.com/scholar?q=%22author:Lord+M%22>
    (2011) Three's company: the fission yeast actin cytoskeleton
    <https://doi.org/10.1016/j.tcb.2010.11.001> /Trends in Cell Biology/
    *21*:177–187.
    https://doi.org/10.1016/j.tcb.2010.11.001
    <https://doi.org/10.1016/j.tcb.2010.11.001>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/21145239>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Three%27s+company%3A+the+fission+yeast+actin+cytoskeleton&author=Kovar+DR&author=Sirotkin+V&author%5B2%5D=Lord+M&publication_year=2011&journal=Trends+in+Cell+Biology&volume=21&pages=pp.+177%E2%80%93187&pmid=21145239>
18.
     1. Laporte D
        <https://scholar.google.com/scholar?q=%22author:Laporte+D%22>
     2. Coffman VC
        <https://scholar.google.com/scholar?q=%22author:Coffman+VC%22>
     3. Lee IJ <https://scholar.google.com/scholar?q=%22author:Lee+IJ%22>
     4. Wu JQ <https://scholar.google.com/scholar?q=%22author:Wu+JQ%22>
    (2011) Assembly and architecture of precursor nodes during fission
    yeast cytokinesis <https://doi.org/10.1083/jcb.201008171> /The
    Journal of Cell Biology/ *192*:1005–1021.
    https://doi.org/10.1083/jcb.201008171
    <https://doi.org/10.1083/jcb.201008171>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/21422229>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Assembly+and+architecture+of+precursor+nodes+during+fission+yeast+cytokinesis&author=Laporte+D&author=Coffman+VC&author%5B2%5D=Lee+IJ&author%5B3%5D=Wu+JQ&publication_year=2011&journal=The+Journal+of+Cell+Biology&volume=192&pages=pp.+1005%E2%80%931021&pmid=21422229>
19.
     1. Li Y <https://scholar.google.com/scholar?q=%22author:Li+Y%22>
     2. Christensen JR
        <https://scholar.google.com/scholar?q=%22author:Christensen+JR%22>
     3. Homa KE <https://scholar.google.com/scholar?q=%22author:Homa+KE%22>
     4. Hocky GM
        <https://scholar.google.com/scholar?q=%22author:Hocky+GM%22>
     5. Fok A <https://scholar.google.com/scholar?q=%22author:Fok+A%22>
     6. Sees JA <https://scholar.google.com/scholar?q=%22author:Sees+JA%22>
     7. Voth GA <https://scholar.google.com/scholar?q=%22author:Voth+GA%22>
     8. Kovar DR
        <https://scholar.google.com/scholar?q=%22author:Kovar+DR%22>
    (2016) The F-actin bundler α-actinin Ain1 is tailored for ring
    assembly and constriction during cytokinesis in fission yeast
    <https://doi.org/10.1091/mbc.e16-01-0010> /Molecular Biology of the
    Cell/ *27*:1821–1833.
    https://doi.org/10.1091/mbc.e16-01-0010
    <https://doi.org/10.1091/mbc.e16-01-0010>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/27075176>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=The+F-actin+bundler+%CE%B1-actinin+Ain1+is+tailored+for+ring+assembly+and+constriction+during+cytokinesis+in+fission+yeast&author=Li+Y&author=Christensen+JR&author%5B2%5D=Homa+KE&author%5B3%5D=Hocky+GM&author%5B4%5D=Fok+A&author%5B5%5D=Sees+JA&author%5B6%5D=Voth+GA&author%5B7%5D=Kovar+DR&publication_year=2016&journal=Molecular+Biology+of+the+Cell&volume=27&pages=pp.+1821%E2%80%931833&pmid=27075176>
20.
     1. McGough A
        <https://scholar.google.com/scholar?q=%22author:McGough+A%22>
     2. Way M <https://scholar.google.com/scholar?q=%22author:Way+M%22>
     3. DeRosier D
        <https://scholar.google.com/scholar?q=%22author:DeRosier+D%22>
    (1994) Determination of the alpha-actinin-binding site on actin
    filaments by cryoelectron microscopy and image analysis
    <https://doi.org/10.1083/jcb.126.2.433> /The Journal of Cell
    Biology/ *126*:433–443.
    https://doi.org/10.1083/jcb.126.2.433
    <https://doi.org/10.1083/jcb.126.2.433>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/8034744>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Determination+of+the+alpha-actinin-binding+site+on+actin+filaments+by+cryoelectron+microscopy+and+image+analysis&author=McGough+A&author=Way+M&author%5B2%5D=DeRosier+D&publication_year=1994&journal=The+Journal+of+Cell+Biology&volume=126&pages=pp.+433%E2%80%93443&pmid=8034744>
21.
     1. Miao Y <https://scholar.google.com/scholar?q=%22author:Miao+Y%22>
     2. Han X <https://scholar.google.com/scholar?q=%22author:Han+X%22>
     3. Zheng L <https://scholar.google.com/scholar?q=%22author:Zheng+L%22>
     4. Xie Y <https://scholar.google.com/scholar?q=%22author:Xie+Y%22>
     5. Mu Y <https://scholar.google.com/scholar?q=%22author:Mu+Y%22>
     6. Yates JR
        <https://scholar.google.com/scholar?q=%22author:Yates+JR%22>
     7. Drubin DG
        <https://scholar.google.com/scholar?q=%22author:Drubin+DG%22>
    (2016) Fimbrin phosphorylation by metaphase Cdk1 regulates actin
    cable dynamics in budding yeast
    <https://doi.org/10.1038/ncomms11265> /Nature Communications/ *7*:11265.
    https://doi.org/10.1038/ncomms11265
    <https://doi.org/10.1038/ncomms11265>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/27068241>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Fimbrin+phosphorylation+by+metaphase+Cdk1+regulates+actin+cable+dynamics+in+budding+yeast&author=Miao+Y&author=Han+X&author%5B2%5D=Zheng+L&author%5B3%5D=Xie+Y&author%5B4%5D=Mu+Y&author%5B5%5D=Yates+JR&author%5B6%5D=Drubin+DG&publication_year=2016&journal=Nature+Communications&volume=7&pages=11265&pmid=27068241>
22.
     1. Michelot A
        <https://scholar.google.com/scholar?q=%22author:Michelot+A%22>
     2. Drubin DG
        <https://scholar.google.com/scholar?q=%22author:Drubin+DG%22>
    (2011) Building distinct actin filament networks in a common
    cytoplasm <https://doi.org/10.1016/j.cub.2011.06.019> /Current
    Biology/ *21*:R560–R569.
    https://doi.org/10.1016/j.cub.2011.06.019
    <https://doi.org/10.1016/j.cub.2011.06.019>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/21783039>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Building+distinct+actin+filament+networks+in+a+common+cytoplasm&author=Michelot+A&author=Drubin+DG&publication_year=2011&journal=Current+Biology&volume=21&pages=pp.+R560%E2%80%93R569&pmid=21783039>
23.
     1. Morita R
        <https://scholar.google.com/scholar?q=%22author:Morita+R%22>
     2. Takaine M
        <https://scholar.google.com/scholar?q=%22author:Takaine+M%22>
     3. Numata O
        <https://scholar.google.com/scholar?q=%22author:Numata+O%22>
     4. Nakano K
        <https://scholar.google.com/scholar?q=%22author:Nakano+K%22>
    (2017) Molecular dissection of the actin-binding ability of the
    fission yeast α-actinin, Ain1, in vitro and in vivo
    <https://doi.org/10.1093/jb/mvx008> /The Journal of Biochemistry/
    *162*:93–102.
    https://doi.org/10.1093/jb/mvx008 <https://doi.org/10.1093/jb/mvx008>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/28338873>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Molecular+dissection+of+the+actin-binding+ability+of+the+fission+yeast+%CE%B1-actinin%2C+Ain1%2C+in+vitro+and+in+vivo&author=Morita+R&author=Takaine+M&author%5B2%5D=Numata+O&author%5B3%5D=Nakano+K&publication_year=2017&journal=The+Journal+of+Biochemistry&volume=162&pages=pp.+93%E2%80%93102&pmid=28338873>
24.
     1. Nakano K
        <https://scholar.google.com/scholar?q=%22author:Nakano+K%22>
     2. Satoh K <https://scholar.google.com/scholar?q=%22author:Satoh+K%22>
     3. Morimatsu A
        <https://scholar.google.com/scholar?q=%22author:Morimatsu+A%22>
     4. Ohnuma M
        <https://scholar.google.com/scholar?q=%22author:Ohnuma+M%22>
     5. Mabuchi I
        <https://scholar.google.com/scholar?q=%22author:Mabuchi+I%22>
    (2001) Interactions among a fimbrin, a capping protein, and an
    actin-depolymerizing factor in organization of the fission yeast
    actin cytoskeleton <https://doi.org/10.1091/mbc.12.11.3515>
    /Molecular Biology of the Cell/ *12*:3515–3526.
    https://doi.org/10.1091/mbc.12.11.3515
    <https://doi.org/10.1091/mbc.12.11.3515>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/11694585>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Interactions+among+a+fimbrin%2C+a+capping+protein%2C+and+an+actin-depolymerizing+factor+in+organization+of+the+fission+yeast+actin+cytoskeleton&author=Nakano+K&author=Satoh+K&author%5B2%5D=Morimatsu+A&author%5B3%5D=Ohnuma+M&author%5B4%5D=Mabuchi+I&publication_year=2001&journal=Molecular+Biology+of+the+Cell&volume=12&pages=pp.+3515%E2%80%933526&pmid=11694585>
25.
     1. Nolen BJ
        <https://scholar.google.com/scholar?q=%22author:Nolen+BJ%22>
     2. Tomasevic N
        <https://scholar.google.com/scholar?q=%22author:Tomasevic+N%22>
     3. Russell A
        <https://scholar.google.com/scholar?q=%22author:Russell+A%22>
     4. Pierce DW
        <https://scholar.google.com/scholar?q=%22author:Pierce+DW%22>
     5. Jia Z <https://scholar.google.com/scholar?q=%22author:Jia+Z%22>
     6. McCormick CD
        <https://scholar.google.com/scholar?q=%22author:McCormick+CD%22>
     7. Hartman J
        <https://scholar.google.com/scholar?q=%22author:Hartman+J%22>
     8. Sakowicz R
        <https://scholar.google.com/scholar?q=%22author:Sakowicz+R%22>
     9. Pollard TD
        <https://scholar.google.com/scholar?q=%22author:Pollard+TD%22>
    (2009) Characterization of two classes of small molecule inhibitors
    of Arp2/3 complex <https://doi.org/10.1038/nature08231> /Nature/
    *460*:1031–1034.
    https://doi.org/10.1038/nature08231
    <https://doi.org/10.1038/nature08231>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/19648907>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Characterization+of+two+classes+of+small+molecule+inhibitors+of+Arp2%2F3+complex&author=Nolen+BJ&author=Tomasevic+N&author%5B2%5D=Russell+A&author%5B3%5D=Pierce+DW&author%5B4%5D=Jia+Z&author%5B5%5D=McCormick+CD&author%5B6%5D=Hartman+J&author%5B7%5D=Sakowicz+R&author%5B8%5D=Pollard+TD&publication_year=2009&journal=Nature&volume=460&pages=pp.+1031%E2%80%931034&pmid=19648907>
26.
     1. Oda T <https://scholar.google.com/scholar?q=%22author:Oda+T%22>
     2. Maéda Y
        <https://scholar.google.com/scholar?q=%22author:Ma%C3%A9da+Y%22>
    (2010) Multiple conformations of F-actin
    <https://doi.org/10.1016/j.str.2010.05.009> /Structure/ *18*:761–767.
    https://doi.org/10.1016/j.str.2010.05.009
    <https://doi.org/10.1016/j.str.2010.05.009>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/20637412>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Multiple+conformations+of+F-actin&author=Oda+T&author=Ma%C3%A9da+Y&publication_year=2010&journal=Structure&volume=18&pages=pp.+761%E2%80%93767&pmid=20637412>
27.
     1. Papp G <https://scholar.google.com/scholar?q=%22author:Papp+G%22>
     2. Bugyi B <https://scholar.google.com/scholar?q=%22author:Bugyi+B%22>
     3. Ujfalusi Z
        <https://scholar.google.com/scholar?q=%22author:Ujfalusi+Z%22>
     4. Barkó S
        <https://scholar.google.com/scholar?q=%22author:Bark%C3%B3+S%22>
     5. Hild G <https://scholar.google.com/scholar?q=%22author:Hild+G%22>
     6. Somogyi B
        <https://scholar.google.com/scholar?q=%22author:Somogyi+B%22>
     7. Nyitrai M
        <https://scholar.google.com/scholar?q=%22author:Nyitrai+M%22>
    (2006) Conformational changes in actin filaments induced by formin
    binding to the barbed end
    <https://doi.org/10.1529/biophysj.106.087775> /Biophysical Journal/
    *91*:2564–2572.
    https://doi.org/10.1529/biophysj.106.087775
    <https://doi.org/10.1529/biophysj.106.087775>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/16829561>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Conformational+changes+in+actin+filaments+induced+by+formin+binding+to+the+barbed+end&author=Papp+G&author=Bugyi+B&author%5B2%5D=Ujfalusi+Z&author%5B3%5D=Bark%C3%B3+S&author%5B4%5D=Hild+G&author%5B5%5D=Somogyi+B&author%5B6%5D=Nyitrai+M&publication_year=2006&journal=Biophysical+Journal&volume=91&pages=pp.+2564%E2%80%932572&pmid=16829561>
28.
     1. Risca VI
        <https://scholar.google.com/scholar?q=%22author:Risca+VI%22>
     2. Wang EB <https://scholar.google.com/scholar?q=%22author:Wang+EB%22>
     3. Chaudhuri O
        <https://scholar.google.com/scholar?q=%22author:Chaudhuri+O%22>
     4. Chia JJ <https://scholar.google.com/scholar?q=%22author:Chia+JJ%22>
     5. Geissler PL
        <https://scholar.google.com/scholar?q=%22author:Geissler+PL%22>
     6. Fletcher DA
        <https://scholar.google.com/scholar?q=%22author:Fletcher+DA%22>
    (2012) Actin filament curvature biases branching direction
    <https://doi.org/10.1073/pnas.1114292109> /PNAS/ *109*:2913–2918.
    https://doi.org/10.1073/pnas.1114292109
    <https://doi.org/10.1073/pnas.1114292109>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/22308368>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Actin+filament+curvature+biases+branching+direction&author=Risca+VI&author=Wang+EB&author%5B2%5D=Chaudhuri+O&author%5B3%5D=Chia+JJ&author%5B4%5D=Geissler+PL&author%5B5%5D=Fletcher+DA&publication_year=2012&journal=PNAS&volume=109&pages=pp.+2913%E2%80%932918&pmid=22308368>
29.
     1. Romet-Lemonne G
        <https://scholar.google.com/scholar?q=%22author:Romet-Lemonne+G%22>
     2. Jégou A
        <https://scholar.google.com/scholar?q=%22author:J%C3%A9gou+A%22>
    (2013) Mechanotransduction down to individual actin filaments
    <https://doi.org/10.1016/j.ejcb.2013.10.011> /European Journal of
    Cell Biology/ *92*:333–338.
    https://doi.org/10.1016/j.ejcb.2013.10.011
    <https://doi.org/10.1016/j.ejcb.2013.10.011>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/24252518>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Mechanotransduction+down+to+individual+actin+filaments&author=Romet-Lemonne+G&author=J%C3%A9gou+A&publication_year=2013&journal=European+Journal+of+Cell+Biology&volume=92&pages=pp.+333%E2%80%93338&pmid=24252518>
30.
     1. Schindelin J
        <https://scholar.google.com/scholar?q=%22author:Schindelin+J%22>
     2. Arganda-Carreras I
        <https://scholar.google.com/scholar?q=%22author:Arganda-Carreras+I%22>
     3. Frise E <https://scholar.google.com/scholar?q=%22author:Frise+E%22>
     4. Kaynig V
        <https://scholar.google.com/scholar?q=%22author:Kaynig+V%22>
     5. Longair M
        <https://scholar.google.com/scholar?q=%22author:Longair+M%22>
     6. Pietzsch T
        <https://scholar.google.com/scholar?q=%22author:Pietzsch+T%22>
     7. Preibisch S
        <https://scholar.google.com/scholar?q=%22author:Preibisch+S%22>
     8. Rueden C
        <https://scholar.google.com/scholar?q=%22author:Rueden+C%22>
     9. Saalfeld S
        <https://scholar.google.com/scholar?q=%22author:Saalfeld+S%22>
    10. Schmid B
        <https://scholar.google.com/scholar?q=%22author:Schmid+B%22>
    11. Tinevez JY
        <https://scholar.google.com/scholar?q=%22author:Tinevez+JY%22>
    12. White DJ
        <https://scholar.google.com/scholar?q=%22author:White+DJ%22>
    13. Hartenstein V
        <https://scholar.google.com/scholar?q=%22author:Hartenstein+V%22>
    14. Eliceiri K
        <https://scholar.google.com/scholar?q=%22author:Eliceiri+K%22>
    15. Tomancak P
        <https://scholar.google.com/scholar?q=%22author:Tomancak+P%22>
    16. Cardona A
        <https://scholar.google.com/scholar?q=%22author:Cardona+A%22>
    (2012) Fiji: an open-source platform for biological-image analysis
    <https://doi.org/10.1038/nmeth.2019> /Nature Methods/ *9*:676–682.
    https://doi.org/10.1038/nmeth.2019 <https://doi.org/10.1038/nmeth.2019>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/22743772>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Fiji%3A+an+open-source+platform+for+biological-image+analysis&author=Schindelin+J&author=Arganda-Carreras+I&author%5B2%5D=Frise+E&author%5B3%5D=Kaynig+V&author%5B4%5D=Longair+M&author%5B5%5D=Pietzsch+T&author%5B6%5D=Preibisch+S&author%5B7%5D=Rueden+C&author%5B8%5D=Saalfeld+S&author%5B9%5D=Schmid+B&author%5B10%5D=Tinevez+JY&author%5B11%5D=White+DJ&author%5B12%5D=Hartenstein+V&author%5B13%5D=Eliceiri+K&author%5B14%5D=Tomancak+P&author%5B15%5D=Cardona+A&publication_year=2012&journal=Nature+Methods&volume=9&pages=pp.+676%E2%80%93682&pmid=22743772>
31.
     1. Schneider CA
        <https://scholar.google.com/scholar?q=%22author:Schneider+CA%22>
     2. Rasband WS
        <https://scholar.google.com/scholar?q=%22author:Rasband+WS%22>
     3. Eliceiri KW
        <https://scholar.google.com/scholar?q=%22author:Eliceiri+KW%22>
    (2012) NIH image to ImageJ: 25 years of image analysis
    <https://doi.org/10.1038/nmeth.2089> /Nature Methods/ *9*:671–675.
    https://doi.org/10.1038/nmeth.2089 <https://doi.org/10.1038/nmeth.2089>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/22930834>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=NIH+image+to+ImageJ%3A+25+years+of+image+analysis&author=Schneider+CA&author=Rasband+WS&author%5B2%5D=Eliceiri+KW&publication_year=2012&journal=Nature+Methods&volume=9&pages=pp.+671%E2%80%93675&pmid=22930834>
32.
     1. Sirotkin V
        <https://scholar.google.com/scholar?q=%22author:Sirotkin+V%22>
     2. Berro J <https://scholar.google.com/scholar?q=%22author:Berro+J%22>
     3. Macmillan K
        <https://scholar.google.com/scholar?q=%22author:Macmillan+K%22>
     4. Zhao L <https://scholar.google.com/scholar?q=%22author:Zhao+L%22>
     5. Pollard TD
        <https://scholar.google.com/scholar?q=%22author:Pollard+TD%22>
    (2010) Quantitative analysis of the mechanism of endocytic actin
    patch assembly and disassembly in fission yeast
    <https://doi.org/10.1091/mbc.e10-02-0157> /Molecular Biology of the
    Cell/ *21*:2894–2904.
    https://doi.org/10.1091/mbc.e10-02-0157
    <https://doi.org/10.1091/mbc.e10-02-0157>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/20587778>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Quantitative+analysis+of+the+mechanism+of+endocytic+actin+patch+assembly+and+disassembly+in+fission+yeast&author=Sirotkin+V&author=Berro+J&author%5B2%5D=Macmillan+K&author%5B3%5D=Zhao+L&author%5B4%5D=Pollard+TD&publication_year=2010&journal=Molecular+Biology+of+the+Cell&volume=21&pages=pp.+2894%E2%80%932904&pmid=20587778>
33.
     1. Skau CT <https://scholar.google.com/scholar?q=%22author:Skau+CT%22>
     2. Courson DS
        <https://scholar.google.com/scholar?q=%22author:Courson+DS%22>
     3. Bestul AJ
        <https://scholar.google.com/scholar?q=%22author:Bestul+AJ%22>
     4. Winkelman JD
        <https://scholar.google.com/scholar?q=%22author:Winkelman+JD%22>
     5. Rock RS <https://scholar.google.com/scholar?q=%22author:Rock+RS%22>
     6. Sirotkin V
        <https://scholar.google.com/scholar?q=%22author:Sirotkin+V%22>
     7. Kovar DR
        <https://scholar.google.com/scholar?q=%22author:Kovar+DR%22>
    (2011) Actin filament bundling by fimbrin is important for
    endocytosis, Cytokinesis, and polarization in fission yeast
    <https://doi.org/10.1074/jbc.M111.239004> /Journal of Biological
    Chemistry/ *286*:26964–26977.
    https://doi.org/10.1074/jbc.M111.239004
    <https://doi.org/10.1074/jbc.M111.239004>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/21642440>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Actin+filament+bundling+by+fimbrin+is+important+for+endocytosis%2C+Cytokinesis%2C+and+polarization+in+fission+yeast&author=Skau+CT&author=Courson+DS&author%5B2%5D=Bestul+AJ&author%5B3%5D=Winkelman+JD&author%5B4%5D=Rock+RS&author%5B5%5D=Sirotkin+V&author%5B6%5D=Kovar+DR&publication_year=2011&journal=Journal+of+Biological+Chemistry&volume=286&pages=pp.+26964%E2%80%9326977&pmid=21642440>
34.
     1. Skau CT <https://scholar.google.com/scholar?q=%22author:Skau+CT%22>
     2. Kovar DR
        <https://scholar.google.com/scholar?q=%22author:Kovar+DR%22>
    (2010) Competition between fimbrin and tropomyosin regulates
    endocytosis and cytokinesis kinetics in fission yeast
    <https://doi.org/10.1016/j.cub.2010.06.020> /Current Biology/
    *20*:1415–1422.
    https://doi.org/10.1016/j.cub.2010.06.020
    <https://doi.org/10.1016/j.cub.2010.06.020>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Competition+between+fimbrin+and+tropomyosin+regulates+endocytosis+and+cytokinesis+kinetics+in+fission+yeast&author=Skau+CT&author=Kovar+DR&publication_year=2010&journal=Current+Biology&volume=20&pages=pp.+1415%E2%80%931422>
35.
     1. Skoumpla K
        <https://scholar.google.com/scholar?q=%22author:Skoumpla+K%22>
     2. Coulton AT
        <https://scholar.google.com/scholar?q=%22author:Coulton+AT%22>
     3. Lehman W
        <https://scholar.google.com/scholar?q=%22author:Lehman+W%22>
     4. Geeves MA
        <https://scholar.google.com/scholar?q=%22author:Geeves+MA%22>
     5. Mulvihill DP
        <https://scholar.google.com/scholar?q=%22author:Mulvihill+DP%22>
    (2007) Acetylation regulates tropomyosin function in the fission
    yeast /Schizosaccharomyces pombe/
    <https://doi.org/10.1242/jcs.001115> /Journal of Cell Science/
    *120*:1635–1645.
    https://doi.org/10.1242/jcs.001115 <https://doi.org/10.1242/jcs.001115>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/17452625>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Acetylation+regulates+tropomyosin+function+in+the+fission+yeast+Schizosaccharomyces+pombe&author=Skoumpla+K&author=Coulton+AT&author%5B2%5D=Lehman+W&author%5B3%5D=Geeves+MA&author%5B4%5D=Mulvihill+DP&publication_year=2007&journal=Journal+of+Cell+Science&volume=120&pages=pp.+1635%E2%80%931645&pmid=17452625>
36.
     1. Spudich JA
        <https://scholar.google.com/scholar?q=%22author:Spudich+JA%22>
     2. Watt S <https://scholar.google.com/scholar?q=%22author:Watt+S%22>
    (1971) The regulation of rabbit skeletal muscle contraction. I.
    biochemical studies of the interaction of the tropomyosin-troponin
    complex with actin and the proteolytic fragments of myosin
    /The Journal of Biological Chemistry/ *246*:4866–4871.
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/4254541>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=The+regulation+of+rabbit+skeletal+muscle+contraction.+I.+biochemical+studies+of+the+interaction+of+the+tropomyosin-troponin+complex+with+actin+and+the+proteolytic+fragments+of+myosin&author=Spudich+JA&author=Watt+S&publication_year=1971&journal=The+Journal+of+Biological+Chemistry&volume=246&pages=pp.+4866%E2%80%934871&pmid=4254541>
37.
     1. Swaffer MP
        <https://scholar.google.com/scholar?q=%22author:Swaffer+MP%22>
     2. Jones AW
        <https://scholar.google.com/scholar?q=%22author:Jones+AW%22>
     3. Flynn HR
        <https://scholar.google.com/scholar?q=%22author:Flynn+HR%22>
     4. Snijders AP
        <https://scholar.google.com/scholar?q=%22author:Snijders+AP%22>
     5. Nurse P <https://scholar.google.com/scholar?q=%22author:Nurse+P%22>
    (2018) Quantitative phosphoproteomics reveals the signaling dynamics
    of Cell-Cycle kinases in the fission yeast /Schizosaccharomyces
    pombe/ <https://doi.org/10.1016/j.celrep.2018.06.036> /Cell Reports/
    *24*:503–514.
    https://doi.org/10.1016/j.celrep.2018.06.036
    <https://doi.org/10.1016/j.celrep.2018.06.036>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/29996109>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Quantitative+phosphoproteomics+reveals+the+signaling+dynamics+of+Cell-Cycle+kinases+in+the+fission+yeast%C2%A0Schizosaccharomyces+pombe&author=Swaffer+MP&author=Jones+AW&author%5B2%5D=Flynn+HR&author%5B3%5D=Snijders+AP&author%5B4%5D=Nurse+P&publication_year=2018&journal=Cell+Reports&volume=24&pages=pp.+503%E2%80%93514&pmid=29996109>
38.
     1. Winkelman JD
        <https://scholar.google.com/scholar?q=%22author:Winkelman+JD%22>
     2. Suarez C
        <https://scholar.google.com/scholar?q=%22author:Suarez+C%22>
     3. Hocky GM
        <https://scholar.google.com/scholar?q=%22author:Hocky+GM%22>
     4. Harker AJ
        <https://scholar.google.com/scholar?q=%22author:Harker+AJ%22>
     5. Morganthaler AN
        <https://scholar.google.com/scholar?q=%22author:Morganthaler+AN%22>
     6. Christensen JR
        <https://scholar.google.com/scholar?q=%22author:Christensen+JR%22>
     7. Voth GA <https://scholar.google.com/scholar?q=%22author:Voth+GA%22>
     8. Bartles JR
        <https://scholar.google.com/scholar?q=%22author:Bartles+JR%22>
     9. Kovar DR
        <https://scholar.google.com/scholar?q=%22author:Kovar+DR%22>
    (2016) Fascin- and α-Actinin-Bundled networks contain intrinsic
    structural features that drive protein sorting
    <https://doi.org/10.1016/j.cub.2016.07.080> /Current Biology/
    *26*:2697–2706.
    https://doi.org/10.1016/j.cub.2016.07.080
    <https://doi.org/10.1016/j.cub.2016.07.080>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/27666967>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Fascin-+and+%CE%B1-Actinin-Bundled+networks+contain+intrinsic+structural+features+that+drive+protein+sorting&author=Winkelman+JD&author=Suarez+C&author%5B2%5D=Hocky+GM&author%5B3%5D=Harker+AJ&author%5B4%5D=Morganthaler+AN&author%5B5%5D=Christensen+JR&author%5B6%5D=Voth+GA&author%5B7%5D=Bartles+JR&author%5B8%5D=Kovar+DR&publication_year=2016&journal=Current+Biology&volume=26&pages=pp.+2697%E2%80%932706&pmid=27666967>
39.
     1. Wu JQ <https://scholar.google.com/scholar?q=%22author:Wu+JQ%22>
     2. Bähler J
        <https://scholar.google.com/scholar?q=%22author:B%C3%A4hler+J%22>
     3. Pringle JR
        <https://scholar.google.com/scholar?q=%22author:Pringle+JR%22>
    (2001) Roles of a fimbrin and an alpha-actinin-like protein in
    fission yeast cell polarization and cytokinesis
    <https://doi.org/10.1091/mbc.12.4.1061> /Molecular Biology of the
    Cell/ *12*:1061–1077.
    https://doi.org/10.1091/mbc.12.4.1061
    <https://doi.org/10.1091/mbc.12.4.1061>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/11294907>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Roles+of+a+fimbrin+and+an+alpha-actinin-like+protein+in+fission+yeast+cell+polarization+and+cytokinesis&author=Wu+JQ&author=B%C3%A4hler+J&author%5B2%5D=Pringle+JR&publication_year=2001&journal=Molecular+Biology+of+the+Cell&volume=12&pages=pp.+1061%E2%80%931077&pmid=11294907>
40.
     1. Wu JQ <https://scholar.google.com/scholar?q=%22author:Wu+JQ%22>
     2. Kuhn JR <https://scholar.google.com/scholar?q=%22author:Kuhn+JR%22>
     3. Kovar DR
        <https://scholar.google.com/scholar?q=%22author:Kovar+DR%22>
     4. Pollard TD
        <https://scholar.google.com/scholar?q=%22author:Pollard+TD%22>
    (2003) Spatial and temporal pathway for assembly and constriction of
    the contractile ring in fission yeast cytokinesis
    <https://doi.org/10.1016/S1534-5807(03)00324-1> /Developmental Cell/
    *5*:723–734.
    https://doi.org/10.1016/S1534-5807(03)00324-1
    <https://doi.org/10.1016/S1534-5807(03)00324-1>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/14602073>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Spatial+and+temporal+pathway+for+assembly+and+constriction+of+the+contractile+ring+in+fission+yeast+cytokinesis&author=Wu+JQ&author=Kuhn+JR&author%5B2%5D=Kovar+DR&author%5B3%5D=Pollard+TD&publication_year=2003&journal=Developmental+Cell&volume=5&pages=pp.+723%E2%80%93734&pmid=14602073>
41.
     1. Wu JQ <https://scholar.google.com/scholar?q=%22author:Wu+JQ%22>
     2. Pollard TD
        <https://scholar.google.com/scholar?q=%22author:Pollard+TD%22>
    (2005) Counting cytokinesis proteins globally and locally in fission
    yeast <https://doi.org/10.1126/science.1113230> /Science/ *310*:310–314.
    https://doi.org/10.1126/science.1113230
    <https://doi.org/10.1126/science.1113230>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/16224022>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=Counting+cytokinesis+proteins+globally+and+locally+in+fission+yeast&author=Wu+JQ&author=Pollard+TD&publication_year=2005&journal=Science&volume=310&pages=pp.+310%E2%80%93314&pmid=16224022>
42.
     1. Zimmermann D
        <https://scholar.google.com/scholar?q=%22author:Zimmermann+D%22>
     2. Morganthaler AN
        <https://scholar.google.com/scholar?q=%22author:Morganthaler+AN%22>
     3. Kovar DR
        <https://scholar.google.com/scholar?q=%22author:Kovar+DR%22>
     4. Suarez C
        <https://scholar.google.com/scholar?q=%22author:Suarez+C%22>
    (2016) In vitro biochemical characterization of cytokinesis
    Actin-Binding proteins
    <https://doi.org/10.1007/978-1-4939-3145-3_12> /Methods in Molecular
    Biology/ *1369*:151–179.
    https://doi.org/10.1007/978-1-4939-3145-3_12
    <https://doi.org/10.1007/978-1-4939-3145-3_12>
      * PubMed <https://www.ncbi.nlm.nih.gov/pubmed/26519312>
      * Google Scholar
        <https://scholar.google.com/scholar_lookup?title=In+vitro+biochemical+characterization+of+cytokinesis+Actin-Binding+proteins&author=Zimmermann+D&author=Morganthaler+AN&author%5B2%5D=Kovar+DR&author%5B3%5D=Suarez+C&publication_year=2016&journal=Methods+in+Molecular+Biology&volume=1369&pages=pp.+151%E2%80%93179&pmid=26519312>


    Decision letter

<#>

 1.
    Mohan K Balasubramanian
    Reviewing Editor; University of Warwick, United Kingdom
 2.
    Anna Akhmanova
    Senior Editor; Utrecht University, Netherlands

In the interests of transparency, eLife includes the editorial decision
letter and accompanying author responses. A lightly edited version of
the letter sent to the authors after peer review is shown, indicating
the most substantive concerns; minor comments are not usually included.

[Editors’ note: a previous version of this study was rejected after peer
review, but the authors submitted for reconsideration. The first
decision letter after peer review is shown below.]

Thank you for submitting your work entitled "Tropomyosin and a-actinin
cooperation inhibits fimbrin association with actin filament networks in
fission yeast" for consideration by /eLife/. Your article has been
reviewed by three peer reviewers, including Mohan Balasubramanian as the
Reviewing Editor, and the evaluation has been overseen by a Senior
Editor. The reviewers have opted to remain anonymous.

Our decision has been reached after consultation between the reviewers.
Based on these discussions and the individual reviews below, we regret
to inform you that your work will not be considered further for
publication in /eLife/.

The reviewers appreciate the novel approaches you are combining to
address how distinct actin cross-linkers localize to distinct cellular
locations and actin structures and how cdc8 and ain1 cooperate to
exclude fimbrin from actomyosin rings and therefore themselves occupying
the actomyosin ring. They find the conclusions to be of definite
interest. However, it was felt that this paper did not reach the same
standards set by the previous work. In light of this, we are unable to
consider the work further presently, since the conclusions may change
after the quantitations recommended by the reviewers.

However, if you feel you can address the issues raised satisfactorily
through additional experiments we will be happy to consider this story
as a new submission and will try to send it to the same three reviewers.

/Reviewer #1:/

Christensen et al. found that fission yeast α-actinin Ain1 competes with
fimbrin Fim1 and tropomyosin Cdc8 is important for the competition.
Using fission yeast genetics and single molecule experiments they make
the conclusion that α-actinin and cdc8 outcompete fimbrin in vivo and in
vitro. The results are quite robust and convincing in general and the in
vitro competition studies with multi-color TIRF microscope are
particularly interesting. However, I believe the manuscript needs
significant revisions before further consideration in /eLife/ Research
Advance. Despite this work being an advance it is the feeling of this
reviewer that the "advance" does not have the same high standards set by
the previous work, in terms of quantitation and discussion.

1) The authors do not provide any explanation of how CK666 treatment
causes Fim1 to move from cytoplasm to the AM ring while opposite effect
is observed for Ain1.

2) The authors claim that Cdc8 enhances Ain1 activity 10 fold but
provide only in vitro evidence with very questionable quantification
approach. Moreover authors do not show where and how much of Ain1 is
bound to F-actin or address why they can't label Ain1 directly. Finally,
no biochemical evidences provided to confirm Cdc8 mediated increase of
Ain1 bundling activity (e.g. pulldown assay).

3) Manuscript does not provide any information about the means of Cdc8
mediated augmentation of Ain1 bundling activity.

4) Figure 1 and later in vitro findings seem to contradict each other.
On one hand CK666 treated cells display replacement of Ain1 with Fim1 in
the ring while Cdc8 stays in the AM ring. On the other hand Fim1 fails
to decorate Ain1/Cdc8 decorated F-actin bundles in TIRF experiments. Why
is Fim1 able to displace Ain1 in vivo but not in vitro? Authors briefly
speculate that increase of Fim1 concentration in the cytoplasm might be
a factor but do not look into that in details.

5) Contractile ring fluorescence quantification in Fim1-GFP expressing cells

They quantified fluorescence of contractile rings but I could not
understand how can they measure the ring fluorescence in Fim1-GFP cells
treated with DMSO (Figure 1A). How can they differentiate the signals
from the patches and rings? They can show how did they draw the ROI for
this quantification.

6) Ain1-GFP patches: The results showing Ain1-GFP patches in fim1-1∆
cells (Figure 1F) are too preliminary to demonstrate their conclusions.
Could they express Ain1-GFP more by using stronger promoters (e.g.
Pnmt1, Padh1, Ptif51 and Pef1a)?

7) Expression levels of Ain1-GFP. The authors over-expressed Ain1-GFP
and its R216E mutant in some experiments (Figure 1E-G and Figure 2C).
Since they analysed localization of Ain1-GFP in different strains, they
should show the expression levels of Ain1-GFP are comparable in these
strains.

8) Cdc8-Cy5 and Cdc8-TMR: The author should use Cdc8-TMR for Figure 3E
to demonstrate TMR did not inhibit the competition between Cdc8 and
actin bundling proteins.

9) Affinity of Ain1 on F-actin filaments in the presence of Cdc8: The
author should show the k_off values of Ain1 on F-actin in the presence
of Cdc8 and Fim1 in the presence of Ain1 and Cdc8.

10) Can Ain1(R216E) release Fim1 from F-Actin in the absence of Cdc8?

The authors clearly demonstrate that Ain1 and Cdc8 outcompete Fim1 for
F-actin binding in vitro (Figure 5). They can also test whether Ain1
R216E mutation can bypath the requirement of Cdc8 to outcompete Fim1 in
vitro. If R216E can bypath the Cdc8 requirements, they can conclude that
this out-competition between Ain1 and Fim1 is simply due to increased
affinity of interaction between Ain1 and F-actin by Cdc8.

11) Genetic interaction between cdc8 and ain1: Related to 10. The author
should test whether expression of Ain1 R216E can rescue cdc8 mutant or not.

/Reviewer #2:/

In this manuscript, Christensen et al., analyze the mechanisms
regulating selective exclusion of Fim1 from the contractile ring. Using
broad design principles of yeast genetics, live cell fluorescent
imaging, and invitro TIRF microscopy, the authors propose that
contractile ring component Cdc8 enhances bundling activity of Ain1 and
together these actin binding components inhibit association of Fim1 to
the ring.

Fim1 localizes to actin patches in cells. This localization is altered
in presence of Arp2/3 inhibitor CK666, that disrupts actin patches,
resulting in increase in the soluble pool of Fim1 which now localizes to
the contractile ring. Fim1 recruitment to the ring displaces Ain1, which
seems to become more diffused. This phenotype of Ain1 can be rescued in
fim1-1Δ cells.

The invitro data in Figure 2, 3, 4 and 5 demonstrates a reduction in
binding of Fim1 on single and 2-filament actin bundles in presence of
Ain1 and Ain1 (R216E), suggesting competition between the two actin
partners. Fim1 binding to Actin can be further lowered in presence of
Cdc8. Based on this the authors conclude that Cdc8 and Ain1 cooperate to
displace Fim1 from contractile ring.

This paper is a continuation of an earlier article from the same author.
In all, the manuscript is well-written with appropriate conclusions. The
quality of data is of very good. However, their main finding that Cdc8
displaces Fim1 has been discussed in detail in their earlier paper (as
part of Figure 5). It would have made the manuscript more impactful if
the authors had probed possibilities of an upstream adaptor at the actin
patches that could be sequestering Fim1, and preventing its recruitment
to the ring in addition to the synergistic inhibition by Ain1 and Cdc8.
The paper needs more data if one needs to understand the mechanisms
regulating Fim1's selective exclusion from the contractile ring.

Fim1 binding to F actin is very stable unlike Ain1 which is an order of
magnitude weaker and highly dynamic. This would have implication in the
contractile ring dynamics and rates of contraction in conditions (both
genetic and pharmacologic) when Fim1 is localized to the contractile
ring. The authors either show or discuss such consequence.

/Reviewer #3:/

In this manuscript, the Kovar lab follows up on their previous paper
regarding how different actin-binding proteins are sorted to different
actin networks in vivo through competition for actin filament binding.
Here, they address how the very strong actin filament binding protein
Fim1 (fimbrin) does not localize to acto-myosin cytokinetic rings,
unlike the weaker binders Cdc8 (tropomyosin) and Ain1 (a-actinin). They
show that Ain1 and Cdc8 do not compete with each other for binding to
actin filaments, but they both compete with Fim1. Although Fim1
outcompetes each individual protein, the combination of Ain1 and Cdc8
outcompetes Fim1. A major strength of this work is showing this
competition both in vitro and in cells. It should be noted that the
concentrations used in vitro do not reflect the cellular concentrations
of these proteins in most cases. On balance, I found the manuscript an
important contribution to the field, and a nice extension of their
previous publication. I have listed several comments below that might be
addressed to strengthen some claims and data.

1) The authors propose that actin patches "sequester" Fim1 to prevent it
from outcompeting Cdc8 and Ain1 at contractile rings. The measured
cellular concentration of Fim1 is approximately 4 μM, of which 68% is
free in the cytoplasm (PMID: 20587778). This suggests that actin patch
disruption (e.g. Figure 1) does not free up a significant number of Fim1
molecules. Clearly, the presence of actin patches prevents Fim1
localization from the contractile ring, but strict competition for
binding to actin filaments might be less important than other regulatory
mechanisms.

2) Figure 5: the presence of Ain1 causes a 90% decrease in Fim1 levels
on actin filaments, but only 16% increase Cdc8 levels on actin
filaments. The difference in these two numbers leaves me wondering how
much Ain1 is on these filaments, which has implications for the
underlying mechanism. The authors could test the levels of
filament-bound Ain1 in this experiment. On one hand, there might be lots
of Ain1 that causes the drastic decrease in Fim1 due to simple
competition. On the other hand, low levels of Ain1 (with Cdc8) might
alter the structure of filaments and bundles in a manner that prevents
Fim1 binding.

3) The authors use off-rates to explain competition with Ain1. Cdc8 and
Ain1 cooperatively compete with Fim1, despite Ain1's fast off-rate for
naked filaments. Does Ain1 have a slower off-rate for Cdc8-bound filaments?

https://doi.org/10.7554/eLife.47279.022
<https://doi.org/10.7554/eLife.47279.022>


    Author response

<#>

[Editors’ note: the author responses to the first round of peer review
follow.]

    Reviewer #1:

    1) The authors do not provide any explanation of how CK666 treatment
    causes Fim1 to move from cytoplasm to the AM ring while opposite
    effect is observed for Ain1.

We have provided an explanation in the first Results section, which is
the basis for the remainder of the study. Increasing the concentration
of soluble Fim1 in the cell via CK666 treatment allows Fim1 to associate
with the contractile ring and outcompete its contractile ring competitor
Ain1.

    2) The authors claim that Cdc8 enhances Ain1 activity 10 fold but
    provide only in vitro evidence with very questionable quantification
    approach. Moreover authors do not show where and how much of Ain1 is
    bound to F-actin or address why they can't label Ain1 directly.
    Finally, no biochemical evidences provided to confirm Cdc8 mediated
    increase of Ain1 bundling activity (e.g. pulldown assay).

We do not agree with the reviewer that TIRF microscopy imaging of
Ain1-mediated F-actin bundling in the absence and presence of Cdc8 in
vitro is a “very questionable quantification approach”, which we have
used in previous studies (Li et al., 2016) and allows a direct
visualization of bundling.

However, we do agree with the reviewer that using multiple approaches
with further quantitative analysis and direct visualization of Ain1
would enhance the impact of our study. Therefore, we did everything
suggested by the reviewer, as described in detail below, which has made
the manuscript significantly more mechanistically insightful.

First, to provide biochemical evidence that Cdc8 increases the bundling
efficiency of Ain1, we performed new low speed sedimentation assays that
demonstrate how increasing concentrations of Cdc8 significantly enhance
Ain1-mediated bundling (new Figure 6A, B). We also performed new
low-speed sedimentation assays with Cdc8 and either Fim1, Ain1, or both
(new Figure 7). The following text has been added to reflect these
results: “Almost two-fold more Cdc8 pellets with Ain1-mediated F-actin
bundles than with Fim1-mediated bundles (Figure 7B). Additionally, in
the presence of both Fim1 and Ain1, an intermediate amount of Cdc8
pellets in the presence of both Fim1 and Ain1, suggesting that Ain1
allows Cdc8 to better associate with F-actin in the presence of Fim1.”

Second, we labeled Ain1 directly on cysteine residues and performed new
4-color TIRFM actin assembly assays to directly visualize the interplay
between Ain1, Cdc8, and Fim1 (see new Figure 8). We added the following
text describing these data: “To directly investigate the effect of Ain1
and Cdc8 cooperation on competition with Fim1, we performed four-color
TIRFM with fluorescently labeled ABPs and quantified Fim1 association
with F-actin in the presence of Cdc8 and/or Ain1. […] The synergy
between Cdc8 and Ain1 may explain why Fim1 only poorly associates with
contractile rings in fission yeast cells (Figure 8E).”

    3) Manuscript does not provide any information about the means of
    Cdc8 mediated augmentation of Ain1 bundling activity.

In the Discussion section of the original submission, we proposed two
possible mechanisms for Cdc8-mediated enhancement of Ain1 bundling
activity: either (A) Cdc8 is changing the binding dynamics of Ain1 on
F-actin bundles or (B) Cdc8 enhances the persistence length of F-actin,
indirectly facilitating enhanced bundle formation.

To test model (A), we conducted new rapid TIRFM imaging experiments of
sparsely-labeled Ain1 on uncoated versus Cdc8-coated F-actin (new Figure
6E-G). We discovered that “Three-fold more Ain1 binding events were
observed on Cdc8-coated F-actin compared to uncoated F-actin (Figure
6E-G), suggesting that Cdc8 enhances the binding of Ain1 to F-actin,
thereby increasing the F-actin bundling ability of Ain1.”

We cannot rule out that the persistence length of F-actin also plays a
role in the increased Ain1-mediated bundling seen in the presence of
Cdc8. Therefore, we also added the following text to the Discussion: “We
suspect that conformational changes in the actin filament may also
affect the observed increase of αactinin Ain1-mediated bundling in the
presence of tropomyosin Cdc8 (Figure 6A-D). […] Tropomyosin’s increase
of the persistence length of F-actin may have an additional effect on
F-actin bundling by promoting bundle stability and inter-filament contacts.”

    4) Figure 1 and later in vitro findings seem to contradict each
    other. On one hand CK666 treated cells display replacement of Ain1
    with Fim1 in the ring while Cdc8 stays in the AM ring. On the other
    hand Fim1 fails to decorate Ain1/Cdc8 decorated F-actin bundles in
    TIRF experiments. Why is Fim1 able to displace Ain1 in vivo but not
    in vitro? Authors briefly speculate that increase of Fim1
    concentration in the cytoplasm might be a factor but do not look
    into that in details.

Our new 4-color TIRFM actin assembly assays, suggested by Review #1,
reveal that Fim1 and Ain1 do compete for binding F-actin in vitro (new
Figure 8), consistent with in vivo results. The kymograph in Figure 8B
(bottom panel) shows that Ain1 and Fim1 associating with F-actin bundles
is mutually exclusively as they sort to non-overlapping domains on
F-actin bundles. This is consistent with our in vivo results in Figure 1
and Figure 1—figure supplement 3, as Ain1 is still present on the ring
after CK-666 treatment, but in significantly lower amounts.

    5) Contractile ring fluorescence quantification in Fim1-GFP
    expressing cells

    They quantified fluorescence of contractile rings but I could not
    understand how can they measure the ring fluorescence in Fim1-GFP
    cells treated with DMSO (Figure 1A). How can they differentiate the
    signals from the patches and rings? They can show how did they draw
    the ROI for this quantification.

We agree with the reviewer that the predominant association of Fim1 with
actin patches near the ring makes quantification of ring fluorescence
difficult. This is precisely why we had also looked at different
contractile ring stages (Figure 1—figure supplement 3), as there are
fewer actin patches at the contractile ring site in Stage 1 and 2 cells.
We have added a yellow box to Figure 1B to highlight the region measured
for contractile ring fluorescence intensity in Fim1GFP cells. We also
added the following text: “At all stages of contractile ring assembly
and constriction, Fim1 similarly localizes to the contractile ring and
displaces Ain1 following CK-666 treatment (Figure 1—figure supplement
3), though it is most prominent in stages with fully-developed
contractile rings (stages 2 and 3).” Furthermore, if anything, the
increase of Fim1 ring fluorescence in the presence of CK-666 is
underreported given that a significant amount of Fim1 ring fluorescence
in the absence of CK-666 is likely due to actin patches at the division
site.

We also provided an explanation of quantification in the “Contractile
ring fluorescence quantification” section of the Materials and methods:
“Contractile ring maturation was divided into three stages by measuring
the distance between spindle pole bodies (SPBs, visualized by
Sad1-tdTomato) and noting constriction of the contractile ring. […]
Maximum project were used for images in figures and sum projections were
used for quantification.”

    6) Ain1-GFP patches: The results showing Ain1-GFP patches in fim1-1∆
    cells (Figure 1F) are too preliminary to demonstrate their
    conclusions. Could they express Ain1-GFP more by using stronger
    promoters (e.g. Pnmt1, Padh1, Ptif51 and Pef1a)?

We disagree that the results showing 41X-Ain1-GFP in actin patches
(Figure 2 of the revised manuscript) are not conclusive, since Ain1-GFP
is clearly observed to colocalize with ArpC5mCherry over time. For
clarity, we added the following: “To increase the expression of
Ain1-GFP, we introduced an additional copy of Ain1-GFP at the leu1-32
locus under the medium-strength 41Xnmt promoter (Li et al., 2016).”
Furthermore, consistent with these findings, the less-dynamic mutant
41X-Ain1(R216E) localizes to actin patches even more strongly (Figure 4).

    7) Expression levels of Ain1-GFP. The authors over-expressed
    Ain1-GFP and its R216E mutant in some experiments (Figure 1E-G and
    Figure 2C). Since they analysed localization of Ain1-GFP in
    different strains, they should show the expression levels of
    Ain1-GFP are comparable in these strains.

We appreciate this important comment from the reviewer. We performed an
analysis of Ain1 expression in all strain backgrounds used (new Figure
2—figure supplement 1) and added the following text: “We observed that
while most Ain1-GFP constructs are expressed similarly to
endogenously-tagged Ain1, the amount of Ain1-GFP overexpressed under the
41Xnmt promoter is almost two-fold higher (Figure 2—figure supplement
1). […] Therefore, Fim1 and Ain1 appear to compete for association with
F-actin at both actin patches and the contractile ring.”

    8) Cdc8-Cy5 and Cdc8-TMR: The author should use Cdc8-TMR for Figure
    3E to demonstrate TMR did not inhibit the competition between Cdc8
    and actin bundling proteins.

Throughout the course of this study and previous work (Christensen et
al. 2017), we have found that the specific identity of the fluorophore
(TMR vs. Cy5) does not change the behavior or binding dynamics of Cdc8
on F-actin. Figure 5C (formerly Figure 3E) serves as a control to show
that Fim1 and Cdc8 compete for binding to F-actin, as we have shown
previously (the subject of Christensen et al., 2017). The true takeaway
from Figure 5 is that neither Ain1 (Figure 5A) nor Ain1(R216E) (Figure
5B) displace Cdc8. That Ain1 and Cdc8 cooperate, whereas Fim1 and Cdc8
compete for associating with F-actin, is also observed in new Figure 7
and new Figure 8.

    9) Affinity of Ain1 on F-actin filaments in the presence of Cdc8:
    The author should show the k_off values of Ain1 on F-actin in the
    presence of Cdc8 and Fim1 in the presence of Ain1 and Cdc8.

We appreciate the reviewer’s suggestions, and have therefore determined
whether the F-actin binding dynamics of Ain1 were altered in the
presence of Cdc8 with new experiments. We conducted rapid TIRFM of
sparsely-labeled Ain1 on uncoated versus Cdc8-coated F-actin (new Figure
6E-G). We found that “Three-fold more Ain1 binding events were observed
on Cdc8 coated F-actin compared to uncoated F-actin (Figure 6E-G),
suggesting that Cdc8 enhances the binding of Ain1 to F-actin, thereby
increasing the F-actin bundling ability of Ain1.” However, we were not
able to determine whether the 3-fold increase in observed Ain1
association events in the presence of Cdc8 are due to an increase in the
rate of Ain1 binding and/or a decrease in the rate of Ain1 dissociation.

    10) Can Ain1(R216E) release Fim1 from F-Actin in the absence of Cdc8?

    The authors clearly demonstrate that Ain1 and Cdc8 outcompete Fim1
    for F-actin binding in vitro (Figure 5). They can also test whether
    Ain1 R216E mutation can bypath the requirement of Cdc8 to outcompete
    Fim1 in vitro. If R216E can bypath the Cdc8 requirements, they can
    conclude that this out-competition between Ain1 and Fim1 is simply
    due to increased affinity of interaction between Ain1 and F-actin by
    Cdc8.

In Figure 3 (was Figure 2), we show that Ain1R216E does not outcompete
Fim1 more than WT Ain1.

    11) Genetic interaction between cdc8 and ain1: Related to 10. The
    author should test whether expression of Ain1 R216E can rescue cdc8
    mutant or not.

We did not do the suggested genetic experiment as we think this is very
unlikely as Cdc8 is involved with regulating the function of several
other essential ABPs, such as myosin-2 Myo2.

    Reviewer #2:

    […] This paper is a continuation of an earlier article from the same
    author. In all, the manuscript is well-written with appropriate
    conclusions. The quality of data is of very good. However, their
    main finding that Cdc8 displaces Fim1 has been discussed in detail
    in their earlier paper (as part of Figure 5). It would have made the
    manuscript more impactful if the authors had probed possibilities of
    an upstream adaptor at the actin patches that could be sequestering
    Fim1, and preventing its recruitment to the ring in addition to the
    synergistic inhibition by Ain1 and Cdc8. The paper needs more data
    if one needs to understand the mechanisms regulating Fim1's
    selective exclusion from the contractile ring.

    Fim1 binding to F actin is very stable unlike Ain1 which is an order
    of magnitude weaker and highly dynamic. This would have implication
    in the contractile ring dynamics and rates of contraction in
    conditions (both genetic and pharmacologic) when Fim1 is localized
    to the contractile ring. The authors either show or discuss such
    consequence.

We thank the reviewer for the favorable comments. However, we disagree
that the main point of this manuscript “has been discussed in detail in
their earlier paper…”. The main points of this important new manuscript
are that although Fim1 outcompetes Cdc8 in isolation (the main point of
our earlier /eLife/ paper), (1) the association of fimbrin Fim1 with
actin patches acts as a sink that contributes to preventing Fim1 from
associating with contractile rings more strongly, (2) the contractile
ring ABP a-actinin Ain1 is another important player in ABP sorting, (3)
fimbrin Fim1 also outcompetes Ain1 for binding actin filaments in
isolation, like it does tropomyosin Cdc8, but (4) tropomyosin Cdc8
enhances the F-actin bundling activity of a-actinin Ain1, and (5) the
combination of tropomyosin Cdc8 and a-actinin Ain1 cooperate to
successfully compete with fimbrin Fim1 for binding actin filaments,
further explaining why fimbrin Fim1 does not associate more strongly
with contractile rings in vivo.

Understanding how Fim1 might be sequestered in actin patches or
inactivated by posttranslational modifications is an ongoing topic of
study in our lab, but outside the scope of the current manuscript. We
suspect that several mechanisms, including post-translational
modifications, actin filament conformation, and the dynamics of ABPs,
contribute to Fim1’s preferential association with actin patches and
affect the competitive and cooperative interactions between ABPs.

Regarding the dynamics of bundling proteins, we thank the reviewer for
highlighting this important point and have added the following new
passage to the Discussion: “We discovered that the contractile ring ABP
α-actinin Ain1 competes with fimbrin Fim1, and that dynamic association
of both Ain1 and Fim1 with actin filaments affects their ability to
compete at different F-actin networks. […] Though our results suggest
that Ain1 is more dynamic on uncoated actin filaments, we showed that
wild-type Ain1 is still a dynamic bundler on Cdc8-coated actin filaments
and is likely capable of allowing contractile ring ABPs to remain
associated with and function optimally at the contractile ring.”

    Reviewer #3:

    […] [1) The authors propose that actin patches "sequester" Fim1 to
    prevent it from outcompeting Cdc8 and Ain1 at contractile rings. The
    measured cellular concentration of Fim1 is approximately 4 μM, of
    which 68% is free in the cytoplasm (PMID: 20587778). This suggests
    that actin patch disruption (e.g. Figure 1) does not free up a
    significant number of Fim1 molecules. Clearly, the presence of actin
    patches prevents Fim1 localization from the contractile ring, but
    strict competition for binding to actin filaments might be less
    important than other regulatory mechanisms.

We definitely agree with the reviewer about the importance of additional
regulatory mechanisms, and we have updated the Discussion to reflect
this. It is currently unknown how much of the free pool of Fim1 is
active. We have added a section discussing how posttranslational
modification of Fim1 could also contribute to regulation of its
activity: “in vitro, the combined efforts of α-actinin Ain1 and
tropomyosin Cdc8 prevent ~40% of fimbrin Fim1 association with F-actin
bundles. […] Fission yeast Fim1 might be similarly post-translationally
modified (Kettenbach et al., 2015; Swaffer et al., 2018), and therefore
a portion of the cytoplasmic Fim1 pool might be more or less active.”

    2) Figure 5: the presence of Ain1 causes a 90% decrease in Fim1
    levels on actin filaments, but only 16% increase Cdc8 levels on
    actin filaments. The difference in these two numbers leaves me
    wondering how much Ain1 is on these filaments, which has
    implications for the underlying mechanism. The authors could test
    the levels of filament-bound Ain1 in this experiment. On one hand,
    there might be lots of Ain1 that causes the drastic decrease in Fim1
    due to simple competition. On the other hand, low levels of Ain1
    (with Cdc8) might alter the structure of filaments and bundles in a
    manner that prevents Fim1 binding.

We agree completely with the reviewer that direct visualization of Ain1
would greatly increase the impact of our study. Therefore, we labeled
Ain1 directly on cysteine residues and performed new 4-color TIRFM actin
assembly assays to directly visualize the interplay between Ain1, Cdc8,
and Fim1 (see new Figure 8). We added the following text describing
these data: “To directly investigate the effect of Ain1 and Cdc8
cooperation on competition with Fim1, we performed four-color TIRFM with
fluorescently labeled ABPs and quantified Fim1 association with F-actin
in the presence of Cdc8 and/or Ain1. […] The synergy between Cdc8 and
Ain1 may explain why Fim1 only poorly associates with contractile rings
in fission yeast cells (Figure 8E).”

    3) The authors use off-rates to explain competition with Ain1. Cdc8
    and Ain1 cooperatively compete with Fim1, despite Ain1's fast
    off-rate for naked filaments. Does Ain1 have a slower off-rate for
    Cdc8-bound filaments?

Please see our response to reviewer 1, comment 9.

https://doi.org/10.7554/eLife.47279.023
<https://doi.org/10.7554/eLife.47279.023>


    Article and author information

<#>


      Author details

 1.


            Jenna R Christensen

    Department of Molecular Genetics and Cell Biology, The University of
    Chicago, Chicago, United States


    Contribution

    Conceptualization, Data curation, Formal analysis, Investigation,
    Writing—original draft, Writing—review and editing


    Contributed equally with

    Kaitlin E Homa


    Competing interests

    No competing interests declared

    ORCID icon "This ORCID iD identifies the author of this article:"
    0000-0003-0323-6169 <https://orcid.org/0000-0003-0323-6169>

 2.


            Kaitlin E Homa

    Department of Molecular Genetics and Cell Biology, The University of
    Chicago, Chicago, United States


    Contribution

    Conceptualization, Formal analysis, Investigation, Writing—original
    draft, Writing—review and editing


    Contributed equally with

    Jenna R Christensen


    Competing interests

    No competing interests declared

 3.


            Alisha N Morganthaler

    Department of Molecular Genetics and Cell Biology, The University of
    Chicago, Chicago, United States


    Contribution

    Formal analysis, Investigation


    Competing interests

    No competing interests declared

 4.


            Rachel R Brown

    Department of Molecular Genetics and Cell Biology, The University of
    Chicago, Chicago, United States


    Contribution

    Formal analysis, Investigation


    Competing interests

    No competing interests declared

 5.


            Cristian Suarez

    Department of Molecular Genetics and Cell Biology, The University of
    Chicago, Chicago, United States


    Contribution

    Formal analysis, Investigation


    Competing interests

    No competing interests declared

 6.


            Alyssa J Harker

    Department of Biochemistry and Molecular Biology, The University of
    Chicago, Chicago, United States


    Contribution

    Formal analysis, Investigation


    Competing interests

    No competing interests declared

 7.


            Meghan E O'Connell

    Department of Molecular Genetics and Cell Biology, The University of
    Chicago, Chicago, United States


    Contribution

    Formal analysis, Investigation


    Competing interests

    No competing interests declared

 8.


            David R Kovar

     1. Department of Molecular Genetics and Cell Biology, The
        University of Chicago, Chicago, United States
     2. Department of Biochemistry and Molecular Biology, The University
        of Chicago, Chicago, United States


    Contribution

    Conceptualization, Supervision, Funding acquisition, Writing—review
    and editing


    For correspondence

    drkovar@uchicago.edu <mailto:drkovar@uchicago.edu>


    Competing interests

    No competing interests declared

    ORCID icon "This ORCID iD identifies the author of this article:"
    0000-0002-5747-0949 <https://orcid.org/0000-0002-5747-0949>


      Funding


        National Science Foundation (Graduate Research Fellowship
        DGE-1144082)

  * Jenna Christensen
  * Alyssa J Harker


        National Institutes of Health (T32 GM0071832)

  * Jenna Christensen
  * Kaitlin E Homa
  * Meghan O'Connell


        National Science Foundation (Graduate Research Fellowship
        DGE-1746045)

  * Alyssa J Harker


        National Institutes of Health (IMSD R25GM109439)

  * Meghan O'Connell


        National Institutes of Health (R01 GM079265)

  * David R Kovar


        American Cancer Society (RSG-11-126-01-CSM)

  * David R Kovar

The funders had no role in study design, data collection and
interpretation, or the decision to submit the work for publication.


      Acknowledgements

This work was supported by NIH R01 GM079265 and ACS RSG-11-126-01-CSM
(to DRK), NIH MCB Training Grant T32 GM0071832 (to JRC, KEH, and MEO),
Initiative for Maximizing Student Development (IMSD) NIGMS R25GM109439
(to MEO) and NSF Graduate Student Fellowships DGE-1144082 (to JRC),
DGE-1144082 and DGE-1746045 (to AJH). Additional support was provided to
DRK by the University of Chicago MRSEC, funded by the NSF through grant
DMR-1420709. We thank Charlie Dulberger and Yujie Li for assistance with
Ain1 purification. We also thank Jonathan Winkelman and the Kovar lab
for helpful discussions.


      Senior Editor

 1. Anna Akhmanova, Utrecht University, Netherlands


      Reviewing Editor

 1. Mohan K Balasubramanian, University of Warwick, United Kingdom


      Version history

 1. Received: March 30, 2019
 2. Accepted: May 29, 2019
 3. Version of Record published: June 10, 2019 (version 1)
    <https://elifesciences.org/articles/47279>


      Copyright

© 2019, Christensen et al.

This article is distributed under the terms of the Creative Commons
Attribution License <http://creativecommons.org/licenses/by/4.0/>, which
permits unrestricted use and redistribution provided that the original
author and source are credited.


    Metrics

<#>

  *

        1,409 
    Page views

  *

        179 
    Downloads

  *

        14 
    Citations

Created with Highcharts 9.1.0(Monthly)Page
Views05101520253001/2302/2303/2304/2305/2306/2307/2308/2309/23

Toggle chartsNavigate left icon <#>Navigate right icon <#>

 1. Daily <#>
 2. Monthly <#>

Created with Highcharts
9.1.0Downloads(Monthly)00.511.522.501/2302/2303/2304/2305/2306/2307/2308/2309/23

Toggle chartsNavigate left icon <#>Navigate right icon <#>

 1. Daily <#>
 2. Monthly <#>

Article citation count generated by polling the highest count across the
following sources: Crossref <https://doi.org/10.7554/eLife.47279>,
PubMed Central <https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6557641/>,
Scopus.


    Download links


        Categories and tags

  * Research Article <https://elifesciences.org/articles/research-article>
  * Cell Biology <https://elifesciences.org/subjects/cell-biology>
  * endocytosis <https://elifesciences.org/search?for=endocytosis>
  * actin patch <https://elifesciences.org/search?for=actin%20patch>
  * cytokinesis <https://elifesciences.org/search?for=cytokinesis>
  * contractile ring
    <https://elifesciences.org/search?for=contractile%20ring>
  * Ain1 <https://elifesciences.org/search?for=Ain1>
  * Cdc8 <https://elifesciences.org/search?for=Cdc8>


        Research organism

  * /S. pombe/ <https://elifesciences.org/search?for=S.%20pombe>


      Further reading

 1.
     1. Cell Biology


      BATF relieves hepatic steatosis by inhibiting PD1 and promoting
      energy metabolism <https://elifesciences.org/articles/88521>

    Zhiwang Zhang, Qichao Liao ... Lei Zhou
    Research Article
    <https://elifesciences.org/articles/research-article> Sep 15, 2023

    The rising prevalence of nonalcoholic fatty liver disease (NAFLD)
    has become a global health threat that needs to be addressed
    urgently. Basic leucine zipper ATF-like transcription factor (BATF)
    is commonly thought to be involved in immunity, but its effect on
    lipid metabolism is not clear. Here, we investigated the function of
    BATF in hepatic lipid metabolism. BATF alleviated high-fat diet
    (HFD)-induced hepatic steatosis and inhibited elevated programmed
    cell death protein (PD)1 expression induced by HFD. A mechanistic
    study confirmed that BATF regulated fat accumulation by inhibiting
    PD1 expression and promoting energy metabolism. PD1 antibodies
    alleviated hepatic lipid deposition. In conclusion, we identified
    the regulatory role of BATF in hepatic lipid metabolism and that PD1
    is a target for alleviation of NAFLD. This study provides new
    insights into the relationship between BATF, PD1, and NAFLD.

 2.
     1. Biochemistry and Chemical Biology
     2. Cell Biology


      Divergent regulation of KCNQ1/E1 by targeted recruitment of
      protein kinase A to distinct sites on the channel complex
      <https://elifesciences.org/articles/83466>

    Xinle Zou, Sri Karthika Shanmugam ... Henry M Colecraft
    Research Article
    <https://elifesciences.org/articles/research-article> Updated Sep
    13, 2023

    The slow delayed rectifier potassium current, /I/_Ks , conducted
    through pore-forming Q1 and auxiliary E1 ion channel complexes is
    important for human cardiac action potential repolarization. During
    exercise or fright, /I/_Ks is up-regulated by protein kinase A
    (PKA)-mediated Q1 phosphorylation to maintain heart rhythm and
    optimum cardiac performance. Sympathetic up-regulation of /I/_Ks
    requires recruitment of PKA holoenzyme (two regulatory – RI or RII –
    and two catalytic Cα subunits) to Q1 C-terminus by an A kinase
    anchoring protein (AKAP9). Mutations in Q1 or AKAP9 that abolish
    their functional interaction result in long QT syndrome type 1 and
    11, respectively, which increases the risk of sudden cardiac death
    during exercise. Here, we investigated the utility of a targeted
    protein phosphorylation (TPP) approach to reconstitute PKA
    regulation of /I/_Ks in the absence of AKAP9. Targeted recruitment
    of endogenous Cα to E1-YFP using a GFP/YFP nanobody (nano) fused to
    RIIα enabled acute cAMP-mediated enhancement of /I/_Ks ,
    reconstituting physiological regulation of the channel complex. By
    contrast, nano-mediated tethering of RIIα or Cα to Q1-YFP
    constitutively inhibited /I/_Ks by retaining the channel
    intracellularly in the endoplasmic reticulum and Golgi. Proteomic
    analysis revealed that distinct phosphorylation sites are modified
    by Cα targeted to Q1-YFP compared to free Cα. Thus, functional
    outcomes of synthetically recruited PKA on /I/_Ks regulation is
    critically dependent on the site of recruitment within the channel
    complex. The results reveal insights into divergent regulation of
    /I/_Ks by phosphorylation across different spatial and time scales,
    and suggest a TPP approach to develop new drugs to prevent
    exercise-induced sudden cardiac death.

 3.
     1. Cell Biology
     2. Microbiology and Infectious Disease


      A CRISPR-based rapid DNA repositioning strategy and the early
      intranuclear life of HSV-1 <https://elifesciences.org/articles/85412>

    Juan Xiang, Zhaoyang Fan ... Pei Xu
    Research Article
    <https://elifesciences.org/articles/research-article> Sep 13, 2023

    The relative positions of viral DNA genomes to the host intranuclear
    environment play critical roles in determining virus fate. Recent
    advances in the application of chromosome conformation capture-based
    sequencing analysis (3C technologies) have revealed valuable aspects
    of the spatiotemporal interplay of viral genomes with host
    chromosomes. However, to elucidate the causal relationship between
    the subnuclear localization of viral genomes and the pathogenic
    outcome of an infection, manipulative tools are needed. Rapid
    repositioning of viral DNAs to specific subnuclear compartments amid
    infection is a powerful approach to synchronize and interrogate this
    dynamically changing process in space and time. Herein, we report an
    inducible CRISPR-based two-component platform that relocates
    extrachromosomal DNA pieces (5 kb to 170 kb) to the nuclear
    periphery in minutes (CRISPR-nuPin). Based on this strategy,
    investigations of herpes simplex virus 1 (HSV-1), a prototypical
    member of the human herpesvirus family, revealed unprecedently
    reported insights into the early intranuclear life of the pathogen:
    I) Viral genomes tethered to the nuclear periphery upon entry,
    compared with those freely infecting the nucleus, were wrapped
    around histones with increased suppressive modifications and
    subjected to stronger transcriptional silencing and prominent growth
    inhibition. II) Relocating HSV-1 genomes at 1 hour post infection
    significantly promoted the transcription of viral genes, termed an
    'Escaping' effect. III) Early accumulation of ICP0 was a sufficient
    but not necessary condition for 'Escaping'. IV) Subnuclear
    localization was only critical during early infection. Importantly,
    the CRISPR-nuPin tactic, in principle, is applicable to many other
    DNA viruses.


    Be the first to read new articles from eLife

Sign up for email alerts <https://elifesciences.org/content-alerts>
Privacy notice <https://elifesciences.org/privacy>

 1.
    Howard Hughes Medical Institute

 2.
    Wellcome Trust

 3.
    Max-Planck-Gesellschaft

 4.
    Knut and Alice Wallenberg Foundation

  * About <https://elifesciences.org/about>
  * Jobs <https://elifesciences.org/jobs>
  * Who we work with <https://elifesciences.org/who-we-work-with>
  * Alerts <https://elifesciences.org/alerts>
  * Contact <https://elifesciences.org/contact>
  * Terms and conditions <https://elifesciences.org/terms>
  * Privacy notice <https://elifesciences.org/privacy>
  * Inside eLife <https://elifesciences.org/inside-elife>
  * Monthly archive <https://elifesciences.org/archive/2023>
  * For the press <https://elifesciences.org/for-the-press>
  * Resources <https://elifesciences.org/resources>
  * XML and Data <http://developers.elifesciences.org/>

  * <https://twitter.com/elife>
  * <https://www.facebook.com/elifesciences>
  * <https://www.instagram.com/elifesciences/>
  * <https://www.youtube.com/channel/UCNEHLtAc_JPI84xW8V4XWyw>
  * <https://www.linkedin.com/company/elife-sciences-publications-ltd>
  * <https://fediscience.org/@eLife>

Find us on GitHub
<https://github.com/elifesciences>
Positive Planet - Certified Carbon Neutral
<https://positiveplanet.uk/company-dashboards/elife-sciences/>
eLife is a non-profit organisation inspired by research funders and led
by scientists. Our mission is to help scientists accelerate discovery by
operating a platform for research communication that encourages and
recognises the most responsible behaviours in science.

eLife Sciences Publications, Ltd is a limited liability non-profit
non-stock corporation incorporated in the State of Delaware, USA, with
company number 5030732, and is registered in the UK with company number
FC030576 and branch number BR015634 at the address:

eLife Sciences Publications, Ltd
Westbrook Centre, Milton Road
Cambridge CB4 1YG
UK

© 2023 eLife Sciences Publications Ltd. Subject to a Creative Commons
Attribution license <https://creativecommons.org/licenses/by/4.0/>,
except where otherwise noted. ISSN: 2050-084X

