Skip to main content <#main-content>

Advertisement

SpringerLink <https://link.springer.com/>
Log in
<https://link.springer.com/signup-login?previousUrl=https%3A%2F%2Flink.springer.com%2Farticle%2F10.1007%2Fs12551-018-0474-8>
Menu <javascript:;>
Find a journal <https://link.springer.com/journals/a/1> Publish with us
<https://www.springernature.com/gp/authors>
Search <javascript:;>
Cart<https://order.springer.com/public/cart>


Search

Search by keyword or author
Search


Navigation

  * Find a journal <https://link.springer.com/journals/a/1>
  * Publish with us <https://www.springernature.com/gp/authors>

 1. Home <https://link.springer.com/>
 2. Biophysical Reviews <https://link.springer.com/journal/12551>
 3. Article

Multiscale simulation of actin filaments and actin-associated proteins
Download PDF

<https://link.springer.com/content/pdf/10.1007/s12551-018-0474-8.pdf?pdf=button%20sticky>
Your content has downloaded


    Similar content being viewed by others

Slider with three content items shown per slide. Use the Previous and
Next buttons to navigate the slides or the slide controller buttons at
the end to navigate through each slide.


      Advances in the mechanical modeling of filamentous actin
      and its cross-linked networks on multiple scales
      <https://link.springer.com/10.1007/s10237-014-0578-4>

Article04 April 2014

Michael J. Unterberger & Gerhard A. Holzapfel


      Connecting Actin Polymer Dynamics Across Multiple Scales
      <https://link.springer.com/10.1007/978-3-030-57129-0_2>

Chapter© 2021


      Computer Simulations of Mechano-Chemical Networks Choreographing
      Actin Dynamics in Cell Motility
      <https://link.springer.com/10.1007/978-1-4614-2146-7_10>

Chapter© 2012


      From solution to surface to filament: actin flux into branched
      networks <https://link.springer.com/10.1007/s12551-018-0469-5>

Article23 November 2018

R. Dyche Mullins, Peter Bieling & Daniel A. Fletcher


      Mechanisms of formin-mediated actin assembly and dynamics
      <https://link.springer.com/10.1007/s12551-018-0468-6>

Article03 November 2018

Naomi Courtemanche


      Modeling spatiotemporally varying protein–protein interactions in
      CyLaKS, the Cytoskeleton Lattice-based Kinetic Simulator
      <https://link.springer.com/10.1140/epje/s10189-021-00097-8>

Article18 August 2021

Shane A. Fiorenza, Daniel G. Steckhahn & Meredith D. Betterton


      Multiscale impact of nucleotides and cations on the conformational
      equilibrium, elasticity and rheology of actin filaments and
      crosslinked networks
      <https://link.springer.com/10.1007/s10237-015-0660-6>

Article24 February 2015

Tamara Carla Bidone, Taeyoon Kim, … Roger D. Kamm


      Comparison of explicit and mean-field models of cytoskeletal
      filaments with crosslinking motors
      <https://link.springer.com/10.1140/epje/s10189-021-00042-9>

Article29 March 2021

Adam R. Lamson, Jeffrey M. Moore, … Meredith D. Betterton


      Numerical Treatment of the Filament-Based Lamellipodium Model
      (FBLM) <https://link.springer.com/10.1007/978-3-319-45833-5_7>

Chapter© 2017

Download PDF

<https://link.springer.com/content/pdf/10.1007/s12551-018-0474-8.pdf?pdf=button>

  * Review
  * Published: 31 October 2018 <#article-info>


  Multiscale simulation of actin filaments and actin-associated proteins

  * Fikret Aydin <#auth-Fikret-Aydin-Aff1>^1 <#Aff1> ^ na1 <#na1> ,
  * Harshwardhan H. Katkar <#auth-Harshwardhan_H_-Katkar-Aff1>^1 <#Aff1>
    ^ na1 <#na1> &
  * Gregory A. Voth<#auth-Gregory_A_-Voth-Aff1>  ORCID:
    orcid.org/0000-0002-3267-6748
    <http://orcid.org/0000-0002-3267-6748>^1 <#Aff1>  

/Biophysical Reviews/ <https://link.springer.com/journal/12551>
*volume 10*, pages 1521–1535 (2018)Cite this article <#citeas>

  *

    896 Accesses

  *

    4 Citations

  *

    Metrics details
    <https://link.springer.com/article/10.1007/s12551-018-0474-8/metrics>


    Abstract

Actin is an important cytoskeletal protein that serves as a building
block to form filament networks that span across the cell. These
networks are orchestrated by a myriad of other cytoskeletal entities
including the unbranched filament–forming protein formin and branched
network–forming protein complex Arp2/3. Computational models have been
able to provide insights into many important structural transitions that
are involved in forming these networks, and into the nature of
interactions essential for actin filament formation and for regulating
the behavior of actin-associated proteins. In this review, we summarize
a subset of such models that focus on the atomistic features and those
that can integrate atomistic features into a larger picture in a
multiscale fashion.

Access provided by Indian Institute of Science Education and Research
(IISER), Pune


      Working on a manuscript?

Avoid the common mistakes

<https://beta.springernature.com/pre-submission?journalId=12551>


    Introduction

Actin is a filament-forming protein that contributes to various parts of
the cellular cytoskeleton via complicated architectures. It co-exists as
a monomer as well as in parallel, anti-parallel, and branched filament
networks that assemble and disassemble to enable cell motility. The
dynamic nature of the actin cytoskeleton results from the ability of
actin to undergo transition between monomeric or globular (G-actin) and
filamentous (F-actin) states, and it is critical to many important
processes such as cell division (Pollard and Cooper 2009
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR62>).
The single-domain G-actin monomer consists of 375 residues, a nucleotide
in its binding pocket, and divalent cations (Fig. 1
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig1>a).
The nucleotide is predominantly adenosine triphosphate (ATP) in G-actin.
Polymerization of G-actin leads to the formation of actin filaments
consisting of F-actin subunits that have a well-characterized structure
well represented using a 13 subunit periodic repeat. The actin subunit
structure is commonly divided into four major subdomains (labeled as 1,
2, 3, and 4 in Fig. 2
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig2>d) and
is asymmetric. G-actin has a more twisted structure that is typically
characterized by a large dihedral angle between the subdomains 2-1-3-4
and a wide cleft represented by the distance between subdomains 2 and 4
(see Fig. 2
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig2>d for
labels). In F-actin, the structure is more flattened, characterized by a
smaller dihedral angle and a narrower cleft. The filament made of the
asymmetric F-actin subunits is polar in nature and consists of a
“faster-growing” barbed end and a “slower-growing” pointed end (Fig. 1
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig1>a).
Moreover, the bound nucleotide ATP hydrolyzes in F-actin to form an
intermediate structure consisting of adenosine diphosphate (ADP) with a
bound inorganic phosphate (P_i ) that is released at a much slower
timescale to leave behind purely ADP-bound subunits (Fig. 1
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig1>a).
Due to the difference in growth rate of actin filaments at the two ends
and the ATP hydrolysis, measuring growth kinetics of the filament is
challenging. Seminal work by Pollard and co-workers established the
macroscopic rate constants for the polymerization of ATP-, ADP-P_i -,
and ADP-bound actin at the barbed and pointed ends of actin filaments
and the rate constant for ATP hydrolysis and phosphate release (Fujiwara
et al. 2007
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR24>;
Pollard and Weeds 1984
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR60>;
Pollard 1986
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR61>).

*Fig. 1

*

figure 1
<https://link.springer.com/article/10.1007/s12551-018-0474-8/figures/1>

Actin filaments and associated proteins. *a* A single actin filament and
a magnified view of a section of the filament. The nucleotide in the
binding pocket undergoes ATP hydrolysis and P_i release. In addition to
the nucleotide-binding pocket, divalent cation binds at other sites to
assist polymerization and provide stiffness to the filament structure.
*b* Formin-mediated growth of a straight actin filament. *c* The Arp2/3
complex binds to the side of a mother filament and grows a daughter
filament, leading to a branched filament network

Full size image

<https://link.springer.com/article/10.1007/s12551-018-0474-8/figures/1>

*Fig. 2

*

figure 2
<https://link.springer.com/article/10.1007/s12551-018-0474-8/figures/2>

Computational modeling of actin. *a* Water crystals near the Mg^+2 in
G-actin versus F-actin observed in AA simulations (reproduced from
Saunders and Voth (2011
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR68>)). *b* Water wires observed in the metadynamics simulations and water wire length distributions in metadynamics simulations (reproduced from McCullagh et al. (2014 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR41>)). *c* DB-loop conformations observed in AA simulations, without and with additional divalent ions (reproduced from Hocky et al. (2016 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR29>)). *d* Intuitive 12-sites-per-subunit CG model of actin, shown in a subunit and a section of the filament. *e* Cooperativity in phosphate release rate (reproduced from Katkar et al. (2018 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR36>))

Full size image

<https://link.springer.com/article/10.1007/s12551-018-0474-8/figures/2>

The cell utilizes different active mechanisms to regulate actin filament
formation within the cytoskeletal network. Such mechanisms usually
involve regulation of the complex interactions between actin filaments
and other cytoskeletal proteins (Campellone and Welch 2010
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR9>).
Thus, any imbalance in these delicate regulatory mechanisms such as a
mutation disrupting a functional component of the system can result in
malfunction of cells and occurrence of serious diseases. A central part
of this regulatory system is composed of formin proteins and the
actin-related protein (Arp) 2/3 complex. The hydrolysis of ATP has
implications on binding kinetics of actin-associated proteins, including
the proteins formin and Arp2/3 that play an important role in nucleation
and growth of unbranched and branched actin filament networks (Fig. 1
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig1>b, c).
Breakthrough work from the Pollard lab, including the discovery of the
Arp2/3 complex and the gating mechanism in formins, has steered our
understanding of how these proteins polymerize actin in different
architectures (Espinoza-Sanchez et al. 2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR18>;
Machesky et al. 1994
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR40>;
Paul and Pollard 2009b
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR53>).

Computer simulations have widely been used to study various aspects of
actin filaments ranging from macroscopic properties of the actin network
(Bidone et al. 2015
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR5>;
Popov et al. 2016
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR63>)
through single-filament properties (Baker and Voth 2013
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR4>;
Chu and Voth 2005
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR12>;
Chu and Voth 2006
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR13>;
Fan et al. 2013
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR19>;
Fan et al. 2012
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR20>;
Hocky et al. 2016
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR29>;
Lyman et al. 2008
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR39>;
Pfaendtner et al. 2010a
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR55>;
Pfaendtner et al. 2010b
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR56>;
Pfaendtner et al. 2012
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR57>;
Saunders and Voth 2012a
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR69>;
Saunders and Voth 2012b
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR70>;
Schramm et al. 2017
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR72>;
Yogurtcu et al. 2012
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR83>)
and properties of the single subunit (Chu and Voth 2005
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR12>;
Mehrafrooz and Shamloo 2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR42>;
Pfaendtner et al. 2009
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR54>;
Saunders et al. 2014
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR67>;
Saunders and Voth 2011
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR68>)
to the hydrolysis of the bound nucleotide (Akola and Jones 2006
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR1>;
Freedman et al. 2012
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR21>;
McCullagh et al. 2014
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR41>;
Sun et al. 2017
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR76>).
At the highest resolution, ab initio molecular dynamics (AIMD), density
functional theory, or quantum mechanics/molecular mechanics (QM/MM)
simulations allow one to simulate the reactive nature of chemical bonds
in ATP and surrounding species using electronic structure theory and the
rest of the system using molecular mechanics to make the simulations
computationally more efficient. The free energy landscape of the ATP
hydrolysis reaction has been successfully studied using QM/MM
simulations (McCullagh et al. 2014
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR41>;
Sun et al. 2017
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR76>).
At the next level of “resolution” are the classical all-atom (AA)
molecular dynamics (MD) simulations, where atoms are represented by
particles whose motions are governed by Newton’s second law and are
typically simulated using a general empirical (and non-reactive)
force-field such as CHARMM. These AA simulations have been used to
simulate filaments of actin and have been able to provide valuable
insights pertaining to the characteristics of single filaments,
including their mechanical properties and the structural changes
associated with polymerization (Baker and Voth 2013
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR4>;
Chu and Voth 2005
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR12>;
Chu and Voth 2006
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR13>;
Fan et al. 2013
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR19>;
Fan et al. 2012
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR20>;
Hocky et al. 2016
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR29>;
Lyman et al. 2008
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR39>;
Pfaendtner et al. 2010a
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR55>;
Pfaendtner et al. 2010b
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR56>;
Pfaendtner et al. 2012
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR57>;
Saunders and Voth 2012a
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR69>;
Saunders and Voth 2012b
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR70>;
Schramm et al. 2017
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR72>).
However, AA simulations are typically limited to tens of subunits long
filaments. Coarse-grained (CG) models have been widely used to push
beyond the limits of AA simulations in terms of the timescales and
length scales that they can explore, by an extent that depends on the
level of coarse-graining (Saunders and Voth 2013
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR71>).

Top-down CG models, parameterized to reproduce macroscopic properties of
the system, have been widely used to study actin (Caby et al. 2012
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR8>;
Freedman et al. 2017
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR22>;
Guo et al. 2011
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR28>;
Popov et al. 2016
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR63>).
Particle-based CG models, especially constructed using a bottom-up
technique using underlying fine resolution simulations (e.g., AA),
reproduce the collective behavior of the atomistic system with a reduced
number of particles. Additionally, another class of models at a similar
resolution is Markov state models, in which a state-based representation
of the system is used. Such kinetic models that look at G-actin and
F-actin with different states for each nucleotide have been widely used
to study polymerization and ATP hydrolysis in actin filaments (Ranjith
et al. 2010
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR64>;
Stukalin and Kolomeisky 2006
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR75>;
Vavylonis et al. 2005
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR78>).
In this review, we focus on QM/MM and AA simulations of actin filaments,
formins, and Arp2/3 complex and briefly talk about bottom-up CG models
of these proteins. We specifically highlight computational studies from
the Voth group that were highly motivated by the pioneering work of Tom
Pollard, many of which were a direct result of either formal
collaborations or informal interactions with him.


    Actin

Many models of actin filament structure have been proposed in the
literature (Galkin et al. 2011
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR25>;
Kabsch et al. 1990
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR33>;
Murakami et al. 2010
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR44>;
Narita et al. 2011
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR45>;
Oda et al. 2009
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR46>).
Although significant efforts were dedicated to obtain improved
structures and to gain insights of the effect of states of the bound
nucleotide (e.g., Galkin et al. (2011
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR25>)), no high-resolution structure of either ATP- or ADP-P_i -bound actin filaments existed until recently. High-resolution structures of two ATP analogs were almost simultaneously solved using cryogenic electron microscopy (cryo-EM) in two recent independent studies, including one from the Pollard lab (Chou and Pollard 2018 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR11>; Merino et al. 2018 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR43>). In addition, the Pollard lab was successful in obtaining the only high-resolution structure of ADP-P_i -bound actin filament available today (Chou and Pollard 2018 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR11>).

Prior to the availability of these structures from cryo-EM, Pollard
helped to guide the efforts of us and others to use rigorous computer
simulations to study the structural differences of actin filaments with
different bound nucleotides. A typical method of constructing actin
filament for a given nucleotide state in computational simulations was
to superpose the ATP- (Graceffa and Dominguez 2003
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR27>)
or ADP (Otterbein et al. 2001
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR49>)-bound G-actin structure in the filament structure model (e.g., Fujii et al. (2010 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR23>), Kabsch et al. (1990 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR33>), and Oda et al. (2009 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR46>)), or for the nucleotide state ADP-P_i , by replacing P_γ from ATP (see Fig. 1 <https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig1>a) with P_i with AA simulations to further refine the resulting structure (Chu and Voth 2005 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR12>; Pfaendtner et al. 2009 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR54>; Pfaendtner et al. 2010b <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR56>). The effect of nucleotide state on structure and dynamics of actin subunits in G-actin and in F-actin was established using AA simulations accelerated with metadynamics in Pfaendtner et al. (2009 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR54>). The DNase-binding loop (DB-loop, see Fig. 1 <https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig1>a) in actin monomers as well as in F-actin has been shown to exhibit multiple conformations (Otterbein et al. 2001 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR49>; Rould et al. 2006 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR66>) and was studied using computer simulations in a series of collaborative work involving Thomas Pollard (Pfaendtner et al. 2009 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR54>; Pfaendtner et al. 2010b <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR56>). AA metadynamics simulations of G-actin revealed that DB-loop can coexist in unfolded and folded conformations in ADP-bound G-actin. Further, it was discovered that the lowest free energy corresponds to an unfolded DB-loop in ATP F-actin filaments, while in ADP F-actin, the folded DB-loop is stable and that folding of the DB-loop results into a softer filament with a lower persistence length (Hocky et al. 2016 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR29>; Pfaendtner et al. 2009 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR54>; Pfaendtner et al. 2010b <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR56>). Key amino acid residues including Met44 involved in the folding of DB-loop were identified through these simulations. AA MD simulations of the Oda filament model (Oda et al. 2009 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR46>) were found to be more stable compared to the Holmes filament model (Kabsch et al. 1990 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR33>) based on the root-mean-square fluctuations in these simulations relative initial structures, and CG analysis of these simulations demonstrated the effect of DB-loop conformations and the bound nucleotide on the dihedral angle distributions. Motivated by these results, extensive self-learning adaptive umbrella-sampling simulations were performed on single-actin subunits starting from the G-actin and F-actin conformations, using the dihedral angle and cleft width as collective variables (Saunders et al. 2014 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR67>). These simulations were not able to explore the full structural transformation between G-actin and F-actin structure of the subunit. Yet, several important aspects of the transformation were revealed. Based on the area of configurational space explored in these simulations, ADP-bound actin was found to be conformationally more mobile than ATP-bound actin. AA simulations of ADP-bound actin filaments were also used to refine two other binding sites—polymerization and stiffness sites—for divalent cations in addition to the nucleotide-binding pocket in the subunits and study their effects on the structure of subunits and properties of the filament (Hocky et al. 2016 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR29>) (see Fig. 1 <https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig1>a for locations). The DB-loop becomes relatively less flexible in the presence of a divalent ion at the stiffness site between the DB-loop of a given subunit and the binding cleft of the target interacting subunit (representative conformations are shown in Fig. 2 <https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig2>c). Persistence length and torsional rigidity of actin filaments increase due to the presence of the additional cations (Hocky et al. 2016 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR29>; Kang et al. 2013 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR34>; Kang et al. 2012 <https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR35>).


      Coarse-grained modeling of actin

Simplified mechanistic models, typically parameterized using a top-down
approach in which model parameters are tuned to obtain certain target
filament properties, have been successfully used to study actin
filaments and their mechanical properties (Bidone et al. 2015
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR5>;
Yogurtcu et al. 2012
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR83>).
Such models reduce large-scale complex process into simpler
phenomenology, but typically have somewhat limited predictive power and
can potentially lack a rigorous justification for the choice of
parameters involved. On the other hand, “bottom-up” CG models of actin
filaments with a CG force-field parameterized using reference AA
simulations, such as the hetero-elastic network model (hENM), have been
found to be adequate in studying actin filaments in terms of their
structure and mechanical properties (Chu and Voth 2005
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR12>;
Chu and Voth 2006
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR13>;
Deriu et al. 2011
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR17>;
Katkar et al. 2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR36>;
Lyman et al. 2008
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR39>;
Saunders and Voth 2012b
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR70>;
Schramm et al. 2017
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR72>;
Voth 2017
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR80>;
Zhang et al. 2008
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR84>).
We limit this review below to such bottom-up CG models.

Bottom-up coarse-graining of actin involves two steps. The first step is
to define a CG map that translates the AA structure into CG sites. The
map can either be defined using biological intuition or knowledge of the
structure of the protein, giving rise to the /intuitive/ map, or can be
obtained using information from the dynamics observed in the AA
trajectory. An example of the latter approach is the essential dynamics
coarse-graining (ED-CG) technique (Zhang et al. 2008
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR84>).
The idea behind ED-CG is to find the optimum mapping by variationally
minimizing a mathematical residual, defined as the total difference
between the displacement of pairs of CG beads averaged across AA data,
with the idea that if the two CG sites move in a correlated fashion in
the AA data, their contribution to the residual should be small.
Numerical implementation of the minimization algorithm involves a
combination of simulated annealing to obtain a global minimum of the
residual, and a steepest descend algorithm to find the accurate local
minimum. The mapping is constrained to the primary sequence of actin to
limit the search space, with the belief that preserving the primary
sequence can allow using the same mapping even with large conformational
changes in the protein. (The latest iteration of this code is available
for public use at https://github.com/uchicago-voth/CGMGC.p
<https://github.com/uchicago-voth/CGMGC.p>). Once the map is defined,
the second step consists of parameterizing the force-field that governs
the interactions between the CG sites. In the hENM (Lyman et al. 2008
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR39>),
the interactions between CG sites are assumed to be pair-wise and
further approximated as harmonic potentials. The harmonic spring
parameters—the spring constant and the equilibrium spring length—are
optimized to reproduce the mean square fluctuations of CG sites between
the mapped AA and the CG simulations.

A number of CG actin filament models with different levels of
coarse-graining based on either ED-CG or intuitive mapping and hENM have
been developed. The highest level of coarse-graining in actin filaments
in this class of models is the four-sites-per-subunit model. The
intuitive CG map at this resolution is readily obtained such that each
subdomain in actin is represented by one CG site (Kabsch et al. 1990
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR33>).
A specific force-field consisting of three bonds between the CG sites
representing the connectivity between the four subdomains along with
additional angle and dihedral potentials defining the opening and
closing of the nucleotide pocket in actin subunits can be parameterized
using the AA simulations. CG modeling at this resolution can help gain
insights into the subdomain level changes in the average structure of
actin and more importantly, into how “stiff” or “loose” these subdomains
are relative to each other, through the dynamics governing their
collective motions. Such a CG model, along with the underlying AA
simulations, was used to discover that the DB-loop conformation
undergoes a loop-to-helix transformation as a result of ATP hydrolysis
in actin filament. The resulting CG model parameters also provided
insights into key structural differences such as the softened dihedral
angle in ADP-bound G-actin versus an ATP-bound G-actin (Chu and Voth
2005
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR12>).
Long CG actin filaments can be constructed by replicating the mapping
from short actin filaments in the AA simulations. Certain choices can be
made to keep the CG force-field local. However, it is essential to
include interactions between CG sites that extend beyond the nearest
neighboring actin subunits in order to reproduce the correct mechanical
properties of the filaments (Chu and Voth 2006
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR13>).

A more detailed CG model of actin consisting of 12-sites-per-subunit has
also been developed (Saunders and Voth 2012b
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR70>)
(Fig. 2
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig2>d).
The novel CG map used in this model consists of four CG sites dedicated
to the four major subdomains of actin, but also used seven other minor
sites that represent various smaller structured domains in the subunit
and a dedicated CG site for the bound nucleotide. Including these minor
sites enables studying the relative orientation of the major subdomains
in the subunit and across subunits in the actin filament. A systematic
comparison of various models for the structure of actin filaments (Oda
(Oda et al. 2009
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR46>),
Namba (Fujii et al. 2010
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR23>),
and Holmes (Kabsch et al. 1990
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR33>)
models) was performed using a CG model at this resolution to study the
dynamics implied by each of these filament models in molecular
simulations. This analysis identified the most accurate structure based
on its stability in molecular simulations (Oda model). The key
structural transformation between G-actin and F-actin subunit in the Oda
model was identified to be the rotation of subdomain 2, and the
positioning of the DB-loop within the subunit and across subunits in the
filament was also identified to be important. The CG model further
showed that largest change is observed in the relative position of
subdomains 2 and 4 in actin, but other minor sites also rearrange
(Saunders et al. 2014
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR67>).
A distinction between the flattening of ATP- and ADP-bound actin
monomers was later discovered using this model. In presence of the
nucleotide ATP, CG sites 1 and 2 shift and rotate relative to each
other, and CG sites 3 and 1 rotate in order to flatten the structure. In
presence of the nucleotide ADP, the flattening is characterized by a
shift and rotation in CG sites 3 and 4 instead. Thus, in ATP bound
subunits, flattening is coupled with twisting of the nucleotide-binding
cleft, while in ADP reorientation of subdomain 2 results into
flattening. A recent improvement of this CG model parameterized for
ATP-, ADP-P_i -, and ADP-bound actin filaments was used as an input for
the ultra-coarse-graining technique (Dama et al. 2017
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR14>;
Dama et al. 2013
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR15>;
Davtyan et al. 2014
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR16>)
to study the mechanism of ATP hydrolysis in actin filaments (Katkar et
al. 2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR36>).

In contrast to the intuitive CG maps used in the above models, a
systematic ED-CG map for a given CG resolution can be obtained that best
represents collective motions in AA simulations. The ED-CG map with
four-sites-per-subunit was shown to preserve the ability of seeing the
two most essential modes in the structural transformation between
G-actin and F-actin. However, a rigorous analysis of the resulting CG
maps based on AA simulations of G-actin showed similarities with the
intuitive map only at the level of seven-sites-per-subunit. A key
strength of ED-CG over biological intuition is, for example, the
identification of DB-loop as a separate CG site in the seven-site model,
which is not separated out in any of the seven contiguous domains in the
intuitive CG map. Although biological intuition can help in describing
either four- or seven-site CG maps, ED-CG can be used to define
arbitrary number of CG sites depending on the required resolution for
the problem being studied. For example, ED-CG maps with four to twenty
CG sites per actin subunit were generated from the same underlying AA
simulations of actin filaments with either ATP or ADP as the bound
nucleotide. This analysis showed that although the same ED-CG map can be
used to define each subunit in the filament, heterogeneity is required
in the hENM parameters for corresponding pairs of CG beads to reproduce
mechanical properties of filament that are consistent with AA
simulations (Fan et al. 2012
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR20>).


      ATP hydrolysis

The nucleotide ATP bound to actin subunits predominantly hydrolyzes in
filaments rather than in G-actin. The rate of ATP hydrolysis in G-actin
is about 7 × 10^−6  s^−1 , which is about four orders of magnitude
slower than in F-actin (~0.3 s^−1 ). Clearly, the difference between
G-actin and F-actin structures was believed to be responsible for the
dramatic increase in the rate of hydrolysis in F-actin. In spite of the
availability of high-resolution structures of G-actin and filament
models, no direct structural evidence at the residue-level resolution
can be obtained to understand these differences. Molecular simulations
are a perfect tool to build upon experimentally observed structures in
order to probe into the important structural changes and the key
residues involved in ATP hydrolysis.


      Ordering of water around the nucleotide

Structured water has been identified in the nucleotide-binding pocket of
actin (see Fig. 1
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig1> for
location of the pocket). This water was shown to be stable in long AA
simulations of G-actin (McCullagh et al. 2014
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR41>).
Moreover, ordering of the water around the divalent cation Mg^+2
provides “lubrication” to assist the twisting of a single actin subunit,
isolated from the filament, from F-actin structure to relax to a G-actin
structure, highlighting the important role of water in preventing direct
salt bridges between the key acidic residues of actin in the
nucleotide-binding pocket. Instead, these residues are found to interact
with the cation through water molecules in order to facilitate smoother
transition between a twisted G-actin and a flat F-actin conformation.

The hydrolysis reaction involves the cleavage of the P_γ  – O_β bond
(see Fig. 1
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig1>a)
with a lytic water added to P_γ . In addition to organization of the
first solvation shell around P_γ for the lytic water, the sub-sequent
water solvation shells are important for facilitating proton transport
via Grotthuss shutting from the lytic water to P_γ and this process is
catalyzed by key coordinating residues highlighted in the following
paragraph. The structural transformation from G-actin to F-actin results
into an extended network of stable waters in the nucleotide-binding
pocket (Saunders and Voth 2011
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR68>).
This change in stability of waters between G-actin and F-actin in
filaments was observed in AA MD simulations. Figure 2
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig2>a
shows waters in the nucleotide-binding pocket, colored in red and
labeled as W1, W2, etc., and the Oda filament model is used to represent
F-actin. These simulations highlighted the importance of including
waters observed in the original crystal structure in the initial setup
of MD simulations. In absence of these waters in the initial simulation
setup, typical timescales of MD simulations were found to be
insufficient to observe the formation of structured water. Although the
translational diffusion of water is not affected to a significant degree
in the structural transformation from G-actin to F-actin, the waters in
the nucleotide-binding pocket are freer to rearrange rotationally in
F-actin (McCullagh et al. 2014
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR41>).
These waters form shorter proton shuttling water wires in F-actin that
facilitate the proton transport between the lytic water, the key
residues involved, and the P_γ (Fig. 2
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig2>b
shows a representative snapshot of the AA simulation and the statistical
distribution of water wire length).

Several important residues have been identified to be involved in
structural transformation from G-actin to F-actin, including the
nucleotide-binding Gln137 and His161, and the nearby bases Asp11,
Asp154, Asp157 (Graceffa and Dominguez 2003
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR27>;
Iwasa et al. 2008
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR30>;
Oda et al. 2009
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR46>;
Otterbein et al. 2001
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR49>;
Schuler 2001
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR73>;
Schuler et al. 1999
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR74>;
Vorobiev et al. 2003
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR79>;
Wertman et al. 1992
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR81>)
(labeled in Fig. 2
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig2>a). Of
these residues, Gln137 is believed to stabilize the lytic water, while
His161 is believed to act as a catalytic base (Graceffa and Dominguez
2003
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR27>;
Vorobiev et al. 2003
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR79>).
AA MD simulations have been successfully used to gain a better
understanding of the connection between structural transformation and
the ATP hydrolysis reaction. These simulations revealed that Asp11 has a
closer coordination with the cation Mg^+2 , while Gln137 moves relative
to the P_γ . The detailed mechanism of ATP hydrolysis in both G-actin
and F-actin involves the cleavage of the P_γ  – O_β bond and the
nucleophilic attack on P_γ by the lytic water. This involves transfer of
the proton from the lytic water to the Asp154 carboxylate oxygen,
followed by protonation of O_γ to form the species \(
{\mathrm{H}}_2\mathrm{P}{\mathrm{O}}_4^{-} \) (Akola and Jones 2006
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR1>;
Freedman et al. 2012
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR21>;
McCullagh et al. 2014
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR41>).
The flattened structure of F-actin and the resulting rearrangement of
key residues help lower the free energy barrier, including altering the
water wires in the nucleotide-binding pocket to be more favorable for
the transport of this proton, as highlighted in the data shown in Fig. 2
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig2>b
(McCullagh et al. 2014
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR41>).


      Hydrolysis mechanism at the filament level

Thanks to the computational studies described in the previous paragraph,
the detailed mechanism of hydrolysis of the nucleotide ATP is now
believed to be well understood at the level of an individual actin
subunit. However, two contradicting views have existed about the
mechanism of collective ATP hydrolysis at the level of the actin
filament. In a classic study that established the kinetic rate constants
of ATP hydrolysis, Pollard and Weeds first proposed the random
hydrolysis mechanism in actin filaments (Pollard and Weeds 1984
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR60>).
In this mechanism, ATP hydrolysis is assumed to be equally likely in all
actin subunits in a filament and hence a single rate governs hydrolysis
of ATP bound to any subunit throughout the filament. This implies that
the conformation of the neighboring subunits does not significantly
affect the rate constant for hydrolysis, even if the neighboring
nucleotide state modulates the local conformations of each subunit.

In contrast to this view, the nucleotide state of neighboring subunits
may instead be assumed to significantly modulate the rate constant
through modulation of the local conformations that each subunit
explores, giving rise to a spectrum of cooperative hydrolysis models.
Among these cooperative models, the vectorial model is the most distinct
from the random hydrolysis model, since it assumes that ATP hydrolysis
can occur predominantly in those subunits in the filament that have an
adjoining hydrolyzed subunit, and hence are at an interface between
un-hydrolyzed and hydrolyzed sections of the filament (Pantaloni et al.
1985
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR50>).
A similar contradiction has existed for the P_i release reaction, which
could be assumed to be either random or cooperative at the filament
level. Several attempts have been made to perform a systematic
comparison between these views (Burnett and Carlsson 2012
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR6>;
Burnett and Carlsson 2013
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR7>;
Carlier et al. 1987
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR10>;
Jégou et al. 2013
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR31>;
Korn et al. 1987
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR37>;
Ohm and Wegner 1994
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR47>;
Pieper and Wegner 1996
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR59>;
Ranjith et al. 2010
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR64>;
Stukalin and Kolomeisky 2006
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR75>;
Vavylonis et al. 2005
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR78>).
However, common experimentally measured quantities such as rate of
filament elongation, fluctuations in filament length, and size of the
un-hydrolyzed ATP cap near the filament end are found to be insensitive
to the mechanism of hydrolysis, except for small quantitative difference
near critical G-actin concentration. Using a mixture of ATP- and
ADP-bound G-actin to introduce large number of interfaces in the
filament suggested that the rate of hydrolysis at the interface must be
less than 100 times faster than the rate of hydrolysis away from the
interface, although the predictions of a random hydrolysis mechanism
were also shown to be able to explain the observed experimental data
(Ohm and Wegner 1994
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR47>;
Pieper and Wegner 1996
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR59>).
By using the nearest-neighbor cooperativity in such a model as a
parameter to fit experimentally measured P_i release profiles, it was
shown that the possibility of a high degree of cooperativity in P_i
release could not be completely excluded, however with some ambiguity in
the data analysis involved (Burnett and Carlsson 2012
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR6>;
Burnett and Carlsson 2013
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR7>;
Jégou et al. 2013
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR31>;
Jégou et al. 2011
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR32>).

Extracting a complex cooperative model in order to probe the true
mechanism of ATP hydrolysis and P_i release is limited by the lack of
any direct experimental evidence to empirically justify and parameterize
a model constructed for any assumed cooperative mechanism. Although
detailed computational simulations at the resolution of AIMD, QM/MM, and
AA have proven to be useful in investigating the mechanism of ATP
hydrolysis at the level of an actin subunit, these simulations cannot be
used to directly investigate the mechanism at the filament level because
these are typically limited to systems of the size of a few actin
subunits. A bottom-up CG modeling approach is a promising candidate in
this respect. In this approach, coarser resolution models are
parameterized in a systematic way to reproduce essential behavior of
underlying finer resolution models. A multiscale bottom-up CG model of
the actin filament was recently successfully developed and employed to
predict the nature and extent of such complex cooperativity present in
both ATP hydrolysis and P_i release reactions (Katkar et al. 2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR36>).
AA simulations of actin filaments were used to predict the nature and
extent to which neighboring nucleotide states modulate the local
conformations of each subunit in the filament. Figure 1
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig1>a
shows a schematic highlighting the fluctuations of neighboring subunits
relative to a given subunit. Using the state-of-the-art
ultra-coarse-graining (UCG) technique (Dama et al. 2013
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR15>;
Davtyan et al. 2014
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR16>;
Voth 2017
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR80>),
these predictions from AA simulations were used to study how the rate
constant for hydrolysis is modulated by the nucleotide state of the
neighboring subunits. Both ATP hydrolysis and P_i release reactions were
predicted to exhibit cooperativity, to the extent of up to ±20% change
in local rate constants of these reactions in the filament relative to
the average rate constant across the entire filament, depending on the
local nucleotide state of neighboring subunits (Fig. 2
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig2>e
shows the variation in rate of phosphate release as a function of
neighboring environment of a given subunit). The random and vectorial
hydrolysis models are attempts to simplify a much more complex
cooperativity into a simpler picture that looks at the difference
between rate constants of hydrolysis and P_i release in subunits at
interfaces and in the bulk of the actin filament. At the level of this
simplification, the UCG model predictions showed no evidence in support
of the vectorial mechanism and seemed to converge toward the random
hydrolysis mechanism proposed by Pollard and Weeds.


    Filament-forming proteins

The following sections will discuss the computational studies from Voth
group in collaboration with Tom Pollard that have shed light on the
structural and dynamical properties of formins, their interactions with
the actin filaments, and important mechanisms related to nucleation and
polymerization of actin.


      Unbranched filaments

Formins are large proteins (120–220 kDa) with multiple domains that
regulate the elongation of unbranched actin filaments. The dimeric
formin homology 2 (FH2) domain forms the core structure of formins (Xu
et al. 2004
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR82>).
They nucleate actin filament by stabilizing actin monomers and
processively move with the growing filament (Paul and Pollard 2009b
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR53>).
FH2 domains, in the absence of FH1 domain and profilin-bound actin
monomers, have been shown to slow down elongation of actin filament
(Paul and Pollard 2009a
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR52>).
These domains are capable of polymerizing actin filament when they are
in “open” state as opposed to “closed” state. The Pollard lab first
suggested that two states of FH2 domain at the barbed end can have
different effects on the conformation of terminal subunits, which would
determine the addition of a new subunit (Paul et al. 2008
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR51>;
Paul and Pollard 2009b
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR53>).
Their single-molecule experiments demonstrated that filament ends with
associated formins grow steadily; thus, they proposed that there is
rapid fluctuation between an open and a closed state during actin
filament growth at the barbed end. This phenomenon of the equilibrium
between the open and closed states of the FH2 domain at the barbed end
of the filament is called “gating” (Vavylonis et al. 2006
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR77>).
The fraction of time spent by the FH2 domain in an open state is
expressed by a gating factor. FH2 domains that belong to different types
of organisms have variability in gating factor although they have
similar three-dimensional structure and mechanistic behavior (Kovar et
al. 2006
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR38>).
For example, mDia1 (mammalian formin) has a gating factor of ~ 0.95
which means that it is mostly found in an open state and competent of
polymerizing actin filament. On the other hand, Bni1p (budding yeast
formin) and Cdc12 (fission yeast formin) have “gating” factors of ~ 0.6
and ~ 0.05, respectively.


      Building computational models of formins bound to the barbed ends
      of actin filaments

With the combined efforts of the Pollard and Voth Labs, a detailed model
of Bni1 FH2 domain bound to actin filament barbed end was created (Baker
et al. 2015
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR3>)
by using a filament composed of seven actin subunits with a twist angle
of 167° based on the bare actin filament structure solved by X-ray fiber
diffraction (Oda et al. 2009
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR46>)
and intermolecular contacts observed in the co-crystal structure of Bni1
FH2 domain—actin with a twist angle of 180° (Otomo et al. 2005
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR48>)
(Fig. 3
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig3>a).
Here, twist angle is defined as the angle by which consecutive subunits
rotate along the filament axis. Baker et al. carried out 160 ns of AA MD
simulation to refine the model, which resulted in conformational changes
in the proteins and flattening of the helical twist at the barbed end of
the filament from a twist angle of 167° (bare actin filament) toward
larger twist angles (Bni1-actin filament). Alternative modeling
approaches can be used to generate the initial structures of formins
that have no available crystal structures (such as Cdc12 or mDia1). For
instance, Aydin et al. used template-based homology modeling to generate
the initial structures of mDia1 and Cdc12 FH2 domains interacting with
the barbed end of actin filament by using the existing structure of Bni1
FH2 domain bound at the barbed end of the filament as a template (Aydin
et al. 2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR2>).
These atomistic models of Cdc12, Bni1, and mDia1 correspond to the
conformations just after the addition of a new actin subunit (labeled A1
in Fig. 1
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig1>b). To
build structures corresponding to the conformation prior to the addition
of a new subunit, actin subunit A1 was removed from the barbed ends of
the seven-mer filaments. Overall, these works led to generation of the
structures of different FH2 domains bound to the barbed ends of
filaments with 167° twist angle, which was not known previously.

*Fig. 3

*

figure 3
<https://link.springer.com/article/10.1007/s12551-018-0474-8/figures/3>

Formin FH2 domains associated with actin filament seven-mer. *a* Initial
all-atom structure of the formin Bni1p FH2 domain interacting with actin
filament composed of seven actin subunits. The structure corresponds to
a state just after addition of a new actin subunit (shown in gray color)
and before stepping of trailing FH2 domain (shown in red color)
(reproduced from Baker et al. (2015
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR3>)).
*b* (Top) All-atom and CG hENM models of an actin subunit and Bni1p FH2
domain. (Bottom) Two different CG models of the FH2 domains associated
with the actin filament seven-mer: one residue per CG site (left) and
multiple residues per CG site (right) (reproduced from Baker et al.
(2015
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR3>)
and Aydin et al. (2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR2>)).
*c* Structural alignment of the FH2 domains of Bni1, Cdc12, and mDia1 at
the barbed end of the actin filament seven-mer (top), and separate
structural alignment of the FHL and FHT domains (bottom) (reproduced
from Aydin et al. (2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR2>)).
*d* Individual alignment of important regions in FHL and FHT domains
(reproduced from Aydin et al. (2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR2>)).
*e* (Top) Twist angles between actin subunits A2 and A3 as a function of
time during all-atom MD simulations of FH2 domains associated with actin
filament five-mer. (Middle and bottom) The distributions of helical
twist angles (A2–A3) during the last time intervals of MD simulations
from two replicas domains (reproduced from Aydin et al. (2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR2>))

Full size image

<https://link.springer.com/article/10.1007/s12551-018-0474-8/figures/3>

AA simulations are not suitable to study phenomena occurring at large
length and time scales, as mentioned in the preceding sections. This
limitation can be overcome by generating CG models of formins at the
barbed end of the filament. Baker et al. used a CG model based on the
hENM approach with a resolution of one-CG bead-per-residue to
investigate the effect of the point mutations on the interactions
between Bni1 and actin (Fig. 3
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig3>b).
The intramolecular interactions within proteins were described by hENM,
and intermolecular interactions between formin and actin filament were
modeled by the combination of van der Waals and electrostatic
interactions, via the use of Lennard Jones (LJ) and screened Coulomb
(Yukawa) potentials, respectively. This high-resolution CG model (each
residue is represented by a single CG site) enabled mutating a residue
and studying its effect on the interactions between formin and actin on
longer time scales. Aydin et al. built a lower-resolution CG model by
using the ED-CG method (Fig. 3
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig3>b). CG
sites of formin and actin were determined based on the collective
protein motions via ED-CG, and each CG site contained about eight to
nine residues. The intramolecular and intermolecular interactions were
described similar those to the model in Baker et al. by using hENM, and
LJ and Yukawa, respectively. The lower-resolution CG model smoothens
free energy surface further, and this enables observing conformational
differences between three formins.


      Structural and dynamical properties of formin—actin complexes:
      identification of interfacial binding regions that affect
      nucleation and polymerization of actin

Baker et al. simulations enabled the initial structure of Bni1-actin
complex to undergo conformational changes, which brought the leading
(FHL) and trailing (FHT) FH2 domains into closer to the actin subunits
at the barbed end. They investigated the contacts between formin and
actin residues in the refined structure by measuring distances between
alpha-carbon atoms. The most extensive interactions were found to be
located between the KnA and KnB helices in the FH2 knob region and the
actin target binding cleft (TBC, see Fig. 1
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig1>a) and
DB-loop, and these contacts are stronger in the refined structure of
Bni1-Oda actin seven-mer than the starting Otomo structure. These
regions were investigated in more detail by measuring salt bridges and
hydrogen bonds between actin and the charged residues in the FH2 knob
helices. Two residues (R1423 in KnA and K1467 in KnB) were selected to
test the importance of these electrostatic interactions between Bni1 and
actin by making point substitutions at these residues both via
experiments and simulation. The mutations reduced the total non-bonded
interaction energy between FH2 knob helices and actin TBC by making the
interactions in this region less favorable. The CG simulations of both
wild type (WT) and mutant Bni1 FH2 domain interacting with actin
seven-mer and actin dimer show that there is a larger difference in
number of contacts between the WT and mutant Bni1 FH2 for actin dimer
than actin seven-mer. These results suggest that these two residues can
be considered more important for the nucleation than elongation.
Experiments done in the Pollard lab confirmed this prediction by showing
that mutant Bni1 has no effect on the elongation of actin but
drastically reduces the nucleation activity. Thus, the combined
experimental and computational findings indicate that the electrostatic
interactions between the knob helices of formin FH2 domains and TBC of
actin filament are critical for the nucleation activity of formins
(Baker et al. 2015
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR3>).

In order to understand the gating mechanism and, therefore, factors
affecting polymerization of actin, the homology models of Cdc12 and
mDia1 were generated as discussed in the previous section. AA MD
simulations were run to investigate differences in the structures of
three FH2 domains (Cdc12, Bni1, and mDia1) and their interactions with
the barbed end of actin filament (Aydin et al. 2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR2>).
All FH2 domains have similar three-dimensional structures except small
variations in some regions (Fig. 3
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig3>c, d).
In addition, the differences in the number of contacts between formin
and actin residues do not correlate with their gating factors. The most
notable difference was found in the number of salt bridges formed
between the knob helices and actin TBC. The number of salt bridges
inversely correlates with the gating factors: Cdc12 and mDia1 have the
highest and lowest number of salt bridges, respectively, and the number
of salt bridges for Bni1 varies between them. These findings also
predicted that Cdc12 might be the best actin nucleating formin,
consistent with the idea proposed by Baker et al. that the electrostatic
interactions between knob helices and actin TBC play a critical role in
the nucleation of actin filament.


      Effect of formins on the barbed end configurations of the actin
      filaments in relation with the gating mechanism

FH2 domains cause flattening of actin subunits at the barbed end of the
actin filaments, as hypothesized by the Pollard lab to explain the
gating mechanism (Aydin et al. 2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR2>;
Baker et al. 2015
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR3>).
Extensive AA simulations were used to test this hypothesis. Twist angles
between actin subunits A1 and A2 (see Fig. 1
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig1>b for
definition) were stable at 167° during a control AA MD simulation of an
Oda actin filament 30-mer without FH2 domain (Baker et al. 2015
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR3>).
On the other hand, twist angles between actin subunits bound to Bni1 FH2
were found to shift toward larger values (from ~ 167 to ~ 172°) at the
end of long simulations by Baker et al. These AA simulations captured
the effect of formin at the barbed end of the filament and sampled
transition between twisted “open” state and flatter “closed” state.

Although formin was shown to influence the flattening of the barbed end
in simulations by Baker et al., these simulations did not rigorously
test Paul and Pollard’s hypothesis on the gating mechanism. This is
because Bni1 has a gating factor of ~ 0.6 and the barbed end bound to
Bni1 is in equilibrium between twisted “open” and flattened “closed”
states. A more reliable test would be comparing the twist angle
distributions of actin subunits bound to formins with different gating
factors. Toward this aim, Aydin et al. investigated flattening of the
barbed end by Cdc12, Bni1, and mDia1 FH2 domains with gating factors of
~ 0.05, ~ 0.6, and ~ 0.95 respectively, by comparing their distributions
of twist angles (Aydin et al. 2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR2>).
The findings from this investigation support Pollard’s hypothesis: The
filaments interacting with Cdc12 and mDia1 have the most flattened and
twisted barbed ends, respectively, while the flattening of the barbed
end bound to Bni1 varies between them (Fig. 3
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig3>e). In
addition, the AA and CG simulations revealed an additional contribution
to gating, steric blocking, by showing that some overlaps exist between
alpha-carbon atoms of FH2 domains and incoming actin subunits during the
simulations, and the extent of overlap is consistent with the gating
factors of formins (steric blocking is the largest for Cdc12 and the
smallest for mDia1) (Aydin et al. 2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR2>).
Moreover, metabasin metadynamics (a recently developed variant of
metadynamics) simulations showed that Cdc12 FH2 domains encounter higher
energy barriers than Bni1 and mDia1 FH2 domains to undergo
conformational changes, which would be necessary to escape from a state
with steric blocking.

Overall, these investigations demonstrate the important contributions of
both helix distortion and steric blocking to the gating with
implications for two main models of processive elongation:
“stair-stepping” and “stepping second.” Aydin et al. revealed that the
formins can switch between “open” and “closed” states without
dissociation of an FH2 domain (Aydin et al. 2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR2>),
as proposed by the stepping second model of the Pollard Lab. On the
other hand, these simulations have no large-scale conformational changes
so they do not completely rule out the “stair-stepping” model, which
requires dissociation of one FH2 domain from the end of the filament for
a transition to “open” state. Indeed, different energy barriers obtained
from metadynamics simulations might be relevant for the “stair-stepping”
model, as the magnitude of energy barrier for a conformational change
can affect ability of FH2 domains to dissociate from the barbed end of
the filament.


      Branched filament networks

One of the key components of the branched cytoskeletal network is the
actin-related protein complex Arp2/3, which consists of seven proteins.
Arp2 and Arp3 are the two actin-like subunits that form the core of the
complex, and the remaining five subunits, ArpC1-5, are considered to
play a role in helping with activation of Arp2/3 and binding of the
complex to a mother filament. Arp2/3 binds to a preexisting actin
filament and nucleates a daughter filament by forming the first two
monomers. The Arp2/3 complex is an inactive state in its native state so
that it cannot initiate filament nucleation without getting activated by
other cellular components. There are only small number of computational
studies that studied the Arp2/3 complex, given the size and complexity
of this system, despite its importance in the cytoskeletal system and
role in the cell motility.


        Effect of bound nucleotide on the structure and dynamics of the
        Arp2/3 complex

Pfaendtner and Voth (2008
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR58>)
performed the first AA MD simulations of the Arp2/3 complex and
investigated the role of the Arp3-bound nucleotide on the dynamics of
the entire complex by using ATP-bound, ADP-bound, and nucleotide free
systems. Moreover, they generated the missing subdomains (subdomains 1
and 2) of Arp2 by using a homology model and studied the effect of bound
nucleotide in Arp2 as an additional analysis and further verification of
the effect of nucleotide. The stability of Arp3 with different
nucleotide states was investigated by analyzing the width of
nucleotide-binding cleft. It was found to remain in closed state for
ADP- and ATP-bound Arp3 within Arp2/3 complex, whereas adopted an open
conformation in nucleotide free Arp2/3 and ATP-bound Arp3-only
simulations. A CG analysis further demonstrated that the shape of
ATP-bound Arp3 within the complex is much more compact than ATP-bound
Arp3-only. The investigations on the nucleotide-binding cleft showed
that ADP-bound Arp3 is much more stable. This study also found that the
contacts between adjacent subunits are not affected by hydrolysis of the
nucleotide ATP. Therefore, the bound nucleotide has no role in
initiating the movements between the subunits for the activation process
of Arp2/3. Overall, this work supported the hypothesis suggesting that
conformational differences in the ATP- and ADP-bound structures point to
the role of ATP hydrolysis in regulating Arp2/3 function and structure
(Espinoza-Sanchez et al. 2018
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR18>;
Goley et al. 2004
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR26>).


      Structural and functional properties of the Arp2/3 branch junction
      complex

As an extension of this work, Pfaendtner et al. (2012
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR57>)
performed large-scale MD simulations of the Arp2/3 complex branch
junction based on the electron microscopy (EM) structure proposed by
Rouiller et al. (2008
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR65>)
to gain insight into the molecular level details of the interfacial
binding region between Arp2/3 complex and mother/daughter filaments and
understand the structural and dynamical properties of the branch
junction. This simulation system consisted of a periodic actin mother
filament with 13 ADP-actin subunits, a daughter filament with 11
ADP-actin subunits, and the Arp2/3 complex in a solvated simulation box
(Fig. 4
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig4>a).
Two different branch junction models (each system has ~ 3 million atoms)
were investigated by using MD simulations totaling 175 ns. One of the
models (Branch08) was based on the EM structure obtained by a fitting
procedure (Rouiller et al. 2008
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR65>),
and the other model (Branch10) was constructed by using the same Arp2/3
branch junction complex but different subunits in the mother filament
and daughter filament based on the Namba filament model (Fujii et al.
2010
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR23>),
except mother filament subunits M6 and M8 that bind Arp2/3 complex. No
significant differences between two models were detected at the end of
the MD simulations of Pfaendtner et al. (2012
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR57>).

*Fig. 4

*

figure 4
<https://link.springer.com/article/10.1007/s12551-018-0474-8/figures/4>

Structure and interactions of the Arp2/3 branch junction domains
(reproduced from Pfaendtner et al. (2012
<https://link.springer.com/article/10.1007/s12551-018-0474-8#ref-CR57>)). *a* The Arp2/3 complex subunits (colored) associated with mother (M1 to M13) and daughter filaments (D1 to D11). *b* Close up views of the Arp2/3 complex associated with mother and daughter filaments. *c* (Top) Salt bridge formation between ARPC2 (colored in cyan) and mother filament subunits M6 and M8 during all-atom MD simulations. (Bottom) Close up views of residues forming salt bridges. *d* Comparison between the density maps of the Arp2/3 complex from electron microscopy (EM) tomography (colored in gray) and the starting model based on EM data (blue) and the best-fitting molecular dynamics (MD) model (red). *e* Local changes in number of alpha-carbons inside the density map shown in (*d*)

Full size image

<https://link.springer.com/article/10.1007/s12551-018-0474-8/figures/4>

The MD simulations of the full branch junction resulted in a relatively
low root-mean-squared deviation (RMSD), which verifies the quality of
the initial EM structure. In addition, the angles between the mother
filament and daughter filament from the simulations were found to be in
good agreement with the angle derived from experimental imaging
techniques. The binding interface between the Arp2/3 complex and
neighboring actin subunits was further seen to be dynamic despite a
comparably low RMSD of the Arp2/3 complex. The difference contact map
was used to track the change in the contacts during both the Branch08
and Branch10 simulations. The contacts between the Arp2/3 complex and
the mother filament were overall lost during the MD simulations due to
structural relaxation. A large number of salt bridges between charged
residues and hydrophobic contacts between hydrophobic residues were
found in the interface between the Arp2/3 complex and the mother
filament, and these strong interactions were considered to stabilize the
binding interface so that the branch junction can reliably transmit
forces during cell motility (Fig. 4
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig4>c).

The Pfaendtner et al. simulations revealed that the DB-loop in actin and
Arp2/3 complex play an important role in stabilizing interface between
Arp2/3 and the mother filament. In addition, strong contacts between the
D-loops in the first two subunits of daughter filament (D1 and D2 in
Fig. 4
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig4>a, b)
and Arp2/3 result in a binding interface at the region where the
daughter filament and mother filament link together. Finally, this work
established a new method for a more reliable comparison between
simulation and experiments by filtering the MD simulations with
experimentally measured structural data (Fig. 4
<https://link.springer.com/article/10.1007/s12551-018-0474-8#Fig4>d, e).
Combining MD simulations of large biomolecules with low-resolution
experimental structure can be helpful to overcome the timescale problems
of the simulations, and also to test the structures of large
biomolecular complexes obtained by using fitting methods.


    Summary and future prospects

This review mostly highlights the work done in the Voth group to
understand the actin cytoskeleton from multiscale computational
approaches, as well as the important contributions of Tom Pollard, both
in collaboration with his group and also inspired by his other
contributions. We have summarized the key structural and chemical
aspects of actin subunits at various resolutions, ranging from quantum
mechanical modeling to study ATP hydrolysis to detailed AA simulations
to characterize the structural transformations that actin subunits
undergo as a result of (a) polymerization and (b) hydrolysis of the
bound nucleotide ATP and phosphate release. In addition, a summary of CG
models of actin at various resolutions of coarse-graining and the
insights gained through these models were also discussed, including the
methods used, such as ED-CG, hENM, and the UCG modeling approaches.
Moreover, we discussed the modeling of actin-associating proteins such
as formins and the Arp2/3 complex to understand the important mechanisms
at the molecular level affecting the architecture of actin network at
larger length scales by using these approaches. These collective works
also highlight the fruitful collaborations between the Voth and Pollard
labs, e.g., by providing direct evidence at the molecular level that
supports the random hydrolysis mechanism for ATP hydrolysis in filaments
as well as the gating mechanism of formins.

Many mysteries still remain to be unraveled related to the cytoskeleton.
For example, there are numerous studies showing seemingly contradicting
effects of applying external force on formin-mediated polymerization of
actin filaments. One of the important challenges in modeling this system
is connecting the residue-level properties with responses at the
filament level while overcoming the size limitations in simulating this
system at atomistic detail. In spite of recent advances in multiscale
modeling approaches that have been able to push the limits of the system
sizes that one can simulate, a real challenge is in cleverly combining
these approaches, while carefully following the fundamental physical and
chemical laws, to be able to address more challenging problems that
would require, for example, simulating the full actin network dynamics
at the mesoscopic level of a cell, including polymerization and
depolymerization, ATP hydrolysis, and P_i release at the level of
individual subunits, binding/unbinding of actin binding proteins
(cross-linkers, severing proteins, etc.) with precise dependence on the
nucleotide state, all while accurately capturing the competition between
these actin binding proteins. All of these challenges await combined
simulation and experimental work in the future.


    References

  *

    Akola J, Jones RO (2006) Density functional calculations of ATP
    systems. 2. ATP hydrolysis at the active site of actin. J Phys Chem
    B 110:8121–8129. https://doi.org/10.1021/jp054921d
    <https://doi.org/10.1021/jp054921d>

    Article <https://doi.org/10.1021%2Fjp054921d>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD28XislOgtrg%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16610915>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Density%20functional%20calculations%20of%20ATP%20systems.%202.%20ATP%20hydrolysis%20at%20the%20active%20site%20of%20actin&journal=J%20Phys%20Chem%20B&doi=10.1021%2Fjp054921d&volume=110&pages=8121-8129&publication_year=2006&author=Akola%2CJ&author=Jones%2CRO> 

  *

    Aydin F, Courtemanche N, Pollard TD, Voth GA (2018) Gating
    mechanisms during actin filament elongation by formins Elife 7
    https://doi.org/10.7554/eLife.37342
    <https://doi.org/10.7554/eLife.37342>

  *

    Baker JL, Courtemanche N, Parton DL, McCullagh M, Pollard TD, Voth
    GA (2015) Electrostatic interactions between the Bni1p Formin FH2
    domain and actin influence actin filament nucleation. Structure
    23:68–79. https://doi.org/10.1016/j.str.2014.10.014
    <https://doi.org/10.1016/j.str.2014.10.014>

    Article <https://doi.org/10.1016%2Fj.str.2014.10.014>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitFCrtL3O>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25482541>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Electrostatic%20interactions%20between%20the%20Bni1p%20Formin%20FH2%20domain%20and%20actin%20influence%20actin%20filament%20nucleation&journal=Structure&doi=10.1016%2Fj.str.2014.10.014&volume=23&pages=68-79&publication_year=2015&author=Baker%2CJL&author=Courtemanche%2CN&author=Parton%2CDL&author=McCullagh%2CM&author=Pollard%2CTD&author=Voth%2CGA> 

  *

    Baker JL, Voth GA (2013) Effects of ATP and actin-filament binding
    on the dynamics of the myosin II S1 domain. Biophys J 105:1624–1634.
    https://doi.org/10.1016/j.bpj.2013.08.023
    <https://doi.org/10.1016/j.bpj.2013.08.023>

    Article <https://doi.org/10.1016%2Fj.bpj.2013.08.023>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsFGru7fE>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3791303>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24094403>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Effects%20of%20ATP%20and%20actin-filament%20binding%20on%20the%20dynamics%20of%20the%20myosin%20II%20S1%20domain&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2013.08.023&volume=105&pages=1624-1634&publication_year=2013&author=Baker%2CJL&author=Voth%2CGA> 

  *

    Bidone TC, Kim T, Deriu MA, Morbiducci U, Kamm RD (2015) Multiscale
    impact of nucleotides and cations on the conformational equilibrium,
    elasticity and rheology of actin filaments and crosslinked networks.
    Biomech Model Mechanobiol 14:1143–1155.
    https://doi.org/10.1007/s10237-015-0660-6
    <https://doi.org/10.1007/s10237-015-0660-6>

    Article <https://doi.org/10.1007%2Fs10237-015-0660-6>  PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25708806>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Multiscale%20impact%20of%20nucleotides%20and%20cations%20on%20the%20conformational%20equilibrium%2C%20elasticity%20and%20rheology%20of%20actin%20filaments%20and%20crosslinked%20networks&journal=Biomech%20Model%20Mechanobiol&doi=10.1007%2Fs10237-015-0660-6&volume=14&pages=1143-1155&publication_year=2015&author=Bidone%2CTC&author=Kim%2CT&author=Deriu%2CMA&author=Morbiducci%2CU&author=Kamm%2CRD> 

  *

    Burnett MM, Carlsson AE (2012) Quantitative analysis of approaches
    to measure cooperative phosphate release in polymerized actin.
    Biophys J 103:2369–2378. https://doi.org/10.1016/j.bpj.2012.10.032
    <https://doi.org/10.1016/j.bpj.2012.10.032>

    Article <https://doi.org/10.1016%2Fj.bpj.2012.10.032>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhslyntbnI>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3514521>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23283236>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Quantitative%20analysis%20of%20approaches%20to%20measure%20cooperative%20phosphate%20release%20in%20polymerized%20actin&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2012.10.032&volume=103&pages=2369-2378&publication_year=2012&author=Burnett%2CMM&author=Carlsson%2CAE> 

  *

    Burnett MM, Carlsson AE (2013) Response to “on phosphate release in
    actin filaments”. Biophys J 104:2780.
    https://doi.org/10.1016/j.bpj.2013.05.020
    <https://doi.org/10.1016/j.bpj.2013.05.020>

    Article <https://doi.org/10.1016%2Fj.bpj.2013.05.020>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXpslKqsrc%3D>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3686334>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23790389>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Response%20to%20%E2%80%9Con%20phosphate%20release%20in%20actin%20filaments%E2%80%9D&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2013.05.020&volume=104&publication_year=2013&author=Burnett%2CMM&author=Carlsson%2CAE> 

  *

    Caby M, Hardas P, Ramachandran S, Ryckaert JP (2012) Hybrid
    molecular dynamics simulations of living filaments. J Chem Phys 136
    https://doi.org/10.1063/1.3694672 <https://doi.org/10.1063/1.3694672>

  *

    Campellone KG, Welch MD (2010) A nucleator arms race: cellular
    control of actin assembly. Nat Rev Mol Cell Biol 11:237–251.
    https://doi.org/10.1038/nrm2867 <https://doi.org/10.1038/nrm2867>

    Article <https://doi.org/10.1038%2Fnrm2867>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXjsFSqu7g%3D>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2929822>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20237478>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=A%20nucleator%20arms%20race%3A%20cellular%20control%20of%20actin%20assembly&journal=Nat%20Rev%20Mol%20Cell%20Biol&doi=10.1038%2Fnrm2867&volume=11&pages=237-251&publication_year=2010&author=Campellone%2CKG&author=Welch%2CMD> 

  *

    Carlier M, Pantaloni D, Korn E (1987) The mechanisms of Atp
    hydrolysis accompanying the polymerization of mg-actin and Ca-actin.
    J Biol Chem 262:3052–3059

    CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaL2sXhtlCjsLc%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=3818633>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20mechanisms%20of%20Atp%20hydrolysis%20accompanying%20the%20polymerization%20of%20mg-actin%20and%20Ca-actin&journal=J%20Biol%20Chem&volume=262&pages=3052-3059&publication_year=1987&author=Carlier%2CM&author=Pantaloni%2CD&author=Korn%2CE> 

  *

    Chou SZ, Pollard TD (2018) Mechanism of actin polymerization
    revealed by cryo-EM structures of actin filaments with three
    different bound nucleotides. bioRxiv https://doi.org/10.1101/309534
    <https://doi.org/10.1101/309534> (preprint posted April 27, 2018)

  *

    Chu JW, Voth GA (2005) Allostery of actin filaments: molecular
    dynamics simulations and coarse-grained analysis. P Natl Acad Sci
    USA 102:13111–13116. https://doi.org/10.1073/pnas.0503732102
    <https://doi.org/10.1073/pnas.0503732102>

    Article <https://doi.org/10.1073%2Fpnas.0503732102>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtVaqu7fN>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Allostery%20of%20actin%20filaments%3A%20molecular%20dynamics%20simulations%20and%20coarse-grained%20analysis&journal=P%20Natl%20Acad%20Sci%20USA&doi=10.1073%2Fpnas.0503732102&volume=102&pages=13111-13116&publication_year=2005&author=Chu%2CJW&author=Voth%2CGA> 

  *

    Chu JW, Voth GA (2006) Coarse-grained modeling of the actin filament
    derived from atomistic-scale simulations. Biophys J 90:1572–1582.
    https://doi.org/10.1529/biophysj.105.073924
    <https://doi.org/10.1529/biophysj.105.073924>

    Article <https://doi.org/10.1529%2Fbiophysj.105.073924>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD28XitVahtLg%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16361345>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Coarse-grained%20modeling%20of%20the%20actin%20filament%20derived%20from%20atomistic-scale%20simulations&journal=Biophys%20J&doi=10.1529%2Fbiophysj.105.073924&volume=90&pages=1572-1582&publication_year=2006&author=Chu%2CJW&author=Voth%2CGA> 

  *

    Dama JF, Jin J, Voth GA (2017) The theory of ultra-coarse-graining.
    3. Coarse-grained sites with rapid local equilibrium of internal
    states. J Chem Theory Comput 13:1010–1022.
    https://doi.org/10.1021/acs.jctc.6b01081
    <https://doi.org/10.1021/acs.jctc.6b01081>

    Article <https://doi.org/10.1021%2Facs.jctc.6b01081>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsVahurs%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28112956>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20theory%20of%20ultra-coarse-graining.%203.%20Coarse-grained%20sites%20with%20rapid%20local%20equilibrium%20of%20internal%20states&journal=J%20Chem%20Theory%20Comput&doi=10.1021%2Facs.jctc.6b01081&volume=13&pages=1010-1022&publication_year=2017&author=Dama%2CJF&author=Jin%2CJ&author=Voth%2CGA> 

  *

    Dama JF, Sinitskiy AV, McCullagh M, Weare J, Roux B, Dinner AR, Voth
    GA (2013) The theory of ultra-coarse-graining. 1. General
    principles. J Chem Theory Comput 9:2466–2480.
    https://doi.org/10.1021/ct4000444 <https://doi.org/10.1021/ct4000444>

    Article <https://doi.org/10.1021%2Fct4000444>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXktlyltLc%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26583735>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20theory%20of%20ultra-coarse-graining.%201.%20General%20principles&journal=J%20Chem%20Theory%20Comput&doi=10.1021%2Fct4000444&volume=9&pages=2466-2480&publication_year=2013&author=Dama%2CJF&author=Sinitskiy%2CAV&author=McCullagh%2CM&author=Weare%2CJ&author=Roux%2CB&author=Dinner%2CAR&author=Voth%2CGA> 

  *

    Davtyan A, Dama JF, Sinitskiy AV, Voth GA (2014) The theory of
    ultra-coarse-graining. 2. Numerical implementation. J Chem Theory
    Comput 10:5265–5275. https://doi.org/10.1021/ct500834t
    <https://doi.org/10.1021/ct500834t>

    Article <https://doi.org/10.1021%2Fct500834t>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFKqtbbN>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26583210>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20theory%20of%20ultra-coarse-graining.%202.%20Numerical%20implementation&journal=J%20Chem%20Theory%20Comput&doi=10.1021%2Fct500834t&volume=10&pages=5265-5275&publication_year=2014&author=Davtyan%2CA&author=Dama%2CJF&author=Sinitskiy%2CAV&author=Voth%2CGA> 

  *

    Deriu MA, Bidone TC, Mastrangelo F, Di Benedetto G, Soncini M,
    Montevecchi FM, Morbiducci U (2011) Biomechanics of actin filaments:
    a computational multi-level study. J Biomech 44:630–636.
    https://doi.org/10.1016/j.jbiomech.2010.11.014
    <https://doi.org/10.1016/j.jbiomech.2010.11.014>

    Article <https://doi.org/10.1016%2Fj.jbiomech.2010.11.014>  PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21130998>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Biomechanics%20of%20actin%20filaments%3A%20a%20computational%20multi-level%20study&journal=J%20Biomech&doi=10.1016%2Fj.jbiomech.2010.11.014&volume=44&pages=630-636&publication_year=2011&author=Deriu%2CMA&author=Bidone%2CTC&author=Mastrangelo%2CF&author=Benedetto%2CG&author=Soncini%2CM&author=Montevecchi%2CFM&author=Morbiducci%2CU> 

  *

    Espinoza-Sanchez S, Metskas LA, Chou SZ, Rhoades E, Pollard TD
    (2018) Conformational changes in Arp2/3 complex induced by ATP,
    WASp-VCA, and actin filaments. Proc Natl Acad Sci U S A
    https://doi.org/10.1073/pnas.1717594115
    <https://doi.org/10.1073/pnas.1717594115>

  *

    Fan J, Saunders MG, Haddadian EJ, Freed KF, De La Cruz EM, Voth GA
    (2013) Molecular origins of cofilin-linked changes in actin filament
    mechanics. J Mol Biol 425:1225–1240.
    https://doi.org/10.1016/j.jmb.2013.01.020
    <https://doi.org/10.1016/j.jmb.2013.01.020>

    Article <https://doi.org/10.1016%2Fj.jmb.2013.01.020>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXisVOnur0%3D>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3740545>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23352932>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Molecular%20origins%20of%20cofilin-linked%20changes%20in%20actin%20filament%20mechanics&journal=J%20Mol%20Biol&doi=10.1016%2Fj.jmb.2013.01.020&volume=425&pages=1225-1240&publication_year=2013&author=Fan%2CJ&author=Saunders%2CMG&author=Haddadian%2CEJ&author=Freed%2CKF&author=Cruz%2CEM&author=Voth%2CGA> 

  *

    Fan J, Saunders MG, Voth GA (2012) Coarse-graining provides insights
    on the essential nature of heterogeneity in actin filaments. Biophys
    J 103:1334–1342

    Article <https://doi.org/10.1016%2Fj.bpj.2012.08.029>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtlOns7nK>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22995506>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3446683>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Coarse-graining%20provides%20insights%20on%20the%20essential%20nature%20of%20heterogeneity%20in%20actin%20filaments&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2012.08.029&volume=103&pages=1334-1342&publication_year=2012&author=Fan%2CJ&author=Saunders%2CMG&author=Voth%2CGA> 

  *

    Freedman H, Laino T, Curioni A (2012) Reaction dynamics of ATP
    hydrolysis in actin determined by ab initio molecular dynamics
    simulations. J Chem Theory Comput 8:3373–3383.
    https://doi.org/10.1021/ct3003282 <https://doi.org/10.1021/ct3003282>

    Article <https://doi.org/10.1021%2Fct3003282>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtFChtr%2FE>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26605743>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Reaction%20dynamics%20of%20ATP%20hydrolysis%20in%20actin%20determined%20by%20ab%20initio%20molecular%20dynamics%20simulations&journal=J%20Chem%20Theory%20Comput&doi=10.1021%2Fct3003282&volume=8&pages=3373-3383&publication_year=2012&author=Freedman%2CH&author=Laino%2CT&author=Curioni%2CA> 

  *

    Freedman SL, Banerjee S, Hocky GM, Dinner AR (2017) A versatile
    framework for simulating the dynamic mechanical structure of
    cytoskeletal networks. Biophys J 113:448–460.
    https://doi.org/10.1016/j.bpj.2017.06.003
    <https://doi.org/10.1016/j.bpj.2017.06.003>

    Article <https://doi.org/10.1016%2Fj.bpj.2017.06.003>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVGktLfM>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5529201>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28746855>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=A%20versatile%20framework%20for%20simulating%20the%20dynamic%20mechanical%20structure%20of%20cytoskeletal%20networks&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2017.06.003&volume=113&pages=448-460&publication_year=2017&author=Freedman%2CSL&author=Banerjee%2CS&author=Hocky%2CGM&author=Dinner%2CAR> 

  *

    Fujii T, Iwane AH, Yanagida T, Namba K (2010) Direct visualization
    of secondary structures of F-actin by electron cryomicroscopy.
    Nature 467:724–U117. https://doi.org/10.1038/nature09372
    <https://doi.org/10.1038/nature09372>

    Article <https://doi.org/10.1038%2Fnature09372>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFGntL%2FK>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20844487>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Direct%20visualization%20of%20secondary%20structures%20of%20F-actin%20by%20electron%20cryomicroscopy&journal=Nature&doi=10.1038%2Fnature09372&volume=467&pages=724-U117&publication_year=2010&author=Fujii%2CT&author=Iwane%2CAH&author=Yanagida%2CT&author=Namba%2CK> 

  *

    Fujiwara I, Vavylonis D, Pollard TD (2007) Polymerization kinetics
    of ADP- and ADP-Pi-actin determined by fluorescence microscopy. Proc
    Natl Acad Sci 104:8827–8832. https://doi.org/10.1073/pnas.0702510104
    <https://doi.org/10.1073/pnas.0702510104>

    Article <https://doi.org/10.1073%2Fpnas.0702510104>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXmt1akurk%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17517656>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1885587>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Polymerization%20kinetics%20of%20ADP-%20and%20ADP-Pi-actin%20determined%20by%20fluorescence%20microscopy&journal=Proc%20Natl%20Acad%20Sci&doi=10.1073%2Fpnas.0702510104&volume=104&pages=8827-8832&publication_year=2007&author=Fujiwara%2CI&author=Vavylonis%2CD&author=Pollard%2CTD> 

  *

    Galkin VE, Orlova A, Schroder G, Egelman EH (2011) Structural
    polymorphism in F-actin. Biophys J 100:33–33

    Article <https://doi.org/10.1016%2Fj.bpj.2010.12.384>  Google
    Scholar
    <http://scholar.google.com/scholar_lookup?&title=Structural%20polymorphism%20in%20F-actin&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2010.12.384&volume=100&pages=33-33&publication_year=2011&author=Galkin%2CVE&author=Orlova%2CA&author=Schroder%2CG&author=Egelman%2CEH> 

  *

    Goley ED, Rodenbusch SE, Martin AC, Welch MD (2004) Critical
    conformational changes in the Arp2/3 complex are induced by
    nucleotide and nucleation promoting factor. Mol Cell 16:269–279.
    https://doi.org/10.1016/j.molcel.2004.09.018
    <https://doi.org/10.1016/j.molcel.2004.09.018>

    Article <https://doi.org/10.1016%2Fj.molcel.2004.09.018>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXhtVWit73L>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15494313>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Critical%20conformational%20changes%20in%20the%20Arp2%2F3%20complex%20are%20induced%20by%20nucleotide%20and%20nucleation%20promoting%20factor&journal=Mol%20Cell&doi=10.1016%2Fj.molcel.2004.09.018&volume=16&pages=269-279&publication_year=2004&author=Goley%2CED&author=Rodenbusch%2CSE&author=Martin%2CAC&author=Welch%2CMD> 

  *

    Graceffa P, Dominguez R (2003) Crystal structure of monomeric actin
    in the ATP state - structural basis of nucleotide-dependent actin
    dynamics. J Biol Chem 278:34172–34180.
    https://doi.org/10.1074/jbc.M303689200
    <https://doi.org/10.1074/jbc.M303689200>

    Article <https://doi.org/10.1074%2Fjbc.M303689200>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXmvVCqs78%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12813032>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Crystal%20structure%20of%20monomeric%20actin%20in%20the%20ATP%20state%20-%20structural%20basis%20of%20nucleotide-dependent%20actin%20dynamics&journal=J%20Biol%20Chem&doi=10.1074%2Fjbc.M303689200&volume=278&pages=34172-34180&publication_year=2003&author=Graceffa%2CP&author=Dominguez%2CR> 

  *

    Guo KK, Xiao WJ, Qiu D (2011) Polymerization of actin filaments
    coupled with adenosine triphosphate hydrolysis: Brownian dynamics
    and theoretical analysis. J Chem Phys 135
    https://doi.org/10.1063/1.3634006 <https://doi.org/10.1063/1.3634006>

  *

    Hocky GM, Baker JL, Bradley MJ, Sinitskiy AV, De La Cruz EM, Voth GA
    (2016) Cations stiffen actin filaments by adhering a key structural
    element to adjacent subunits. J Phys Chem B 120:4558–4567.
    https://doi.org/10.1021/acs.jpcb.6b02741
    <https://doi.org/10.1021/acs.jpcb.6b02741>

    Article <https://doi.org/10.1021%2Facs.jpcb.6b02741>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xnt12rsL4%3D>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4959277>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27146246>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Cations%20stiffen%20actin%20filaments%20by%20adhering%20a%20key%20structural%20element%20to%20adjacent%20subunits&journal=J%20Phys%20Chem%20B&doi=10.1021%2Facs.jpcb.6b02741&volume=120&pages=4558-4567&publication_year=2016&author=Hocky%2CGM&author=Baker%2CJL&author=Bradley%2CMJ&author=Sinitskiy%2CAV&author=Cruz%2CEM&author=Voth%2CGA> 

  *

    Iwasa M, Maeda K, Narita A, Maeda Y, Oda T (2008) Dual roles of
    Gln(137) of actin revealed by recombinant human cardiac muscle
    alpha-actin mutants. J Biol Chem 283:21045–21053.
    https://doi.org/10.1074/jbc.M800570200
    <https://doi.org/10.1074/jbc.M800570200>

    Article <https://doi.org/10.1074%2Fjbc.M800570200>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXoslWrt7g%3D>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3258932>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18515362>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Dual%20roles%20of%20Gln%28137%29%20of%20actin%20revealed%20by%20recombinant%20human%20cardiac%20muscle%20alpha-actin%20mutants&journal=J%20Biol%20Chem&doi=10.1074%2Fjbc.M800570200&volume=283&pages=21045-21053&publication_year=2008&author=Iwasa%2CM&author=Maeda%2CK&author=Narita%2CA&author=Maeda%2CY&author=Oda%2CT> 

  *

    Jégou A, Niedermayer T, Lipowsky R, Carlier M-F, Romet-Lemonne G
    (2013) On phosphate release in actin filaments. Biophys J
    104:2778–2779. https://doi.org/10.1016/j.bpj.2013.05.019
    <https://doi.org/10.1016/j.bpj.2013.05.019>

    Article <https://doi.org/10.1016%2Fj.bpj.2013.05.019>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXpslKltLw%3D>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3686342>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23790388>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=On%20phosphate%20release%20in%20actin%20filaments&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2013.05.019&volume=104&pages=2778-2779&publication_year=2013&author=J%C3%A9gou%2CA&author=Niedermayer%2CT&author=Lipowsky%2CR&author=Carlier%2CM-F&author=Romet-Lemonne%2CG> 

  *

    Jégou A, Niedermayer T, Orban J, Didry D, Lipowsky R, M-f C,
    Romet-lemonne G (2011) Individual actin filaments in a microfluidic
    flow reveal the mechanism of Atp hydrolysis and give insight into
    the properties of profilin. PLoS Biol 9:1001161

    Article <https://doi.org/10.1371%2Fjournal.pbio.1001161>  Google
    Scholar
    <http://scholar.google.com/scholar_lookup?&title=Individual%20actin%20filaments%20in%20a%20microfluidic%20flow%20reveal%20the%20mechanism%20of%20Atp%20hydrolysis%20and%20give%20insight%20into%20the%20properties%20of%20profilin&journal=PLoS%20Biol&doi=10.1371%2Fjournal.pbio.1001161&volume=9&publication_year=2011&author=J%C3%A9gou%2CA&author=Niedermayer%2CT&author=Orban%2CJ&author=Didry%2CD&author=Lipowsky%2CR&author=M-f%2CC&author=Romet-lemonne%2CG> 

  *

    Kabsch W, Mannherz HG, Suck D, Pai EF, Holmes KC (1990) Atomic
    structure of the actin: DNase I complex. Nature 347:37–44

    Article <https://doi.org/10.1038%2F347037a0>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaK3cXls1yjsLk%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=2395459>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Atomic%20structure%20of%20the%20actin%3A%20DNase%20I%20complex&journal=Nature&doi=10.1038%2F347037a0&volume=347&pages=37-44&publication_year=1990&author=Kabsch%2CW&author=Mannherz%2CHG&author=Suck%2CD&author=Pai%2CEF&author=Holmes%2CKC> 

  *

    Kang H, Bradley MJ, Elam WA, De La Cruz EM (2013) Regulation of
    actin by ion-linked equilibria. Biophys J 105:2621–2628.
    https://doi.org/10.1016/j.bpj.2013.10.032
    <https://doi.org/10.1016/j.bpj.2013.10.032>

    Article <https://doi.org/10.1016%2Fj.bpj.2013.10.032>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFOlsL3N>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3882474>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24359734>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Regulation%20of%20actin%20by%20ion-linked%20equilibria&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2013.10.032&volume=105&pages=2621-2628&publication_year=2013&author=Kang%2CH&author=Bradley%2CMJ&author=Elam%2CWA&author=Cruz%2CEM> 

  *

    Kang HR, Bradley MJ, McCullough BR, Pierre A, Grintsevich EE,
    Reisler E, De La Cruz EM (2012) Identification of cation-binding
    sites on actin that drive polymerization and modulate bending
    stiffness. P Natl Acad Sci USA 109:16923–16927.
    https://doi.org/10.1073/pnas.1211078109
    <https://doi.org/10.1073/pnas.1211078109>

    Article <https://doi.org/10.1073%2Fpnas.1211078109>  Google Scholar
    <http://scholar.google.com/scholar_lookup?&title=Identification%20of%20cation-binding%20sites%20on%20actin%20that%20drive%20polymerization%20and%20modulate%20bending%20stiffness&journal=P%20Natl%20Acad%20Sci%20USA&doi=10.1073%2Fpnas.1211078109&volume=109&pages=16923-16927&publication_year=2012&author=Kang%2CHR&author=Bradley%2CMJ&author=McCullough%2CBR&author=Pierre%2CA&author=Grintsevich%2CEE&author=Reisler%2CE&author=Cruz%2CEM> 

  *

    Katkar HH, Davtyan A, Durumeric AEP, Hocky GM, Schramm AC, De La
    Cruz EM, Voth GA (2018) Insights into the cooperative nature of ATP
    hydrolysis in actin filaments. Biophys J in press
    https://doi.org/10.1016/j.bpj.2018.08.034
    <https://doi.org/10.1016/j.bpj.2018.08.034>

  *

    Korn E, Carlier M, Pantaloni D (1987) Actin polymerization and Atp
    hydrolysis. Science 238:638–644

    Article <https://doi.org/10.1126%2Fscience.3672117>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaL1cXltFehuw%3D%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=3672117>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20polymerization%20and%20Atp%20hydrolysis&journal=Science&doi=10.1126%2Fscience.3672117&volume=238&pages=638-644&publication_year=1987&author=Korn%2CE&author=Carlier%2CM&author=Pantaloni%2CD> 

  *

    Kovar DR, Harris ES, Mahaffy R, Higgs HN, Pollard TD (2006) Control
    of the assembly of ATP- and ADP-actin by formins and profilin. Cell
    124:423–435

    Article <https://doi.org/10.1016%2Fj.cell.2005.11.038>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xht1Kktbk%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16439214>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Control%20of%20the%20assembly%20of%20ATP-%20and%20ADP-actin%20by%20formins%20and%20profilin&journal=Cell&doi=10.1016%2Fj.cell.2005.11.038&volume=124&pages=423-435&publication_year=2006&author=Kovar%2CDR&author=Harris%2CES&author=Mahaffy%2CR&author=Higgs%2CHN&author=Pollard%2CTD> 

  *

    Lyman E, Pfaendtner J, Voth GA (2008) Systematic multiscale
    parameterization of heterogeneous elastic network models of
    proteins. Biophys J 95:4183–4192.
    https://doi.org/10.1529/biophysj.108.139733
    <https://doi.org/10.1529/biophysj.108.139733>

    Article <https://doi.org/10.1529%2Fbiophysj.108.139733>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlagu7vF>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2567941>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18658214>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Systematic%20multiscale%20parameterization%20of%20heterogeneous%20elastic%20network%20models%20of%20proteins&journal=Biophys%20J&doi=10.1529%2Fbiophysj.108.139733&volume=95&pages=4183-4192&publication_year=2008&author=Lyman%2CE&author=Pfaendtner%2CJ&author=Voth%2CGA> 

  *

    Machesky LM, Atkinson SJ, Ampe C, Vandekerckhove J, Pollard TD
    (1994) Purification of a cortical complex containing 2
    unconventional actins from Acanthamoeba by affinity-chromatography
    on profilin-agarose. J Cell Biol 127:107–115.
    https://doi.org/10.1083/jcb.127.1.107
    <https://doi.org/10.1083/jcb.127.1.107>

    Article <https://doi.org/10.1083%2Fjcb.127.1.107>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaK2cXmslKgsLs%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=7929556>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Purification%20of%20a%20cortical%20complex%20containing%202%20unconventional%20actins%20from%20Acanthamoeba%20by%20affinity-chromatography%20on%20profilin-agarose&journal=J%20Cell%20Biol&doi=10.1083%2Fjcb.127.1.107&volume=127&pages=107-115&publication_year=1994&author=Machesky%2CLM&author=Atkinson%2CSJ&author=Ampe%2CC&author=Vandekerckhove%2CJ&author=Pollard%2CTD> 

  *

    McCullagh M, Saunders MG, Voth GA (2014) Unraveling the mystery of
    ATP hydrolysis in actin filaments. J Am Chem Soc 136:13053–13058.
    https://doi.org/10.1021/ja507169f <https://doi.org/10.1021/ja507169f>

    Article <https://doi.org/10.1021%2Fja507169f>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsVOnt7vI>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4183606>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25181471>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Unraveling%20the%20mystery%20of%20ATP%20hydrolysis%20in%20actin%20filaments&journal=J%20Am%20Chem%20Soc&doi=10.1021%2Fja507169f&volume=136&pages=13053-13058&publication_year=2014&author=McCullagh%2CM&author=Saunders%2CMG&author=Voth%2CGA> 

  *

    Mehrafrooz B, Shamloo A (2018) Mechanical differences between ATP
    and ADP actin states: a molecular dynamics study. J Theor Biol
    448:94–103. https://doi.org/10.1016/j.jtbi.2018.04.010
    <https://doi.org/10.1016/j.jtbi.2018.04.010>

    Article <https://doi.org/10.1016%2Fj.jtbi.2018.04.010>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXotFKgsbc%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29634959>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Mechanical%20differences%20between%20ATP%20and%20ADP%20actin%20states%3A%20a%20molecular%20dynamics%20study&journal=J%20Theor%20Biol&doi=10.1016%2Fj.jtbi.2018.04.010&volume=448&pages=94-103&publication_year=2018&author=Mehrafrooz%2CB&author=Shamloo%2CA> 

  *

    Merino F et al (2018) Structural transitions of F-actin upon ATP
    hydrolysis at near-atomic resolution revealed by cryo-EM. Nat Struct
    Mol Biol 25:531−+. https://doi.org/10.1038/s41594-018-0074-0
    <https://doi.org/10.1038/s41594-018-0074-0>

    Article <https://doi.org/10.1038%2Fs41594-018-0074-0>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlKqu7nP>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20transitions%20of%20F-actin%20upon%20ATP%20hydrolysis%20at%20near-atomic%20resolution%20revealed%20by%20cryo-EM&journal=Nat%20Struct%20Mol%20Biol&doi=10.1038%2Fs41594-018-0074-0&volume=25&publication_year=2018&author=Merino%2CF> 

  *

    Murakami K, Yasunaga T, Noguchi TQP, Gomibuchi Y, Ngo KX, Uyeda TQP,
    Wakabayashi T (2010) Structural basis for actin assembly, activation
    of ATP hydrolysis, and delayed phosphate release. Cell 143:275–287.
    https://doi.org/10.1016/j.cell.2010.09.034
    <https://doi.org/10.1016/j.cell.2010.09.034>

    Article <https://doi.org/10.1016%2Fj.cell.2010.09.034>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXht12qurnL>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20946985>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20for%20actin%20assembly%2C%20activation%20of%20ATP%20hydrolysis%2C%20and%20delayed%20phosphate%20release&journal=Cell&doi=10.1016%2Fj.cell.2010.09.034&volume=143&pages=275-287&publication_year=2010&author=Murakami%2CK&author=Yasunaga%2CT&author=Noguchi%2CTQP&author=Gomibuchi%2CY&author=Ngo%2CKX&author=Uyeda%2CTQP&author=Wakabayashi%2CT> 

  *

    Narita A, Oda T, Maeda Y (2011) Structural basis for the slow
    dynamics of the actin filament pointed end. EMBO J 30:1230–1237.
    https://doi.org/10.1038/emboj.2011.48
    <https://doi.org/10.1038/emboj.2011.48>

    Article <https://doi.org/10.1038%2Femboj.2011.48>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXislyjsbY%3D>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3094112>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21378753>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20for%20the%20slow%20dynamics%20of%20the%20actin%20filament%20pointed%20end&journal=EMBO%20J&doi=10.1038%2Femboj.2011.48&volume=30&pages=1230-1237&publication_year=2011&author=Narita%2CA&author=Oda%2CT&author=Maeda%2CY> 

  *

    Oda T, Iwasa M, Aihara T, Maeda Y, Narita A (2009) The nature of the
    globular-to fibrous-actin transition. Nature 457:441–445.
    https://doi.org/10.1038/nature07685
    <https://doi.org/10.1038/nature07685>

    Article <https://doi.org/10.1038%2Fnature07685>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXotlOnsQ%3D%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19158791>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20nature%20of%20the%20globular-to%20fibrous-actin%20transition&journal=Nature&doi=10.1038%2Fnature07685&volume=457&pages=441-445&publication_year=2009&author=Oda%2CT&author=Iwasa%2CM&author=Aihara%2CT&author=Maeda%2CY&author=Narita%2CA> 

  *

    Ohm T, Wegner A (1994) Mechanism of Atp hydrolysis by polymeric
    actin. Bba-Protein Struct M 1208:8–14

    Article <https://doi.org/10.1016%2F0167-4838%2894%2990153-8>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaK2cXmsV2gs78%3D>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Mechanism%20of%20Atp%20hydrolysis%20by%20polymeric%20actin&journal=Bba-Protein%20Struct%20M&doi=10.1016%2F0167-4838%2894%2990153-8&volume=1208&pages=8-14&publication_year=1994&author=Ohm%2CT&author=Wegner%2CA> 

  *

    Otomo T, Tomchick DR, Otomo C, Panchal SC, Machius M, Rosen MK
    (2005) Structural basis of actin filament nucleation and processive
    capping by a formin homology 2 domain. Nature 433:488–494

    Article <https://doi.org/10.1038%2Fnature03251>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXpt1Clsg%3D%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15635372>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20of%20actin%20filament%20nucleation%20and%20processive%20capping%20by%20a%20formin%20homology%202%20domain&journal=Nature&doi=10.1038%2Fnature03251&volume=433&pages=488-494&publication_year=2005&author=Otomo%2CT&author=Tomchick%2CDR&author=Otomo%2CC&author=Panchal%2CSC&author=Machius%2CM&author=Rosen%2CMK> 

  *

    Otterbein LR, Graceffa P, Dominguez R (2001) The crystal structure
    of uncomplexed actin in the ADP state. Science 293:708–711.
    https://doi.org/10.1126/science.1059700
    <https://doi.org/10.1126/science.1059700>

    Article <https://doi.org/10.1126%2Fscience.1059700>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXls1Kmt7Y%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11474115>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20crystal%20structure%20of%20uncomplexed%20actin%20in%20the%20ADP%20state&journal=Science&doi=10.1126%2Fscience.1059700&volume=293&pages=708-711&publication_year=2001&author=Otterbein%2CLR&author=Graceffa%2CP&author=Dominguez%2CR> 

  *

    Pantaloni D, Hill T, Carlier M, Korn E (1985) A model for actin
    polymerization and the kinetic effects of Atp hydrolysis. P Natl
    Acad Sci USA 82:7207–7211

    Article <https://doi.org/10.1073%2Fpnas.82.21.7207>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaL2MXmtV2ht7o%3D>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=A%20model%20for%20actin%20polymerization%20and%20the%20kinetic%20effects%20of%20Atp%20hydrolysis&journal=P%20Natl%20Acad%20Sci%20USA&doi=10.1073%2Fpnas.82.21.7207&volume=82&pages=7207-7211&publication_year=1985&author=Pantaloni%2CD&author=Hill%2CT&author=Carlier%2CM&author=Korn%2CE> 

  *

    Paul AS, Paul A, Pollard TD, Pollard T (2008) The role of the FH1
    domain and profilin in formin-mediated actin-filament elongation and
    nucleation. Curr Biol : CB 18:9–19

    Article <https://doi.org/10.1016%2Fj.cub.2007.11.062>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXivFagsw%3D%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18160294>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20role%20of%20the%20FH1%20domain%20and%20profilin%20in%20formin-mediated%20actin-filament%20elongation%20and%20nucleation&journal=Curr%20Biol%20%3A%20CB&doi=10.1016%2Fj.cub.2007.11.062&volume=18&pages=9-19&publication_year=2008&author=Paul%2CAS&author=Paul%2CA&author=Pollard%2CTD&author=Pollard%2CT> 

  *

    Paul AS, Pollard TD (2009a) Energetic requirements for processive
    elongation of actin filaments by FH1FH2-formins. J Biol Chem
    284:12533–12540

    Article <https://doi.org/10.1074%2Fjbc.M808587200>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXltVClsL4%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19251693>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2673319>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Energetic%20requirements%20for%20processive%20elongation%20of%20actin%20filaments%20by%20FH1FH2-formins&journal=J%20Biol%20Chem&doi=10.1074%2Fjbc.M808587200&volume=284&pages=12533-12540&publication_year=2009&author=Paul%2CAS&author=Pollard%2CTD> 

  *

    Paul AS, Pollard TD (2009b) Review of the mechanism of processive
    actin filament elongation by formins. Cell Motil Cytoskeleton 66:606–617

    Article <https://doi.org/10.1002%2Fcm.20379>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVKntr%2FK>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19459187>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2871153>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Review%20of%20the%20mechanism%20of%20processive%20actin%20filament%20elongation%20by%20formins&journal=Cell%20Motil%20Cytoskeleton&doi=10.1002%2Fcm.20379&volume=66&pages=606-617&publication_year=2009&author=Paul%2CAS&author=Pollard%2CTD> 

  *

    Pfaendtner J, Branduardi D, Parrinello M, Pollard TD, Voth GA (2009)
    Nucleotide-dependent conformational states of actin. P Natl Acad Sci
    USA 106:12723–12728. https://doi.org/10.1073/pnas.0902092106
    <https://doi.org/10.1073/pnas.0902092106>

    Article <https://doi.org/10.1073%2Fpnas.0902092106>  Google Scholar
    <http://scholar.google.com/scholar_lookup?&title=Nucleotide-dependent%20conformational%20states%20of%20actin&journal=P%20Natl%20Acad%20Sci%20USA&doi=10.1073%2Fpnas.0902092106&volume=106&pages=12723-12728&publication_year=2009&author=Pfaendtner%2CJ&author=Branduardi%2CD&author=Parrinello%2CM&author=Pollard%2CTD&author=Voth%2CGA> 

  *

    Pfaendtner J, De La Cruz EM, Voth GA (2010a) Actin filament
    remodeling by actin depolymerization factor/cofilin. P Natl Acad Sci
    USA 107:7299–7304. https://doi.org/10.1073/pnas.0911675107
    <https://doi.org/10.1073/pnas.0911675107>

    Article <https://doi.org/10.1073%2Fpnas.0911675107>  Google Scholar
    <http://scholar.google.com/scholar_lookup?&title=Actin%20filament%20remodeling%20by%20actin%20depolymerization%20factor%2Fcofilin&journal=P%20Natl%20Acad%20Sci%20USA&doi=10.1073%2Fpnas.0911675107&volume=107&pages=7299-7304&publication_year=2010&author=Pfaendtner%2CJ&author=Cruz%2CEM&author=Voth%2CGA> 

  *

    Pfaendtner J, Lyman E, Pollard TD, Voth GA (2010b) Structure and
    dynamics of the actin filament. J Mol Biol 396:252–263.
    https://doi.org/10.1016/j.jmb.2009.11.034
    <https://doi.org/10.1016/j.jmb.2009.11.034>

    Article <https://doi.org/10.1016%2Fj.jmb.2009.11.034>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXht12mtb0%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19931282>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structure%20and%20dynamics%20of%20the%20actin%20filament&journal=J%20Mol%20Biol&doi=10.1016%2Fj.jmb.2009.11.034&volume=396&pages=252-263&publication_year=2010&author=Pfaendtner%2CJ&author=Lyman%2CE&author=Pollard%2CTD&author=Voth%2CGA> 

  *

    Pfaendtner J, Volkmann N, Hanein D, Dalhaimer P, Pollard TD, Voth GA
    (2012) Key structural features of the actin filament Arp2/3 complex
    branch junction revealed by molecular simulation. J Mol Biol
    416:148–161. https://doi.org/10.1016/j.jmb.2011.12.025
    <https://doi.org/10.1016/j.jmb.2011.12.025>

    Article <https://doi.org/10.1016%2Fj.jmb.2011.12.025>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xhs1Cqsrc%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22206989>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Key%20structural%20features%20of%20the%20actin%20filament%20Arp2%2F3%20complex%20branch%20junction%20revealed%20by%20molecular%20simulation&journal=J%20Mol%20Biol&doi=10.1016%2Fj.jmb.2011.12.025&volume=416&pages=148-161&publication_year=2012&author=Pfaendtner%2CJ&author=Volkmann%2CN&author=Hanein%2CD&author=Dalhaimer%2CP&author=Pollard%2CTD&author=Voth%2CGA> 

  *

    Pfaendtner J, Voth GA (2008) Molecular dynamics simulation and
    coarse-grained analysis of the Arp2/3 complex. Biophys J
    95:5324–5333. https://doi.org/10.1529/biophysj.108.143313
    <https://doi.org/10.1529/biophysj.108.143313>

    Article <https://doi.org/10.1529%2Fbiophysj.108.143313>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsVymsbnO>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2586551>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18805923>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Molecular%20dynamics%20simulation%20and%20coarse-grained%20analysis%20of%20the%20Arp2%2F3%20complex&journal=Biophys%20J&doi=10.1529%2Fbiophysj.108.143313&volume=95&pages=5324-5333&publication_year=2008&author=Pfaendtner%2CJ&author=Voth%2CGA> 

  *

    Pieper U, Wegner A (1996) The end of a polymerizing actin filament
    contains numerous Atp-subunit segments that are disconnected by
    Adp-subunits resulting from Atp hydrolysis. Biochemistry 35:4396–4402

    Article <https://doi.org/10.1021%2Fbi9527045>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaK28Xhs12ms7w%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8605188>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20end%20of%20a%20polymerizing%20actin%20filament%20contains%20numerous%20Atp-subunit%20segments%20that%20are%20disconnected%20by%20Adp-subunits%20resulting%20from%20Atp%20hydrolysis&journal=Biochemistry&doi=10.1021%2Fbi9527045&volume=35&pages=4396-4402&publication_year=1996&author=Pieper%2CU&author=Wegner%2CA> 

  *

    Pollard T, Weeds A (1984) The rate-constant for Atp hydrolysis by
    polymerized actin. FEBS Lett 170:94–98

    Article <https://doi.org/10.1016%2F0014-5793%2884%2981376-9>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaL2cXksFSgt7c%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=6427006>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20rate-constant%20for%20Atp%20hydrolysis%20by%20polymerized%20actin&journal=FEBS%20Lett&doi=10.1016%2F0014-5793%2884%2981376-9&volume=170&pages=94-98&publication_year=1984&author=Pollard%2CT&author=Weeds%2CA> 

  *

    Pollard TD (1986) Rate constants for the reactions of Atp-actin and
    Adp-actin with the ends of actin-filaments. J Cell Biol
    103:2747–2754. https://doi.org/10.1083/jcb.103.6.2747
    <https://doi.org/10.1083/jcb.103.6.2747>

    Article <https://doi.org/10.1083%2Fjcb.103.6.2747>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaL2sXhtFGnsrg%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=3793756>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Rate%20constants%20for%20the%20reactions%20of%20Atp-actin%20and%20Adp-actin%20with%20the%20ends%20of%20actin-filaments&journal=J%20Cell%20Biol&doi=10.1083%2Fjcb.103.6.2747&volume=103&pages=2747-2754&publication_year=1986&author=Pollard%2CTD> 

  *

    Pollard TD, Cooper JA (2009) Actin, a central player in cell shape
    and movement. Science (New York, NY) 326:1208–1212

    Article <https://doi.org/10.1126%2Fscience.1175862>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhsVKhur7L>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%2C%20a%20central%20player%20in%20cell%20shape%20and%20movement&journal=Science%20%28New%20York%2C%20NY%29&doi=10.1126%2Fscience.1175862&volume=326&pages=1208-1212&publication_year=2009&author=Pollard%2CTD&author=Cooper%2CJA> 

  *

    Popov K, Komianos J, Papoian GA (2016) MEDYAN: mechanochemical
    simulations of contraction and polarity alignment in actomyosin
    networks. PLoS Comput Biol 12
    https://doi.org/10.1371/journal.pcbi.1004877
    <https://doi.org/10.1371/journal.pcbi.1004877>

  *

    Ranjith P, Mallick K, J-f J, Lacoste D (2010) Role of Atp-hydrolysis
    in the dynamics of a single actin filament. Biophys J 98:1418–1427

    Article <https://doi.org/10.1016%2Fj.bpj.2009.12.4306>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXht1Wnsr3O>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20409460>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2856186>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Role%20of%20Atp-hydrolysis%20in%20the%20dynamics%20of%20a%20single%20actin%20filament&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2009.12.4306&volume=98&pages=1418-1427&publication_year=2010&author=Ranjith%2CP&author=Mallick%2CK&author=J-f%2CJ&author=Lacoste%2CD> 

  *

    Rouiller I et al (2008) The structural basis of actin filament
    branching by the Arp2/3 complex. J Cell Biol 180:887–895.
    https://doi.org/10.1083/jcb.200709092
    <https://doi.org/10.1083/jcb.200709092>

    Article <https://doi.org/10.1083%2Fjcb.200709092>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXjtlKmtb0%3D>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2265399>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18316411>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20structural%20basis%20of%20actin%20filament%20branching%20by%20the%20Arp2%2F3%20complex&journal=J%20Cell%20Biol&doi=10.1083%2Fjcb.200709092&volume=180&pages=887-895&publication_year=2008&author=Rouiller%2CI> 

  *

    Rould MA, Wan Q, Joel PB, Lowey S, Trybus KM (2006) Crystal
    structures of expressed non-polymerizable monomeric actin in the ADP
    and ATP states. J Biol Chem 281:31909–31919.
    https://doi.org/10.1074/jbc.M601973200
    <https://doi.org/10.1074/jbc.M601973200>

    Article <https://doi.org/10.1074%2Fjbc.M601973200>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD28XhtVOrtrvE>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16920713>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Crystal%20structures%20of%20expressed%20non-polymerizable%20monomeric%20actin%20in%20the%20ADP%20and%20ATP%20states&journal=J%20Biol%20Chem&doi=10.1074%2Fjbc.M601973200&volume=281&pages=31909-31919&publication_year=2006&author=Rould%2CMA&author=Wan%2CQ&author=Joel%2CPB&author=Lowey%2CS&author=Trybus%2CKM> 

  *

    Saunders MG, Tempkin J, Weare J, Dinner AR, Roux B, Voth GA (2014)
    Nucleotide regulation of the structure and dynamics of G-actin.
    Biophys J 106:1710–1720. https://doi.org/10.1016/j.bpj.2014.03.012
    <https://doi.org/10.1016/j.bpj.2014.03.012>

    Article <https://doi.org/10.1016%2Fj.bpj.2014.03.012>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXmslahu7k%3D>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4008840>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24739170>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Nucleotide%20regulation%20of%20the%20structure%20and%20dynamics%20of%20G-actin&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2014.03.012&volume=106&pages=1710-1720&publication_year=2014&author=Saunders%2CMG&author=Tempkin%2CJ&author=Weare%2CJ&author=Dinner%2CAR&author=Roux%2CB&author=Voth%2CGA> 

  *

    Saunders MG, Voth GA (2011) Water molecules in the nucleotide
    binding cleft of actin: effects on subunit conformation and
    implications for ATP hydrolysis. J Mol Biol 413:279–291.
    https://doi.org/10.1016/j.jmb.2011.07.068
    <https://doi.org/10.1016/j.jmb.2011.07.068>

    Article <https://doi.org/10.1016%2Fj.jmb.2011.07.068>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1Grtr3K>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21856312>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Water%20molecules%20in%20the%20nucleotide%20binding%20cleft%20of%20actin%3A%20effects%20on%20subunit%20conformation%20and%20implications%20for%20ATP%20hydrolysis&journal=J%20Mol%20Biol&doi=10.1016%2Fj.jmb.2011.07.068&volume=413&pages=279-291&publication_year=2011&author=Saunders%2CMG&author=Voth%2CGA> 

  *

    Saunders MG, Voth GA (2012a) Coarse-grained analysis and modeling of
    nucleotide-dependent changes in F-actin. Biophys J 102:237a.
    https://doi.org/10.1016/j.bpj.2011.11.1306
    <https://doi.org/10.1016/j.bpj.2011.11.1306>

    Article <https://doi.org/10.1016%2Fj.bpj.2011.11.1306>  Google
    Scholar
    <http://scholar.google.com/scholar_lookup?&title=Coarse-grained%20analysis%20and%20modeling%20of%20nucleotide-dependent%20changes%20in%20F-actin&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2011.11.1306&volume=102&publication_year=2012&author=Saunders%2CMG&author=Voth%2CGA> 

  *

    Saunders MG, Voth GA (2012b) Comparison between actin filament
    models: coarse-graining reveals essential differences. Structure
    20:641–653

    Article <https://doi.org/10.1016%2Fj.str.2012.02.008>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xltl2ktbo%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22483111>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Comparison%20between%20actin%20filament%20models%3A%20coarse-graining%20reveals%20essential%20differences&journal=Structure&doi=10.1016%2Fj.str.2012.02.008&volume=20&pages=641-653&publication_year=2012&author=Saunders%2CMG&author=Voth%2CGA> 

  *

    Saunders MG, Voth GA (2013) Coarse-graining methods for
    computational biology. Annu Rev Biophys 42:73–93.
    https://doi.org/10.1146/annurev-biophys-083012-130348
    <https://doi.org/10.1146/annurev-biophys-083012-130348>

    Article <https://doi.org/10.1146%2Fannurev-biophys-083012-130348> 
    CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFGrs7bN>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23451897>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Coarse-graining%20methods%20for%20computational%20biology&journal=Annu%20Rev%20Biophys&doi=10.1146%2Fannurev-biophys-083012-130348&volume=42&pages=73-93&publication_year=2013&author=Saunders%2CMG&author=Voth%2CGA> 

  *

    Schramm AC, Hocky GM, Voth GA, Blanchoin L, Martiel JL, De La Cruz
    EM (2017) Actin filament strain promotes severing and cofilin
    dissociation. Biophys J 112:2624–2633.
    https://doi.org/10.1016/j.bpj.2017.05.016
    <https://doi.org/10.1016/j.bpj.2017.05.016>

    Article <https://doi.org/10.1016%2Fj.bpj.2017.05.016>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXovVKhtLs%3D>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5479148>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28636918>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20filament%20strain%20promotes%20severing%20and%20cofilin%20dissociation&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2017.05.016&volume=112&pages=2624-2633&publication_year=2017&author=Schramm%2CAC&author=Hocky%2CGM&author=Voth%2CGA&author=Blanchoin%2CL&author=Martiel%2CJL&author=Cruz%2CEM> 

  *

    Schuler H (2001) ATPase activity and conformational changes in the
    regulation of actin. Bba-Protein Struct M 1549:137–147.
    https://doi.org/10.1016/S0167-4838(01)00255-2
    <https://doi.org/10.1016/S0167-4838(01)00255-2>

    Article <https://doi.org/10.1016%2FS0167-4838%2801%2900255-2>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXnvFCrsrY%3D>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=ATPase%20activity%20and%20conformational%20changes%20in%20the%20regulation%20of%20actin&journal=Bba-Protein%20Struct%20M&doi=10.1016%2FS0167-4838%2801%2900255-2&volume=1549&pages=137-147&publication_year=2001&author=Schuler%2CH> 

  *

    Schuler H, Korenbaum E, Schutt CE, Lindberg U, Karlsson R (1999)
    Mutational analysis of Ser14 and Asp157 in the nucleotide-binding
    site of beta-actin. Eur J Biochem 265:210–220.
    https://doi.org/10.1046/j.1432-1327.1999.00716.x
    <https://doi.org/10.1046/j.1432-1327.1999.00716.x>

    Article <https://doi.org/10.1046%2Fj.1432-1327.1999.00716.x>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaK1MXmsFKks7w%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10491176>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Mutational%20analysis%20of%20Ser14%20and%20Asp157%20in%20the%20nucleotide-binding%20site%20of%20beta-actin&journal=Eur%20J%20Biochem&doi=10.1046%2Fj.1432-1327.1999.00716.x&volume=265&pages=210-220&publication_year=1999&author=Schuler%2CH&author=Korenbaum%2CE&author=Schutt%2CCE&author=Lindberg%2CU&author=Karlsson%2CR> 

  *

    Stukalin E, Kolomeisky A (2006) Atp hydrolysis stimulates large
    length fluctuations in single actin filaments. Biophys J 90:2673–2685

    Article <https://doi.org/10.1529%2Fbiophysj.105.074211>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD28XjsFeqtLw%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16443647>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1414574>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Atp%20hydrolysis%20stimulates%20large%20length%20fluctuations%20in%20single%20actin%20filaments&journal=Biophys%20J&doi=10.1529%2Fbiophysj.105.074211&volume=90&pages=2673-2685&publication_year=2006&author=Stukalin%2CE&author=Kolomeisky%2CA> 

  *

    Sun R, Sode O, Dama JF, Voth GA (2017) Simulating protein mediated
    hydrolysis of ATP and other nucleoside triphosphates by combining
    QM/MM molecular dynamics with advances in metadynamics. J Chem
    Theory Comput 13:2332–2341. https://doi.org/10.1021/acs.jctc.7b00077
    <https://doi.org/10.1021/acs.jctc.7b00077>

    Article <https://doi.org/10.1021%2Facs.jctc.7b00077>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXltVSktLo%3D>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5425946>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28345907>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Simulating%20protein%20mediated%20hydrolysis%20of%20ATP%20and%20other%20nucleoside%20triphosphates%20by%20combining%20QM%2FMM%20molecular%20dynamics%20with%20advances%20in%20metadynamics&journal=J%20Chem%20Theory%20Comput&doi=10.1021%2Facs.jctc.7b00077&volume=13&pages=2332-2341&publication_year=2017&author=Sun%2CR&author=Sode%2CO&author=Dama%2CJF&author=Voth%2CGA> 

  *

    Vavylonis D, Kovar DR, O'Shaughnessy B, Pollard TD (2006) Model of
    formin-associated actin filament elongation. Mol Cell 21:455–466

    Article <https://doi.org/10.1016%2Fj.molcel.2006.01.016>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD28XisVansLc%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16483928>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3716371>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Model%20of%20formin-associated%20actin%20filament%20elongation&journal=Mol%20Cell&doi=10.1016%2Fj.molcel.2006.01.016&volume=21&pages=455-466&publication_year=2006&author=Vavylonis%2CD&author=Kovar%2CDR&author=O%27Shaughnessy%2CB&author=Pollard%2CTD> 

  *

    Vavylonis D, Yang Q, O'Shaughnessy B (2005) Actin polymerization
    kinetics, cap structure, and fluctuations. P Natl Acad Sci USA
    102:8543–8548

    Article <https://doi.org/10.1073%2Fpnas.0501435102>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXlsFCgtr4%3D>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20polymerization%20kinetics%2C%20cap%20structure%2C%20and%20fluctuations&journal=P%20Natl%20Acad%20Sci%20USA&doi=10.1073%2Fpnas.0501435102&volume=102&pages=8543-8548&publication_year=2005&author=Vavylonis%2CD&author=Yang%2CQ&author=O%27Shaughnessy%2CB> 

  *

    Vorobiev S et al (2003) The structure of nonvertebrate actin:
    implications for the ATP hydrolytic mechanism. P Natl Acad Sci USA
    100:5760–5765. https://doi.org/10.1073/pnas.0832273100
    <https://doi.org/10.1073/pnas.0832273100>

    Article <https://doi.org/10.1073%2Fpnas.0832273100>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXjvFOls7o%3D>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20structure%20of%20nonvertebrate%20actin%3A%20implications%20for%20the%20ATP%20hydrolytic%20mechanism&journal=P%20Natl%20Acad%20Sci%20USA&doi=10.1073%2Fpnas.0832273100&volume=100&pages=5760-5765&publication_year=2003&author=Vorobiev%2CS> 

  *

    Voth GA (2017) A multiscale description of biomolecular active
    matter: the chemistry underlying many life processes. Acc Chem Res
    50:594–598. https://doi.org/10.1021/acs.accounts.6b00572
    <https://doi.org/10.1021/acs.accounts.6b00572>

    Article <https://doi.org/10.1021%2Facs.accounts.6b00572>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXkt1Wkt7c%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28945406>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=A%20multiscale%20description%20of%20biomolecular%20active%20matter%3A%20the%20chemistry%20underlying%20many%20life%20processes&journal=Acc%20Chem%20Res&doi=10.1021%2Facs.accounts.6b00572&volume=50&pages=594-598&publication_year=2017&author=Voth%2CGA> 

  *

    Wertman KF, Drubin DG, Botstein D (1992) Systematic mutational
    analysis of the yeast. ACT1 gene Genetics 132:337–350

    CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaK3sXhvVajug%3D%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=1427032>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Systematic%20mutational%20analysis%20of%20the%20yeast&journal=ACT1%20gene%20Genetics&volume=132&pages=337-350&publication_year=1992&author=Wertman%2CKF&author=Drubin%2CDG&author=Botstein%2CD> 

  *

    Xu YW, Moseley JB, Sagot I, Poy F, Pellman D, Goode BL, Eck MJ
    (2004) Crystal structures of a formin homology-2 domain reveal a
    tethered dimer architecture. Cell 116:711–723.
    https://doi.org/10.1016/S0092-8674(04)00210-7
    <https://doi.org/10.1016/S0092-8674(04)00210-7>

    Article <https://doi.org/10.1016%2FS0092-8674%2804%2900210-7>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXitFKisLg%3D>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15006353>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=Crystal%20structures%20of%20a%20formin%20homology-2%20domain%20reveal%20a%20tethered%20dimer%20architecture&journal=Cell&doi=10.1016%2FS0092-8674%2804%2900210-7&volume=116&pages=711-723&publication_year=2004&author=Xu%2CYW&author=Moseley%2CJB&author=Sagot%2CI&author=Poy%2CF&author=Pellman%2CD&author=Goode%2CBL&author=Eck%2CMJ> 

  *

    Yogurtcu ON, Kim JS, Sun SX (2012) A mechanochemical model of actin
    filaments. Biophys J 103:719–727.
    https://doi.org/10.1016/j.bpj.2012.07.020
    <https://doi.org/10.1016/j.bpj.2012.07.020>

    Article <https://doi.org/10.1016%2Fj.bpj.2012.07.020>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1eku7nI>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3443790>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22947933>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=A%20mechanochemical%20model%20of%20actin%20filaments&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2012.07.020&volume=103&pages=719-727&publication_year=2012&author=Yogurtcu%2CON&author=Kim%2CJS&author=Sun%2CSX> 

  *

    Zhang ZY, Lu LY, Noid WG, Krishna V, Pfaendtner J, Voth GA (2008) A
    systematic methodology for defining coarse-grained sites in large
    biomolecules. Biophys J 95:5073–5083.
    https://doi.org/10.1529/biophysj.108.139626
    <https://doi.org/10.1529/biophysj.108.139626>

    Article <https://doi.org/10.1529%2Fbiophysj.108.139626>  CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsVymsbrF>  PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2586547>  PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18757560>  Google Scholar <http://scholar.google.com/scholar_lookup?&title=A%20systematic%20methodology%20for%20defining%20coarse-grained%20sites%20in%20large%20biomolecules&journal=Biophys%20J&doi=10.1529%2Fbiophysj.108.139626&volume=95&pages=5073-5083&publication_year=2008&author=Zhang%2CZY&author=Lu%2CLY&author=Noid%2CWG&author=Krishna%2CV&author=Pfaendtner%2CJ&author=Voth%2CGA> 

Download
references<https://citation-needed.springer.com/v2/references/10.1007/s12551-018-0474-8?format=refman&flavour=references>


    Author information

Author notes

 1.

    Fikret Aydin and Harshwardhan H. Katkar contributed equally to this
    work.


      Authors and Affiliations

 1.

    Department of Chemistry, Institute of Biophysical Dynamics, and
    James Frank Institute, University of Chicago, Chicago, IL, USA

    Fikret Aydin, Harshwardhan H. Katkar & Gregory A. Voth

Authors

 1. Fikret Aydin
    View author publications
    <https://link.springer.com/search?dc.creator=Fikret%20Aydin>

    You can also search for this author in PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Fikret%20Aydin> Google Scholar <http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Fikret%20Aydin%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en>

 2. Harshwardhan H. Katkar
    View author publications
    <https://link.springer.com/search?dc.creator=Harshwardhan%20H.%20Katkar>

    You can also search for this author in PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Harshwardhan%20H.%20Katkar> Google Scholar <http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Harshwardhan%20H.%20Katkar%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en>

 3. Gregory A. Voth
    View author publications
    <https://link.springer.com/search?dc.creator=Gregory%20A.%20Voth>

    You can also search for this author in PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&term=Gregory%20A.%20Voth> Google Scholar <http://scholar.google.co.uk/scholar?as_q=&num=10&btnG=Search+Scholar&as_epq=&as_oq=&as_eq=&as_occt=any&as_sauthors=%22Gregory%20A.%20Voth%22&as_publication=&as_ylo=&as_yhi=&as_allsubj=all&hl=en>


      Corresponding author

Correspondence to Gregory A. Voth <mailto:gavoth@uchicago.edu>.


    Ethics declarations


      Funding information

This study was financially supported in part by the National Science
Foundation through NSF Grant CHE-1465248 and Materials Research Science
and Engineering Center (MRSEC) grant DMR-14207090, and Department of
Defense Army Research Office (ARO) through MURI grant W911NF1410403.


      Conflict of interest

Fikret Aydin declares that he has no conflict of interest. Harshwardhan
H. Katkar declares that he has no conflict of interest. Gregory A. Voth
declares that he has no conflict of interest.


      Ethical approval

This article does not contain any studies with human participants or
animals performed by any of the authors.


    Rights and permissions

Reprints and Permissions
<https://s100.copyright.com/AppDispatchServlet?title=Multiscale%20simulation%20of%20actin%20filaments%20and%20actin-associated%20proteins&author=Fikret%20Aydin%20et%20al&contentID=10.1007%2Fs12551-018-0474-8&copyright=International%20Union%20for%20Pure%20and%20Applied%20Biophysics%20%28IUPAB%29%20and%20Springer-Verlag%20GmbH%20Germany%2C%20part%20of%20Springer%20Nature&publication=1867-2450&publicationDate=2018-10-31&publisherName=SpringerNature&orderBeanReset=true>


    About this article

Check for updates. Verify currency and authenticity via CrossMark
<https://crossmark.crossref.org/dialog/?doi=10.1007/s12551-018-0474-8>


      Cite this article

Aydin, F., Katkar, H.H. & Voth, G.A. Multiscale simulation of actin
filaments and actin-associated proteins. /Biophys Rev/ *10*, 1521–1535
(2018). https://doi.org/10.1007/s12551-018-0474-8

Download
citation<https://citation-needed.springer.com/v2/references/10.1007/s12551-018-0474-8?format=refman&flavour=citation>

  *

    Received: 10 September 2018

  *

    Accepted: 21 October 2018

  *

    Published: 31 October 2018

  *

    Issue Date: December 2018

  *

    DOI: https://doi.org/10.1007/s12551-018-0474-8


      Share this article

Anyone you share the following link with will be able to read this content:

Get shareable link

Sorry, a shareable link is not currently available for this article.

Copy to clipboard

Provided by the Springer Nature SharedIt content-sharing initiative


      Keywords

  * Coarse-graining
  * Cytoskeleton
  * Protein dynamics
  * Protein-protein interactions
  * Molecular dynamics
  * Enhanced sampling

Download PDF

<https://link.springer.com/content/pdf/10.1007/s12551-018-0474-8.pdf?pdf=button>


      Working on a manuscript?

Avoid the common mistakes

<https://beta.springernature.com/pre-submission?journalId=12551>

  * Sections
  * Figures
  * References

  * Abstract <#Abs1>
  * Introduction <#Sec1>
  * Actin <#Sec2>
  * Filament-forming proteins <#Sec7>
  * Summary and future prospects <#Sec15>
  * References <#Bib1>
  * Author information <#author-information>
  * Ethics declarations <#ethics>
  * Rights and permissions <#rightslink>
  * About this article <#article-info>

Advertisement

  *
    *Fig. 1

    *

    figure 1

    View in article <#Fig1>Full size image

    <https://link.springer.com/article/10.1007/s12551-018-0474-8/figures/1>

  *
    *Fig. 2

    *

    figure 2

    View in article <#Fig2>Full size image

    <https://link.springer.com/article/10.1007/s12551-018-0474-8/figures/2>

  *
    *Fig. 3

    *

    figure 3

    View in article <#Fig3>Full size image

    <https://link.springer.com/article/10.1007/s12551-018-0474-8/figures/3>

  *
    *Fig. 4

    *

    figure 4

    View in article <#Fig4>Full size image

    <https://link.springer.com/article/10.1007/s12551-018-0474-8/figures/4>

 1.

    Akola J, Jones RO (2006) Density functional calculations of ATP
    systems. 2. ATP hydrolysis at the active site of actin. J Phys Chem
    B 110:8121–8129. https://doi.org/10.1021/jp054921d
    <https://doi.org/10.1021/jp054921d>

    Article <https://doi.org/10.1021%2Fjp054921d> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD28XislOgtrg%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16610915> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Density%20functional%20calculations%20of%20ATP%20systems.%202.%20ATP%20hydrolysis%20at%20the%20active%20site%20of%20actin&journal=J%20Phys%20Chem%20B&doi=10.1021%2Fjp054921d&volume=110&pages=8121-8129&publication_year=2006&author=Akola%2CJ&author=Jones%2CRO> 

 2.

    Aydin F, Courtemanche N, Pollard TD, Voth GA (2018) Gating
    mechanisms during actin filament elongation by formins Elife 7
    https://doi.org/10.7554/eLife.37342
    <https://doi.org/10.7554/eLife.37342>
 3.

    Baker JL, Courtemanche N, Parton DL, McCullagh M, Pollard TD, Voth
    GA (2015) Electrostatic interactions between the Bni1p Formin FH2
    domain and actin influence actin filament nucleation. Structure
    23:68–79. https://doi.org/10.1016/j.str.2014.10.014
    <https://doi.org/10.1016/j.str.2014.10.014>

    Article <https://doi.org/10.1016%2Fj.str.2014.10.014> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitFCrtL3O> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25482541> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Electrostatic%20interactions%20between%20the%20Bni1p%20Formin%20FH2%20domain%20and%20actin%20influence%20actin%20filament%20nucleation&journal=Structure&doi=10.1016%2Fj.str.2014.10.014&volume=23&pages=68-79&publication_year=2015&author=Baker%2CJL&author=Courtemanche%2CN&author=Parton%2CDL&author=McCullagh%2CM&author=Pollard%2CTD&author=Voth%2CGA> 

 4.

    Baker JL, Voth GA (2013) Effects of ATP and actin-filament binding
    on the dynamics of the myosin II S1 domain. Biophys J 105:1624–1634.
    https://doi.org/10.1016/j.bpj.2013.08.023
    <https://doi.org/10.1016/j.bpj.2013.08.023>

    Article <https://doi.org/10.1016%2Fj.bpj.2013.08.023> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsFGru7fE> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3791303> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24094403> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Effects%20of%20ATP%20and%20actin-filament%20binding%20on%20the%20dynamics%20of%20the%20myosin%20II%20S1%20domain&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2013.08.023&volume=105&pages=1624-1634&publication_year=2013&author=Baker%2CJL&author=Voth%2CGA> 

 5.

    Bidone TC, Kim T, Deriu MA, Morbiducci U, Kamm RD (2015) Multiscale
    impact of nucleotides and cations on the conformational equilibrium,
    elasticity and rheology of actin filaments and crosslinked networks.
    Biomech Model Mechanobiol 14:1143–1155.
    https://doi.org/10.1007/s10237-015-0660-6
    <https://doi.org/10.1007/s10237-015-0660-6>

    Article <https://doi.org/10.1007%2Fs10237-015-0660-6> PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25708806> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Multiscale%20impact%20of%20nucleotides%20and%20cations%20on%20the%20conformational%20equilibrium%2C%20elasticity%20and%20rheology%20of%20actin%20filaments%20and%20crosslinked%20networks&journal=Biomech%20Model%20Mechanobiol&doi=10.1007%2Fs10237-015-0660-6&volume=14&pages=1143-1155&publication_year=2015&author=Bidone%2CTC&author=Kim%2CT&author=Deriu%2CMA&author=Morbiducci%2CU&author=Kamm%2CRD> 

 6.

    Burnett MM, Carlsson AE (2012) Quantitative analysis of approaches
    to measure cooperative phosphate release in polymerized actin.
    Biophys J 103:2369–2378. https://doi.org/10.1016/j.bpj.2012.10.032
    <https://doi.org/10.1016/j.bpj.2012.10.032>

    Article <https://doi.org/10.1016%2Fj.bpj.2012.10.032> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhslyntbnI> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3514521> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23283236> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Quantitative%20analysis%20of%20approaches%20to%20measure%20cooperative%20phosphate%20release%20in%20polymerized%20actin&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2012.10.032&volume=103&pages=2369-2378&publication_year=2012&author=Burnett%2CMM&author=Carlsson%2CAE> 

 7.

    Burnett MM, Carlsson AE (2013) Response to “on phosphate release in
    actin filaments”. Biophys J 104:2780.
    https://doi.org/10.1016/j.bpj.2013.05.020
    <https://doi.org/10.1016/j.bpj.2013.05.020>

    Article <https://doi.org/10.1016%2Fj.bpj.2013.05.020> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXpslKqsrc%3D> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3686334> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23790389> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Response%20to%20%E2%80%9Con%20phosphate%20release%20in%20actin%20filaments%E2%80%9D&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2013.05.020&volume=104&publication_year=2013&author=Burnett%2CMM&author=Carlsson%2CAE> 

 8.

    Caby M, Hardas P, Ramachandran S, Ryckaert JP (2012) Hybrid
    molecular dynamics simulations of living filaments. J Chem Phys 136
    https://doi.org/10.1063/1.3694672 <https://doi.org/10.1063/1.3694672>
 9.

    Campellone KG, Welch MD (2010) A nucleator arms race: cellular
    control of actin assembly. Nat Rev Mol Cell Biol 11:237–251.
    https://doi.org/10.1038/nrm2867 <https://doi.org/10.1038/nrm2867>

    Article <https://doi.org/10.1038%2Fnrm2867> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXjsFSqu7g%3D> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2929822> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20237478> Google Scholar <http://scholar.google.com/scholar_lookup?&title=A%20nucleator%20arms%20race%3A%20cellular%20control%20of%20actin%20assembly&journal=Nat%20Rev%20Mol%20Cell%20Biol&doi=10.1038%2Fnrm2867&volume=11&pages=237-251&publication_year=2010&author=Campellone%2CKG&author=Welch%2CMD> 

10.

    Carlier M, Pantaloni D, Korn E (1987) The mechanisms of Atp
    hydrolysis accompanying the polymerization of mg-actin and Ca-actin.
    J Biol Chem 262:3052–3059

    CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaL2sXhtlCjsLc%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=3818633> Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20mechanisms%20of%20Atp%20hydrolysis%20accompanying%20the%20polymerization%20of%20mg-actin%20and%20Ca-actin&journal=J%20Biol%20Chem&volume=262&pages=3052-3059&publication_year=1987&author=Carlier%2CM&author=Pantaloni%2CD&author=Korn%2CE> 

11.

    Chou SZ, Pollard TD (2018) Mechanism of actin polymerization
    revealed by cryo-EM structures of actin filaments with three
    different bound nucleotides. bioRxiv https://doi.org/10.1101/309534
    <https://doi.org/10.1101/309534> (preprint posted April 27, 2018)
12.

    Chu JW, Voth GA (2005) Allostery of actin filaments: molecular
    dynamics simulations and coarse-grained analysis. P Natl Acad Sci
    USA 102:13111–13116. https://doi.org/10.1073/pnas.0503732102
    <https://doi.org/10.1073/pnas.0503732102>

    Article <https://doi.org/10.1073%2Fpnas.0503732102> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtVaqu7fN> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Allostery%20of%20actin%20filaments%3A%20molecular%20dynamics%20simulations%20and%20coarse-grained%20analysis&journal=P%20Natl%20Acad%20Sci%20USA&doi=10.1073%2Fpnas.0503732102&volume=102&pages=13111-13116&publication_year=2005&author=Chu%2CJW&author=Voth%2CGA> 

13.

    Chu JW, Voth GA (2006) Coarse-grained modeling of the actin filament
    derived from atomistic-scale simulations. Biophys J 90:1572–1582.
    https://doi.org/10.1529/biophysj.105.073924
    <https://doi.org/10.1529/biophysj.105.073924>

    Article <https://doi.org/10.1529%2Fbiophysj.105.073924> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD28XitVahtLg%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16361345> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Coarse-grained%20modeling%20of%20the%20actin%20filament%20derived%20from%20atomistic-scale%20simulations&journal=Biophys%20J&doi=10.1529%2Fbiophysj.105.073924&volume=90&pages=1572-1582&publication_year=2006&author=Chu%2CJW&author=Voth%2CGA> 

14.

    Dama JF, Jin J, Voth GA (2017) The theory of ultra-coarse-graining.
    3. Coarse-grained sites with rapid local equilibrium of internal
    states. J Chem Theory Comput 13:1010–1022.
    https://doi.org/10.1021/acs.jctc.6b01081
    <https://doi.org/10.1021/acs.jctc.6b01081>

    Article <https://doi.org/10.1021%2Facs.jctc.6b01081> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsVahurs%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28112956> Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20theory%20of%20ultra-coarse-graining.%203.%20Coarse-grained%20sites%20with%20rapid%20local%20equilibrium%20of%20internal%20states&journal=J%20Chem%20Theory%20Comput&doi=10.1021%2Facs.jctc.6b01081&volume=13&pages=1010-1022&publication_year=2017&author=Dama%2CJF&author=Jin%2CJ&author=Voth%2CGA> 

15.

    Dama JF, Sinitskiy AV, McCullagh M, Weare J, Roux B, Dinner AR, Voth
    GA (2013) The theory of ultra-coarse-graining. 1. General
    principles. J Chem Theory Comput 9:2466–2480.
    https://doi.org/10.1021/ct4000444 <https://doi.org/10.1021/ct4000444>

    Article <https://doi.org/10.1021%2Fct4000444> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXktlyltLc%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26583735> Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20theory%20of%20ultra-coarse-graining.%201.%20General%20principles&journal=J%20Chem%20Theory%20Comput&doi=10.1021%2Fct4000444&volume=9&pages=2466-2480&publication_year=2013&author=Dama%2CJF&author=Sinitskiy%2CAV&author=McCullagh%2CM&author=Weare%2CJ&author=Roux%2CB&author=Dinner%2CAR&author=Voth%2CGA> 

16.

    Davtyan A, Dama JF, Sinitskiy AV, Voth GA (2014) The theory of
    ultra-coarse-graining. 2. Numerical implementation. J Chem Theory
    Comput 10:5265–5275. https://doi.org/10.1021/ct500834t
    <https://doi.org/10.1021/ct500834t>

    Article <https://doi.org/10.1021%2Fct500834t> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFKqtbbN> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26583210> Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20theory%20of%20ultra-coarse-graining.%202.%20Numerical%20implementation&journal=J%20Chem%20Theory%20Comput&doi=10.1021%2Fct500834t&volume=10&pages=5265-5275&publication_year=2014&author=Davtyan%2CA&author=Dama%2CJF&author=Sinitskiy%2CAV&author=Voth%2CGA> 

17.

    Deriu MA, Bidone TC, Mastrangelo F, Di Benedetto G, Soncini M,
    Montevecchi FM, Morbiducci U (2011) Biomechanics of actin filaments:
    a computational multi-level study. J Biomech 44:630–636.
    https://doi.org/10.1016/j.jbiomech.2010.11.014
    <https://doi.org/10.1016/j.jbiomech.2010.11.014>

    Article <https://doi.org/10.1016%2Fj.jbiomech.2010.11.014> PubMed
    <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21130998> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Biomechanics%20of%20actin%20filaments%3A%20a%20computational%20multi-level%20study&journal=J%20Biomech&doi=10.1016%2Fj.jbiomech.2010.11.014&volume=44&pages=630-636&publication_year=2011&author=Deriu%2CMA&author=Bidone%2CTC&author=Mastrangelo%2CF&author=Benedetto%2CG&author=Soncini%2CM&author=Montevecchi%2CFM&author=Morbiducci%2CU> 

18.

    Espinoza-Sanchez S, Metskas LA, Chou SZ, Rhoades E, Pollard TD
    (2018) Conformational changes in Arp2/3 complex induced by ATP,
    WASp-VCA, and actin filaments. Proc Natl Acad Sci U S A
    https://doi.org/10.1073/pnas.1717594115
    <https://doi.org/10.1073/pnas.1717594115>
19.

    Fan J, Saunders MG, Haddadian EJ, Freed KF, De La Cruz EM, Voth GA
    (2013) Molecular origins of cofilin-linked changes in actin filament
    mechanics. J Mol Biol 425:1225–1240.
    https://doi.org/10.1016/j.jmb.2013.01.020
    <https://doi.org/10.1016/j.jmb.2013.01.020>

    Article <https://doi.org/10.1016%2Fj.jmb.2013.01.020> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXisVOnur0%3D> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3740545> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23352932> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Molecular%20origins%20of%20cofilin-linked%20changes%20in%20actin%20filament%20mechanics&journal=J%20Mol%20Biol&doi=10.1016%2Fj.jmb.2013.01.020&volume=425&pages=1225-1240&publication_year=2013&author=Fan%2CJ&author=Saunders%2CMG&author=Haddadian%2CEJ&author=Freed%2CKF&author=Cruz%2CEM&author=Voth%2CGA> 

20.

    Fan J, Saunders MG, Voth GA (2012) Coarse-graining provides insights
    on the essential nature of heterogeneity in actin filaments. Biophys
    J 103:1334–1342

    Article <https://doi.org/10.1016%2Fj.bpj.2012.08.029> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtlOns7nK> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22995506> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3446683> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Coarse-graining%20provides%20insights%20on%20the%20essential%20nature%20of%20heterogeneity%20in%20actin%20filaments&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2012.08.029&volume=103&pages=1334-1342&publication_year=2012&author=Fan%2CJ&author=Saunders%2CMG&author=Voth%2CGA> 

21.

    Freedman H, Laino T, Curioni A (2012) Reaction dynamics of ATP
    hydrolysis in actin determined by ab initio molecular dynamics
    simulations. J Chem Theory Comput 8:3373–3383.
    https://doi.org/10.1021/ct3003282 <https://doi.org/10.1021/ct3003282>

    Article <https://doi.org/10.1021%2Fct3003282> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtFChtr%2FE> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=26605743> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Reaction%20dynamics%20of%20ATP%20hydrolysis%20in%20actin%20determined%20by%20ab%20initio%20molecular%20dynamics%20simulations&journal=J%20Chem%20Theory%20Comput&doi=10.1021%2Fct3003282&volume=8&pages=3373-3383&publication_year=2012&author=Freedman%2CH&author=Laino%2CT&author=Curioni%2CA> 

22.

    Freedman SL, Banerjee S, Hocky GM, Dinner AR (2017) A versatile
    framework for simulating the dynamic mechanical structure of
    cytoskeletal networks. Biophys J 113:448–460.
    https://doi.org/10.1016/j.bpj.2017.06.003
    <https://doi.org/10.1016/j.bpj.2017.06.003>

    Article <https://doi.org/10.1016%2Fj.bpj.2017.06.003> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVGktLfM> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5529201> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28746855> Google Scholar <http://scholar.google.com/scholar_lookup?&title=A%20versatile%20framework%20for%20simulating%20the%20dynamic%20mechanical%20structure%20of%20cytoskeletal%20networks&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2017.06.003&volume=113&pages=448-460&publication_year=2017&author=Freedman%2CSL&author=Banerjee%2CS&author=Hocky%2CGM&author=Dinner%2CAR> 

23.

    Fujii T, Iwane AH, Yanagida T, Namba K (2010) Direct visualization
    of secondary structures of F-actin by electron cryomicroscopy.
    Nature 467:724–U117. https://doi.org/10.1038/nature09372
    <https://doi.org/10.1038/nature09372>

    Article <https://doi.org/10.1038%2Fnature09372> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFGntL%2FK> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20844487> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Direct%20visualization%20of%20secondary%20structures%20of%20F-actin%20by%20electron%20cryomicroscopy&journal=Nature&doi=10.1038%2Fnature09372&volume=467&pages=724-U117&publication_year=2010&author=Fujii%2CT&author=Iwane%2CAH&author=Yanagida%2CT&author=Namba%2CK> 

24.

    Fujiwara I, Vavylonis D, Pollard TD (2007) Polymerization kinetics
    of ADP- and ADP-Pi-actin determined by fluorescence microscopy. Proc
    Natl Acad Sci 104:8827–8832. https://doi.org/10.1073/pnas.0702510104
    <https://doi.org/10.1073/pnas.0702510104>

    Article <https://doi.org/10.1073%2Fpnas.0702510104> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXmt1akurk%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=17517656> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1885587> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Polymerization%20kinetics%20of%20ADP-%20and%20ADP-Pi-actin%20determined%20by%20fluorescence%20microscopy&journal=Proc%20Natl%20Acad%20Sci&doi=10.1073%2Fpnas.0702510104&volume=104&pages=8827-8832&publication_year=2007&author=Fujiwara%2CI&author=Vavylonis%2CD&author=Pollard%2CTD> 

25.

    Galkin VE, Orlova A, Schroder G, Egelman EH (2011) Structural
    polymorphism in F-actin. Biophys J 100:33–33

    Article <https://doi.org/10.1016%2Fj.bpj.2010.12.384> Google Scholar
    <http://scholar.google.com/scholar_lookup?&title=Structural%20polymorphism%20in%20F-actin&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2010.12.384&volume=100&pages=33-33&publication_year=2011&author=Galkin%2CVE&author=Orlova%2CA&author=Schroder%2CG&author=Egelman%2CEH> 

26.

    Goley ED, Rodenbusch SE, Martin AC, Welch MD (2004) Critical
    conformational changes in the Arp2/3 complex are induced by
    nucleotide and nucleation promoting factor. Mol Cell 16:269–279.
    https://doi.org/10.1016/j.molcel.2004.09.018
    <https://doi.org/10.1016/j.molcel.2004.09.018>

    Article <https://doi.org/10.1016%2Fj.molcel.2004.09.018> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXhtVWit73L> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15494313> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Critical%20conformational%20changes%20in%20the%20Arp2%2F3%20complex%20are%20induced%20by%20nucleotide%20and%20nucleation%20promoting%20factor&journal=Mol%20Cell&doi=10.1016%2Fj.molcel.2004.09.018&volume=16&pages=269-279&publication_year=2004&author=Goley%2CED&author=Rodenbusch%2CSE&author=Martin%2CAC&author=Welch%2CMD> 

27.

    Graceffa P, Dominguez R (2003) Crystal structure of monomeric actin
    in the ATP state - structural basis of nucleotide-dependent actin
    dynamics. J Biol Chem 278:34172–34180.
    https://doi.org/10.1074/jbc.M303689200
    <https://doi.org/10.1074/jbc.M303689200>

    Article <https://doi.org/10.1074%2Fjbc.M303689200> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXmvVCqs78%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=12813032> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Crystal%20structure%20of%20monomeric%20actin%20in%20the%20ATP%20state%20-%20structural%20basis%20of%20nucleotide-dependent%20actin%20dynamics&journal=J%20Biol%20Chem&doi=10.1074%2Fjbc.M303689200&volume=278&pages=34172-34180&publication_year=2003&author=Graceffa%2CP&author=Dominguez%2CR> 

28.

    Guo KK, Xiao WJ, Qiu D (2011) Polymerization of actin filaments
    coupled with adenosine triphosphate hydrolysis: Brownian dynamics
    and theoretical analysis. J Chem Phys 135
    https://doi.org/10.1063/1.3634006 <https://doi.org/10.1063/1.3634006>
29.

    Hocky GM, Baker JL, Bradley MJ, Sinitskiy AV, De La Cruz EM, Voth GA
    (2016) Cations stiffen actin filaments by adhering a key structural
    element to adjacent subunits. J Phys Chem B 120:4558–4567.
    https://doi.org/10.1021/acs.jpcb.6b02741
    <https://doi.org/10.1021/acs.jpcb.6b02741>

    Article <https://doi.org/10.1021%2Facs.jpcb.6b02741> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xnt12rsL4%3D> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4959277> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=27146246> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Cations%20stiffen%20actin%20filaments%20by%20adhering%20a%20key%20structural%20element%20to%20adjacent%20subunits&journal=J%20Phys%20Chem%20B&doi=10.1021%2Facs.jpcb.6b02741&volume=120&pages=4558-4567&publication_year=2016&author=Hocky%2CGM&author=Baker%2CJL&author=Bradley%2CMJ&author=Sinitskiy%2CAV&author=Cruz%2CEM&author=Voth%2CGA> 

30.

    Iwasa M, Maeda K, Narita A, Maeda Y, Oda T (2008) Dual roles of
    Gln(137) of actin revealed by recombinant human cardiac muscle
    alpha-actin mutants. J Biol Chem 283:21045–21053.
    https://doi.org/10.1074/jbc.M800570200
    <https://doi.org/10.1074/jbc.M800570200>

    Article <https://doi.org/10.1074%2Fjbc.M800570200> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXoslWrt7g%3D> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3258932> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18515362> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Dual%20roles%20of%20Gln%28137%29%20of%20actin%20revealed%20by%20recombinant%20human%20cardiac%20muscle%20alpha-actin%20mutants&journal=J%20Biol%20Chem&doi=10.1074%2Fjbc.M800570200&volume=283&pages=21045-21053&publication_year=2008&author=Iwasa%2CM&author=Maeda%2CK&author=Narita%2CA&author=Maeda%2CY&author=Oda%2CT> 

31.

    Jégou A, Niedermayer T, Lipowsky R, Carlier M-F, Romet-Lemonne G
    (2013) On phosphate release in actin filaments. Biophys J
    104:2778–2779. https://doi.org/10.1016/j.bpj.2013.05.019
    <https://doi.org/10.1016/j.bpj.2013.05.019>

    Article <https://doi.org/10.1016%2Fj.bpj.2013.05.019> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXpslKltLw%3D> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3686342> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23790388> Google Scholar <http://scholar.google.com/scholar_lookup?&title=On%20phosphate%20release%20in%20actin%20filaments&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2013.05.019&volume=104&pages=2778-2779&publication_year=2013&author=J%C3%A9gou%2CA&author=Niedermayer%2CT&author=Lipowsky%2CR&author=Carlier%2CM-F&author=Romet-Lemonne%2CG> 

32.

    Jégou A, Niedermayer T, Orban J, Didry D, Lipowsky R, M-f C,
    Romet-lemonne G (2011) Individual actin filaments in a microfluidic
    flow reveal the mechanism of Atp hydrolysis and give insight into
    the properties of profilin. PLoS Biol 9:1001161

    Article <https://doi.org/10.1371%2Fjournal.pbio.1001161> Google
    Scholar
    <http://scholar.google.com/scholar_lookup?&title=Individual%20actin%20filaments%20in%20a%20microfluidic%20flow%20reveal%20the%20mechanism%20of%20Atp%20hydrolysis%20and%20give%20insight%20into%20the%20properties%20of%20profilin&journal=PLoS%20Biol&doi=10.1371%2Fjournal.pbio.1001161&volume=9&publication_year=2011&author=J%C3%A9gou%2CA&author=Niedermayer%2CT&author=Orban%2CJ&author=Didry%2CD&author=Lipowsky%2CR&author=M-f%2CC&author=Romet-lemonne%2CG> 

33.

    Kabsch W, Mannherz HG, Suck D, Pai EF, Holmes KC (1990) Atomic
    structure of the actin: DNase I complex. Nature 347:37–44

    Article <https://doi.org/10.1038%2F347037a0> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaK3cXls1yjsLk%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=2395459> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Atomic%20structure%20of%20the%20actin%3A%20DNase%20I%20complex&journal=Nature&doi=10.1038%2F347037a0&volume=347&pages=37-44&publication_year=1990&author=Kabsch%2CW&author=Mannherz%2CHG&author=Suck%2CD&author=Pai%2CEF&author=Holmes%2CKC> 

34.

    Kang H, Bradley MJ, Elam WA, De La Cruz EM (2013) Regulation of
    actin by ion-linked equilibria. Biophys J 105:2621–2628.
    https://doi.org/10.1016/j.bpj.2013.10.032
    <https://doi.org/10.1016/j.bpj.2013.10.032>

    Article <https://doi.org/10.1016%2Fj.bpj.2013.10.032> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFOlsL3N> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3882474> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24359734> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Regulation%20of%20actin%20by%20ion-linked%20equilibria&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2013.10.032&volume=105&pages=2621-2628&publication_year=2013&author=Kang%2CH&author=Bradley%2CMJ&author=Elam%2CWA&author=Cruz%2CEM> 

35.

    Kang HR, Bradley MJ, McCullough BR, Pierre A, Grintsevich EE,
    Reisler E, De La Cruz EM (2012) Identification of cation-binding
    sites on actin that drive polymerization and modulate bending
    stiffness. P Natl Acad Sci USA 109:16923–16927.
    https://doi.org/10.1073/pnas.1211078109
    <https://doi.org/10.1073/pnas.1211078109>

    Article <https://doi.org/10.1073%2Fpnas.1211078109> Google Scholar
    <http://scholar.google.com/scholar_lookup?&title=Identification%20of%20cation-binding%20sites%20on%20actin%20that%20drive%20polymerization%20and%20modulate%20bending%20stiffness&journal=P%20Natl%20Acad%20Sci%20USA&doi=10.1073%2Fpnas.1211078109&volume=109&pages=16923-16927&publication_year=2012&author=Kang%2CHR&author=Bradley%2CMJ&author=McCullough%2CBR&author=Pierre%2CA&author=Grintsevich%2CEE&author=Reisler%2CE&author=Cruz%2CEM> 

36.

    Katkar HH, Davtyan A, Durumeric AEP, Hocky GM, Schramm AC, De La
    Cruz EM, Voth GA (2018) Insights into the cooperative nature of ATP
    hydrolysis in actin filaments. Biophys J in press
    https://doi.org/10.1016/j.bpj.2018.08.034
    <https://doi.org/10.1016/j.bpj.2018.08.034>
37.

    Korn E, Carlier M, Pantaloni D (1987) Actin polymerization and Atp
    hydrolysis. Science 238:638–644

    Article <https://doi.org/10.1126%2Fscience.3672117> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaL1cXltFehuw%3D%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=3672117> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20polymerization%20and%20Atp%20hydrolysis&journal=Science&doi=10.1126%2Fscience.3672117&volume=238&pages=638-644&publication_year=1987&author=Korn%2CE&author=Carlier%2CM&author=Pantaloni%2CD> 

38.

    Kovar DR, Harris ES, Mahaffy R, Higgs HN, Pollard TD (2006) Control
    of the assembly of ATP- and ADP-actin by formins and profilin. Cell
    124:423–435

    Article <https://doi.org/10.1016%2Fj.cell.2005.11.038> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xht1Kktbk%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16439214> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Control%20of%20the%20assembly%20of%20ATP-%20and%20ADP-actin%20by%20formins%20and%20profilin&journal=Cell&doi=10.1016%2Fj.cell.2005.11.038&volume=124&pages=423-435&publication_year=2006&author=Kovar%2CDR&author=Harris%2CES&author=Mahaffy%2CR&author=Higgs%2CHN&author=Pollard%2CTD> 

39.

    Lyman E, Pfaendtner J, Voth GA (2008) Systematic multiscale
    parameterization of heterogeneous elastic network models of
    proteins. Biophys J 95:4183–4192.
    https://doi.org/10.1529/biophysj.108.139733
    <https://doi.org/10.1529/biophysj.108.139733>

    Article <https://doi.org/10.1529%2Fbiophysj.108.139733> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlagu7vF> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2567941> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18658214> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Systematic%20multiscale%20parameterization%20of%20heterogeneous%20elastic%20network%20models%20of%20proteins&journal=Biophys%20J&doi=10.1529%2Fbiophysj.108.139733&volume=95&pages=4183-4192&publication_year=2008&author=Lyman%2CE&author=Pfaendtner%2CJ&author=Voth%2CGA> 

40.

    Machesky LM, Atkinson SJ, Ampe C, Vandekerckhove J, Pollard TD
    (1994) Purification of a cortical complex containing 2
    unconventional actins from Acanthamoeba by affinity-chromatography
    on profilin-agarose. J Cell Biol 127:107–115.
    https://doi.org/10.1083/jcb.127.1.107
    <https://doi.org/10.1083/jcb.127.1.107>

    Article <https://doi.org/10.1083%2Fjcb.127.1.107> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaK2cXmslKgsLs%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=7929556> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Purification%20of%20a%20cortical%20complex%20containing%202%20unconventional%20actins%20from%20Acanthamoeba%20by%20affinity-chromatography%20on%20profilin-agarose&journal=J%20Cell%20Biol&doi=10.1083%2Fjcb.127.1.107&volume=127&pages=107-115&publication_year=1994&author=Machesky%2CLM&author=Atkinson%2CSJ&author=Ampe%2CC&author=Vandekerckhove%2CJ&author=Pollard%2CTD> 

41.

    McCullagh M, Saunders MG, Voth GA (2014) Unraveling the mystery of
    ATP hydrolysis in actin filaments. J Am Chem Soc 136:13053–13058.
    https://doi.org/10.1021/ja507169f <https://doi.org/10.1021/ja507169f>

    Article <https://doi.org/10.1021%2Fja507169f> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsVOnt7vI> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4183606> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=25181471> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Unraveling%20the%20mystery%20of%20ATP%20hydrolysis%20in%20actin%20filaments&journal=J%20Am%20Chem%20Soc&doi=10.1021%2Fja507169f&volume=136&pages=13053-13058&publication_year=2014&author=McCullagh%2CM&author=Saunders%2CMG&author=Voth%2CGA> 

42.

    Mehrafrooz B, Shamloo A (2018) Mechanical differences between ATP
    and ADP actin states: a molecular dynamics study. J Theor Biol
    448:94–103. https://doi.org/10.1016/j.jtbi.2018.04.010
    <https://doi.org/10.1016/j.jtbi.2018.04.010>

    Article <https://doi.org/10.1016%2Fj.jtbi.2018.04.010> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXotFKgsbc%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=29634959> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Mechanical%20differences%20between%20ATP%20and%20ADP%20actin%20states%3A%20a%20molecular%20dynamics%20study&journal=J%20Theor%20Biol&doi=10.1016%2Fj.jtbi.2018.04.010&volume=448&pages=94-103&publication_year=2018&author=Mehrafrooz%2CB&author=Shamloo%2CA> 

43.

    Merino F et al (2018) Structural transitions of F-actin upon ATP
    hydrolysis at near-atomic resolution revealed by cryo-EM. Nat Struct
    Mol Biol 25:531−+. https://doi.org/10.1038/s41594-018-0074-0
    <https://doi.org/10.1038/s41594-018-0074-0>

    Article <https://doi.org/10.1038%2Fs41594-018-0074-0> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlKqu7nP> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20transitions%20of%20F-actin%20upon%20ATP%20hydrolysis%20at%20near-atomic%20resolution%20revealed%20by%20cryo-EM&journal=Nat%20Struct%20Mol%20Biol&doi=10.1038%2Fs41594-018-0074-0&volume=25&publication_year=2018&author=Merino%2CF> 

44.

    Murakami K, Yasunaga T, Noguchi TQP, Gomibuchi Y, Ngo KX, Uyeda TQP,
    Wakabayashi T (2010) Structural basis for actin assembly, activation
    of ATP hydrolysis, and delayed phosphate release. Cell 143:275–287.
    https://doi.org/10.1016/j.cell.2010.09.034
    <https://doi.org/10.1016/j.cell.2010.09.034>

    Article <https://doi.org/10.1016%2Fj.cell.2010.09.034> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXht12qurnL> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20946985> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20for%20actin%20assembly%2C%20activation%20of%20ATP%20hydrolysis%2C%20and%20delayed%20phosphate%20release&journal=Cell&doi=10.1016%2Fj.cell.2010.09.034&volume=143&pages=275-287&publication_year=2010&author=Murakami%2CK&author=Yasunaga%2CT&author=Noguchi%2CTQP&author=Gomibuchi%2CY&author=Ngo%2CKX&author=Uyeda%2CTQP&author=Wakabayashi%2CT> 

45.

    Narita A, Oda T, Maeda Y (2011) Structural basis for the slow
    dynamics of the actin filament pointed end. EMBO J 30:1230–1237.
    https://doi.org/10.1038/emboj.2011.48
    <https://doi.org/10.1038/emboj.2011.48>

    Article <https://doi.org/10.1038%2Femboj.2011.48> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXislyjsbY%3D> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3094112> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21378753> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20for%20the%20slow%20dynamics%20of%20the%20actin%20filament%20pointed%20end&journal=EMBO%20J&doi=10.1038%2Femboj.2011.48&volume=30&pages=1230-1237&publication_year=2011&author=Narita%2CA&author=Oda%2CT&author=Maeda%2CY> 

46.

    Oda T, Iwasa M, Aihara T, Maeda Y, Narita A (2009) The nature of the
    globular-to fibrous-actin transition. Nature 457:441–445.
    https://doi.org/10.1038/nature07685
    <https://doi.org/10.1038/nature07685>

    Article <https://doi.org/10.1038%2Fnature07685> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXotlOnsQ%3D%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19158791> Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20nature%20of%20the%20globular-to%20fibrous-actin%20transition&journal=Nature&doi=10.1038%2Fnature07685&volume=457&pages=441-445&publication_year=2009&author=Oda%2CT&author=Iwasa%2CM&author=Aihara%2CT&author=Maeda%2CY&author=Narita%2CA> 

47.

    Ohm T, Wegner A (1994) Mechanism of Atp hydrolysis by polymeric
    actin. Bba-Protein Struct M 1208:8–14

    Article <https://doi.org/10.1016%2F0167-4838%2894%2990153-8> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaK2cXmsV2gs78%3D> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Mechanism%20of%20Atp%20hydrolysis%20by%20polymeric%20actin&journal=Bba-Protein%20Struct%20M&doi=10.1016%2F0167-4838%2894%2990153-8&volume=1208&pages=8-14&publication_year=1994&author=Ohm%2CT&author=Wegner%2CA> 

48.

    Otomo T, Tomchick DR, Otomo C, Panchal SC, Machius M, Rosen MK
    (2005) Structural basis of actin filament nucleation and processive
    capping by a formin homology 2 domain. Nature 433:488–494

    Article <https://doi.org/10.1038%2Fnature03251> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXpt1Clsg%3D%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15635372> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structural%20basis%20of%20actin%20filament%20nucleation%20and%20processive%20capping%20by%20a%20formin%20homology%202%20domain&journal=Nature&doi=10.1038%2Fnature03251&volume=433&pages=488-494&publication_year=2005&author=Otomo%2CT&author=Tomchick%2CDR&author=Otomo%2CC&author=Panchal%2CSC&author=Machius%2CM&author=Rosen%2CMK> 

49.

    Otterbein LR, Graceffa P, Dominguez R (2001) The crystal structure
    of uncomplexed actin in the ADP state. Science 293:708–711.
    https://doi.org/10.1126/science.1059700
    <https://doi.org/10.1126/science.1059700>

    Article <https://doi.org/10.1126%2Fscience.1059700> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXls1Kmt7Y%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=11474115> Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20crystal%20structure%20of%20uncomplexed%20actin%20in%20the%20ADP%20state&journal=Science&doi=10.1126%2Fscience.1059700&volume=293&pages=708-711&publication_year=2001&author=Otterbein%2CLR&author=Graceffa%2CP&author=Dominguez%2CR> 

50.

    Pantaloni D, Hill T, Carlier M, Korn E (1985) A model for actin
    polymerization and the kinetic effects of Atp hydrolysis. P Natl
    Acad Sci USA 82:7207–7211

    Article <https://doi.org/10.1073%2Fpnas.82.21.7207> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaL2MXmtV2ht7o%3D> Google Scholar <http://scholar.google.com/scholar_lookup?&title=A%20model%20for%20actin%20polymerization%20and%20the%20kinetic%20effects%20of%20Atp%20hydrolysis&journal=P%20Natl%20Acad%20Sci%20USA&doi=10.1073%2Fpnas.82.21.7207&volume=82&pages=7207-7211&publication_year=1985&author=Pantaloni%2CD&author=Hill%2CT&author=Carlier%2CM&author=Korn%2CE> 

51.

    Paul AS, Paul A, Pollard TD, Pollard T (2008) The role of the FH1
    domain and profilin in formin-mediated actin-filament elongation and
    nucleation. Curr Biol : CB 18:9–19

    Article <https://doi.org/10.1016%2Fj.cub.2007.11.062> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXivFagsw%3D%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18160294> Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20role%20of%20the%20FH1%20domain%20and%20profilin%20in%20formin-mediated%20actin-filament%20elongation%20and%20nucleation&journal=Curr%20Biol%20%3A%20CB&doi=10.1016%2Fj.cub.2007.11.062&volume=18&pages=9-19&publication_year=2008&author=Paul%2CAS&author=Paul%2CA&author=Pollard%2CTD&author=Pollard%2CT> 

52.

    Paul AS, Pollard TD (2009a) Energetic requirements for processive
    elongation of actin filaments by FH1FH2-formins. J Biol Chem
    284:12533–12540

    Article <https://doi.org/10.1074%2Fjbc.M808587200> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXltVClsL4%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19251693> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2673319> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Energetic%20requirements%20for%20processive%20elongation%20of%20actin%20filaments%20by%20FH1FH2-formins&journal=J%20Biol%20Chem&doi=10.1074%2Fjbc.M808587200&volume=284&pages=12533-12540&publication_year=2009&author=Paul%2CAS&author=Pollard%2CTD> 

53.

    Paul AS, Pollard TD (2009b) Review of the mechanism of processive
    actin filament elongation by formins. Cell Motil Cytoskeleton 66:606–617

    Article <https://doi.org/10.1002%2Fcm.20379> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVKntr%2FK> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19459187> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2871153> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Review%20of%20the%20mechanism%20of%20processive%20actin%20filament%20elongation%20by%20formins&journal=Cell%20Motil%20Cytoskeleton&doi=10.1002%2Fcm.20379&volume=66&pages=606-617&publication_year=2009&author=Paul%2CAS&author=Pollard%2CTD> 

54.

    Pfaendtner J, Branduardi D, Parrinello M, Pollard TD, Voth GA (2009)
    Nucleotide-dependent conformational states of actin. P Natl Acad Sci
    USA 106:12723–12728. https://doi.org/10.1073/pnas.0902092106
    <https://doi.org/10.1073/pnas.0902092106>

    Article <https://doi.org/10.1073%2Fpnas.0902092106> Google Scholar
    <http://scholar.google.com/scholar_lookup?&title=Nucleotide-dependent%20conformational%20states%20of%20actin&journal=P%20Natl%20Acad%20Sci%20USA&doi=10.1073%2Fpnas.0902092106&volume=106&pages=12723-12728&publication_year=2009&author=Pfaendtner%2CJ&author=Branduardi%2CD&author=Parrinello%2CM&author=Pollard%2CTD&author=Voth%2CGA> 

55.

    Pfaendtner J, De La Cruz EM, Voth GA (2010a) Actin filament
    remodeling by actin depolymerization factor/cofilin. P Natl Acad Sci
    USA 107:7299–7304. https://doi.org/10.1073/pnas.0911675107
    <https://doi.org/10.1073/pnas.0911675107>

    Article <https://doi.org/10.1073%2Fpnas.0911675107> Google Scholar
    <http://scholar.google.com/scholar_lookup?&title=Actin%20filament%20remodeling%20by%20actin%20depolymerization%20factor%2Fcofilin&journal=P%20Natl%20Acad%20Sci%20USA&doi=10.1073%2Fpnas.0911675107&volume=107&pages=7299-7304&publication_year=2010&author=Pfaendtner%2CJ&author=Cruz%2CEM&author=Voth%2CGA> 

56.

    Pfaendtner J, Lyman E, Pollard TD, Voth GA (2010b) Structure and
    dynamics of the actin filament. J Mol Biol 396:252–263.
    https://doi.org/10.1016/j.jmb.2009.11.034
    <https://doi.org/10.1016/j.jmb.2009.11.034>

    Article <https://doi.org/10.1016%2Fj.jmb.2009.11.034> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXht12mtb0%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=19931282> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Structure%20and%20dynamics%20of%20the%20actin%20filament&journal=J%20Mol%20Biol&doi=10.1016%2Fj.jmb.2009.11.034&volume=396&pages=252-263&publication_year=2010&author=Pfaendtner%2CJ&author=Lyman%2CE&author=Pollard%2CTD&author=Voth%2CGA> 

57.

    Pfaendtner J, Volkmann N, Hanein D, Dalhaimer P, Pollard TD, Voth GA
    (2012) Key structural features of the actin filament Arp2/3 complex
    branch junction revealed by molecular simulation. J Mol Biol
    416:148–161. https://doi.org/10.1016/j.jmb.2011.12.025
    <https://doi.org/10.1016/j.jmb.2011.12.025>

    Article <https://doi.org/10.1016%2Fj.jmb.2011.12.025> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xhs1Cqsrc%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22206989> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Key%20structural%20features%20of%20the%20actin%20filament%20Arp2%2F3%20complex%20branch%20junction%20revealed%20by%20molecular%20simulation&journal=J%20Mol%20Biol&doi=10.1016%2Fj.jmb.2011.12.025&volume=416&pages=148-161&publication_year=2012&author=Pfaendtner%2CJ&author=Volkmann%2CN&author=Hanein%2CD&author=Dalhaimer%2CP&author=Pollard%2CTD&author=Voth%2CGA> 

58.

    Pfaendtner J, Voth GA (2008) Molecular dynamics simulation and
    coarse-grained analysis of the Arp2/3 complex. Biophys J
    95:5324–5333. https://doi.org/10.1529/biophysj.108.143313
    <https://doi.org/10.1529/biophysj.108.143313>

    Article <https://doi.org/10.1529%2Fbiophysj.108.143313> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsVymsbnO> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2586551> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18805923> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Molecular%20dynamics%20simulation%20and%20coarse-grained%20analysis%20of%20the%20Arp2%2F3%20complex&journal=Biophys%20J&doi=10.1529%2Fbiophysj.108.143313&volume=95&pages=5324-5333&publication_year=2008&author=Pfaendtner%2CJ&author=Voth%2CGA> 

59.

    Pieper U, Wegner A (1996) The end of a polymerizing actin filament
    contains numerous Atp-subunit segments that are disconnected by
    Adp-subunits resulting from Atp hydrolysis. Biochemistry 35:4396–4402

    Article <https://doi.org/10.1021%2Fbi9527045> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaK28Xhs12ms7w%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=8605188> Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20end%20of%20a%20polymerizing%20actin%20filament%20contains%20numerous%20Atp-subunit%20segments%20that%20are%20disconnected%20by%20Adp-subunits%20resulting%20from%20Atp%20hydrolysis&journal=Biochemistry&doi=10.1021%2Fbi9527045&volume=35&pages=4396-4402&publication_year=1996&author=Pieper%2CU&author=Wegner%2CA> 

60.

    Pollard T, Weeds A (1984) The rate-constant for Atp hydrolysis by
    polymerized actin. FEBS Lett 170:94–98

    Article <https://doi.org/10.1016%2F0014-5793%2884%2981376-9> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaL2cXksFSgt7c%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=6427006> Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20rate-constant%20for%20Atp%20hydrolysis%20by%20polymerized%20actin&journal=FEBS%20Lett&doi=10.1016%2F0014-5793%2884%2981376-9&volume=170&pages=94-98&publication_year=1984&author=Pollard%2CT&author=Weeds%2CA> 

61.

    Pollard TD (1986) Rate constants for the reactions of Atp-actin and
    Adp-actin with the ends of actin-filaments. J Cell Biol
    103:2747–2754. https://doi.org/10.1083/jcb.103.6.2747
    <https://doi.org/10.1083/jcb.103.6.2747>

    Article <https://doi.org/10.1083%2Fjcb.103.6.2747> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaL2sXhtFGnsrg%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=3793756> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Rate%20constants%20for%20the%20reactions%20of%20Atp-actin%20and%20Adp-actin%20with%20the%20ends%20of%20actin-filaments&journal=J%20Cell%20Biol&doi=10.1083%2Fjcb.103.6.2747&volume=103&pages=2747-2754&publication_year=1986&author=Pollard%2CTD> 

62.

    Pollard TD, Cooper JA (2009) Actin, a central player in cell shape
    and movement. Science (New York, NY) 326:1208–1212

    Article <https://doi.org/10.1126%2Fscience.1175862> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhsVKhur7L> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%2C%20a%20central%20player%20in%20cell%20shape%20and%20movement&journal=Science%20%28New%20York%2C%20NY%29&doi=10.1126%2Fscience.1175862&volume=326&pages=1208-1212&publication_year=2009&author=Pollard%2CTD&author=Cooper%2CJA> 

63.

    Popov K, Komianos J, Papoian GA (2016) MEDYAN: mechanochemical
    simulations of contraction and polarity alignment in actomyosin
    networks. PLoS Comput Biol 12
    https://doi.org/10.1371/journal.pcbi.1004877
    <https://doi.org/10.1371/journal.pcbi.1004877>
64.

    Ranjith P, Mallick K, J-f J, Lacoste D (2010) Role of Atp-hydrolysis
    in the dynamics of a single actin filament. Biophys J 98:1418–1427

    Article <https://doi.org/10.1016%2Fj.bpj.2009.12.4306> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXht1Wnsr3O> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=20409460> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2856186> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Role%20of%20Atp-hydrolysis%20in%20the%20dynamics%20of%20a%20single%20actin%20filament&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2009.12.4306&volume=98&pages=1418-1427&publication_year=2010&author=Ranjith%2CP&author=Mallick%2CK&author=J-f%2CJ&author=Lacoste%2CD> 

65.

    Rouiller I et al (2008) The structural basis of actin filament
    branching by the Arp2/3 complex. J Cell Biol 180:887–895.
    https://doi.org/10.1083/jcb.200709092
    <https://doi.org/10.1083/jcb.200709092>

    Article <https://doi.org/10.1083%2Fjcb.200709092> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXjtlKmtb0%3D> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2265399> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18316411> Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20structural%20basis%20of%20actin%20filament%20branching%20by%20the%20Arp2%2F3%20complex&journal=J%20Cell%20Biol&doi=10.1083%2Fjcb.200709092&volume=180&pages=887-895&publication_year=2008&author=Rouiller%2CI> 

66.

    Rould MA, Wan Q, Joel PB, Lowey S, Trybus KM (2006) Crystal
    structures of expressed non-polymerizable monomeric actin in the ADP
    and ATP states. J Biol Chem 281:31909–31919.
    https://doi.org/10.1074/jbc.M601973200
    <https://doi.org/10.1074/jbc.M601973200>

    Article <https://doi.org/10.1074%2Fjbc.M601973200> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD28XhtVOrtrvE> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16920713> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Crystal%20structures%20of%20expressed%20non-polymerizable%20monomeric%20actin%20in%20the%20ADP%20and%20ATP%20states&journal=J%20Biol%20Chem&doi=10.1074%2Fjbc.M601973200&volume=281&pages=31909-31919&publication_year=2006&author=Rould%2CMA&author=Wan%2CQ&author=Joel%2CPB&author=Lowey%2CS&author=Trybus%2CKM> 

67.

    Saunders MG, Tempkin J, Weare J, Dinner AR, Roux B, Voth GA (2014)
    Nucleotide regulation of the structure and dynamics of G-actin.
    Biophys J 106:1710–1720. https://doi.org/10.1016/j.bpj.2014.03.012
    <https://doi.org/10.1016/j.bpj.2014.03.012>

    Article <https://doi.org/10.1016%2Fj.bpj.2014.03.012> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXmslahu7k%3D> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4008840> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=24739170> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Nucleotide%20regulation%20of%20the%20structure%20and%20dynamics%20of%20G-actin&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2014.03.012&volume=106&pages=1710-1720&publication_year=2014&author=Saunders%2CMG&author=Tempkin%2CJ&author=Weare%2CJ&author=Dinner%2CAR&author=Roux%2CB&author=Voth%2CGA> 

68.

    Saunders MG, Voth GA (2011) Water molecules in the nucleotide
    binding cleft of actin: effects on subunit conformation and
    implications for ATP hydrolysis. J Mol Biol 413:279–291.
    https://doi.org/10.1016/j.jmb.2011.07.068
    <https://doi.org/10.1016/j.jmb.2011.07.068>

    Article <https://doi.org/10.1016%2Fj.jmb.2011.07.068> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1Grtr3K> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=21856312> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Water%20molecules%20in%20the%20nucleotide%20binding%20cleft%20of%20actin%3A%20effects%20on%20subunit%20conformation%20and%20implications%20for%20ATP%20hydrolysis&journal=J%20Mol%20Biol&doi=10.1016%2Fj.jmb.2011.07.068&volume=413&pages=279-291&publication_year=2011&author=Saunders%2CMG&author=Voth%2CGA> 

69.

    Saunders MG, Voth GA (2012a) Coarse-grained analysis and modeling of
    nucleotide-dependent changes in F-actin. Biophys J 102:237a.
    https://doi.org/10.1016/j.bpj.2011.11.1306
    <https://doi.org/10.1016/j.bpj.2011.11.1306>

    Article <https://doi.org/10.1016%2Fj.bpj.2011.11.1306> Google
    Scholar
    <http://scholar.google.com/scholar_lookup?&title=Coarse-grained%20analysis%20and%20modeling%20of%20nucleotide-dependent%20changes%20in%20F-actin&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2011.11.1306&volume=102&publication_year=2012&author=Saunders%2CMG&author=Voth%2CGA> 

70.

    Saunders MG, Voth GA (2012b) Comparison between actin filament
    models: coarse-graining reveals essential differences. Structure
    20:641–653

    Article <https://doi.org/10.1016%2Fj.str.2012.02.008> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xltl2ktbo%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22483111> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Comparison%20between%20actin%20filament%20models%3A%20coarse-graining%20reveals%20essential%20differences&journal=Structure&doi=10.1016%2Fj.str.2012.02.008&volume=20&pages=641-653&publication_year=2012&author=Saunders%2CMG&author=Voth%2CGA> 

71.

    Saunders MG, Voth GA (2013) Coarse-graining methods for
    computational biology. Annu Rev Biophys 42:73–93.
    https://doi.org/10.1146/annurev-biophys-083012-130348
    <https://doi.org/10.1146/annurev-biophys-083012-130348>

    Article
    <https://doi.org/10.1146%2Fannurev-biophys-083012-130348> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFGrs7bN> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=23451897> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Coarse-graining%20methods%20for%20computational%20biology&journal=Annu%20Rev%20Biophys&doi=10.1146%2Fannurev-biophys-083012-130348&volume=42&pages=73-93&publication_year=2013&author=Saunders%2CMG&author=Voth%2CGA> 

72.

    Schramm AC, Hocky GM, Voth GA, Blanchoin L, Martiel JL, De La Cruz
    EM (2017) Actin filament strain promotes severing and cofilin
    dissociation. Biophys J 112:2624–2633.
    https://doi.org/10.1016/j.bpj.2017.05.016
    <https://doi.org/10.1016/j.bpj.2017.05.016>

    Article <https://doi.org/10.1016%2Fj.bpj.2017.05.016> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXovVKhtLs%3D> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5479148> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28636918> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20filament%20strain%20promotes%20severing%20and%20cofilin%20dissociation&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2017.05.016&volume=112&pages=2624-2633&publication_year=2017&author=Schramm%2CAC&author=Hocky%2CGM&author=Voth%2CGA&author=Blanchoin%2CL&author=Martiel%2CJL&author=Cruz%2CEM> 

73.

    Schuler H (2001) ATPase activity and conformational changes in the
    regulation of actin. Bba-Protein Struct M 1549:137–147.
    https://doi.org/10.1016/S0167-4838(01)00255-2
    <https://doi.org/10.1016/S0167-4838(01)00255-2>

    Article <https://doi.org/10.1016%2FS0167-4838%2801%2900255-2> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXnvFCrsrY%3D> Google Scholar <http://scholar.google.com/scholar_lookup?&title=ATPase%20activity%20and%20conformational%20changes%20in%20the%20regulation%20of%20actin&journal=Bba-Protein%20Struct%20M&doi=10.1016%2FS0167-4838%2801%2900255-2&volume=1549&pages=137-147&publication_year=2001&author=Schuler%2CH> 

74.

    Schuler H, Korenbaum E, Schutt CE, Lindberg U, Karlsson R (1999)
    Mutational analysis of Ser14 and Asp157 in the nucleotide-binding
    site of beta-actin. Eur J Biochem 265:210–220.
    https://doi.org/10.1046/j.1432-1327.1999.00716.x
    <https://doi.org/10.1046/j.1432-1327.1999.00716.x>

    Article <https://doi.org/10.1046%2Fj.1432-1327.1999.00716.x> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaK1MXmsFKks7w%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=10491176> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Mutational%20analysis%20of%20Ser14%20and%20Asp157%20in%20the%20nucleotide-binding%20site%20of%20beta-actin&journal=Eur%20J%20Biochem&doi=10.1046%2Fj.1432-1327.1999.00716.x&volume=265&pages=210-220&publication_year=1999&author=Schuler%2CH&author=Korenbaum%2CE&author=Schutt%2CCE&author=Lindberg%2CU&author=Karlsson%2CR> 

75.

    Stukalin E, Kolomeisky A (2006) Atp hydrolysis stimulates large
    length fluctuations in single actin filaments. Biophys J 90:2673–2685

    Article <https://doi.org/10.1529%2Fbiophysj.105.074211> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD28XjsFeqtLw%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16443647> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1414574> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Atp%20hydrolysis%20stimulates%20large%20length%20fluctuations%20in%20single%20actin%20filaments&journal=Biophys%20J&doi=10.1529%2Fbiophysj.105.074211&volume=90&pages=2673-2685&publication_year=2006&author=Stukalin%2CE&author=Kolomeisky%2CA> 

76.

    Sun R, Sode O, Dama JF, Voth GA (2017) Simulating protein mediated
    hydrolysis of ATP and other nucleoside triphosphates by combining
    QM/MM molecular dynamics with advances in metadynamics. J Chem
    Theory Comput 13:2332–2341. https://doi.org/10.1021/acs.jctc.7b00077
    <https://doi.org/10.1021/acs.jctc.7b00077>

    Article <https://doi.org/10.1021%2Facs.jctc.7b00077> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXltVSktLo%3D> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5425946> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28345907> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Simulating%20protein%20mediated%20hydrolysis%20of%20ATP%20and%20other%20nucleoside%20triphosphates%20by%20combining%20QM%2FMM%20molecular%20dynamics%20with%20advances%20in%20metadynamics&journal=J%20Chem%20Theory%20Comput&doi=10.1021%2Facs.jctc.7b00077&volume=13&pages=2332-2341&publication_year=2017&author=Sun%2CR&author=Sode%2CO&author=Dama%2CJF&author=Voth%2CGA> 

77.

    Vavylonis D, Kovar DR, O'Shaughnessy B, Pollard TD (2006) Model of
    formin-associated actin filament elongation. Mol Cell 21:455–466

    Article <https://doi.org/10.1016%2Fj.molcel.2006.01.016> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD28XisVansLc%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16483928> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3716371> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Model%20of%20formin-associated%20actin%20filament%20elongation&journal=Mol%20Cell&doi=10.1016%2Fj.molcel.2006.01.016&volume=21&pages=455-466&publication_year=2006&author=Vavylonis%2CD&author=Kovar%2CDR&author=O%27Shaughnessy%2CB&author=Pollard%2CTD> 

78.

    Vavylonis D, Yang Q, O'Shaughnessy B (2005) Actin polymerization
    kinetics, cap structure, and fluctuations. P Natl Acad Sci USA
    102:8543–8548

    Article <https://doi.org/10.1073%2Fpnas.0501435102> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXlsFCgtr4%3D> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Actin%20polymerization%20kinetics%2C%20cap%20structure%2C%20and%20fluctuations&journal=P%20Natl%20Acad%20Sci%20USA&doi=10.1073%2Fpnas.0501435102&volume=102&pages=8543-8548&publication_year=2005&author=Vavylonis%2CD&author=Yang%2CQ&author=O%27Shaughnessy%2CB> 

79.

    Vorobiev S et al (2003) The structure of nonvertebrate actin:
    implications for the ATP hydrolytic mechanism. P Natl Acad Sci USA
    100:5760–5765. https://doi.org/10.1073/pnas.0832273100
    <https://doi.org/10.1073/pnas.0832273100>

    Article <https://doi.org/10.1073%2Fpnas.0832273100> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXjvFOls7o%3D> Google Scholar <http://scholar.google.com/scholar_lookup?&title=The%20structure%20of%20nonvertebrate%20actin%3A%20implications%20for%20the%20ATP%20hydrolytic%20mechanism&journal=P%20Natl%20Acad%20Sci%20USA&doi=10.1073%2Fpnas.0832273100&volume=100&pages=5760-5765&publication_year=2003&author=Vorobiev%2CS> 

80.

    Voth GA (2017) A multiscale description of biomolecular active
    matter: the chemistry underlying many life processes. Acc Chem Res
    50:594–598. https://doi.org/10.1021/acs.accounts.6b00572
    <https://doi.org/10.1021/acs.accounts.6b00572>

    Article <https://doi.org/10.1021%2Facs.accounts.6b00572> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXkt1Wkt7c%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=28945406> Google Scholar <http://scholar.google.com/scholar_lookup?&title=A%20multiscale%20description%20of%20biomolecular%20active%20matter%3A%20the%20chemistry%20underlying%20many%20life%20processes&journal=Acc%20Chem%20Res&doi=10.1021%2Facs.accounts.6b00572&volume=50&pages=594-598&publication_year=2017&author=Voth%2CGA> 

81.

    Wertman KF, Drubin DG, Botstein D (1992) Systematic mutational
    analysis of the yeast. ACT1 gene Genetics 132:337–350

    CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DyaK3sXhvVajug%3D%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=1427032> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Systematic%20mutational%20analysis%20of%20the%20yeast&journal=ACT1%20gene%20Genetics&volume=132&pages=337-350&publication_year=1992&author=Wertman%2CKF&author=Drubin%2CDG&author=Botstein%2CD> 

82.

    Xu YW, Moseley JB, Sagot I, Poy F, Pellman D, Goode BL, Eck MJ
    (2004) Crystal structures of a formin homology-2 domain reveal a
    tethered dimer architecture. Cell 116:711–723.
    https://doi.org/10.1016/S0092-8674(04)00210-7
    <https://doi.org/10.1016/S0092-8674(04)00210-7>

    Article <https://doi.org/10.1016%2FS0092-8674%2804%2900210-7> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXitFKisLg%3D> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=15006353> Google Scholar <http://scholar.google.com/scholar_lookup?&title=Crystal%20structures%20of%20a%20formin%20homology-2%20domain%20reveal%20a%20tethered%20dimer%20architecture&journal=Cell&doi=10.1016%2FS0092-8674%2804%2900210-7&volume=116&pages=711-723&publication_year=2004&author=Xu%2CYW&author=Moseley%2CJB&author=Sagot%2CI&author=Poy%2CF&author=Pellman%2CD&author=Goode%2CBL&author=Eck%2CMJ> 

83.

    Yogurtcu ON, Kim JS, Sun SX (2012) A mechanochemical model of actin
    filaments. Biophys J 103:719–727.
    https://doi.org/10.1016/j.bpj.2012.07.020
    <https://doi.org/10.1016/j.bpj.2012.07.020>

    Article <https://doi.org/10.1016%2Fj.bpj.2012.07.020> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1eku7nI> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3443790> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=22947933> Google Scholar <http://scholar.google.com/scholar_lookup?&title=A%20mechanochemical%20model%20of%20actin%20filaments&journal=Biophys%20J&doi=10.1016%2Fj.bpj.2012.07.020&volume=103&pages=719-727&publication_year=2012&author=Yogurtcu%2CON&author=Kim%2CJS&author=Sun%2CSX> 

84.

    Zhang ZY, Lu LY, Noid WG, Krishna V, Pfaendtner J, Voth GA (2008) A
    systematic methodology for defining coarse-grained sites in large
    biomolecules. Biophys J 95:5073–5083.
    https://doi.org/10.1529/biophysj.108.139626
    <https://doi.org/10.1529/biophysj.108.139626>

    Article <https://doi.org/10.1529%2Fbiophysj.108.139626> CAS
    <https://link.springer.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsVymsbrF> PubMed Central <http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2586547> PubMed <http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=18757560> Google Scholar <http://scholar.google.com/scholar_lookup?&title=A%20systematic%20methodology%20for%20defining%20coarse-grained%20sites%20in%20large%20biomolecules&journal=Biophys%20J&doi=10.1529%2Fbiophysj.108.139626&volume=95&pages=5073-5083&publication_year=2008&author=Zhang%2CZY&author=Lu%2CLY&author=Noid%2CWG&author=Krishna%2CV&author=Pfaendtner%2CJ&author=Voth%2CGA> 


      Discover content

  * Journals A-Z <https://link.springer.com/journals/a/1>
  * Books A-Z <https://link.springer.com/books/a/1>


      Publish with us

  * Publish your research <https://www.springernature.com/gp/authors>
  * Open access publishing
    <https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research>


      Products and services

  * Our products <https://www.springernature.com/gp/products>
  * Librarians <https://www.springernature.com/gp/librarians>
  * Societies <https://www.springernature.com/gp/societies>
  * Partners and advertisers <https://www.springernature.com/gp/partners>


      Our imprints

  * Springer <https://www.springer.com/>
  * Nature Portfolio <https://www.nature.com/>
  * BMC <https://www.biomedcentral.com/>
  * Palgrave Macmillan <https://www.palgrave.com/>
  * Apress <https://www.apress.com/>

  * Your privacy choices/Manage cookies
  * Your US state privacy rights
    <https://www.springernature.com/gp/legal/ccpa>
  * Accessibility statement
    <https://www.springernature.com/gp/info/accessibility>
  * Terms and conditions <https://link.springer.com/termsandconditions>
  * Privacy policy <https://link.springer.com/privacystatement>
  * Help and support <https://support.springernature.com/en/support/home>

14.139.123.36

Indian Institute of Science Education and Research (IISER), Pune
(2000532537) - Information and Library Network (INFLIBNET) Centre
(3994475188) - IISER Consortium (3000180695) - Indian Institute of
Science and Education Research (3000174968)

Springer Nature <https://www.springernature.com/>

© 2023 Springer Nature

