Skip to main content <#screen-reader-main-content>Skip to article
<#screen-reader-main-title>
Elsevier logo
ScienceDirect

<https://www.sciencedirect.com/>

  * Journals & Books
    <https://www.sciencedirect.com/browse/journals-and-books>

  *
  * <https://www.sciencedirect.com/search>Search<https://www.sciencedirect.com/search>

Register
<https://www.sciencedirect.com/user/register?targetURL=%2Fscience%2Farticle%2Fpii%2FS0022283619303493>Sign in <https://www.sciencedirect.com/user/login?targetURL=%2Fscience%2Farticle%2Fpii%2FS0022283619303493&from=globalheader>
Brought to you by:IISER PUNE LIBRARY

  *

    View *PDF*
    <https://www.sciencedirect.com/science/article/pii/S0022283619303493/pdfft?md5=94e7fbf71911d8aa14de2bd7e12d8ef9&pid=1-s2.0-S0022283619303493-main.pdf>
  *
    Download full issue

Search ScienceDirect


    Outline

 1. Highlights <#ab0015>
 2. Abstract <#ab0005>
 3. Graphical Abstract <#ab0010>
 4. Abbreviations <#ks0005>
 5. Keywords <#ks0010>
 6. Introduction <#s0005>
 7. Results <#s0010>
 8. Discussion <#s0040>
 9. Materials and Methods <#s0045>
10. CRediT authorship contribution statement <#s6115>
11. Acknowledgments <#ac0005>
12. Appendix A. Supplementary data <#s0065>
13. References <#bi0005>

Show full outline


    Cited by (15) <#section-cited-by>


    Figures (7)

 1.
    Unlabelled figure
     <#f0035>
 2.
    Fig. 1. Definition of the core domains
     <#f0005>
 3.
    Fig. 2. Motion tree for the outer domain (A) and inner domain (B)
    using the MD…
     <#f0010>
 4.
    Fig. 3. Clustering of outer domain orientations
     <#f0015>
 5.
    Fig. 4. Outer domain orientations in the GCF space
     <#f0020>
 6.
    Fig. 5. Outer domain orientation of G-form in the GCF space
     <#f0025>

Show 1 more figure


    Extras (1)

 1. Supplementary material. <#ec0005>

Journal of Molecular Biology


Journal of Molecular Biology

<https://www.sciencedirect.com/journal/journal-of-molecular-biology>
Volume 431, Issue 17
<https://www.sciencedirect.com/journal/journal-of-molecular-biology/vol/431/issue/17>, 9 August 2019, Pages 3217-3228
Journal home page for Journal of Molecular Biology
<https://www.sciencedirect.com/journal/journal-of-molecular-biology/vol/431/issue/17>


  Structural Polymorphism of Actin

Author links open overlay panelToshiro Oda ^1 , Shuichi Takeda ^2 ,
Akihiro Narita ^2 , Yuichiro Maéda ^2 ^3
Show more
Outline
Add to Mendeley
Share
Cite
https://doi.org/10.1016/j.jmb.2019.05.048<https://doi.org/10.1016/j.jmb.2019.05.048>Get rights and content<https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&contentID=S0022283619303493&orderBeanReset=true>


    Highlights

  * •

    Over 200 atomic structures of actin have been reported.

  * •

    Actin structures can be classified to four groups on the basis of
    orientation of two major domains.

  * •

    The groups are F-form in F-actin, C-form in cofilactin, O-form with
    a wide open cleft, and G-form.

  * •

    F-form and C-form are inaccessible form G-form by thermal
    fluctuation in a short period.

  * •

    The analysis shows allowed global conformations of actin molecule.


    Abstract

Information on the structural polymorphism of a protein is essential to
understand the mechanisms of how it functions at an atomic level.
Numerous studies on actin have accumulated substantial amounts of
information about its polymorphism, and there are over 200 published
atomic structures of different forms of actin using crystallography
<https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/crystallography>, fiber diffraction <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/fiber-diffraction>, and electron microscopy <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/electron-microscopy>. To characterize all the reported structures, we proposed simple parameters based on the discrete rigid bodies within the actin molecule and identified four conformation groups by cluster analysis: the F-form in naked F-actin, the C-form in cofilactin <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cofilin>, the O-form in profilin–actin, and the G-form in the majority of actin-containing crystal structures. The G-form group included the most variations, but each conformational variation was convertible via a thermal fluctuation, whereas the F- and C-forms were not accessible from the G-form. The convertibility and accessibility of the structures were evaluated using molecular dynamics simulations. Information about conformational conversion among each group is useful for understanding the mechanisms of actin function.


    Graphical Abstract

Unlabelled Image

 1. Download : Download high-res image (144KB)
    <https://ars.els-cdn.com/content/image/1-s2.0-S0022283619303493-ga1_lrg.jpg>
 2. Download : Download full-size image
    <https://ars.els-cdn.com/content/image/1-s2.0-S0022283619303493-ga1.jpg>

  * Previous articlein issue
    <https://www.sciencedirect.com/science/article/pii/S002228361930350X>
  * Next articlein
    issue<https://www.sciencedirect.com/science/article/pii/S0022283619303481>


    Abbreviations

G-actinglobular actin
F-actinfibrous actin
D-loopDNase I binding loop
ODouter domain
IDinner domain
Cαα-carbon
MDmolecular dynamics
/B/-factortemperature factor
ABPactin binding protein
gelsolin S1gelsolin sub-fragment 1
RPELRPxxxEL motif
WH2Wiskott–Aldrich syndrome protein-homology domain 2
TMR-actintetra-methyl-rhodamine conjugated actin
VASPvasodilator- stimulated phosphoprotein
SpvB/Salmonella enterica/ virulence-associated protein
MSDmean square displacement
Arpactin-related protein
CAPcyclase-associated protein
CARPCAP and X-linked retinitis pigmentosa 2 protein


    Keywords

domain orientation
rigid bodies
protein assembly
clustering
conformational space


    Introduction

Actin is a central player in dynamic cellular activities such as cell
division and cell motility
<https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cell-motility> [1] <#bb0005>, [2] <#bb0010>. Elucidation of the basic functions of actin is essential to completely understand these events. Generally, conformational changes <https://www.sciencedirect.com/topics/immunology-and-microbiology/conformational-transition> are often involved in protein functions. To understand the mechanism(s) of how a particular protein functions at an atomic level, it is essential to obtain information on the various conformations that it can adopt, that is, its polymorphism. Numerous studies have revealed that actin is highly polymorphic. Over 200 atomic structures of actin have been reported by crystallography <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/crystallography>, fiber diffraction <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/fiber-diffraction>, and electron microscopy <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/electron-microscopy>, and to date, at least four groups of distinct actin conformations have been identified. The first conformation, the G-form, has been reported in many crystal structures of monomeric actin and is characterized by a closed nucleotide-binding cleft. The second group, the F-form, has been observed in structures of naked actin filaments <https://www.sciencedirect.com/topics/immunology-and-microbiology/actin-filament> [3] <#bb0015>, [4] <#bb0020>, [5] <#bb0025>and actin filaments complexed with tropomyosin <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/tropomyosin> [6] <#bb0030>. This conformation has a closed cleft, and, compared with the G-form, one domain is rotated in an anti-clockwise direction with respect to the other, generating a flatter overall shape (Figs. S1 and S2). The third group, the C-form, has been observed in cofilin-decorated actin filaments (cofilactin). This conformation also possesses a closed cleft, but has a distinctive clockwise domain rotation [7] <#bb0035> (Figs. S1 and S2). The fourth conformation, the O-form, has been reported in phosphate-treated crystals of profilin–actin complexes and is characterized by a wide-open nucleotide-binding cleft [8] <#bb0040>, [9] <#bb0045> (Fig. S2). Other minor conformational variations have also been reported [10] <#bb0050>, [11] <#bb0055>, [12] <#bb0060>. Structural changes in the actin molecule between these various forms will affect the environment of cleft-bound nucleotides such as ATP, which in turn drives actin functions such as cycles of polymerization and depolymerization <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/depolymerization>through hydrolysis <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/enzymatic-hydrolysis> of bound ATP and exchange of bound nucleotides.

Furthermore, local polymorphisms of loops within the actin structure
have been reported. In particular, the D-loop has been studied
extensively owing to its relatively easy accessibility by biochemical
techniques
<https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/biochemical-techniques> such as restricted enzymatic cleavages. The D-loop is only visible, that is, with significant electron densities, when it is in contact with other molecules in a crystal, and its conformation is also dependent upon contact with molecules such as DNase I and thymosin β-4. Therefore, this polymorphism appears to be passive and may not directly drive actin functions relevant to energy consumption.

To discuss the global polymorphism of actin, it is necessary to provide
a clear definition of its various forms. In 2010, to clearly distinguish
between the G- and F-forms, we proposed a parameter expressing the
rotation of one domain with respect to the other about an axis
determined by the G- to F-actin transition [13] <#bb0065> (Fig. S1).
However, as described above, other actin conformations have recently
been described. Therefore, a new set of descriptions are required to
provide a clear distinction between each of the forms that will aid
discussions of the structure–function relationships. In this report, we
propose simple, visible, and sharply defined parameters based on
discrete rigid bodies identified within the actin molecule [7] <#bb0035>
and discuss their potential applications.


    Results


      Definition of the inner and outer domains of actin

Actin has two major domains enclosing a nucleotide-binding cleft. These
are termed the inner domain (ID) and the outer domain (OD) owing to
their relative distances from the helix axis within an actin filament
<https://www.sciencedirect.com/topics/immunology-and-microbiology/actin-filament>. The ID is composed of subdomain 3 (residues 145–180 and 270–337) and subdomain 4 (181–269), and the OD is composed of subdomain 1 (residues 1–32, 70–144, and 338–375) and subdomain 2 (33–69), as defined by Kabsch /et al/. [14] <#bb0070>.

The main chains in the periphery of the two major domains that include
the D-loop, residues 200–206, and the C-terminus, are flexible and adopt
several different conformations [13] <#bb0065>, [15] <#bb0075>. In
contrast, it appears that the core of each major domain has an invariant
structure and moves as a rigid body. Tanaka /et al/. [7] <#bb0035> used
eight available actin structures [PDB accession numbers:
1J6Z<http://www.rcsb.org/pdb/explore.do?structureId=1J6Z>,
4PKH<http://www.rcsb.org/pdb/explore.do?structureId=4PKH>,
3DAW<http://www.rcsb.org/pdb/explore.do?structureId=3DAW>,
5JLF<http://www.rcsb.org/pdb/explore.do?structureId=5JLF>, fragmin–actin
complex (unpublished data), 2PAV, 4PL7, 5YU8] to define the residues
comprising the rigid bodies of the two major domains by setting a
threshold value (< 0.7 Å) for the standard deviation calculated for each
Cα position. In this study, only the residues defined by Tanaka /et al/.
that are visible in all 127 actin crystal structures at a resolution of
at least 3.5 Å were included in the core domains. The OD core was
specified as composing of residues 7–31, 70–71, 76–108, 115–137,
338–347, and 355–364, and the ID core as residues 147–166, 171–194,
205–229, 237–238, 251–322, and 327–337 (Fig. 1 <#f0005>A).

Fig. 1

 1. Download : Download high-res image (347KB)
    <https://ars.els-cdn.com/content/image/1-s2.0-S0022283619303493-gr1_lrg.jpg>
 2. Download : Download full-size image
    <https://ars.els-cdn.com/content/image/1-s2.0-S0022283619303493-gr1.jpg>

Fig. 1. Definition of the core domains. (A) Structure of actin (PDB
accession number:
1J6Z<http://www.rcsb.org/pdb/explore.do?structureId=1J6Z>). The outer
domain (OD) and the inner domain (ID) cores are colored red and blue,
respectively. The cyan lines represent the three Cartesian axes used for
describing the OD core orientation. (B) Graph showing standardized
/B/-factor averages for 79 actin crystal structures plotted against the
residue number. The red and blue points represent the OD and ID domain
core residues, respectively, and the blue line represents the mean value
of averages of the standardized /B/-factors.

To characterize the domain cores by the internal motions of the two
major domains, we created a “motion tree” to describe the structural
dynamics using hierarchically defined rigid-body pairs on the basis of
RMSD values [16] <#bb0080>. Figure 2 <#f0010>A shows a typical motion
tree for the OD, which was calculated from frames of 200–500 ns in the
trajectory of molecular dynamics (MD) simulation of actin (chain A of
1WUA); the details of the MD simulation are described in the later
section. The root of the tree denotes the cluster composed all residues
in the OD of actin. The root is divided into the N-terminus (residues
1–6) and the larger rest of the OD (residues 7–144, 338–375) at node 1,
the height of which from the bottom represents a magnitude of the motion
of N-terminus relative to the rest part. The rest part is subdivided,
and small parts are sequentially removed: the D-loop (residues 41–52) at
node 2, the C-terminus (residues 372–375) at node 3, and the neighbor
region of the D-loop (residues 30–40 and 53–77) at node 4. The rest part
obtained at node 4 (residues 7–29, 78–144, and 338–371) corresponds to
the OD core described above. Further removal of small parts only reduces
the height of nodes slightly. This suggests that magnitudes of motion
for parts constituting the OD core are comparable. Thus, motions of the
OD core can be assumed to be representative of the global motions of the
OD, excluding local motions of the N-terminus, D-loop, and C-terminus.
Similarly, detected motions of the ID core would be unaffected by local
motions of the 198–206 and 223–250 regions (Fig. 2 <#f0010>B). Motion
trees derived from the MD trajectories of other actin structures
exhibited similar patterns.

Fig. 2

 1. Download : Download high-res image (456KB)
    <https://ars.els-cdn.com/content/image/1-s2.0-S0022283619303493-gr2_lrg.jpg>
 2. Download : Download full-size image
    <https://ars.els-cdn.com/content/image/1-s2.0-S0022283619303493-gr2.jpg>

Fig. 2. Motion tree for the outer domain (A) and inner domain (B) using
the MD trajectory of actin (PDB accession number:
1WUA<http://www.rcsb.org/pdb/explore.do?structureId=1WUA>). Each node
shows integration of two rigid bodies [the L (large) and S (small)
part]. The height from the bottom indicates the magnitudes of a relative
motion of two rigid bodies integrated at the node. Red dashed lines
indicate 1-Å level of distance between clusters.

Furthermore, we calculated the standardized /B/-factor for each Cα in 79
reported crystal structures of skeletal muscle α-actin at a minimum
resolution of 3.0 Å. Each Cα present in the core domains had a lower
/B/-factor value, which is consistent with lower mobility (Fig. 1
<#f0005>B). Although a set of residues in the domain core was altered by
a change of the RMSD threshold value and/or altering the sets of
analyzed actin structures, the overall results were generally unaffected.


      Determination of the orientation of the actin outer domain

As actin is composed of two major domains that are composed of a rigid
body-like core, actin conformations can be specified by defining the
relative configuration of the two cores, that is, the orientation of the
OD core relative to the ID core. The orientation was determined using
the following procedures (Fig. 1 <#f0005>A). First, the ID core of a
test structure was superimposed on the reference structure (1J6Z) by
translation and rotation of the whole molecule. The structure 1J6Z is a
commonly used reference structure of actin that is not complexed with
ABPs. Second, the center of mass of the OD core of the superimposed
structure was moved onto that of the reference structure by translation.
The determined OD shift vector was minute and negligible. Third, the
shifted OD core was superimposed on the OD of the reference structure by
rotation around the mass center of the shifted OD core. The rotation
matrix was calculated using Kabsch's algorithm [17] <#bb0085>, [18]
<#bb0090>. Finally, three rotation angles (β, α, and γ) were calculated
from the rotation matrix. These were defined by sequential rotations
around the /Y/, /X/, and /Z/ axes of the coordinate system fixed to the
reference structure (Fig. 1 <#f0005>A).


      Actin molecule conformations can be classified into four groups

Hierarchical cluster analysis was performed on the orientations of the
OD cores using the grouped average method in the R software package [19]
<#bb0095>, [20] <#bb0100>. The distance measure used was the square of
the Euclidean distance in the αβγ-space. The resulting dendrogram
<https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/phylogenetic-tree>, shown in Fig. 3 <#f0015>A [21] <#bb0105>, was categorized into four clusters using the “Dynamic Tree Cut” package for R to detect clusters on the basis of the dendrogram only [22] <#bb0110>. Jaccard bootstrap means were calculated for each of the four clusters using the “cboot” function of R to assess whether the clusters represent a true structure [20] <#bb0100>, [23] <#bb0115>. The values for clusters 1–4 were 0.99, 0.84, 0.80, and 0.97, respectively, suggesting that these clusters were valid. In addition, each of the four clusters could be interpreted as one of the four actin forms already described. As shown in Fig. 3 <#f0015>B, we observed large gaps separating the four actin clusters in the αβγ-space. Cluster 1 was centered at (α, β, γ) = (12, −10, 0.82) and consisted of F-actin conformations (F-form, which includes the structures 2ZWH, 5JLF, and 5ONV). The set of parameters used means that the actin OD in F-actin is superimposed onto the reference structure via sequential rotations of -10 degrees around the /Y/ axis, 12 degrees around the /X/ axis, and 0.81 degrees around the /Z/axis. Cluster 2 was located at (α, β, γ) = (9.5, 5.5, −6.8) and was characterized by a wide-open nucleotide-binding cleft, a conformation adopted in profilin–actin complexes (O-form: 1HLU and 3UB5). Cluster 3 was located at (α, β, γ) = (−3.5, −3.3, 2.0) and corresponded to the actin conformation in the cofilactin <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cofilin> filament and the twinfilin–monomeric actin complex (C-form: 5YU8 and 3DAW). Cluster 4 was widely distributed in the area enclosed by the F-, C-, and O-forms and contained the majority of the actin conformations reported from crystal structures (G-form).

Fig. 3

 1. Download : Download high-res image (522KB)
    <https://ars.els-cdn.com/content/image/1-s2.0-S0022283619303493-gr3_lrg.jpg>
 2. Download : Download full-size image
    <https://ars.els-cdn.com/content/image/1-s2.0-S0022283619303493-gr3.jpg>

Fig. 3. Clustering of outer domain orientations. (A) Dendrogram
<https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/phylogenetic-tree> showing classification of actin conformations using the group average method. Four clusters separated are colored [21] <#bb0105>. Cluster 1: blue, Cluster 2: orange, Cluster 3: red, and Cluster 4: black. (B) Orientations of the outer domain (OD) core in the αβγ-space. F-form: blue, C-form: red, O-form: orange, and G-form: black. The pink dot represents the origin of the plot, indicating the OD orientation of 1J6Z.

Interestingly, the direction of the F-form cluster from the origin of
the coordinate system (1J6Z) was almost perpendicular to the direction
of the C-form cluster. This suggests that the rotation axis of a hinge
motion from the G- to the F-form is almost perpendicular to the rotation
axis of a hinge motion from the G- to the C-form (Fig. S1). For ease of
OD core characterization, the αβγ-space was rotationally transformed to
the space specified by a new orthogonal unit vector set (the GCF-space;
see supplementary note): the GC axis was directed to 5YU8 (C-form) from
1J6Z (origin), the GF axis was almost directed to the F-form cluster
(which was perpendicular to the GC axis), and the corresponding
orthogonal axis (Fig. 4 <#f0020>A). The GF axis represents the degree of
flattening (propeller motion [3] <#bb0015>), and the GC axis denotes the
degree of the cleft closure (scissors motion [7] <#bb0035>). As a result
of this transformation, the centers of the F-, C-, O-, and G-form
clusters are (15, − 1.4, 0), (− 0.19, 5.2, 0.26), (2.9, − 12, 1.5), and
(4.5, − 3.1, − 0.58), respectively. The O-form cluster has a large
negative value along the GC axis, indicating that the nucleotide-binding
cleft is open (Fig. 4 <#f0020>A and B). Furthermore, the two-dimensional
projection with the GC axis and the GF axis (the 2D-GCF plot) appears to
be sufficient to characterize each of the actin conformations (Figs. 4
<#f0020>B, 5 <#f0025>A–D, and S3–S6; the F-, C-, and O-forms are colored
blue, red, and orange, respectively).

Fig. 4

 1. Download : Download high-res image (339KB)
    <https://ars.els-cdn.com/content/image/1-s2.0-S0022283619303493-gr4_lrg.jpg>
 2. Download : Download full-size image
    <https://ars.els-cdn.com/content/image/1-s2.0-S0022283619303493-gr4.jpg>

Fig. 4. Outer domain orientations in the GCF space. (A) Orientations of
the OD core in the GCF-space specified by the GF, GC and orthogonal axes
(black lines). F-form: blue, C-form: red, O-form: orange, and G-form:
black. (B) 2D projection of panel A along the orthogonal axis.

Fig. 5

 1. Download : Download high-res image (351KB)
    <https://ars.els-cdn.com/content/image/1-s2.0-S0022283619303493-gr5_lrg.jpg>
 2. Download : Download full-size image
    <https://ars.els-cdn.com/content/image/1-s2.0-S0022283619303493-gr5.jpg>

Fig. 5. Outer domain orientation of G-form in the GCF space. The figures
are 2D map of panel B in Fig. 4 <#f0020>along the orthogonal axis.
F-form: blue, C-form: red, and O-form: orange in (A) and (B). In panel
A, actin in complex with gelsolin
<https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/gelsolin>S1: green; with DNase I: pink; with profilin <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/profilin>: maroon; with gelsolin S1 and DNase I in a ternary complex <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/ternary-complex> (/3j3c/): grey; In panel B, actin in complex with the WH2 domain: yellow; and with the RPEL domain: maroon. Uncomplexed actin is represented as cyan in panel B. In panel C, actin with bound ADP: orange; and with bound ATP/AMPPNP: cyan. In panel D, actin with a visible D-loop: cyan.


      Bound ABP-dependent subclassification of the G-form cluster

As shown in Fig. 5 <#f0025>A and B, OD orientations in the G-form
cluster form small sub-groups that are dependent upon the bound ABP. For
example, one G-form sub-group (shown in maroon in Fig. 5 <#f0025>A)
contains the profilin–actin complex, which also tends to have an open
ATP-binding cleft between the two major domains [11] <#bb0055> that is
analogous to the O-form cluster. In addition, bound gelsolin S1 (green
data points in Fig. 5 <#f0025>A) and DNase I (pink in Fig. 5 <#f0025>A)
clearly form distinct groups from one other, and actin molecules bound
to gelsolin S1 are flatter. Intriguingly, the actin–gelsolin S1-DNase I
ternary complex
<https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/ternary-complex> adopts the domain orientation of the gelsolin S1 group despite the typical D-loop conformation for DNase I binding (/3cjc/; grey in Fig. 5 <#f0025>A). This suggests that gelsolin S1, rather than DNase I, dominantly dictates the actin conformation in the ternary complex. Furthermore, the actin–gelsolin S1 region (green in Fig. 5 <#f0025>A) barely overlaps with regions comprising the actin–RPEL domain complex that includes the RPxxxEL actin-binding motif (maroon in Fig. 5 <#f0025>B) [24] <#bb0120> and actin-WH2 (Wiskott–Aldrich syndrome protein-homology domain 2)-like domain complexes (yellow in Fig. 5 <#f0025>B) [25] <#bb0125>.

Crystal structures of uncomplexed, non-polymerizable actin molecules,
either by amino acid replacements
<https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/amino-acid-substitution> (e.g., A204E/P243K) and/or by chemical modifications such as ADP-ribosylation and TMR-modification (cyan in Fig. 5 <#f0025>B) also appear to form a discrete group. In contrast, the majority of actin structures with bound ADP (orange in Fig. 5 <#f0025>C) were distributed in the same area as those with bound ATP or AMPPNP (cyan in Fig. 5 <#f0025>C); however, in contrast to bound ADP, bound ATP appears to allow conformations with an open cleft. In addition, actins that possess a D-loop with continuous electron density did not show a unique distribution of OD orientations (cyan in Fig. 5 <#f0025>D). These findings indicate that actin conformations are primarily determined via their interaction with other proteins.


      Relationship between thermal fluctuations and variations of the
      G-form cluster

To investigate the relationship between the conformations within the
G-form cluster and thermal fluctuations of domain orientation, MD
simulations were performed on six isolated high-resolution crystal
structures of actin at 300 K and 1 atm: 18 short runs for a duration of
100 ns and six long runs for a duration of 500 ns. The actin structures
selected were chain A of 2FXU (a bistramide A–actin complex; 1.35 Å [26]
<#bb0130>), chain A of 1WUA (an Aplyronine A–actin complex; 1.45 Å [27]
<#bb0135>), chain A of 2PBD (a profilin–actin complex with the
poly-PRO-GAB domain of VASP; 1.5 Å [28] <#bb0140>), chain A of 4K41 (a
kabiramide–actin complex; 1.4 Å [29] <#bb0145>), chain A of 2GWK (SpvB
ADP-ribosylated, uncomplexed actin; 2.0 Å [30] <#bb0150>), and chain A
of 4PKG (actin complexed with the N-terminal actin-binding domain of
tropomodulin
<https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/tropomodulin>; 1.8 Å [31] <#bb0155>). These structures were scattered in the G-form cluster region. After the systems were energy minimized [followed by short runs to equilibrate temperature and pressure with harmonic position restraints applied for the heavy atoms of actin (C, N, O, S, and Ca)], OD core orientations were plotted every 0.1 ns from trajectories of 200–500 ns (Fig. 6 <#f0030>A). From the six MD trajectories in Fig. 6 <#f0030>A, total distribution of the OD orientation was calculated (Fig. 6 <#f0030>B), and distribution for each MD trajectory is shown in Fig. S5. To characterize the fluctuating motion of the OD core in the αβγ-space during the MD simulations, we performed a logarithmic plot of mean square displacement (MSD) as a function of time (Fig. 6 <#f0030>C). MSD increased linearly with time up to the order of 10 ps, at which point the slope became smaller, suggesting that the displacement has an upper limit. Our results indicated that the OD orientation fluctuates within a confined space when viewed on a 1- to 500-ns timescale.

Fig. 6

 1. Download : Download high-res image (578KB)
    <https://ars.els-cdn.com/content/image/1-s2.0-S0022283619303493-gr6_lrg.jpg>
 2. Download : Download full-size image
    <https://ars.els-cdn.com/content/image/1-s2.0-S0022283619303493-gr6.jpg>

Fig. 6. Characterization of the outer domain fluctuating motion in the
αβγ-space (or GCF-space). (A) 500-ns MD trajectories (6 in total)
calculated from six G-form actin structures, which have been added to
Fig. 4 <#f0020>A: 2FXU (blue line and dots), 1WUA (green line and dots),
2PBD (cyan line and dots), 4K41 (pink line and dots), 2GWJ (magenta line
and dots), and 4PKG (brown line and dots). Each dot represents the outer
domain (OD) orientation sampled every 0.1 ns in the trajectory of
200–500 ns. (B) The two-dimensional (2D-GCF) surface plot for
distribution of OD orientation sampled every 0.1 ns in 200–500 ns. The
distribution was calculated from the numbers of points present in the
grid defined by the GC and GF axes from panel A. The color map
progresses from green (1) to red (437) in 32 steps. F-form: blue,
C-form: red, O-form: orange, and G-form: black. (C) Logarithmic plot of
MSD as a function of time, created from the MD trajectory of chain A of
4K41. (D) Pathway of actin ATPase
<https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/atpase> cycle in the GCF-space. Actin F-form: blue, C-form: red, O-from: orange, actin in complex with profilin <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/profilin>: maroon; with the WH2 domain: yellow; and with the CAP-CARP domain: deep-green. Arrows represent the pathway of exchanging ABPs. *G-actin-WH2(CAP) has not been crystalized.

Conformational fluctuations were observed throughout almost the entire
G-form cluster region (Fig. 6 <#f0030>A and B). For quantitative
analysis
<https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/quantitative-technique>, we divided the αβγ-space into sets of 1.5 Degree voxels and counted the number of voxels in which at least one data point resides (Table S1). The entire G-form cluster region (shown in Fig. 6 <#f0030>B) was covered by 55 voxels, and MD trajectories (data points sampled every 0.1 ns from one trajectory of 200–500 ns) were covered by 300 voxels on average. The data points selected from the MD runs were widely spread toward large negative values along the GC axis (the open cleft) (Fig. 6 <#f0030>B). Voxels picked by six MD simulation long runs partially overlapped each other and the cumulative voxels covered 95% of the entire G-form cluster voxels. This suggests that various conformations within the G-form cluster are accessible via thermal fluctuations.

Accessibility across separated clusters is summarized in Table S1.
Overlapping of MD trajectories within the O-form region was observed in
2 of the 6 MD simulation long runs as shown in Fig. 5 <#f0025>S (and in
3 of the 18 short runs). In contrast, we did not observe any overlapping
within the F- and the C-form regions. These results indicate that,
within the 500-ns timescale tested, the O-form is accessible from the
G-form by a thermal fluctuation, whereas the F- and the C-forms are not
accessible from the G-form under equivalent conditions. However, the
G-form region was accessible from the F- and C-forms (Figs. S4 and S6).


    Discussion

Proteins can change their overall conformations by shear and hinge
motions, which, in recent years, have been determined by the use of
computer programs such as DynDom [32] <#bb0160>, [33] <#bb0165>. To
date, over 200 actin structures have been determined, which exhibit a
variety of hinge motions of the two main domains when compared against a
representative G-actin reference structure (typically 1J6Z). This
approach is excellent for categorizing the characteristics of each
structure. However, the relationship between actin conformations and
their relevant functions are still not understood. We previously
proposed two alternative parameters to summarize each structure: the
distance between Cαs of G15 and of D157 as an indicator of cleft closure
and the rotational angle of two major domains around the G/F axis as a
measure of flattening [13] <#bb0065> (Fig. S1). However, these two
parameters are not suitable for plotting in a conformational space to
allow a collective overview of the available data. Xue /et al/. [34]
<#bb0170>used principal component analysis
<https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/principal-component-analysis> to analyze MD-computational and experimental actin structures, and plotted data points for each structure in a two-dimensional space of the first and second principal components. This clearly showed bound ABP-dependent and filament-specific actin conformations; however, a clear separation between the F- and G-forms was not evident. In contrast, our study shows discrete F-and G-form clusters in the αβγ-space of the orientation of the OD core that are separated by a large gap, which may reflect the existence of an energy barrier between the two clusters. Principle component analysis-based analysis of a set that mainly includes G-form actins is likely to be insufficient to accurately describe the F-form. To conclude, we believe that OD core orientation is a suitable parameter to delineate all reported actin conformations.

In this study, we showed that actin conformations can be described by
two parameters (flattening and cleft closing) and determined ranges of
allowed values for these parameters that represent all the reported
actin conformations (Figs. 3 <#f0015>B and 4 <#f0020>A–B). The allowed
space was the conformational space of the actin molecule, which could be
used for further bioinformatical analyses such as energy landscape
calculations despite some limitations. The conformational space was then
divided into G-, C-, and F-form regions. O-form appears to be a
satellite cluster of G-form as the O-form is accessible from G-form. The
G-form has a wide conformational space toward the open cleft with likely
multiple energy minima; four of the six MD simulation long runs remained
in the original G-form cluster area, while the nucleotide-binding cleft
in the other two runs partially opened at early stage and retained the
state (Fig. S5 and Table S1). The F- and C-forms were not accessible
from the G-form via a thermal fluctuation when viewed on a 500-ns
timescale (Figs. 6 <#f0030>A and B, S4 and S5, and Table S1). This
suggests that there is an energy barrier from the G- to F-forms and from
the G- to the C-forms that cannot be overcome by a thermal fluctuation
in this timescale. However, transition from the G- to the F-form would
occur in the polymerization process: upon incorporation into the
filament end, a monomer would overcome the energy barrier by interacting
with two other actin molecules constituting the filament end. Nucleators
such as the Arp2/3 complex mediate a similar effect. Conversely, of the
four MD simulation long runs on the isolated F-form of actin, two
proceeded to the G-form, while the other two remained unchanged (2 of 18
short runs proceeded to the G-form) (Figs. S4 and S6). However, both the
two MD simulation long runs and six of the short runs on the isolated
C-form proceeded to the G-form (Figs. S4 and S6). It is feasible that
there is an energy barrier from the F- to the G-form and no barrier from
the C- to the G-form: the F-form is a semi-stable state, while the
C-form is an unstable state induced by cofilin binding.

Each ABP binds to its respective binding site on actin and fixes the two
core domains to a particular unique configuration in the conformational
space, which allosterically affects its binding affinity
<https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/binding-affinity> to another ABP. When an ABP–actin complex comes into contact with another ABP with a different binding site, two ABPs inducing different configurations would show a negative cooperativity. As shown in Fig. 5 <#f0025>A, the configuration of the two cores of actin in complex with DNase I is vastly different from that with gelsolin S1 or profilin, and, indeed, a negative cooperativity of their binding has been observed [35] <#bb0175>, [36] <#bb0180>. All cellular processes involving actin are reliant upon the exchange of bound ABPs. Our GCF-space plot is convenient for summarizing bound ABPs in addition to actin conformations. For example, Fig. 6 <#f0030>D shows an actin ATPase <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/atpase> cycle pathway in the GCF-space: actin exchanges a bound ABP to profilin for polymerization, then to cofilin for depolymerization <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/depolymerization>, further to CAP (cyclase-associated protein) for ATP-recharging, then to profilin again [37] <#bb0185>. CAP is a multi-domain protein, the C-terminus domain of which has multiple actin binding domains including the WH2 and CARP domains [38] <#bb0190>. The CAP catalyzes the exchange of actin-bound ADP to ATP during change of the actin binding sites from CARP to WH2 domains [37] <#bb0185>. The GCF-space plot suggests that exchange of bound ABP from cofilin to the CARP domain as well as from the WH2 domain to profilin occur without actin undergoing a global conformational change <https://www.sciencedirect.com/topics/immunology-and-microbiology/conformational-transition>. Conversely, an exchange of CARP to the WH2 domain would involve a conformational change; this corresponds to a key process for the exchange of ADP to ATP.

Finally, the F-actin D-loop strengthens inter- and intra-strand
interactions and controls F-actin stability [3] <#bb0015>, [6]
<#bb0030>; indeed, Try53 phosphorylation [11] <#bb0055>, [39]
<#bb0195>or Mical-mediated oxidation
<https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/alpha-oxidation> [40] <#bb0200>impairs actin polymerization <https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/actin-polymerization>. However, in crystal structures, D-loop conformations appear to be variable depending on the contact partners, and visibility of the D-loop has no correlation with the configuration of the two core domains, or what nucleotide is bound (Fig. 5 <#f0025>C and D). Furthermore, the D-loop in naked F-actin is visible, while it is invisible in cofilin-coated F-actin as the subunits are differently arranged. This suggests that the D-loop can adopt multiple conformations that are principally determined by the shape and position of contact partners. However, a recent report has indicated that the D-loop and the C-terminus are responsible for the conformational difference between ATP–F-actin and ADP–F-actin [4] <#bb0020>. In the present study, the global conformation (the relative configuration of the two major domains) of isolated actin was comprehensively analyzed by clustering and MD simulations. To determine a factor that dictates multiple D-loop conformations in actin function, it would be necessary to perform in-depth MD simulations of systems including actin complexed with its various binding partners.


    Materials and Methods


      Software

Software for calculations of actin OD orientations was run on Scilab 6.0
[41] <#bb0205>. Hierarchical cluster analyses were performed using
RStudio (R 3.4.3; Boston, MA, USA) [19] <#bb0095>. MD simulations of
protein were performed using GROMACS
<https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/gromacs> 2016.4 (GPU version) for Tegara workstation in our laboratory and NEC LX 108Th-4G in the Research Center for Computational Science, Okazaki, and using GROMACS version 5.0.4 (MPI version) for Fujitsu CX4000/2550 in Nagoya University Information and Communications department [42] <#bb0210>.


      Analysis of /B/-factors

Each actin structure was selected in turn and the mean (/m/) and
standard deviation (/σ/^2 ) of the Cα /B/-factors (bf) in the
non-missing regions (comprising residues 7–31, 70–196, 205–229, 237,
238, 251–324, and 327–364) were calculated. Next, using the mean and the
standard deviation, we standardized each Cα /B/-factor contained within
the actin structure [(bf − /m/)/σ]. After performing this procedure for
the 79 α-actin crystal structures with a minimum resolution of 3.0 Å, we
averaged the standardized /B/-factors of the actin structures for each Cα.


      MD simulations

All simulations were performed using the GROMACS package [42] <#bb0210>
using the CHARMM27/CMAP all-atom force field and the TIP3P water model
[43] <#bb0215>. The missing region of the initial crystal structure was
added using the Modeller 9.0 package [44] <#bb0220>. At the beginning of
each simulation, the protein was solvated in a rectangular box with a
minimum distance of 1.0 nm between any protein atom and the edges of the
box. Potassium ions and chloride ions (100 mM) were added to compensate
the net charge of the system. Electrostatic interactions were calculated
using the particle-mesh Ewald (PMA) algorithm [45] <#bb0225>. All bond
lengths were constrained using the linear constraint solver (LINCS)
algorithm [46] <#bb0230>. The temperature and pressure were stabilized
at 300 K and 1 atm using a v-rescale thermostat [47] <#bb0235> and the
Parrinello–Rahman barostat [48] <#bb0240>, respectively. The system was
equilibrated under constant volume and temperature (NVT) conditions for
100 ps with position restraints on the protein heavy atoms. Next, the
system was repeatedly equilibrated under constant pressure and
temperature (NPT) conditions for 100 ps with position restraints on the
protein heavy atoms while the force constants for the position
restraints were gradually decreased from 1000 to 5 kJ mol^−1  nm^−2 .
After the equilibrations were performed, the simulation was started. For
G-form simulations, three independent simulations were obtained from one
structure using different initial velocities generated by random seeds.


    CRediT authorship contribution statement

*Toshiro Oda:* Conceptualization, Data curation, Software, Funding
acquisition, Writing, review & editing. *Shuichi Takeda:* Data curation,
Writing, review & editing. *Akihiro Narita:* Conceptualization, Data
curation, Writing, review & editing. *Yuichiro Maéda:*
Conceptualization, Writing, review & editing.


    Acknowledgments

We thank Prof. Dr. Ota (Nagoya University) for comments on the
manuscript, Dr. Koike for advice on Motion Tree analysis, and Emma
Andrew, PhD, from Edanz Group
(www.edanzediting.com/ac<http://www.edanzediting.com/ac>) for editing a
draft of this manuscript. This work was supported by JSPS (Japan Society
for the Promotion of Science, Japan), KAKENHI (Grants-in-Aid for
Scientific Research) Grant Number 17 K07373 <#gts0005>. Some
computations were partially performed at the Research Center for
Computational Science, Okazaki, Japan. In addition, other computations
were partially performed at the Information Technology Center of Nagoya
University, Nagoya, Japan.

*Declaration of Competing Interest:* The authors declare no competing
interests.


    Appendix A. Supplementary data

Download : Download Acrobat PDF file (1MB)
<https://ars.els-cdn.com/content/image/1-s2.0-S0022283619303493-mmc1.pdf>

Supplementary material.

Recommended articles


    References

 1. [1] <#bbb0005>
    T.D. Pollard
    Regulation of actin filament assembly by Arp2/3 complex and formins
    Annu. Rev. Biophys. Biomol. Struct., 36 (2007), pp. 451-477,
    10.1146/annurev.biophys.35.040405.101936<https://doi.org/10.1146/annurev.biophys.35.040405.101936>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-34247644614&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Regulation%20of%20actin%20filament%20assembly%20by%20Arp23%20complex%20and%20formins&publication_year=2007&author=T.D.%20Pollard>
 2. [2] <#bbb0010>
    M.-F. Carlier, D. Pantaloni
    Control of actin assembly dynamics in cell motility
    J. Biol. Chem., 282 (2007), pp. 23005-23009,
    10.1074/jbc.R700020200<https://doi.org/10.1074/jbc.R700020200>
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0021925820545146/pdfft?md5=bf96a4245f756257fdf4dcbbe2bdf4b3&pid=1-s2.0-S0021925820545146-main.pdf>View article <https://www.sciencedirect.com/science/article/pii/S0021925820545146>View in Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-34548160002&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Control%20of%20actin%20assembly%20dynamics%20in%20cell%20motility&publication_year=2007&author=M.-F.%20Carlier&author=D.%20Pantaloni>
 3. [3] <#bbb0015>
    T. Oda, M. Iwasa, T. Aihara, Y. Maéda, A. Narita
    The nature of the globular- to fibrous-actin transition
    Nature., 457 (2009), pp. 441-445,
    10.1038/nature07685<https://doi.org/10.1038/nature07685>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-58749091822&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=The%20nature%20of%20the%20globular-%20to%20fibrous-actin%20transition&publication_year=2009&author=T.%20Oda&author=M.%20Iwasa&author=T.%20Aihara&author=Y.%20Ma%C3%A9da&author=A.%20Narita>
 4. [4] <#bbb0020>
    F. Merino, S. Pospich, J. Funk, T. Wagner, F. Küllmer, H.-D. Arndt,
    P. Bieling, S. Raunser
    Structural transitions of F-actin upon ATP hydrolysis at near-atomic
    resolution revealed by cryo-EM
    Nat. Struct. & Mol. Biol., 25 (2018), pp. 528-537,
    10.1038/s41594-018-0074-0<https://doi.org/10.1038/s41594-018-0074-0>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-85047986609&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Structural%20transitions%20of%20F-actin%20upon%20ATP%20hydrolysis%20at%20near-atomic%20resolution%20revealed%20by%20cryo-EM&publication_year=2018&author=F.%20Merino&author=S.%20Pospich&author=J.%20Funk&author=T.%20Wagner&author=F.%20K%C3%BCllmer&author=H.-D.%20Arndt&author=P.%20Bieling&author=S.%20Raunser>
 5. [5] <#bbb0025>
    S.Z. Chou, T.D. Pollard
    Mechanism of actin polymerization revealed by cryo-EM structures of
    actin filaments with three different bound nucleotides
    Proc. Natl. Acad. Sci. United States Am. (2019),
    10.1073/pnas.1807028115<https://doi.org/10.1073/pnas.1807028115>
    Google
    Scholar<https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20actin%20polymerization%20revealed%20by%20cryo-EM%20structures%20of%20actin%20filaments%20with%20three%20different%20bound%20nucleotides&publication_year=2019&author=S.Z.%20Chou&author=T.D.%20Pollard>
 6. [6] <#bbb0030>
    J. von der Ecken, M. Müller, W. Lehman, D.J. Manstein, P.A. Penczek,
    S. Raunser
    Structure of the F-actin–tropomyosin complex
    Nature., 519 (2015), pp. 114-117,
    10.1038/nature14033<https://doi.org/10.1038/nature14033>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-84925512097&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Structure%20of%20the%20F-actintropomyosin%20complex&publication_year=2015&author=J.%20von%20der%20Ecken&author=M.%20M%C3%BCller&author=W.%20Lehman&author=D.J.%20Manstein&author=P.A.%20Penczek&author=S.%20Raunser>
 7. [7] <#bbb0035>
    K. Tanaka, S. Takeda, K. Mitsuoka, T. Oda, C. Kimura-Sakiyama, Y.
    Maéda, A. Narita
    Structural basis for cofilin binding and actin filament disassembly
    Nat. Commun., 9 (2018), Article 1860,
    10.1038/s41467-018-04290-w<https://doi.org/10.1038/s41467-018-04290-w>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-85047074507&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Structural%20basis%20for%20cofilin%20binding%20and%20actin%20filament%20disassembly&publication_year=2018&author=K.%20Tanaka&author=S.%20Takeda&author=K.%20Mitsuoka&author=T.%20Oda&author=C.%20Kimura-Sakiyama&author=Y.%20Ma%C3%A9da&author=A.%20Narita>
 8. [8] <#bbb0040>
    J.C. Porta, G.E.O. Borgstahl
    Structural basis for profilin-mediated actin nucleotide exchange
    J. Mol. Biol., 418 (2012), pp. 103-116,
    10.1016/j.jmb.2012.02.012<https://doi.org/10.1016/j.jmb.2012.02.012>
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0022283612001532/pdfft?md5=0059ed98d738ac741c0aa8a89325e5bd&pid=1-s2.0-S0022283612001532-main.pdf>View article <https://www.sciencedirect.com/science/article/pii/S0022283612001532>View in Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-84858698489&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Structural%20basis%20for%20profilin-mediated%20actin%20nucleotide%20exchange&publication_year=2012&author=J.C.%20Porta&author=G.E.O.%20Borgstahl>
 9. [9] <#bbb0045>
    J.K. Chik, U. Lindberg, C.E. Schutt
    The structure of an open state of beta-actin at 2.65 Å resolution
    J. Mol. Biol., 263 (1996), pp. 607-623,
    10.1006/jmbi.1996.0602<https://doi.org/10.1006/jmbi.1996.0602>
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0022283696906026/pdf?md5=184f4377ad000c98f62846eb38321702&pid=1-s2.0-S0022283696906026-main.pdf>View article <https://www.sciencedirect.com/science/article/pii/S0022283696906026>View in Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-0030575783&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=The%20structure%20of%20an%20open%20state%20of%20beta-actin%20at%202.65%20%C3%85%20resolution&publication_year=1996&author=J.K.%20Chik&author=U.%20Lindberg&author=C.E.%20Schutt>
10. [10] <#bbb0050>
    U.B. Nair, P.B. Joel, Q. Wan, S. Lowey, M.A. Rould, K.M. Trybus
    Crystal structures of monomeric actin bound to cytochalasin D
    J. Mol. Biol., 384 (2008), pp. 848-864,
    10.1016/j.jmb.2008.09.082<https://doi.org/10.1016/j.jmb.2008.09.082>
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0022283608012539/pdfft?md5=dcc23724a9d4aeac92b10611afad69ce&pid=1-s2.0-S0022283608012539-main.pdf>View article <https://www.sciencedirect.com/science/article/pii/S0022283608012539>View in Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-56249092074&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Crystal%20structures%20of%20monomeric%20actin%20bound%20to%20cytochalasin%20D&publication_year=2008&author=U.B.%20Nair&author=P.B.%20Joel&author=Q.%20Wan&author=S.%20Lowey&author=M.A.%20Rould&author=K.M.%20Trybus>
11. [11] <#bbb0055>
    K. Baek, X. Liu, F. Ferron, S. Shu, E.D. Korn, R. Dominguez
    Modulation of actin structure and function by phosphorylation of
    Tyr-53 and profilin binding
    Proc. Natl. Acad. Sci. United States Am., 105 (2008), pp.
    11748-11753,
    10.1073/pnas.0805852105<https://doi.org/10.1073/pnas.0805852105>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-50149116323&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Modulation%20of%20actin%20structure%20and%20function%20by%20phosphorylation%20of%20Tyr-53%20and%20profilin%20binding&publication_year=2008&author=K.%20Baek&author=X.%20Liu&author=F.%20Ferron&author=S.%20Shu&author=E.D.%20Korn&author=R.%20Dominguez>
12. [12] <#bbb0060>
    J.-F. Gaucher, C. Maugé, D. Didry, B. Guichard, L. Renault, M.-F.
    Carlier
    Interactions of isolated C-terminal fragments of neural
    Wiskott–Aldrich syndrome protein (N-WASP) with actin and Arp2/3 complex
    J. Biol. Chem., 287 (2012), pp. 34646-34659,
    10.1074/jbc.M112.394361<https://doi.org/10.1074/jbc.M112.394361>
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0021925820627971/pdfft?md5=cf23b447ab9ce9f4071ce428d2821ad9&pid=1-s2.0-S0021925820627971-main.pdf>View article <https://www.sciencedirect.com/science/article/pii/S0021925820627971>Google Scholar<https://scholar.google.com/scholar_lookup?title=Interactions%20of%20isolated%20C-terminal%20fragments%20of%20neural%20WiskottAldrich%20syndrome%20protein%20%20with%20actin%20and%20Arp23%20complex&publication_year=2012&author=J.-F.%20Gaucher&author=C.%20Maug%C3%A9&author=D.%20Didry&author=B.%20Guichard&author=L.%20Renault&author=M.-F.%20Carlier>
13. [13] <#bbb0065>
    T. Oda, Y. Maéda
    Multiple conformations of F-actin
    Struct., 18 (2010), pp. 761-767,
    10.1016/j.str.2010.05.009<https://doi.org/10.1016/j.str.2010.05.009>
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0969212610001875/pdfft?md5=a7d834e0aa6d543ebb32c9ae97e0467b&pid=1-s2.0-S0969212610001875-main.pdf>View article <https://www.sciencedirect.com/science/article/pii/S0969212610001875>View in Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-77954649029&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Multiple%20conformations%20of%20F-actin&publication_year=2010&author=T.%20Oda&author=Y.%20Ma%C3%A9da>
14. [14] <#bbb0070>
    W. Kabsch, H.G. Mannherz, D. Suck, E.F. Pai, K.C. Holmes
    Atomic structure of the actin:DNase I complex
    Nature., 347 (1990), pp. 37-44,
    10.1038/347037a0<https://doi.org/10.1038/347037a0>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-0024988340&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Atomic%20structure%20of%20the%20actin%3ADNase%20I%20complex&publication_year=1990&author=W.%20Kabsch&author=H.G.%20Mannherz&author=D.%20Suck&author=E.F.%20Pai&author=K.C.%20Holmes>
15. [15] <#bbb0075>
    A. Narita, T. Oda, Y. Maéda
    Structural basis for the slow dynamics of the actin filament pointed end
    EMBO J., 30 (2011), pp. 1230-1237,
    10.1038/emboj.2011.48<https://doi.org/10.1038/emboj.2011.48>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-79953790685&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Structural%20basis%20for%20the%20slow%20dynamics%20of%20the%20actin%20filament%20pointed%20end&publication_year=2011&author=A.%20Narita&author=T.%20Oda&author=Y.%20Ma%C3%A9da>
16. [16] <#bbb0080>
    R. Koike, M. Ota, A. Kidera
    Hierarchical description and extensive classification of protein
    structural changes by motion tree
    J. Mol. Biol., 426 (2014), pp. 752-762,
    10.1016/j.jmb.2013.10.034<https://doi.org/10.1016/j.jmb.2013.10.034>
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0022283613006918/pdfft?md5=1959e5c713a208a7c1ddf39fe59517b7&pid=1-s2.0-S0022283613006918-main.pdf>View article <https://www.sciencedirect.com/science/article/pii/S0022283613006918>View in Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-84892510091&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Hierarchical%20description%20and%20extensive%20classification%20of%20protein%20structural%20changes%20by%20motion%20tree&publication_year=2014&author=R.%20Koike&author=M.%20Ota&author=A.%20Kidera>
17. [17] <#bbb0085>
    W. Kabsch
    A solution for the best rotation to relate two sets of vectors
    Acta Cryst., A32 (1976), p. 922,
    10.1107/S0567739476001873<https://doi.org/10.1107/S0567739476001873>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-84893482610&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=A%20solution%20for%20the%20best%20rotation%20to%20relate%20two%20sets%20of%20vectors&publication_year=1976&author=W.%20Kabsch>
18. [18] <#bbb0090>
    W. Kabsch
    A discussion of the solution for the best rotation to relate two
    sets of vectors
    Acta Cryst., A34 (1978), p. 827,
    10.1107/S0567739478001680<https://doi.org/10.1107/S0567739478001680>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-12944249776&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=A%20discussion%20of%20the%20solution%20for%20the%20best%20rotation%20to%20relate%20two%20sets%20of%20vectors&publication_year=1978&author=W.%20Kabsch>
19. [19] <#bbb0095>
    R Core Team, The R project for statistical computing, 2017.
    Google
    Scholar<https://scholar.google.com/scholar?q=R%20Core%20Team%2C%20The%20R%20project%20for%20statistical%20computing%2C%202017.>
20. [20] <#bbb0100>
    N. Zumel
    Practical Data Science With R
    Manning Publications (2014)
    Google
    Scholar<https://scholar.google.com/scholar_lookup?title=Practical%20Data%20Science%20With%20R&publication_year=2014&author=N.%20Zumel>
21. [21] <#bbb0105>
    T. Galili
    Dendextend: an R package for visualizing, adjusting and comparing
    trees of hierarchical clustering
    Bioinforma., 31 (2015), pp. 3718-3720,
    10.1093/bioinformatics/btv428<https://doi.org/10.1093/bioinformatics/btv428>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-84947476425&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Dendextend%3A%20an%20R%20package%20for%20visualizing%2C%20adjusting%20and%20comparing%20trees%20of%20hierarchical%20clustering&publication_year=2015&author=T.%20Galili>
22. [22] <#bbb0110>
    P. Langfelder, B. Zhang, with contibutions from S. Horath
    dynamicTreeCut: Method for Detection of Cluster in Hierarchical
    Clustering Dendrogram
    R package version 1.63-1.
    (2016)
    Google
    Scholar<https://scholar.google.com/scholar_lookup?title=dynamicTreeCut%3A%20Method%20for%20Detection%20of%20Cluster%20in%20Hierarchical%20Clustering%20Dendrogram&publication_year=2016&author=P.%20Langfelder&author=B.%20Zhang&author=with%20contibutions%20from%20S.%20Horath>
23. [23] <#bbb0115>
    C. Henning
    Fpc: Flexible Procedures for Clustering
    R package version 2.1-11.1.
    (2018)
    Google
    Scholar<https://scholar.google.com/scholar_lookup?title=Fpc%3A%20Flexible%20Procedures%20for%20Clustering&publication_year=2018&author=C.%20Henning>
24. [24] <#bbb0120>
    S. Mouilleron, S. Guettler, C.A. Langer, R. Treisman, N.Q. McDonald
    Molecular basis for G-actin binding to RPEL motifs from the serum
    response factor coactivator MAL
    EMBO J., 27 (2008), pp. 3198-3208,
    10.1038/emboj.2008.235<https://doi.org/10.1038/emboj.2008.235>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-57149087850&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Molecular%20basis%20for%20G-actin%20binding%20to%20RPEL%20motifs%20from%20the%20serum%20response%20factor%20coactivator%20MAL&publication_year=2008&author=S.%20Mouilleron&author=S.%20Guettler&author=C.A.%20Langer&author=R.%20Treisman&author=N.Q.%20McDonald>
25. [25] <#bbb0125>
    E. Paunola, P.K. Mattila, P. Lappalainen
    WH2 domain: a small, versatile adapter for actin monomers
    FEBS Lett., 513 (2002), pp. 92-97
    https://www.ncbi.nlm.nih.gov/pubmed/11911886<https://www.ncbi.nlm.nih.gov/pubmed/11911886>
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0014579301032422/pdfft?md5=9fbe5f9787041b6ca909ff068783cb3d&pid=1-s2.0-S0014579301032422-main.pdf>View article <https://www.sciencedirect.com/science/article/pii/S0014579301032422>View in Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-0037138384&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=WH2%20domain%3A%20a%20small%2C%20versatile%20adapter%20for%20actin%20monomers&publication_year=2002&author=E.%20Paunola&author=P.K.%20Mattila&author=P.%20Lappalainen>
26. [26] <#bbb0130>
    S.A. Rizvi, V. Tereshko, A.A. Kossiakoff, S.A. Kozmin
    Structure of bistramide A–actin complex at a 1.35 angstroms resolution
    J. Am. Chem. Soc., 128 (2006), pp. 3882-3883,
    10.1021/ja058319c<https://doi.org/10.1021/ja058319c>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-33645453004&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Structure%20of%20bistramide%20Aactin%20complex%20at%20a%201.35%20angstroms%20resolution&publication_year=2006&author=S.A.%20Rizvi&author=V.%20Tereshko&author=A.A.%20Kossiakoff&author=S.A.%20Kozmin>
27. [27] <#bbb0135>
    K. Hirata, S. Muraoka, K. Suenaga, T. Kuroda, K. Kato, H. Tanaka, M.
    Yamamoto, M. Takata, K. Yamada, H. Kigoshi
    Structure basis for antitumor effect of aplyronine a
    J. Mol. Biol., 356 (2006), pp. 945-954,
    10.1016/j.jmb.2005.12.031<https://doi.org/10.1016/j.jmb.2005.12.031>
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0022283605015937/pdfft?md5=4a805e672e55e3db6088d6776e864ddb&pid=1-s2.0-S0022283605015937-main.pdf>View article <https://www.sciencedirect.com/science/article/pii/S0022283605015937>View in Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-31444433185&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Structure%20basis%20for%20antitumor%20effect%20of%20aplyronine%20a&publication_year=2006&author=K.%20Hirata&author=S.%20Muraoka&author=K.%20Suenaga&author=T.%20Kuroda&author=K.%20Kato&author=H.%20Tanaka&author=M.%20Yamamoto&author=M.%20Takata&author=K.%20Yamada&author=H.%20Kigoshi>
28. [28] <#bbb0140>
    F. Ferron, G. Rebowski, S.H. Lee, R. Dominguez
    Structural basis for the recruitment of profilin–actin complexes
    during filament elongation by Ena/VASP
    EMBO J., 26 (2007), pp. 4597-4606,
    10.1038/sj.emboj.7601874<https://doi.org/10.1038/sj.emboj.7601874>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-35649021883&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Structural%20basis%20for%20the%20recruitment%20of%20profilinactin%20complexes%20during%20filament%20elongation%20by%20EnaVASP&publication_year=2007&author=F.%20Ferron&author=G.%20Rebowski&author=S.H.%20Lee&author=R.%20Dominguez>
29. [29] <#bbb0145>
    J.H. Pereira, C. Petchprayoon, A.C. Hoepker, N.W. Moriarty, S.J.
    Fink, G. Cecere, I. Paterson, P.D. Adams, G. Marriott
    Structural and biochemical studies of actin in complex with
    synthetic macrolide tail analogues
    ChemMedChem., 9 (2014), pp. 2286-2293,
    10.1002/cmdc.201402150<https://doi.org/10.1002/cmdc.201402150>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-84907875840&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Structural%20and%20biochemical%20studies%20of%20actin%20in%20complex%20with%20synthetic%20macrolide%20tail%20analogues&publication_year=2014&author=J.H.%20Pereira&author=C.%20Petchprayoon&author=A.C.%20Hoepker&author=N.W.%20Moriarty&author=S.J.%20Fink&author=G.%20Cecere&author=I.%20Paterson&author=P.D.%20Adams&author=G.%20Marriott>
30. [30] <#bbb0150>
    S.M. Margarit, W. Davidson, L. Frego, C.E. Stebbins
    A steric antagonism of actin polymerization by a salmonella
    virulence protein
    Struct., 14 (2006), pp. 1219-1229,
    10.1016/j.str.2006.05.022<https://doi.org/10.1016/j.str.2006.05.022>
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0969212606002887/pdfft?md5=f88e1ef9fa65c91210b35a2ea7c87233&pid=1-s2.0-S0969212606002887-main.pdf>View article <https://www.sciencedirect.com/science/article/pii/S0969212606002887>View in Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-33746855102&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=A%20steric%20antagonism%20of%20actin%20polymerization%20by%20a%20salmonella%20virulence%20protein&publication_year=2006&author=S.M.%20Margarit&author=W.%20Davidson&author=L.%20Frego&author=C.E.%20Stebbins>
31. [31] <#bbb0155>
    J.N. Rao, Y. Madasu, R. Dominguez
    Mechanism of actin filament pointed-end capping by tropomodulin
    Sci., 345 (2014), pp. 463-467,
    10.1126/science.1256159<https://doi.org/10.1126/science.1256159>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-84904805269&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20actin%20filament%20pointed-end%20capping%20by%20tropomodulin&publication_year=2014&author=J.N.%20Rao&author=Y.%20Madasu&author=R.%20Dominguez>
32. [32] <#bbb0160>
    S. Hayward, A. Kitao, H.J. Berendsen
    Model-free methods of analyzing domain motions in proteins from
    simulation: a comparison of normal mode analysis and molecular
    dynamics simulation of lysozyme
    Proteins., 27 (1997), pp. 425-437
    https://www.ncbi.nlm.nih.gov/pubmed/9094744<https://www.ncbi.nlm.nih.gov/pubmed/9094744>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-0030888546&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Model-free%20methods%20of%20analyzing%20domain%20motions%20in%20proteins%20from%20simulation%3A%20a%20comparison%20of%20normal%20mode%20analysis%20and%20molecular%20dynamics%20simulation%20of%20lysozyme&publication_year=1997&author=S.%20Hayward&author=A.%20Kitao&author=H.J.%20Berendsen>
33. [33] <#bbb0165>
    S. Hayward, H.J. Berendsen
    Systematic analysis of domain motions in proteins from
    conformational change: new results on citrate synthase and T4 lysozyme
    Proteins., 30 (1998), pp. 144-154
    https://www.ncbi.nlm.nih.gov/pubmed/9489922<https://www.ncbi.nlm.nih.gov/pubmed/9489922>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-0032006209&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Systematic%20analysis%20of%20domain%20motions%20in%20proteins%20from%20conformational%20change%3A%20new%20results%20on%20citrate%20synthase%20and%20T4%20lysozyme&publication_year=1998&author=S.%20Hayward&author=H.J.%20Berendsen>
34. [34] <#bbb0170>
    B. Xue, C. Leyrat, J.M. Grimes, R.C. Robinson
    Structural basis of thymosin-β4/profilin exchange leading to actin
    filament polymerization
    Proc. Natl. Acad. Sci. United States Am., 111 (2014), pp.
    E4596-E4605,
    10.1073/pnas.1412271111<https://doi.org/10.1073/pnas.1412271111>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-84908274470&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Structural%20basis%20of%20thymosin-4profilin%20exchange%20leading%20to%20actin%20filament%20polymerization&publication_year=2014&author=B.%20Xue&author=C.%20Leyrat&author=J.M.%20Grimes&author=R.C.%20Robinson>
35. [35] <#bbb0175>
    E. Ballweber, E. Hannappel, T. Huff, H.G. Mannherz
    Mapping the binding site of thymosin beta4 on actin by competition
    with G-actin binding proteins indicates negative co-operativity
    between binding sites located on opposite subdomains of actin
    Biochem. J., 327 (1997), pp. 787-793
    Pt 3
    https://www.ncbi.nlm.nih.gov/pubmed/9581557<https://www.ncbi.nlm.nih.gov/pubmed/9581557>
    CrossRef<https://doi.org/10.1042/bj3270787>View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-0030659761&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Mapping%20the%20binding%20site%20of%20thymosin%20beta4%20on%20actin%20by%20competition%20with%20G-actin%20binding%20proteins%20indicates%20negative%20co-operativity%20between%20binding%20sites%20located%20on%20opposite%20subdomains%20of%20actin&publication_year=1997&author=E.%20Ballweber&author=E.%20Hannappel&author=T.%20Huff&author=H.G.%20Mannherz>
36. [36] <#bbb0180>
    E. Ballweber, K. Giehl, E. Hannappel, T. Huff, B.M. Jockusch, H.G.
    Mannherz
    Plant profilin induces actin polymerization from actin :
    beta-thymosin complexes and competes directly with beta-thymosins
    and with negative co-operativity with DNase I for binding to actin
    FEBS Lett., 425 (1998), pp. 251-255
    https://www.ncbi.nlm.nih.gov/pubmed/9559659<https://www.ncbi.nlm.nih.gov/pubmed/9559659>
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0014579398002403/pdfft?md5=a66e4b16c3a0d5fb75e5e118c4d1f47b&pid=1-s2.0-S0014579398002403-main.pdf>View article <https://www.sciencedirect.com/science/article/pii/S0014579398002403>View in Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-0032571566&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Plant%20profilin%20induces%20actin%20polymerization%20from%20actin%E2%80%AF%3A%20beta-thymosin%20complexes%20and%20competes%20directly%20with%20beta-thymosins%20and%20with%20negative%20co-operativity%20with%20DNase%20I%20for%20binding%20to%20actin&publication_year=1998&author=E.%20Ballweber&author=K.%20Giehl&author=E.%20Hannappel&author=T.%20Huff&author=B.M.%20Jockusch&author=H.G.%20Mannherz>
37. [37] <#bbb0185>
    T. Kotila, K. Kogan, G. Enkavi, S. Guo, I. Vattulainen, B.L. Goode,
    P. Lappalainen
    Structural basis of actin monomer re-charging by cyclase-associated
    protein
    Nat. Commun., 9 (2018), Article 1892,
    10.1038/s41467-018-04231-7<https://doi.org/10.1038/s41467-018-04231-7>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-85047005231&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Structural%20basis%20of%20actin%20monomer%20re-charging%20by%20cyclase-associated%20protein&publication_year=2018&author=T.%20Kotila&author=K.%20Kogan&author=G.%20Enkavi&author=S.%20Guo&author=I.%20Vattulainen&author=B.L.%20Goode&author=P.%20Lappalainen>
38. [38] <#bbb0190>
    P.K. Mattila, O. Quintero-Monzon, J. Kugler, J.B. Moseley, S.C.
    Almo, P. Lappalainen, B.L. Goode
    A high-affinity interaction with ADP–actin monomers underlies the
    mechanism and in vivo function of Srv2/cyclase-associated protein
    Mol. Biol. Cell, 15 (2004), pp. 5158-5171,
    10.1091/mbc.e04-06-0444<https://doi.org/10.1091/mbc.e04-06-0444>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-6344225310&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=A%20high-affinity%20interaction%20with%20ADPactin%20monomers%20underlies%20the%20mechanism%20and%20in%20vivo%20function%20of%20Srv2cyclase-associated%20protein&publication_year=2004&author=P.K.%20Mattila&author=O.%20Quintero-Monzon&author=J.%20Kugler&author=J.B.%20Moseley&author=S.C.%20Almo&author=P.%20Lappalainen&author=B.L.%20Goode>
39. [39] <#bbb0195>
    X. Liu, S. Shu, M.-S.S. Hong, B. Yu, E.D. Korn
    Mutation of actin Tyr-53 alters the conformations of the DNase
    I-binding loop and the nucleotide-binding cleft
    J. Biol. Chem., 285 (2010), pp. 9729-9739,
    10.1074/jbc.M109.073452<https://doi.org/10.1074/jbc.M109.073452>
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0021925819550293/pdfft?md5=b841e90f4cc2797952dce400b29232e8&pid=1-s2.0-S0021925819550293-main.pdf>View article <https://www.sciencedirect.com/science/article/pii/S0021925819550293>View in Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-77951219690&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Mutation%20of%20actin%20Tyr-53%20alters%20the%20conformations%20of%20the%20DNase%20I-binding%20loop%20and%20the%20nucleotide-binding%20cleft&publication_year=2010&author=X.%20Liu&author=S.%20Shu&author=M.-S.S.%20Hong&author=B.%20Yu&author=E.D.%20Korn>
40. [40] <#bbb0200>
    E.E. Grintsevich, P. Ge, M.R. Sawaya, H.G. Yesilyurt, J.R. Terman,
    Z.H. Zhou, E. Reisler
    Catastrophic disassembly of actin filaments via Mical-mediated oxidation
    Nat. Commun., 8 (2017), Article 2183,
    10.1038/s41467-017-02357-8<https://doi.org/10.1038/s41467-017-02357-8>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-85038628534&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Catastrophic%20disassembly%20of%20actin%20filaments%20via%20Mical-mediated%20oxidation&publication_year=2017&author=E.E.%20Grintsevich&author=P.%20Ge&author=M.R.%20Sawaya&author=H.G.%20Yesilyurt&author=J.R.%20Terman&author=Z.H.%20Zhou&author=E.%20Reisler>
41. [41] <#bbb0205>
    Scilab Enterprises, Scilab: Free and Open Source software for
    numerical computation (Windows 10, Version 6.0) [Software] Available
    from: http://www.scilab.org<http://www.scilab.org/>,
    www.scilab.org<http://www.scilab.org/>, 2012.
    Google
    Scholar<https://scholar.google.com/scholar?q=Scilab%20Enterprises%2C%20Scilab%3A%20Free%20and%20Open%20Source%20software%20for%20numerical%20computation%20(Windows%2010%2C%20Version%206.0)%20%5BSoftware%5D%20Available%20from%3A%20http%3A%2F%2Fwww.scilab.org%2C%20www.scilab.org%2C%202012.>
42. [42] <#bbb0210>
    M.J. Abraham, D. van der Spoel, E. Lindahl, B. Hess
    GROMACS User Manual Version 2016.1, www.gromacs.org
    (2016)
    Google
    Scholar<https://scholar.google.com/scholar_lookup?title=GROMACS%20User%20Manual%20Version%202016.1%2C%20www.gromacs.org&publication_year=2016&author=M.J.%20Abraham&author=D.%20van%20der%20Spoel&author=E.%20Lindahl&author=B.%20Hess>
43. [43] <#bbb0215>
    A.D. Mackerell, M. Feig, C.L. Brooks
    Extending the treatment of backbone energetics in protein force
    fields: limitations of gas-phase quantum mechanics in reproducing
    protein conformational distributions in molecular dynamics simulations
    J. Comput. Chem., 25 (2004), pp. 1400-1415,
    10.1002/jcc.20065<https://doi.org/10.1002/jcc.20065>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-3142714765&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Extending%20the%20treatment%20of%20backbone%20energetics%20in%20protein%20force%20fields%3A%20limitations%20of%20gas-phase%20quantum%20mechanics%20in%20reproducing%20protein%20conformational%20distributions%20in%20molecular%20dynamics%20simulations&publication_year=2004&author=A.D.%20Mackerell&author=M.%20Feig&author=C.L.%20Brooks>
44. [44] <#bbb0220>
    A. Sali, T.L. Blundell
    Comparative protein modelling by satisfaction of spatial restraints
    J. Mol. Biol., 234 (1993), pp. 779-815,
    10.1006/jmbi.1993.1626<https://doi.org/10.1006/jmbi.1993.1626>
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0022283683716268/pdf?md5=149f50ebcd32c61fbae8da1a471122a7&pid=1-s2.0-S0022283683716268-main.pdf>View article <https://www.sciencedirect.com/science/article/pii/S0022283683716268>View in Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-0027136282&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Comparative%20protein%20modelling%20by%20satisfaction%20of%20spatial%20restraints&publication_year=1993&author=A.%20Sali&author=T.L.%20Blundell>
45. [45] <#bbb0225>
    U. Essmann, L. Perera, M.L. Berkowitz, T. Darden, H. Lee, L.G. Pedersen
    A smooth particle mesh Ewald method
    J. Chem. Phys., 103 (1995), pp. 8577-8592,
    10.1063/1.470117<https://doi.org/10.1063/1.470117>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-33645961739&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=A%20smooth%20particle%20mesh%20Ewald%20method&publication_year=1995&author=U.%20Essmann&author=L.%20Perera&author=M.L.%20Berkowitz&author=T.%20Darden&author=H.%20Lee&author=L.G.%20Pedersen>
46. [46] <#bbb0230>
    B. Hess
    P-LINCS: a parallel linear constraint solver for molecular simulation
    J. Chem. Theory Comput., 4 (2008), pp. 116-122,
    10.1021/ct700200b<https://doi.org/10.1021/ct700200b>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-38749123962&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=P-LINCS%3A%20a%20parallel%20linear%20constraint%20solver%20for%20molecular%20simulation&publication_year=2008&author=B.%20Hess>
47. [47] <#bbb0235>
    G. Bussi, D. Donadio, M. Parrinello
    Canonical sampling through velocity rescaling
    J. Chem. Phys., 126 (2007), Article 014101,
    10.1063/1.2408420<https://doi.org/10.1063/1.2408420>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-33846086933&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Canonical%20sampling%20through%20velocity%20rescaling&publication_year=2007&author=G.%20Bussi&author=D.%20Donadio&author=M.%20Parrinello>
48. [48] <#bbb0240>
    M. Parrinello, A. Rahman
    Polymorphic transitions in single crystal: a new molecular dynamics
    method
    J. Appl. Phys., 52 (1981), pp. 7182-7190,
    10.1063/1.328693<https://doi.org/10.1063/1.328693>
    View in
    Scopus<https://www.scopus.com/inward/record.url?eid=2-s2.0-0019707626&partnerID=10&rel=R3.0.0>Google Scholar<https://scholar.google.com/scholar_lookup?title=Polymorphic%20transitions%20in%20single%20crystal%3A%20a%20new%20molecular%20dynamics%20method&publication_year=1981&author=M.%20Parrinello&author=A.%20Rahman>


    Cited by (15)

  *


          Structural Insights into the Regulation of Actin Capping
          Protein by Twinfilin C-terminal Tail
          <https://www.sciencedirect.com/science/article/pii/S0022283621000851>

    2021, Journal of Molecular Biology
    Citation Excerpt :

    Recent studies demonstrated that twinfilin does not enhance the
    depolymerization, but instead, twinfilin associates with the
    filament barbed ends to prevent their elongation while allowing the
    dissociation of actin subunits.30,38 These filament-destabilizing
    activities of twinfilin are mediated through tandemly linked two
    actin depolymerization factor/cofilin-homology (ADF-H) domains that
    could shift the actin conformation to cofilin-bound C-form that is
    unfavorable for polymerization.39–42 The ADF-H domains of twinfilin
    are followed by ~36 amino acid residues, the TWtail, which has
    sequence similarity with CARMIL CPIs.29

    Show abstract

    Twinfilin is a conserved actin regulator that interacts with actin
    capping protein (CP) via C terminus residues (TWtail) that exhibits
    sequence similarity with the CP interaction (CPI) motif of CARMIL.
    Here we report the crystal structure of TWtail in complex with CP.
    Our structure showed that although TWtail and CARMIL CPI bind CP to
    an overlapping surface via their middle regions, they exhibit
    different CP-binding modes at both termini. Consequently, TWtail and
    CARMIL CPI restrict the CP in distinct conformations of open and
    closed forms, respectively. Interestingly, V-1, which targets CP
    away from the TWtail binding site, also favors the open-form CP.
    Consistently, TWtail forms a stable ternary complex with CP and V-1,
    a striking contrast to CARMIL CPI, which rapidly dissociates V-1
    from CP. Our results demonstrate that TWtail is a unique CP-binding
    motif that regulates CP in a manner distinct from CARMIL CPI.

  *


          Mechanically tuning actin filaments to modulate the action of
          actin-binding proteins
          <https://www.sciencedirect.com/science/article/pii/S0955067420301162>

    2021, Current Opinion in Cell Biology
    Citation Excerpt :

    In addition, the application of force to the actin subunit, mediated
    by numerous bonds with neighboring subunits, can cause the subunit
    to change conformation. In particular, the D-loop of subdomain 2, a
    key motif involved in the interaction with many ABPs, can adopt
    various conformations, thereby changing interactions between
    subunits and changing their propensity to bind ABPs [52–54]. At the
    scale of the filament, thermal fluctuations cause variations of the
    local strain and thus variations of the binding interfaces presented
    to ABPs.

    Show abstract

    In cells, the actin cytoskeleton is regulated by an interplay
    between mechanics and biochemistry. A key mechanism, which has
    emerged based on converging indications from structural, cellular,
    and biophysical data, depicts the actin filament as a mechanically
    tunable substrate: mechanical stress applied to an actin filament
    induces conformational changes, which modify the binding and the
    regulatory action of actin-binding proteins. For a long time,
    however, direct evidence of this mechanotransductive mechanism was
    very scarce. This situation is changing rapidly, and recent
    /in vitro/ single-filament studies using different techniques have
    revealed that several actin-binding proteins are able to sense
    tension, curvature, and/or torsion, applied to actin filaments.
    Here, we discuss these recent advances and their possible implications.

  *


          Diverse cytomotive actins and tubulins share a polymerization
          switch mechanism conferring robust
          dynamics<https://doi.org/10.1126/sciadv.adf3021>

    2023, Science Advances
  *


          Mutagenic analysis of actin reveals the mechanism of His161
          flipping that triggers ATP
          hydrolysis<https://doi.org/10.3389/fcell.2023.1105460>

    2023, Frontiers in Cell and Developmental Biology
  *


          Magic angle spinning NMR structure of human cofilin-2
          assembled on actin filaments reveals isoform-specific
          conformation and binding
          mode<https://doi.org/10.1038/s41467-022-29595-9>

    2022, Nature Communications
  *


          Bending forces and nucleotide state jointly regulate F-actin
          structure<https://doi.org/10.1038/s41586-022-05366-w>

    2022, Nature

View all citing articles on
Scopus<http://www.scopus.com/scopus/inward/citedby.url?partnerID=10&rel=3.0.0&eid=2-s2.0-85067524272&md5=501fa877deeb9c0425815174efaa523>

View Abstract
<https://www.sciencedirect.com/science/article/abs/pii/S0022283619303493>
© 2019 Elsevier Ltd. All rights reserved.


    Recommended articles

  *


          Gain-of-Function Mutation W493R in the Epithelial Sodium
          Channel Allosterically Reconfigures Intersubunit Coupling
          <https://www.sciencedirect.com/science/article/pii/S0021925820436804>

    Journal of Biological Chemistry, Volume 291, Issue 8, 2016, pp.
    3682-3692
    Mahmoud Shobair, …, Nikolay V. Dokholyan
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0021925820436804/pdfft?md5=83ccd38163373bdb912dc684f8ab411e&pid=1-s2.0-S0021925820436804-main.pdf>
  *


          Single Filaments to Reveal the Multiple Flavors of Actin
          <https://www.sciencedirect.com/science/article/pii/S000634951630220X>

    Biophysical Journal, Volume 110, Issue 10, 2016, pp. 2138-2146
    Antoine Jégou, Guillaume Romet-Lemonne
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S000634951630220X/pdfft?md5=4878a7d2017cebae30a1e4316d950a3a&pid=1-s2.0-S000634951630220X-main.pdf>
  *


          Cryo-EM and Molecular Docking Shows Myosin Loop 4 Contacts
          Actin and Tropomyosin on Thin Filaments
          <https://www.sciencedirect.com/science/article/pii/S000634952030535X>

    Biophysical Journal, Volume 119, Issue 4, 2020, pp. 821-830
    Matthew H. Doran, …, William Lehman
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S000634952030535X/pdfft?md5=ac5370ac4b749f061134094de0fddd6e&pid=1-s2.0-S000634952030535X-main.pdf>
  *


          INF2-Mediated Severing through Actin Filament Encirclement and
          Disruption
          <https://www.sciencedirect.com/science/article/pii/S0960982213015777>

    Current Biology, Volume 24, Issue 2, 2014, pp. 156-164
    Pinar S. Gurel, …, Henry N. Higgs
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0960982213015777/pdfft?md5=d50ea14e8fa130145085824745453dc1&pid=1-s2.0-S0960982213015777-main.pdf>
  *


          Clusters of a Few Bound Cofilins Sever Actin Filaments
          <https://www.sciencedirect.com/science/article/pii/S0022283621000279>

    Journal of Molecular Biology, Volume 433, Issue 7, 2021, Article 166833
    Jeffrey P. Bibeau, …, Enrique M. De La Cruz
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0022283621000279/pdfft?md5=64abb105ab7d22b967c7c154158800c0&pid=1-s2.0-S0022283621000279-main.pdf>
  *


          Phylogenetic origin and sequence features of MreB from the
          wall-less swimming bacteria /Spiroplasma/
          <https://www.sciencedirect.com/science/article/pii/S0006291X20318015>

    Biochemical and Biophysical Research Communications, Volume 533,
    Issue 4, 2020, pp. 638-644
    Daichi Takahashi, …, Makoto Miyata
    View PDF
    <https://www.sciencedirect.com/science/article/pii/S0006291X20318015/pdfft?md5=7112352d0b66b300ebbd87ef31a43f11&pid=1-s2.0-S0006291X20318015-main.pdf>

Show 3 more articles


    Article Metrics

plumX logo
Citations

  * Citation Indexes: 14

Captures

  * Readers: 24

plumX logo
View
details<https://plu.mx/plum/a/?doi=10.1016/j.jmb.2019.05.048&theme=plum-sciencedirect-theme&hideUsage=true>

Elsevier logo with wordmark <https://www.elsevier.com/>

  * About ScienceDirect<https://www.elsevier.com/solutions/sciencedirect>
  * Remote access
    <https://www.sciencedirect.com/user/institution/login?targetURL=%2Fscience%2Farticle%2Fpii%2FS0022283619303493>
  * Shopping cart<https://sd-cart.elsevier.com/?>
  * Advertise<http://elsmediakits.com/>
  * Contact and
    support<https://service.elsevier.com/app/contact/supporthub/sciencedirect/>
  * Terms and
    conditions<https://www.elsevier.com/legal/elsevier-website-terms-and-conditions>
  * Privacy policy<https://www.elsevier.com/legal/privacy-policy>

We use cookies to help provide and enhance our service and tailor
content and ads. By continuing you agree to the *use of cookies*
<https://www.sciencedirect.com/legal/use-of-cookies>.

All content on this site: Copyright © 2023 Elsevier B.V., its licensors,
and contributors. All rights are reserved, including those for text and
data mining, AI training, and similar technologies. For all open access
content, the Creative Commons licensing terms apply.

RELX group home page <https://www.relx.com/>

Feedback

